{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2.1.1 SRA "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "os.chdir(\"SRA\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "pd.set_option('display.max_columns', 200)\n",
    "pd.set_option('display.max_rows', 100)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1 Retrieve SRA metadata"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Code to connect Assembly Ids (Genbank) with SRA metadata"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "- 01_entrez_direct - this folder contains the methods to link the assembly accession of a bacterial genome to its paternal raw reads\n",
    "    \n",
    "# 1 - Download the summary of available bacterial genomes on NCBI\n",
    "wget ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/bacteria/assembly_summary.txt;\n",
    "\n",
    "# entrez direct:\n",
    "sh esearch_assembly_to_sra.sh assembly_ids.csv > entrez_output.tsv"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "- 02_srr_to_sradb - this folder contains the methods to link the paternal raw reads (ids) to sradb and assure that the data is only (or most likely) genome sequencing\n",
    "\n",
    "# 2 - Generate list of SRR ids\n",
    "\n",
    "cut -f4 entrez_output.txt | sort | uniq | perl -ne 'chomp; print \"$_,\"' | sed -e \"s/^,//\" -e \"s/,$//\" | tr \",\" \"\\n\" | sort | uniq > srr_ids.txt\n",
    "\n",
    "Run retrieve_table_from_sradb.R\n",
    "output: reads_dataset.txt"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "- 03_sradb_to_assembly_tb/ - this folder contains the methods to link back the \"validated SRRs\" to the assembly list\n",
    "\n",
    "awk '{if(NF==4){print $0}}' entrez_output.txt > entrez_output_reduced.txt\n",
    "\n",
    "cut -f4 reads_dataset.txt | tail -n+2 | tr -d '\"' > valid_srr_ids.txt\n",
    "\n",
    "python joiner.py valid_srr_ids.txt entrez_output_reduced.txt > assembly_srr.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2 Upload SRA metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "sra = pd.read_csv(\"reads_dataset.txt\", sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>run_accession</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>SRR013476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>SRR013477</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  assembly_accession run_accession\n",
       "0    GCA_000023285.1     SRR013476\n",
       "1    GCA_000023285.1     SRR013477"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "code = pd.read_csv(\"assembly_srr.txt\", sep=\"\\t\", names=[\"assembly_accession\", \"run_accession\"])\n",
    "code.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "code = code.drop_duplicates(subset=['assembly_accession']) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "code_sra = pd.merge(code, sra, on=\"run_accession\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>run_accession</th>\n",
       "      <th>sra_ID</th>\n",
       "      <th>run_ID</th>\n",
       "      <th>run_alias</th>\n",
       "      <th>run_date</th>\n",
       "      <th>updated_date</th>\n",
       "      <th>spots</th>\n",
       "      <th>bases</th>\n",
       "      <th>run_center</th>\n",
       "      <th>experiment_name</th>\n",
       "      <th>run_attribute</th>\n",
       "      <th>experiment_ID</th>\n",
       "      <th>experiment_alias</th>\n",
       "      <th>experiment_accession</th>\n",
       "      <th>experiment_title</th>\n",
       "      <th>study_name</th>\n",
       "      <th>sample_name</th>\n",
       "      <th>design_description</th>\n",
       "      <th>library_name</th>\n",
       "      <th>library_strategy</th>\n",
       "      <th>library_source</th>\n",
       "      <th>library_selection</th>\n",
       "      <th>library_layout</th>\n",
       "      <th>library_construction_protocol</th>\n",
       "      <th>read_spec</th>\n",
       "      <th>platform</th>\n",
       "      <th>instrument_model</th>\n",
       "      <th>platform_parameters</th>\n",
       "      <th>experiment_attribute</th>\n",
       "      <th>sample_ID</th>\n",
       "      <th>sample_alias</th>\n",
       "      <th>sample_accession</th>\n",
       "      <th>taxon_id</th>\n",
       "      <th>common_name</th>\n",
       "      <th>description</th>\n",
       "      <th>sample_url_link</th>\n",
       "      <th>sample_attribute</th>\n",
       "      <th>study_ID</th>\n",
       "      <th>study_alias</th>\n",
       "      <th>study_accession</th>\n",
       "      <th>study_title</th>\n",
       "      <th>study_type</th>\n",
       "      <th>study_abstract</th>\n",
       "      <th>center_project_name</th>\n",
       "      <th>study_description</th>\n",
       "      <th>study_url_link</th>\n",
       "      <th>study_attribute</th>\n",
       "      <th>submission_ID</th>\n",
       "      <th>submission_accession</th>\n",
       "      <th>submission_comment</th>\n",
       "      <th>submission_center</th>\n",
       "      <th>submission_lab</th>\n",
       "      <th>submission_date</th>\n",
       "      <th>sradb_updated</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>534</th>\n",
       "      <td>GCA_902804475.1</td>\n",
       "      <td>ERR3888428</td>\n",
       "      <td>0</td>\n",
       "      <td>9069339</td>\n",
       "      <td>ena-RUN-INSTITUT PASTEUR-06-02-2020-13:43:47:1...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-02-25</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ENA-FIRST-PUBLIC: 2020-02-21 || ENA-LAST-UPDAT...</td>\n",
       "      <td>8233124</td>\n",
       "      <td>ena-EXPERIMENT-INSTITUT PASTEUR-06-02-2020-13:...</td>\n",
       "      <td>ERX3898656</td>\n",
       "      <td>NextSeq 500 paired end sequencing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Nextera XT</td>\n",
       "      <td>WGS</td>\n",
       "      <td>GENOMIC</td>\n",
       "      <td>RANDOM</td>\n",
       "      <td>PAIRED - NOMINAL_LENGTH: 317;</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>INSTRUMENT_MODEL: NextSeq 500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>7432552.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ERS4290983</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ERP119591</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1567020</td>\n",
       "      <td>ERA2376428</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>European Nucleotide Archive</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-09-03 11:01:35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>535</th>\n",
       "      <td>GCA_902804485.1</td>\n",
       "      <td>ERR3888429</td>\n",
       "      <td>0</td>\n",
       "      <td>9069340</td>\n",
       "      <td>ena-RUN-INSTITUT PASTEUR-06-02-2020-13:43:47:1...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-02-25</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ENA-FIRST-PUBLIC: 2020-02-21 || ENA-LAST-UPDAT...</td>\n",
       "      <td>8233125</td>\n",
       "      <td>ena-EXPERIMENT-INSTITUT PASTEUR-06-02-2020-13:...</td>\n",
       "      <td>ERX3898657</td>\n",
       "      <td>NextSeq 500 paired end sequencing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Nextera XT</td>\n",
       "      <td>WGS</td>\n",
       "      <td>GENOMIC</td>\n",
       "      <td>RANDOM</td>\n",
       "      <td>PAIRED - NOMINAL_LENGTH: 327;</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ILLUMINA</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>INSTRUMENT_MODEL: NextSeq 500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>7432551.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ERS4290982</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ERP119591</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1567020</td>\n",
       "      <td>ERA2376428</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>European Nucleotide Archive</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-09-03 11:01:35</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    assembly_accession run_accession  sra_ID   run_ID  \\\n",
       "534    GCA_902804475.1    ERR3888428       0  9069339   \n",
       "535    GCA_902804485.1    ERR3888429       0  9069340   \n",
       "\n",
       "                                             run_alias run_date updated_date  \\\n",
       "534  ena-RUN-INSTITUT PASTEUR-06-02-2020-13:43:47:1...      NaN   2020-02-25   \n",
       "535  ena-RUN-INSTITUT PASTEUR-06-02-2020-13:43:47:1...      NaN   2020-02-25   \n",
       "\n",
       "     spots  bases run_center experiment_name  \\\n",
       "534    0.0    0.0        NaN             NaN   \n",
       "535    0.0    0.0        NaN             NaN   \n",
       "\n",
       "                                         run_attribute  experiment_ID  \\\n",
       "534  ENA-FIRST-PUBLIC: 2020-02-21 || ENA-LAST-UPDAT...        8233124   \n",
       "535  ENA-FIRST-PUBLIC: 2020-02-21 || ENA-LAST-UPDAT...        8233125   \n",
       "\n",
       "                                      experiment_alias experiment_accession  \\\n",
       "534  ena-EXPERIMENT-INSTITUT PASTEUR-06-02-2020-13:...           ERX3898656   \n",
       "535  ena-EXPERIMENT-INSTITUT PASTEUR-06-02-2020-13:...           ERX3898657   \n",
       "\n",
       "                      experiment_title study_name sample_name  \\\n",
       "534  NextSeq 500 paired end sequencing        NaN         NaN   \n",
       "535  NextSeq 500 paired end sequencing        NaN         NaN   \n",
       "\n",
       "    design_description library_name library_strategy library_source  \\\n",
       "534                NaN   Nextera XT              WGS        GENOMIC   \n",
       "535                NaN   Nextera XT              WGS        GENOMIC   \n",
       "\n",
       "    library_selection                  library_layout  \\\n",
       "534            RANDOM  PAIRED - NOMINAL_LENGTH: 317;    \n",
       "535            RANDOM  PAIRED - NOMINAL_LENGTH: 327;    \n",
       "\n",
       "    library_construction_protocol read_spec  platform instrument_model  \\\n",
       "534                           NaN       NaN  ILLUMINA      NextSeq 500   \n",
       "535                           NaN       NaN  ILLUMINA      NextSeq 500   \n",
       "\n",
       "               platform_parameters experiment_attribute  sample_ID  \\\n",
       "534  INSTRUMENT_MODEL: NextSeq 500                  NaN  7432552.0   \n",
       "535  INSTRUMENT_MODEL: NextSeq 500                  NaN  7432551.0   \n",
       "\n",
       "    sample_alias sample_accession  taxon_id common_name description  \\\n",
       "534          NaN       ERS4290983       NaN         NaN         NaN   \n",
       "535          NaN       ERS4290982       NaN         NaN         NaN   \n",
       "\n",
       "    sample_url_link sample_attribute  study_ID study_alias study_accession  \\\n",
       "534             NaN              NaN       NaN         NaN       ERP119591   \n",
       "535             NaN              NaN       NaN         NaN       ERP119591   \n",
       "\n",
       "    study_title study_type study_abstract center_project_name  \\\n",
       "534         NaN        NaN            NaN                 NaN   \n",
       "535         NaN        NaN            NaN                 NaN   \n",
       "\n",
       "    study_description study_url_link study_attribute  submission_ID  \\\n",
       "534               NaN            NaN             NaN        1567020   \n",
       "535               NaN            NaN             NaN        1567020   \n",
       "\n",
       "    submission_accession submission_comment submission_center  \\\n",
       "534           ERA2376428                NaN               NaN   \n",
       "535           ERA2376428                NaN               NaN   \n",
       "\n",
       "                  submission_lab submission_date        sradb_updated  \n",
       "534  European Nucleotide Archive             NaN  2020-09-03 11:01:35  \n",
       "535  European Nucleotide Archive             NaN  2020-09-03 11:01:35  "
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "code_sra.tail(2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3 Solve sample attribute column"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>sample_attribute</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>strain: DSM 7271 || type-material: type strain...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>strain: ATCC 35910 || misc_param: HMP supersit...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                    sample_attribute\n",
       "0  strain: DSM 7271 || type-material: type strain...\n",
       "1  strain: ATCC 35910 || misc_param: HMP supersit..."
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "att = pd.DataFrame(code_sra[\"sample_attribute\"])\n",
    "att.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "att.to_csv(\"att.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "d = {}\n",
    "l = []\n",
    "\n",
    "f = open(\"att.csv\", \"r\")\n",
    "lines = f.readlines()\n",
    "\n",
    "for line in lines:\n",
    "    splited = line.split(r\" || \")\n",
    "    for x in splited:\n",
    "        l.append(x)\n",
    "\n",
    "for item in l:\n",
    "    a = item.split(\": \")\n",
    "    key = a[0]\n",
    "    key = key.replace('\"', '')\n",
    "    key = key.replace(' ', '')\n",
    "    value = a[1:]\n",
    "    if key in d:\n",
    "        # append the new number to the existing array at this slot\n",
    "        d[key].append(value)\n",
    "    else:\n",
    "        # create a new array in this slot\n",
    "        d[key] = [value]\n",
    "\n",
    "import pandas as pd\n",
    "\n",
    "df = pd.DataFrame.from_dict(d, orient='index').T\n",
    "df.to_csv(\"from_script.csv\", index=False)\n",
    "\n",
    "f.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "att2 = pd.read_csv(\"from_script.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>,sample_attribute\\n</th>\n",
       "      <th>0,strain</th>\n",
       "      <th>type-material</th>\n",
       "      <th>1,strain</th>\n",
       "      <th>misc_param</th>\n",
       "      <th>project_type</th>\n",
       "      <th>estimated_size</th>\n",
       "      <th>culture-collection</th>\n",
       "      <th>feature</th>\n",
       "      <th>investigation_type</th>\n",
       "      <th>hosttaxid</th>\n",
       "      <th>nucleicacidextraction</th>\n",
       "      <th>finishingstrategy(depthofcoverage)</th>\n",
       "      <th>material</th>\n",
       "      <th>sop</th>\n",
       "      <th>num_replicons</th>\n",
       "      <th>ref_biomaterial</th>\n",
       "      <th>geo_loc_name</th>\n",
       "      <th>biome</th>\n",
       "      <th>latlon</th>\n",
       "      <th>env_package</th>\n",
       "      <th>collectiondate</th>\n",
       "      <th>specifichost</th>\n",
       "      <th>sequencingmethod</th>\n",
       "      <th>assembly</th>\n",
       "      <th>isol_growth_condt</th>\n",
       "      <th>source_mat_id</th>\n",
       "      <th>project_name</th>\n",
       "      <th>isolation-source</th>\n",
       "      <th>BioSampleModel</th>\n",
       "      <th>2,finishingstrategy(depthofcoverage)</th>\n",
       "      <th>lat_lon</th>\n",
       "      <th>investigationtype</th>\n",
       "      <th>strain</th>\n",
       "      <th>environmentalpackage</th>\n",
       "      <th>3,finishingstrategy(depthofcoverage)</th>\n",
       "      <th>4,biotic_relationship</th>\n",
       "      <th>IsolationSite</th>\n",
       "      <th>TemperatureOptimum</th>\n",
       "      <th>BodySampleSite</th>\n",
       "      <th>Motility</th>\n",
       "      <th>GOLDStampID</th>\n",
       "      <th>Diseases</th>\n",
       "      <th>collection_date</th>\n",
       "      <th>environment</th>\n",
       "      <th>IsolationComments</th>\n",
       "      <th>TemperatureRange</th>\n",
       "      <th>TypeStrain</th>\n",
       "      <th>rel_to_oxygen</th>\n",
       "      <th>alt_elev</th>\n",
       "      <th>FundingProgram</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>trophic_level</th>\n",
       "      <th>country</th>\n",
       "      <th>depth</th>\n",
       "      <th>CellShape</th>\n",
       "      <th>HostHealth</th>\n",
       "      <th>HostName</th>\n",
       "      <th>5,finishingstrategy(depthofcoverage)</th>\n",
       "      <th>6,biotic_relationship</th>\n",
       "      <th>pH</th>\n",
       "      <th>GramStaining</th>\n",
       "      <th>GeneCallingMethod</th>\n",
       "      <th>7,\\n</th>\n",
       "      <th>8,\\n</th>\n",
       "      <th>9,finishingstrategy(depthofcoverage)</th>\n",
       "      <th>10,\\n</th>\n",
       "      <th>11,\\n</th>\n",
       "      <th>12,finishingstrategy(depthofcoverage)</th>\n",
       "      <th>13,env_package</th>\n",
       "      <th>biotic_relationship</th>\n",
       "      <th>14,biotic_relationship</th>\n",
       "      <th>15,env_package</th>\n",
       "      <th>16,biotic_relationship</th>\n",
       "      <th>Isolationsource</th>\n",
       "      <th>Host</th>\n",
       "      <th>geo-loc-name</th>\n",
       "      <th>BodySampleSubSite</th>\n",
       "      <th>17,finishingstrategy(depthofcoverage)</th>\n",
       "      <th>18,finishingstrategy(depthofcoverage)</th>\n",
       "      <th>19,finishingstrategy(depthofcoverage)</th>\n",
       "      <th>20,strain</th>\n",
       "      <th>21,strain</th>\n",
       "      <th>22,finishingstrategy(depthofcoverage)</th>\n",
       "      <th>23,strain</th>\n",
       "      <th>24,strain</th>\n",
       "      <th>host</th>\n",
       "      <th>25,env_package</th>\n",
       "      <th>isolationsource</th>\n",
       "      <th>26,strain</th>\n",
       "      <th>note</th>\n",
       "      <th>27,strain</th>\n",
       "      <th>28,strain</th>\n",
       "      <th>29,strain</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>30,strain</th>\n",
       "      <th>31,strain</th>\n",
       "      <th>32,strain</th>\n",
       "      <th>33,strain</th>\n",
       "      <th>34,strain</th>\n",
       "      <th>...</th>\n",
       "      <th>467,strain</th>\n",
       "      <th>468,strain</th>\n",
       "      <th>469,strain</th>\n",
       "      <th>470,strain</th>\n",
       "      <th>471,strain</th>\n",
       "      <th>472,geo_loc_name</th>\n",
       "      <th>473,strain</th>\n",
       "      <th>474,strain</th>\n",
       "      <th>475,geo_loc_name</th>\n",
       "      <th>476,synonym</th>\n",
       "      <th>geographiclocation</th>\n",
       "      <th>latitudeandlongitude</th>\n",
       "      <th>ENA-FIRST-PUBLIC</th>\n",
       "      <th>ENA-LAST-UPDATE</th>\n",
       "      <th>477,synonym</th>\n",
       "      <th>478,synonym</th>\n",
       "      <th>479,strain</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>specific_host</th>\n",
       "      <th>anonymized_name</th>\n",
       "      <th>host_status</th>\n",
       "      <th>supplier_name</th>\n",
       "      <th>480,strain</th>\n",
       "      <th>481,strain</th>\n",
       "      <th>482,geo_loc_name</th>\n",
       "      <th>483,strain</th>\n",
       "      <th>484,synonym</th>\n",
       "      <th>485,strain</th>\n",
       "      <th>486,strain</th>\n",
       "      <th>487,strain</th>\n",
       "      <th>488,strain</th>\n",
       "      <th>489,\\n</th>\n",
       "      <th>490,strain</th>\n",
       "      <th>ArrayExpress-StrainOrLine</th>\n",
       "      <th>ArrayExpress-Species</th>\n",
       "      <th>491,\\n</th>\n",
       "      <th>492,strain</th>\n",
       "      <th>493,strain</th>\n",
       "      <th>494,\\n</th>\n",
       "      <th>495,strain</th>\n",
       "      <th>ArrayExpress-Phenotype</th>\n",
       "      <th>496,Strain</th>\n",
       "      <th>SampleDescription</th>\n",
       "      <th>497,\\n</th>\n",
       "      <th>498,\\n</th>\n",
       "      <th>499,Strain</th>\n",
       "      <th>500,\\n</th>\n",
       "      <th>501,\\n</th>\n",
       "      <th>502,\\n</th>\n",
       "      <th>503,Strain</th>\n",
       "      <th>504,\\n</th>\n",
       "      <th>505,\\n</th>\n",
       "      <th>506,strain</th>\n",
       "      <th>507,strain</th>\n",
       "      <th>508,\\n</th>\n",
       "      <th>509,strain</th>\n",
       "      <th>510,\\n</th>\n",
       "      <th>511,strain</th>\n",
       "      <th>512,strain</th>\n",
       "      <th>513,\\n</th>\n",
       "      <th>514,\\n</th>\n",
       "      <th>515,\\n</th>\n",
       "      <th>516,\\n</th>\n",
       "      <th>517,\\n</th>\n",
       "      <th>518,strain</th>\n",
       "      <th>519,strain</th>\n",
       "      <th>520,strain</th>\n",
       "      <th>521,strain</th>\n",
       "      <th>522,SUBJECT_ID</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>523,\\n</th>\n",
       "      <th>524,\\n</th>\n",
       "      <th>525,\\n</th>\n",
       "      <th>526,\\n</th>\n",
       "      <th>527,Alias</th>\n",
       "      <th>ENAchecklist</th>\n",
       "      <th>INSDCcenteralias</th>\n",
       "      <th>INSDCcentername</th>\n",
       "      <th>INSDCfirstpublic</th>\n",
       "      <th>INSDClastupdate</th>\n",
       "      <th>INSDCstatus</th>\n",
       "      <th>SRAaccession</th>\n",
       "      <th>SampleName</th>\n",
       "      <th>Title</th>\n",
       "      <th>environment(biome)</th>\n",
       "      <th>environment(feature)</th>\n",
       "      <th>environment(material)</th>\n",
       "      <th>geographiclocation(latitude)</th>\n",
       "      <th>geographiclocation(longitude)</th>\n",
       "      <th>plant-associatedenvironmentalpackage</th>\n",
       "      <th>projectname</th>\n",
       "      <th>528,Alias</th>\n",
       "      <th>529,Alias</th>\n",
       "      <th>530,collected_by</th>\n",
       "      <th>531,\\n</th>\n",
       "      <th>532,\\n</th>\n",
       "      <th>533,\\n</th>\n",
       "      <th>534,\\n</th>\n",
       "      <th>535,\\n</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[]</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>type strain of Capnocytophaga ochracea\\n</td>\n",
       "      <td>ATCC 35910</td>\n",
       "      <td>HMP supersite', 'urogenital_tract</td>\n",
       "      <td>Reference Genome</td>\n",
       "      <td>5562000</td>\n",
       "      <td>ATCC:35910</td>\n",
       "      <td>Homo sapiens-associated habitat [ENVO:00009003]</td>\n",
       "      <td>missing</td>\n",
       "      <td>9606</td>\n",
       "      <td>not determined</td>\n",
       "      <td>Level 6', 'Finished33.84x;7</td>\n",
       "      <td>biological product [ENVO:02000043]</td>\n",
       "      <td>http://hmpdacc.org/doc/CommonGeneAnnotation_SO...</td>\n",
       "      <td>not determined</td>\n",
       "      <td>not determined</td>\n",
       "      <td>not determined</td>\n",
       "      <td>terrestrial biome [ENVO:00000446]</td>\n",
       "      <td>not determined</td>\n",
       "      <td>missing</td>\n",
       "      <td>not determined</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>454-GS-FLX, Illumina</td>\n",
       "      <td>Newbler v. 2.0.0-Eclipse</td>\n",
       "      <td>not determined</td>\n",
       "      <td>ATCC 35910, CL4/79, IFO 15054, JCM 2410, NCTC ...</td>\n",
       "      <td>Chryseobacterium gleum F93, ATCC 35910</td>\n",
       "      <td>missing</td>\n",
       "      <td>MIGS.ba\"\\n</td>\n",
       "      <td>Level 2', 'High-Quality Draft19x;65</td>\n",
       "      <td>not determined</td>\n",
       "      <td>missing</td>\n",
       "      <td>Capno</td>\n",
       "      <td>missing</td>\n",
       "      <td>Level 2', 'High-Quality Draft37x;37</td>\n",
       "      <td>symbiont</td>\n",
       "      <td>Duck blood</td>\n",
       "      <td>37C</td>\n",
       "      <td>Blood</td>\n",
       "      <td>Nonmotile</td>\n",
       "      <td>Gi04613</td>\n",
       "      <td>Septicemia</td>\n",
       "      <td>1954</td>\n",
       "      <td>Missing</td>\n",
       "      <td>isolated by Bruner and Fabricant</td>\n",
       "      <td>Mesophile</td>\n",
       "      <td>Yes</td>\n",
       "      <td>microaerophilic</td>\n",
       "      <td>Missing</td>\n",
       "      <td>DOE-GEBA 2007</td>\n",
       "      <td>Nonsporulating</td>\n",
       "      <td>Missing</td>\n",
       "      <td>USA</td>\n",
       "      <td>Missing</td>\n",
       "      <td>Rod-shaped</td>\n",
       "      <td>Septicemic disease in waterfowls. Septicemia a...</td>\n",
       "      <td>Anas platyrhynchos (duck), gucks, geese,turkey...</td>\n",
       "      <td>Level 2', 'High-Quality Draft34.5x;61</td>\n",
       "      <td>free living</td>\n",
       "      <td>7.5</td>\n",
       "      <td>Gram-</td>\n",
       "      <td>Prodigal 1.4, GenePRIMP</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>Level 2', 'High-Quality Draft37.97x;74</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>Level 2', 'High-Quality Draft157</td>\n",
       "      <td>missing</td>\n",
       "      <td>free living</td>\n",
       "      <td>free living</td>\n",
       "      <td>missing</td>\n",
       "      <td>free living</td>\n",
       "      <td>Turkey, respiratory tract; United Kingdom</td>\n",
       "      <td>Turkey</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>Respiratory tract</td>\n",
       "      <td>Level 2', 'High-Quality Draft32x;62</td>\n",
       "      <td>Level 2', 'High-Quality Draft10x;42</td>\n",
       "      <td>Level 2', 'High-Quality Draft11x;38</td>\n",
       "      <td>CF136\\n</td>\n",
       "      <td>CF314\\n</td>\n",
       "      <td>Level 2', 'High-Quality Draft33x;75</td>\n",
       "      <td>AK20\\n</td>\n",
       "      <td>DSM 16469</td>\n",
       "      <td>pigeons</td>\n",
       "      <td>missing</td>\n",
       "      <td>Mouth of a dog; USA, Virginia</td>\n",
       "      <td>SCGC AAA160-P02</td>\n",
       "      <td>single cell isolate</td>\n",
       "      <td>ATCC 13253</td>\n",
       "      <td>DSM 19726</td>\n",
       "      <td>DSM 18436</td>\n",
       "      <td>Gliding; Catalase positive</td>\n",
       "      <td>JM1</td>\n",
       "      <td>DSM 12212</td>\n",
       "      <td>DSM 17708</td>\n",
       "      <td>DSM 15936</td>\n",
       "      <td>DSM 22377</td>\n",
       "      <td>...</td>\n",
       "      <td>DSM 23546</td>\n",
       "      <td>DSM 23405</td>\n",
       "      <td>DSM 23400</td>\n",
       "      <td>DSM 16775</td>\n",
       "      <td>DSM 23401</td>\n",
       "      <td>missing</td>\n",
       "      <td>DSM 21164</td>\n",
       "      <td>CGMCC 1.12708</td>\n",
       "      <td>missing</td>\n",
       "      <td>Gp0194197</td>\n",
       "      <td>South Korea:Busan</td>\n",
       "      <td>35.1 N 129.01 E</td>\n",
       "      <td>2017-05-03</td>\n",
       "      <td>2017-05-03\\n</td>\n",
       "      <td>Gp0194253</td>\n",
       "      <td>Gp0194254</td>\n",
       "      <td>NCTC11014</td>\n",
       "      <td>[]</td>\n",
       "      <td>Duck</td>\n",
       "      <td>2660916</td>\n",
       "      <td>not applicable</td>\n",
       "      <td>NCTC11014</td>\n",
       "      <td>NCTC13490</td>\n",
       "      <td>NCTC12947</td>\n",
       "      <td>missing</td>\n",
       "      <td>DSM 29150</td>\n",
       "      <td>Gp0194240</td>\n",
       "      <td>DSM 27993</td>\n",
       "      <td>DSM 19840</td>\n",
       "      <td>CGMCC 1.12641</td>\n",
       "      <td>DSM 25885</td>\n",
       "      <td>[]</td>\n",
       "      <td>NCTC11660</td>\n",
       "      <td>NCTC11660</td>\n",
       "      <td>Bergeyella zoohelcum\\n</td>\n",
       "      <td>[]</td>\n",
       "      <td>NCTC12948</td>\n",
       "      <td>NCTC11653</td>\n",
       "      <td>[]</td>\n",
       "      <td>NCTC11212</td>\n",
       "      <td>Not applicable</td>\n",
       "      <td>NCTC13533</td>\n",
       "      <td></td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>NCTC13532</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>NCTC10588</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>NCTC11180</td>\n",
       "      <td>NCTC11036</td>\n",
       "      <td>[]</td>\n",
       "      <td>NCTC12242</td>\n",
       "      <td>[]</td>\n",
       "      <td>NCTC10796</td>\n",
       "      <td>NCTC10016</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>NCTC11432</td>\n",
       "      <td>NCTC13529</td>\n",
       "      <td>NCTC11634</td>\n",
       "      <td>NCTC13489</td>\n",
       "      <td>3012STDY6944352</td>\n",
       "      <td>NCTC11097</td>\n",
       "      <td>Capnocytophaga sputigena</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>Flavo_2</td>\n",
       "      <td>ERC000020</td>\n",
       "      <td>NIOO KNAW</td>\n",
       "      <td>NIOO KNAW</td>\n",
       "      <td>2019-07-09T17:06:29Z</td>\n",
       "      <td>2019-07-09T15:28:25Z</td>\n",
       "      <td>public</td>\n",
       "      <td>ERS3568299</td>\n",
       "      <td>ERS3568299</td>\n",
       "      <td>endosphere flavobacterium isolate</td>\n",
       "      <td>area of cropland</td>\n",
       "      <td>area of cropland</td>\n",
       "      <td>endosphere environment</td>\n",
       "      <td>51.9691868</td>\n",
       "      <td>5.665394800000058</td>\n",
       "      <td>plant-associated</td>\n",
       "      <td>Unlock the secrets of disease suppressive soils</td>\n",
       "      <td>Flavo_1</td>\n",
       "      <td>Flavo_4</td>\n",
       "      <td>Institut de CiÃƒÂ¨ncies del Mar (ICM-CSIC)</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Chryseobacterium gleum</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HMP body site', 'Vaginal</td>\n",
       "      <td>Reference Genome</td>\n",
       "      <td>2998000</td>\n",
       "      <td>ATCC:33624</td>\n",
       "      <td>human-associated habitat [ENVO:00009003]</td>\n",
       "      <td>bacteria_archaea</td>\n",
       "      <td>9606</td>\n",
       "      <td>not determined</td>\n",
       "      <td>NaN</td>\n",
       "      <td>biological product [ENVO:02000043]</td>\n",
       "      <td>http://hmpdacc.org/doc/CommonGeneAnnotation_SO...</td>\n",
       "      <td>not determined</td>\n",
       "      <td>not determined</td>\n",
       "      <td>not determined</td>\n",
       "      <td>terrestrial biome [ENVO:00000446]</td>\n",
       "      <td>not determined</td>\n",
       "      <td>missing</td>\n",
       "      <td>not determined</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>454-GS-FLX</td>\n",
       "      <td>Newbler v. 1.1.03.24-64-7</td>\n",
       "      <td>not determined</td>\n",
       "      <td>ATCC 33612</td>\n",
       "      <td>Capnocytophaga sputigena Capno, ATCC 33612</td>\n",
       "      <td>missing</td>\n",
       "      <td>MIGS.ba\"\\n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>not determined</td>\n",
       "      <td>missing</td>\n",
       "      <td>ATCC 33624</td>\n",
       "      <td>missing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>surfaces of algal species from the Antarctic m...</td>\n",
       "      <td>30C</td>\n",
       "      <td>Airways</td>\n",
       "      <td>Motile</td>\n",
       "      <td>Gi04009</td>\n",
       "      <td>None</td>\n",
       "      <td>Missing</td>\n",
       "      <td>Missing</td>\n",
       "      <td>isolated by U. Maschmann</td>\n",
       "      <td>Mesophile</td>\n",
       "      <td>Yes</td>\n",
       "      <td>aerobe</td>\n",
       "      <td>Missing</td>\n",
       "      <td>DOE-GEBA 2009</td>\n",
       "      <td>Nonsporulating</td>\n",
       "      <td>chemoheterotroph</td>\n",
       "      <td>Missing</td>\n",
       "      <td>Missing</td>\n",
       "      <td>Rod-shaped\"\\n</td>\n",
       "      <td>Associated with respiratory disease in pigeons</td>\n",
       "      <td>Pyramimonas gelidicola</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5.5-11.0</td>\n",
       "      <td>Gram-</td>\n",
       "      <td>Prodigal 2.5</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>free living</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>missing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aporrectodea caliginosa</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 13253 = NBRC 12535</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Catalase positive</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2017-05-03</td>\n",
       "      <td>2017-05-03\\n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>[]</td>\n",
       "      <td>not applicable</td>\n",
       "      <td>2755631</td>\n",
       "      <td>not applicable</td>\n",
       "      <td>NCTC13490</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NCTC12948</td>\n",
       "      <td>Capnocytophaga granulosa\\n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>N/A</td>\n",
       "      <td>NaN</td>\n",
       "      <td></td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ERC000020</td>\n",
       "      <td>NIOO KNAW</td>\n",
       "      <td>NIOO KNAW</td>\n",
       "      <td>2019-07-09T17:06:29Z</td>\n",
       "      <td>2019-07-09T15:28:25Z</td>\n",
       "      <td>public</td>\n",
       "      <td>ERS3568298</td>\n",
       "      <td>ERS3568298</td>\n",
       "      <td>endosphere flavobacterium isolate</td>\n",
       "      <td>area of cropland</td>\n",
       "      <td>area of cropland</td>\n",
       "      <td>endosphere environment</td>\n",
       "      <td>51.9691868</td>\n",
       "      <td>5.665394800000058</td>\n",
       "      <td>plant-associated</td>\n",
       "      <td>Unlock the secrets of disease suppressive soils</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 692 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  ,sample_attribute\\n  0,strain                             type-material  \\\n",
       "0                  []  DSM 7271  type strain of Capnocytophaga ochracea\\n   \n",
       "1                 NaN       NaN     type strain of Chryseobacterium gleum   \n",
       "\n",
       "     1,strain                         misc_param      project_type  \\\n",
       "0  ATCC 35910  HMP supersite', 'urogenital_tract  Reference Genome   \n",
       "1         NaN           HMP body site', 'Vaginal  Reference Genome   \n",
       "\n",
       "  estimated_size culture-collection  \\\n",
       "0        5562000         ATCC:35910   \n",
       "1        2998000         ATCC:33624   \n",
       "\n",
       "                                           feature investigation_type  \\\n",
       "0  Homo sapiens-associated habitat [ENVO:00009003]            missing   \n",
       "1         human-associated habitat [ENVO:00009003]   bacteria_archaea   \n",
       "\n",
       "  hosttaxid nucleicacidextraction finishingstrategy(depthofcoverage)  \\\n",
       "0      9606        not determined        Level 6', 'Finished33.84x;7   \n",
       "1      9606        not determined                                NaN   \n",
       "\n",
       "                             material  \\\n",
       "0  biological product [ENVO:02000043]   \n",
       "1  biological product [ENVO:02000043]   \n",
       "\n",
       "                                                 sop   num_replicons  \\\n",
       "0  http://hmpdacc.org/doc/CommonGeneAnnotation_SO...  not determined   \n",
       "1  http://hmpdacc.org/doc/CommonGeneAnnotation_SO...  not determined   \n",
       "\n",
       "  ref_biomaterial    geo_loc_name                              biome  \\\n",
       "0  not determined  not determined  terrestrial biome [ENVO:00000446]   \n",
       "1  not determined  not determined  terrestrial biome [ENVO:00000446]   \n",
       "\n",
       "           latlon env_package  collectiondate  specifichost  \\\n",
       "0  not determined     missing  not determined  Homo sapiens   \n",
       "1  not determined     missing  not determined  Homo sapiens   \n",
       "\n",
       "       sequencingmethod                   assembly isol_growth_condt  \\\n",
       "0  454-GS-FLX, Illumina   Newbler v. 2.0.0-Eclipse    not determined   \n",
       "1            454-GS-FLX  Newbler v. 1.1.03.24-64-7    not determined   \n",
       "\n",
       "                                       source_mat_id  \\\n",
       "0  ATCC 35910, CL4/79, IFO 15054, JCM 2410, NCTC ...   \n",
       "1                                         ATCC 33612   \n",
       "\n",
       "                                 project_name isolation-source BioSampleModel  \\\n",
       "0      Chryseobacterium gleum F93, ATCC 35910          missing     MIGS.ba\"\\n   \n",
       "1  Capnocytophaga sputigena Capno, ATCC 33612          missing     MIGS.ba\"\\n   \n",
       "\n",
       "  2,finishingstrategy(depthofcoverage)         lat_lon investigationtype  \\\n",
       "0  Level 2', 'High-Quality Draft19x;65  not determined           missing   \n",
       "1                                  NaN  not determined           missing   \n",
       "\n",
       "       strain environmentalpackage 3,finishingstrategy(depthofcoverage)  \\\n",
       "0       Capno              missing  Level 2', 'High-Quality Draft37x;37   \n",
       "1  ATCC 33624              missing                                  NaN   \n",
       "\n",
       "  4,biotic_relationship                                      IsolationSite  \\\n",
       "0              symbiont                                         Duck blood   \n",
       "1                   NaN  surfaces of algal species from the Antarctic m...   \n",
       "\n",
       "  TemperatureOptimum BodySampleSite   Motility GOLDStampID    Diseases  \\\n",
       "0                37C          Blood  Nonmotile     Gi04613  Septicemia   \n",
       "1                30C        Airways     Motile     Gi04009        None   \n",
       "\n",
       "  collection_date environment                 IsolationComments  \\\n",
       "0            1954     Missing  isolated by Bruner and Fabricant   \n",
       "1         Missing     Missing          isolated by U. Maschmann   \n",
       "\n",
       "  TemperatureRange TypeStrain    rel_to_oxygen alt_elev FundingProgram  \\\n",
       "0        Mesophile        Yes  microaerophilic  Missing  DOE-GEBA 2007   \n",
       "1        Mesophile        Yes           aerobe  Missing  DOE-GEBA 2009   \n",
       "\n",
       "      Sporulation     trophic_level  country    depth      CellShape  \\\n",
       "0  Nonsporulating           Missing      USA  Missing     Rod-shaped   \n",
       "1  Nonsporulating  chemoheterotroph  Missing  Missing  Rod-shaped\"\\n   \n",
       "\n",
       "                                          HostHealth  \\\n",
       "0  Septicemic disease in waterfowls. Septicemia a...   \n",
       "1     Associated with respiratory disease in pigeons   \n",
       "\n",
       "                                            HostName  \\\n",
       "0  Anas platyrhynchos (duck), gucks, geese,turkey...   \n",
       "1                             Pyramimonas gelidicola   \n",
       "\n",
       "    5,finishingstrategy(depthofcoverage) 6,biotic_relationship        pH  \\\n",
       "0  Level 2', 'High-Quality Draft34.5x;61           free living       7.5   \n",
       "1                                    NaN                   NaN  5.5-11.0   \n",
       "\n",
       "  GramStaining        GeneCallingMethod 7,\\n 8,\\n  \\\n",
       "0        Gram-  Prodigal 1.4, GenePRIMP   []   []   \n",
       "1        Gram-             Prodigal 2.5  NaN  NaN   \n",
       "\n",
       "     9,finishingstrategy(depthofcoverage) 10,\\n 11,\\n  \\\n",
       "0  Level 2', 'High-Quality Draft37.97x;74    []    []   \n",
       "1                                     NaN   NaN   NaN   \n",
       "\n",
       "  12,finishingstrategy(depthofcoverage) 13,env_package biotic_relationship  \\\n",
       "0      Level 2', 'High-Quality Draft157        missing         free living   \n",
       "1                                   NaN            NaN         free living   \n",
       "\n",
       "  14,biotic_relationship 15,env_package 16,biotic_relationship  \\\n",
       "0            free living        missing            free living   \n",
       "1                    NaN            NaN                    NaN   \n",
       "\n",
       "                             Isolationsource    Host    geo-loc-name  \\\n",
       "0  Turkey, respiratory tract; United Kingdom  Turkey  United Kingdom   \n",
       "1                                        NaN     NaN         missing   \n",
       "\n",
       "   BodySampleSubSite 17,finishingstrategy(depthofcoverage)  \\\n",
       "0  Respiratory tract   Level 2', 'High-Quality Draft32x;62   \n",
       "1                NaN                                   NaN   \n",
       "\n",
       "  18,finishingstrategy(depthofcoverage) 19,finishingstrategy(depthofcoverage)  \\\n",
       "0   Level 2', 'High-Quality Draft10x;42   Level 2', 'High-Quality Draft11x;38   \n",
       "1                                   NaN                                   NaN   \n",
       "\n",
       "  20,strain 21,strain 22,finishingstrategy(depthofcoverage) 23,strain  \\\n",
       "0   CF136\\n   CF314\\n   Level 2', 'High-Quality Draft33x;75    AK20\\n   \n",
       "1       NaN       NaN                                   NaN       NaN   \n",
       "\n",
       "   24,strain                     host 25,env_package  \\\n",
       "0  DSM 16469                  pigeons        missing   \n",
       "1        NaN  Aporrectodea caliginosa            NaN   \n",
       "\n",
       "                 isolationsource        26,strain                     note  \\\n",
       "0  Mouth of a dog; USA, Virginia  SCGC AAA160-P02      single cell isolate   \n",
       "1                            NaN              NaN  ATCC 13253 = NBRC 12535   \n",
       "\n",
       "    27,strain  28,strain  29,strain                  Phenotypes 30,strain  \\\n",
       "0  ATCC 13253  DSM 19726  DSM 18436  Gliding; Catalase positive       JM1   \n",
       "1         NaN        NaN        NaN           Catalase positive       NaN   \n",
       "\n",
       "   31,strain  32,strain  33,strain  34,strain  ... 467,strain 468,strain  \\\n",
       "0  DSM 12212  DSM 17708  DSM 15936  DSM 22377  ...  DSM 23546  DSM 23405   \n",
       "1        NaN        NaN        NaN        NaN  ...        NaN        NaN   \n",
       "\n",
       "  469,strain 470,strain 471,strain 472,geo_loc_name 473,strain     474,strain  \\\n",
       "0  DSM 23400  DSM 16775  DSM 23401          missing  DSM 21164  CGMCC 1.12708   \n",
       "1        NaN        NaN        NaN              NaN        NaN            NaN   \n",
       "\n",
       "  475,geo_loc_name 476,synonym geographiclocation latitudeandlongitude  \\\n",
       "0          missing   Gp0194197  South Korea:Busan      35.1 N 129.01 E   \n",
       "1              NaN         NaN                NaN                  NaN   \n",
       "\n",
       "  ENA-FIRST-PUBLIC ENA-LAST-UPDATE 477,synonym 478,synonym 479,strain  \\\n",
       "0       2017-05-03    2017-05-03\\n   Gp0194253   Gp0194254  NCTC11014   \n",
       "1       2017-05-03    2017-05-03\\n         NaN         NaN        NaN   \n",
       "\n",
       "  sample_description   specific_host anonymized_name     host_status  \\\n",
       "0                 []            Duck         2660916  not applicable   \n",
       "1                 []  not applicable         2755631  not applicable   \n",
       "\n",
       "  supplier_name 480,strain 481,strain 482,geo_loc_name 483,strain 484,synonym  \\\n",
       "0     NCTC11014  NCTC13490  NCTC12947          missing  DSM 29150   Gp0194240   \n",
       "1     NCTC13490        NaN        NaN              NaN        NaN         NaN   \n",
       "\n",
       "  485,strain 486,strain     487,strain 488,strain 489,\\n 490,strain  \\\n",
       "0  DSM 27993  DSM 19840  CGMCC 1.12641  DSM 25885     []  NCTC11660   \n",
       "1        NaN        NaN            NaN        NaN    NaN        NaN   \n",
       "\n",
       "  ArrayExpress-StrainOrLine        ArrayExpress-Species 491,\\n 492,strain  \\\n",
       "0                 NCTC11660      Bergeyella zoohelcum\\n     []  NCTC12948   \n",
       "1                 NCTC12948  Capnocytophaga granulosa\\n    NaN        NaN   \n",
       "\n",
       "  493,strain 494,\\n 495,strain ArrayExpress-Phenotype 496,Strain  \\\n",
       "0  NCTC11653     []  NCTC11212         Not applicable  NCTC13533   \n",
       "1        NaN    NaN        NaN                    N/A        NaN   \n",
       "\n",
       "  SampleDescription 497,\\n 498,\\n 499,Strain 500,\\n 501,\\n 502,\\n 503,Strain  \\\n",
       "0                       []     []  NCTC13532     []     []     []  NCTC10588   \n",
       "1                      NaN    NaN        NaN    NaN    NaN    NaN        NaN   \n",
       "\n",
       "  504,\\n 505,\\n 506,strain 507,strain 508,\\n 509,strain 510,\\n 511,strain  \\\n",
       "0     []     []  NCTC11180  NCTC11036     []  NCTC12242     []  NCTC10796   \n",
       "1    NaN    NaN        NaN        NaN    NaN        NaN    NaN        NaN   \n",
       "\n",
       "  512,strain 513,\\n 514,\\n 515,\\n 516,\\n 517,\\n 518,strain 519,strain  \\\n",
       "0  NCTC10016     []     []     []     []     []  NCTC11432  NCTC13529   \n",
       "1        NaN    NaN    NaN    NaN    NaN    NaN        NaN        NaN   \n",
       "\n",
       "  520,strain 521,strain   522,SUBJECT_ID ArrayExpress-STRAIN_OR_LINE  \\\n",
       "0  NCTC11634  NCTC13489  3012STDY6944352                   NCTC11097   \n",
       "1        NaN        NaN              NaN                         NaN   \n",
       "\n",
       "       ArrayExpress-SPECIES 523,\\n 524,\\n 525,\\n 526,\\n 527,Alias  \\\n",
       "0  Capnocytophaga sputigena     []     []     []     []   Flavo_2   \n",
       "1                       NaN    NaN    NaN    NaN    NaN       NaN   \n",
       "\n",
       "  ENAchecklist INSDCcenteralias INSDCcentername      INSDCfirstpublic  \\\n",
       "0    ERC000020        NIOO KNAW       NIOO KNAW  2019-07-09T17:06:29Z   \n",
       "1    ERC000020        NIOO KNAW       NIOO KNAW  2019-07-09T17:06:29Z   \n",
       "\n",
       "        INSDClastupdate INSDCstatus SRAaccession  SampleName  \\\n",
       "0  2019-07-09T15:28:25Z      public   ERS3568299  ERS3568299   \n",
       "1  2019-07-09T15:28:25Z      public   ERS3568298  ERS3568298   \n",
       "\n",
       "                               Title environment(biome) environment(feature)  \\\n",
       "0  endosphere flavobacterium isolate   area of cropland     area of cropland   \n",
       "1  endosphere flavobacterium isolate   area of cropland     area of cropland   \n",
       "\n",
       "    environment(material) geographiclocation(latitude)  \\\n",
       "0  endosphere environment                   51.9691868   \n",
       "1  endosphere environment                   51.9691868   \n",
       "\n",
       "  geographiclocation(longitude) plant-associatedenvironmentalpackage  \\\n",
       "0             5.665394800000058                     plant-associated   \n",
       "1             5.665394800000058                     plant-associated   \n",
       "\n",
       "                                       projectname 528,Alias 529,Alias  \\\n",
       "0  Unlock the secrets of disease suppressive soils   Flavo_1   Flavo_4   \n",
       "1  Unlock the secrets of disease suppressive soils       NaN       NaN   \n",
       "\n",
       "                             530,collected_by 531,\\n 532,\\n 533,\\n 534,\\n  \\\n",
       "0  Institut de CiÃƒÂ¨ncies del Mar (ICM-CSIC)     []     []     []     []   \n",
       "1                                         NaN    NaN    NaN    NaN    NaN   \n",
       "\n",
       "  535,\\n  \n",
       "0     []  \n",
       "1    NaN  \n",
       "\n",
       "[2 rows x 692 columns]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "att2.replace(regex=True,inplace=True,to_replace=r\"\\['\",value=r'')\n",
    "att2.replace(regex=True,inplace=True,to_replace=r\"\\']\",value=r'')\n",
    "att2.head(2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 3.1 Get counts:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "d ={}\n",
    "for col in att2.columns:\n",
    "    tmp_series = att2[col].value_counts()\n",
    "    tmp_series.name = col\n",
    "    d[col]=tmp_series"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.series.Series'>\n"
     ]
    }
   ],
   "source": [
    "df = pd.concat(d.values(), keys=d.keys())\n",
    "print(type(df))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "df.to_csv(\"counts_final.csv\", header=[\"count\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 3.2 Sample attribute division per columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "sample_att = code_sra.copy()\n",
    "sample_att2 = sample_att[\"sample_attribute\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "sample_att2 = sample_att[[\"assembly_accession\",\"sample_attribute\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/gomes/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:1: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  \"\"\"Entry point for launching an IPython kernel.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>sample_attribute</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>strain: DSM 7271,type-material: type strain of...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000143785.1</td>\n",
       "      <td>strain: ATCC 35910,misc_param: HMP supersite: ...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  assembly_accession                                   sample_attribute\n",
       "0    GCA_000023285.1  strain: DSM 7271,type-material: type strain of...\n",
       "1    GCA_000143785.1  strain: ATCC 35910,misc_param: HMP supersite: ..."
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sample_att2[\"sample_attribute\"] = sample_att2.sample_attribute.str.replace(r\" \\|\\| \", \",\")\n",
    "sample_att2.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "assembly_accession = sample_att2[\"assembly_accession\"].to_list()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "splitted = sample_att2.sample_attribute.str.split(\",\", expand=True)\n",
    "splitted.drop(splitted.tail(2).index,inplace=True)\n",
    "splitted = splitted.rename(index=sample_att[\"assembly_accession\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "col_list2 = []\n",
    "for i, cell in splitted.iterrows():\n",
    "    cell2 = cell[0]\n",
    "    cell2 = str(cell2)\n",
    "    key = cell2.split(\": \")[0]\n",
    "    value = cell2.split(\": \")[1:]\n",
    "    if key not in col_list2:\n",
    "        col_list2.append(key)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i, cell in splitted.iteritems():\n",
    "    for x in cell:\n",
    "        if x is None:\n",
    "            continue\n",
    "        else:\n",
    "            x = str(x)\n",
    "            if \":\" in x:\n",
    "                key = x.split(\": \")[0]\n",
    "                #print(key)\n",
    "                #value = x.split(\": \")[1:]\n",
    "                if key not in col_list2:\n",
    "                     col_list2.append(key)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>strain</th>\n",
       "      <th>finishing strategy (depth of coverage)</th>\n",
       "      <th>biotic_relationship</th>\n",
       "      <th>nan</th>\n",
       "      <th>env_package</th>\n",
       "      <th>geo_loc_name</th>\n",
       "      <th>host</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>host_disease</th>\n",
       "      <th>isolation_source</th>\n",
       "      <th>sample_name</th>\n",
       "      <th>isolation-source</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>FDA CVM ID</th>\n",
       "      <th>culture_collection</th>\n",
       "      <th>assembly_method</th>\n",
       "      <th>collection_date</th>\n",
       "      <th>env_feature</th>\n",
       "      <th>synonym</th>\n",
       "      <th>Strain</th>\n",
       "      <th>SUBJECT_ID</th>\n",
       "      <th>Alias</th>\n",
       "      <th>type-material</th>\n",
       "      <th>misc_param</th>\n",
       "      <th>collection date</th>\n",
       "      <th>Isolation Site</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>country</th>\n",
       "      <th>note</th>\n",
       "      <th>GOLD Stamp ID</th>\n",
       "      <th>investigation_type</th>\n",
       "      <th>culture-collection</th>\n",
       "      <th>lat_lon</th>\n",
       "      <th>alt_elev</th>\n",
       "      <th>locus_tag_prefix</th>\n",
       "      <th>nat-host</th>\n",
       "      <th>assembly_method_version</th>\n",
       "      <th>env_material</th>\n",
       "      <th>model</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>Sample Description</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>project_type</th>\n",
       "      <th>estimated_size</th>\n",
       "      <th>Temperature Optimum</th>\n",
       "      <th>isol_growth_condt</th>\n",
       "      <th>Motility</th>\n",
       "      <th>feature</th>\n",
       "      <th>project_name</th>\n",
       "      <th>assembly</th>\n",
       "      <th>num_replicons</th>\n",
       "      <th>isolate</th>\n",
       "      <th>depth</th>\n",
       "      <th>env_biome</th>\n",
       "      <th>altitude</th>\n",
       "      <th>ref_biomaterial</th>\n",
       "      <th>package</th>\n",
       "      <th>geographic location</th>\n",
       "      <th>ArrayExpress-StrainOrLine</th>\n",
       "      <th>ArrayExpress-Phenotype</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>geo-loc-name</th>\n",
       "      <th>environment</th>\n",
       "      <th>host_sex</th>\n",
       "      <th>elev</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>completeness_estimated</th>\n",
       "      <th>geographic location (country and/or sea)</th>\n",
       "      <th>latitude and longitude</th>\n",
       "      <th>specific_host</th>\n",
       "      <th>ArrayExpress-Species</th>\n",
       "      <th>ENA-FIRST-PUBLIC</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>specific host</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>pH</th>\n",
       "      <th>biome</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>isolate_name_alias</th>\n",
       "      <th>contamination_estimated</th>\n",
       "      <th>host scientific name</th>\n",
       "      <th>anonymized_name</th>\n",
       "      <th>ENA-LAST-UPDATE</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>BioSampleModel</th>\n",
       "      <th>environmental_sample</th>\n",
       "      <th>material</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>host health state</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>lat lon</th>\n",
       "      <th>Diseases</th>\n",
       "      <th>strain_name_alias</th>\n",
       "      <th>genome_coverage</th>\n",
       "      <th>host_status</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>host taxid</th>\n",
       "      <th>isolation source</th>\n",
       "      <th>Gram Staining</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>sequencing_meth</th>\n",
       "      <th>Historical Monobactam Produced</th>\n",
       "      <th>identification method</th>\n",
       "      <th>supplier_name</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>nucleic acid extraction</th>\n",
       "      <th>host_disease_outcome</th>\n",
       "      <th>Unique Monobactam Clusters Identified</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>Sample Name</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Gene Calling Method</th>\n",
       "      <th>temp</th>\n",
       "      <th>Title</th>\n",
       "      <th>investigation type</th>\n",
       "      <th>Host</th>\n",
       "      <th>rel_to_oxygen</th>\n",
       "      <th>host_age</th>\n",
       "      <th>Species</th>\n",
       "      <th>mapping_method</th>\n",
       "      <th>serovar</th>\n",
       "      <th>sop</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>finishing_strategy</th>\n",
       "      <th>Genus</th>\n",
       "      <th>mapping_method_version</th>\n",
       "      <th>environment (biome)</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>metagenome_source</th>\n",
       "      <th>environment (feature)</th>\n",
       "      <th>trophic_level</th>\n",
       "      <th>metagenomic</th>\n",
       "      <th>environment (material)</th>\n",
       "      <th>Funding Program</th>\n",
       "      <th>quality_assessment_method</th>\n",
       "      <th>host_health_state</th>\n",
       "      <th>quality_assessment_method_version</th>\n",
       "      <th>geographic location (latitude)</th>\n",
       "      <th>source_mat_id</th>\n",
       "      <th>host_tissue_sampled</th>\n",
       "      <th>host_description</th>\n",
       "      <th>geographic location (longitude)</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>host_disease_stage</th>\n",
       "      <th>subsrc_note</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>value</th>\n",
       "      <th>Laboratory Host</th>\n",
       "      <th>plant-associated environmental package</th>\n",
       "      <th>Extraction Method</th>\n",
       "      <th>project name</th>\n",
       "      <th>sequencing method</th>\n",
       "      <th>Body Sample SubSite</th>\n",
       "      <th>passage_history</th>\n",
       "      <th>Extraction Date</th>\n",
       "      <th>sample type</th>\n",
       "      <th>environmental package</th>\n",
       "      <th>Passage Date</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>assembly_accession</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>GCA_000023285.1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GCA_000143785.1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                   strain finishing strategy (depth of coverage)  \\\n",
       "assembly_accession                                                 \n",
       "GCA_000023285.1       NaN                                    NaN   \n",
       "GCA_000143785.1       NaN                                    NaN   \n",
       "\n",
       "                   biotic_relationship  nan env_package geo_loc_name host  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                    NaN  NaN         NaN          NaN  NaN   \n",
       "GCA_000143785.1                    NaN  NaN         NaN          NaN  NaN   \n",
       "\n",
       "                   collected_by host_disease isolation_source sample_name  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1             NaN          NaN              NaN         NaN   \n",
       "GCA_000143785.1             NaN          NaN              NaN         NaN   \n",
       "\n",
       "                   isolation-source sample_type FDA CVM ID culture_collection  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1                 NaN         NaN        NaN                NaN   \n",
       "GCA_000143785.1                 NaN         NaN        NaN                NaN   \n",
       "\n",
       "                   assembly_method collection_date env_feature synonym Strain  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1                NaN             NaN         NaN     NaN    NaN   \n",
       "GCA_000143785.1                NaN             NaN         NaN     NaN    NaN   \n",
       "\n",
       "                   SUBJECT_ID Alias type-material misc_param collection date  \\\n",
       "assembly_accession                                                             \n",
       "GCA_000023285.1           NaN   NaN           NaN        NaN             NaN   \n",
       "GCA_000143785.1           NaN   NaN           NaN        NaN             NaN   \n",
       "\n",
       "                   Isolation Site Isolation source country note GOLD Stamp ID  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1               NaN              NaN     NaN  NaN           NaN   \n",
       "GCA_000143785.1               NaN              NaN     NaN  NaN           NaN   \n",
       "\n",
       "                   investigation_type culture-collection lat_lon alt_elev  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                   NaN                NaN     NaN      NaN   \n",
       "GCA_000143785.1                   NaN                NaN     NaN      NaN   \n",
       "\n",
       "                   locus_tag_prefix nat-host assembly_method_version  \\\n",
       "assembly_accession                                                     \n",
       "GCA_000023285.1                 NaN      NaN                     NaN   \n",
       "GCA_000143785.1                 NaN      NaN                     NaN   \n",
       "\n",
       "                   env_material model sample_description Sample Description  \\\n",
       "assembly_accession                                                            \n",
       "GCA_000023285.1             NaN   NaN                NaN                NaN   \n",
       "GCA_000143785.1             NaN   NaN                NaN                NaN   \n",
       "\n",
       "                   ArrayExpress-STRAIN_OR_LINE ENA checklist project_type  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                            NaN           NaN          NaN   \n",
       "GCA_000143785.1                            NaN           NaN          NaN   \n",
       "\n",
       "                   estimated_size Temperature Optimum isol_growth_condt  \\\n",
       "assembly_accession                                                        \n",
       "GCA_000023285.1               NaN                 NaN               NaN   \n",
       "GCA_000143785.1               NaN                 NaN               NaN   \n",
       "\n",
       "                   Motility feature project_name assembly num_replicons  \\\n",
       "assembly_accession                                                        \n",
       "GCA_000023285.1         NaN     NaN          NaN      NaN           NaN   \n",
       "GCA_000143785.1         NaN     NaN          NaN      NaN           NaN   \n",
       "\n",
       "                   isolate depth env_biome altitude ref_biomaterial package  \\\n",
       "assembly_accession                                                            \n",
       "GCA_000023285.1        NaN   NaN       NaN      NaN             NaN     NaN   \n",
       "GCA_000143785.1        NaN   NaN       NaN      NaN             NaN     NaN   \n",
       "\n",
       "                   geographic location ArrayExpress-StrainOrLine  \\\n",
       "assembly_accession                                                 \n",
       "GCA_000023285.1                    NaN                       NaN   \n",
       "GCA_000143785.1                    NaN                       NaN   \n",
       "\n",
       "                   ArrayExpress-Phenotype ArrayExpress-SPECIES  \\\n",
       "assembly_accession                                               \n",
       "GCA_000023285.1                       NaN                  NaN   \n",
       "GCA_000143785.1                       NaN                  NaN   \n",
       "\n",
       "                   INSDC center alias geo-loc-name environment host_sex elev  \\\n",
       "assembly_accession                                                             \n",
       "GCA_000023285.1                   NaN          NaN         NaN      NaN  NaN   \n",
       "GCA_000143785.1                   NaN          NaN         NaN      NaN  NaN   \n",
       "\n",
       "                   PublicAccession completeness_estimated  \\\n",
       "assembly_accession                                          \n",
       "GCA_000023285.1                NaN                    NaN   \n",
       "GCA_000143785.1                NaN                    NaN   \n",
       "\n",
       "                   geographic location (country and/or sea)  \\\n",
       "assembly_accession                                            \n",
       "GCA_000023285.1                                         NaN   \n",
       "GCA_000143785.1                                         NaN   \n",
       "\n",
       "                   latitude and longitude specific_host ArrayExpress-Species  \\\n",
       "assembly_accession                                                             \n",
       "GCA_000023285.1                       NaN           NaN                  NaN   \n",
       "GCA_000143785.1                       NaN           NaN                  NaN   \n",
       "\n",
       "                   ENA-FIRST-PUBLIC INSDC center name specific host  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1                 NaN               NaN           NaN   \n",
       "GCA_000143785.1                 NaN               NaN           NaN   \n",
       "\n",
       "                   Body Sample Site   pH biome biomaterial_provider  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1                 NaN  NaN   NaN                  NaN   \n",
       "GCA_000143785.1                 NaN  NaN   NaN                  NaN   \n",
       "\n",
       "                   isolate_name_alias contamination_estimated  \\\n",
       "assembly_accession                                              \n",
       "GCA_000023285.1                   NaN                     NaN   \n",
       "GCA_000143785.1                   NaN                     NaN   \n",
       "\n",
       "                   host scientific name anonymized_name ENA-LAST-UPDATE  \\\n",
       "assembly_accession                                                        \n",
       "GCA_000023285.1                     NaN             NaN             NaN   \n",
       "GCA_000143785.1                     NaN             NaN             NaN   \n",
       "\n",
       "                   INSDC first public Cell Shape BioSampleModel  \\\n",
       "assembly_accession                                                \n",
       "GCA_000023285.1                   NaN        NaN            NaN   \n",
       "GCA_000143785.1                   NaN        NaN            NaN   \n",
       "\n",
       "                   environmental_sample material identified_by  \\\n",
       "assembly_accession                                               \n",
       "GCA_000023285.1                     NaN      NaN           NaN   \n",
       "GCA_000143785.1                     NaN      NaN           NaN   \n",
       "\n",
       "                   host health state INSDC last update lat lon Diseases  \\\n",
       "assembly_accession                                                        \n",
       "GCA_000023285.1                  NaN               NaN     NaN      NaN   \n",
       "GCA_000143785.1                  NaN               NaN     NaN      NaN   \n",
       "\n",
       "                   strain_name_alias genome_coverage host_status INSDC status  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1                  NaN             NaN         NaN          NaN   \n",
       "GCA_000143785.1                  NaN             NaN         NaN          NaN   \n",
       "\n",
       "                   host taxid isolation source Gram Staining  \\\n",
       "assembly_accession                                             \n",
       "GCA_000023285.1           NaN              NaN           NaN   \n",
       "GCA_000143785.1           NaN              NaN           NaN   \n",
       "\n",
       "                   Temperature Range Sporulation sequencing_meth  \\\n",
       "assembly_accession                                                 \n",
       "GCA_000023285.1                  NaN         NaN             NaN   \n",
       "GCA_000143785.1                  NaN         NaN             NaN   \n",
       "\n",
       "                   Historical Monobactam Produced identification method  \\\n",
       "assembly_accession                                                        \n",
       "GCA_000023285.1                               NaN                   NaN   \n",
       "GCA_000143785.1                               NaN                   NaN   \n",
       "\n",
       "                   supplier_name SRA accession nucleic acid extraction  \\\n",
       "assembly_accession                                                       \n",
       "GCA_000023285.1              NaN           NaN                     NaN   \n",
       "GCA_000143785.1              NaN           NaN                     NaN   \n",
       "\n",
       "                   host_disease_outcome Unique Monobactam Clusters Identified  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1                     NaN                                   NaN   \n",
       "GCA_000143785.1                     NaN                                   NaN   \n",
       "\n",
       "                   ProjectAccession Sample Name Type Strain Phenotypes  \\\n",
       "assembly_accession                                                       \n",
       "GCA_000023285.1                 NaN         NaN         NaN        NaN   \n",
       "GCA_000143785.1                 NaN         NaN         NaN        NaN   \n",
       "\n",
       "                   Gene Calling Method temp Title investigation type Host  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                    NaN  NaN   NaN                NaN  NaN   \n",
       "GCA_000143785.1                    NaN  NaN   NaN                NaN  NaN   \n",
       "\n",
       "                   rel_to_oxygen host_age Species mapping_method serovar  sop  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1              NaN      NaN     NaN            NaN     NaN  NaN   \n",
       "GCA_000143785.1              NaN      NaN     NaN            NaN     NaN  NaN   \n",
       "\n",
       "                   Isolation Comments finishing_strategy Genus  \\\n",
       "assembly_accession                                               \n",
       "GCA_000023285.1                   NaN                NaN   NaN   \n",
       "GCA_000143785.1                   NaN                NaN   NaN   \n",
       "\n",
       "                   mapping_method_version environment (biome)  \\\n",
       "assembly_accession                                              \n",
       "GCA_000023285.1                       NaN                 NaN   \n",
       "GCA_000143785.1                       NaN                 NaN   \n",
       "\n",
       "                   attribute_package metagenome_source environment (feature)  \\\n",
       "assembly_accession                                                             \n",
       "GCA_000023285.1                  NaN               NaN                   NaN   \n",
       "GCA_000143785.1                  NaN               NaN                   NaN   \n",
       "\n",
       "                   trophic_level metagenomic environment (material)  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1              NaN         NaN                    NaN   \n",
       "GCA_000143785.1              NaN         NaN                    NaN   \n",
       "\n",
       "                   Funding Program quality_assessment_method  \\\n",
       "assembly_accession                                             \n",
       "GCA_000023285.1                NaN                       NaN   \n",
       "GCA_000143785.1                NaN                       NaN   \n",
       "\n",
       "                   host_health_state quality_assessment_method_version  \\\n",
       "assembly_accession                                                       \n",
       "GCA_000023285.1                  NaN                               NaN   \n",
       "GCA_000143785.1                  NaN                               NaN   \n",
       "\n",
       "                   geographic location (latitude) source_mat_id  \\\n",
       "assembly_accession                                                \n",
       "GCA_000023285.1                               NaN           NaN   \n",
       "GCA_000143785.1                               NaN           NaN   \n",
       "\n",
       "                   host_tissue_sampled host_description  \\\n",
       "assembly_accession                                        \n",
       "GCA_000023285.1                    NaN              NaN   \n",
       "GCA_000143785.1                    NaN              NaN   \n",
       "\n",
       "                   geographic location (longitude) Host Name  \\\n",
       "assembly_accession                                             \n",
       "GCA_000023285.1                                NaN       NaN   \n",
       "GCA_000143785.1                                NaN       NaN   \n",
       "\n",
       "                   host_disease_stage subsrc_note Host Health value  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1                   NaN         NaN         NaN   NaN   \n",
       "GCA_000143785.1                   NaN         NaN         NaN   NaN   \n",
       "\n",
       "                   Laboratory Host plant-associated environmental package  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                NaN                                    NaN   \n",
       "GCA_000143785.1                NaN                                    NaN   \n",
       "\n",
       "                   Extraction Method project name sequencing method  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1                  NaN          NaN               NaN   \n",
       "GCA_000143785.1                  NaN          NaN               NaN   \n",
       "\n",
       "                   Body Sample SubSite passage_history Extraction Date  \\\n",
       "assembly_accession                                                       \n",
       "GCA_000023285.1                    NaN             NaN             NaN   \n",
       "GCA_000143785.1                    NaN             NaN             NaN   \n",
       "\n",
       "                   sample type environmental package Passage Date  \n",
       "assembly_accession                                                 \n",
       "GCA_000023285.1            NaN                   NaN          NaN  \n",
       "GCA_000143785.1            NaN                   NaN          NaN  "
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.DataFrame(columns=col_list2, index=sample_att[\"assembly_accession\"])\n",
    "df.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i, cell in splitted.iterrows():\n",
    "    for x in cell:\n",
    "        cell = x\n",
    "        if type(cell) != None:\n",
    "            cell = str(cell)\n",
    "            cell2 = cell[0]\n",
    "            #print(cell)\n",
    "            key = cell.split(\": \")[0]#\n",
    "            if key in col_list2:\n",
    "                value = cell.split(\": \")[1:]\n",
    "                df.loc[i,key] = str(value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "d = {r'\\[':'', r'\\]':r''}\n",
    "df.replace(d, regex=True, inplace=True)\n",
    "df.drop(labels=[\"nan\"],axis=1, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Index: 536 entries, GCA_000023285.1 to GCA_902804485.1\n",
      "Columns: 162 entries, strain to Passage Date\n",
      "dtypes: object(162)\n",
      "memory usage: 702.6+ KB\n"
     ]
    }
   ],
   "source": [
    "df.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>strain</th>\n",
       "      <th>finishing strategy (depth of coverage)</th>\n",
       "      <th>biotic_relationship</th>\n",
       "      <th>env_package</th>\n",
       "      <th>geo_loc_name</th>\n",
       "      <th>host</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>host_disease</th>\n",
       "      <th>isolation_source</th>\n",
       "      <th>sample_name</th>\n",
       "      <th>isolation-source</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>FDA CVM ID</th>\n",
       "      <th>culture_collection</th>\n",
       "      <th>assembly_method</th>\n",
       "      <th>collection_date</th>\n",
       "      <th>env_feature</th>\n",
       "      <th>synonym</th>\n",
       "      <th>Strain</th>\n",
       "      <th>SUBJECT_ID</th>\n",
       "      <th>Alias</th>\n",
       "      <th>type-material</th>\n",
       "      <th>misc_param</th>\n",
       "      <th>collection date</th>\n",
       "      <th>Isolation Site</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>country</th>\n",
       "      <th>note</th>\n",
       "      <th>GOLD Stamp ID</th>\n",
       "      <th>investigation_type</th>\n",
       "      <th>culture-collection</th>\n",
       "      <th>lat_lon</th>\n",
       "      <th>alt_elev</th>\n",
       "      <th>locus_tag_prefix</th>\n",
       "      <th>nat-host</th>\n",
       "      <th>assembly_method_version</th>\n",
       "      <th>env_material</th>\n",
       "      <th>model</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>Sample Description</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>project_type</th>\n",
       "      <th>estimated_size</th>\n",
       "      <th>Temperature Optimum</th>\n",
       "      <th>isol_growth_condt</th>\n",
       "      <th>Motility</th>\n",
       "      <th>feature</th>\n",
       "      <th>project_name</th>\n",
       "      <th>assembly</th>\n",
       "      <th>num_replicons</th>\n",
       "      <th>isolate</th>\n",
       "      <th>depth</th>\n",
       "      <th>env_biome</th>\n",
       "      <th>altitude</th>\n",
       "      <th>ref_biomaterial</th>\n",
       "      <th>package</th>\n",
       "      <th>geographic location</th>\n",
       "      <th>ArrayExpress-StrainOrLine</th>\n",
       "      <th>ArrayExpress-Phenotype</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>geo-loc-name</th>\n",
       "      <th>environment</th>\n",
       "      <th>host_sex</th>\n",
       "      <th>elev</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>completeness_estimated</th>\n",
       "      <th>geographic location (country and/or sea)</th>\n",
       "      <th>latitude and longitude</th>\n",
       "      <th>specific_host</th>\n",
       "      <th>ArrayExpress-Species</th>\n",
       "      <th>ENA-FIRST-PUBLIC</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>specific host</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>pH</th>\n",
       "      <th>biome</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>isolate_name_alias</th>\n",
       "      <th>contamination_estimated</th>\n",
       "      <th>host scientific name</th>\n",
       "      <th>anonymized_name</th>\n",
       "      <th>ENA-LAST-UPDATE</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>BioSampleModel</th>\n",
       "      <th>environmental_sample</th>\n",
       "      <th>material</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>host health state</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>lat lon</th>\n",
       "      <th>Diseases</th>\n",
       "      <th>strain_name_alias</th>\n",
       "      <th>genome_coverage</th>\n",
       "      <th>host_status</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>host taxid</th>\n",
       "      <th>isolation source</th>\n",
       "      <th>Gram Staining</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>sequencing_meth</th>\n",
       "      <th>Historical Monobactam Produced</th>\n",
       "      <th>identification method</th>\n",
       "      <th>supplier_name</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>nucleic acid extraction</th>\n",
       "      <th>host_disease_outcome</th>\n",
       "      <th>Unique Monobactam Clusters Identified</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>Sample Name</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Gene Calling Method</th>\n",
       "      <th>temp</th>\n",
       "      <th>Title</th>\n",
       "      <th>investigation type</th>\n",
       "      <th>Host</th>\n",
       "      <th>rel_to_oxygen</th>\n",
       "      <th>host_age</th>\n",
       "      <th>Species</th>\n",
       "      <th>mapping_method</th>\n",
       "      <th>serovar</th>\n",
       "      <th>sop</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>finishing_strategy</th>\n",
       "      <th>Genus</th>\n",
       "      <th>mapping_method_version</th>\n",
       "      <th>environment (biome)</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>metagenome_source</th>\n",
       "      <th>environment (feature)</th>\n",
       "      <th>trophic_level</th>\n",
       "      <th>metagenomic</th>\n",
       "      <th>environment (material)</th>\n",
       "      <th>Funding Program</th>\n",
       "      <th>quality_assessment_method</th>\n",
       "      <th>host_health_state</th>\n",
       "      <th>quality_assessment_method_version</th>\n",
       "      <th>geographic location (latitude)</th>\n",
       "      <th>source_mat_id</th>\n",
       "      <th>host_tissue_sampled</th>\n",
       "      <th>host_description</th>\n",
       "      <th>geographic location (longitude)</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>host_disease_stage</th>\n",
       "      <th>subsrc_note</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>value</th>\n",
       "      <th>Laboratory Host</th>\n",
       "      <th>plant-associated environmental package</th>\n",
       "      <th>Extraction Method</th>\n",
       "      <th>project name</th>\n",
       "      <th>sequencing method</th>\n",
       "      <th>Body Sample SubSite</th>\n",
       "      <th>passage_history</th>\n",
       "      <th>Extraction Date</th>\n",
       "      <th>sample type</th>\n",
       "      <th>environmental package</th>\n",
       "      <th>Passage Date</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>assembly_accession</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>GCA_000023285.1</th>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GCA_000143785.1</th>\n",
       "      <td>ATCC 35910</td>\n",
       "      <td>Level 6, Finished33.84x;7</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Chryseobacterium gleum</td>\n",
       "      <td>HMP body site, Vaginal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC:35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Reference Genome</td>\n",
       "      <td>5562000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens-associated habitat ENVO:00009003</td>\n",
       "      <td>Chryseobacterium gleum F93</td>\n",
       "      <td>Newbler v. 2.0.0-Eclipse</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>terrestrial biome ENVO:00000446</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MIGS.ba</td>\n",
       "      <td>NaN</td>\n",
       "      <td>biological product ENVO:02000043</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9606</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>http://hmpdacc.org/doc/CommonGeneAnnotation_SO...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>454-GS-FLX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                        strain finishing strategy (depth of coverage)  \\\n",
       "assembly_accession                                                      \n",
       "GCA_000023285.1       DSM 7271                                    NaN   \n",
       "GCA_000143785.1     ATCC 35910              Level 6, Finished33.84x;7   \n",
       "\n",
       "                   biotic_relationship env_package geo_loc_name host  \\\n",
       "assembly_accession                                                     \n",
       "GCA_000023285.1                    NaN         NaN          NaN  NaN   \n",
       "GCA_000143785.1                    NaN         NaN          NaN  NaN   \n",
       "\n",
       "                   collected_by host_disease isolation_source sample_name  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1             NaN          NaN              NaN         NaN   \n",
       "GCA_000143785.1             NaN          NaN              NaN         NaN   \n",
       "\n",
       "                   isolation-source sample_type FDA CVM ID culture_collection  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1                 NaN         NaN        NaN                NaN   \n",
       "GCA_000143785.1                 NaN         NaN        NaN                NaN   \n",
       "\n",
       "                   assembly_method collection_date env_feature synonym Strain  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1                NaN             NaN         NaN     NaN    NaN   \n",
       "GCA_000143785.1                NaN             NaN         NaN     NaN    NaN   \n",
       "\n",
       "                   SUBJECT_ID Alias                           type-material  \\\n",
       "assembly_accession                                                            \n",
       "GCA_000023285.1           NaN   NaN  type strain of Capnocytophaga ochracea   \n",
       "GCA_000143785.1           NaN   NaN   type strain of Chryseobacterium gleum   \n",
       "\n",
       "                                misc_param collection date Isolation Site  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                        NaN             NaN            NaN   \n",
       "GCA_000143785.1     HMP body site, Vaginal             NaN            NaN   \n",
       "\n",
       "                   Isolation source country note GOLD Stamp ID  \\\n",
       "assembly_accession                                               \n",
       "GCA_000023285.1                 NaN     NaN  NaN           NaN   \n",
       "GCA_000143785.1                 NaN     NaN  NaN           NaN   \n",
       "\n",
       "                   investigation_type culture-collection lat_lon alt_elev  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                   NaN                NaN     NaN      NaN   \n",
       "GCA_000143785.1                   NaN         ATCC:35910     NaN      NaN   \n",
       "\n",
       "                   locus_tag_prefix nat-host assembly_method_version  \\\n",
       "assembly_accession                                                     \n",
       "GCA_000023285.1                 NaN      NaN                     NaN   \n",
       "GCA_000143785.1                 NaN      NaN                     NaN   \n",
       "\n",
       "                   env_material model sample_description Sample Description  \\\n",
       "assembly_accession                                                            \n",
       "GCA_000023285.1             NaN   NaN                NaN                NaN   \n",
       "GCA_000143785.1             NaN   NaN                NaN                NaN   \n",
       "\n",
       "                   ArrayExpress-STRAIN_OR_LINE ENA checklist  \\\n",
       "assembly_accession                                             \n",
       "GCA_000023285.1                            NaN           NaN   \n",
       "GCA_000143785.1                            NaN           NaN   \n",
       "\n",
       "                        project_type estimated_size Temperature Optimum  \\\n",
       "assembly_accession                                                        \n",
       "GCA_000023285.1                  NaN            NaN                 NaN   \n",
       "GCA_000143785.1     Reference Genome        5562000                 NaN   \n",
       "\n",
       "                   isol_growth_condt Motility  \\\n",
       "assembly_accession                              \n",
       "GCA_000023285.1                  NaN      NaN   \n",
       "GCA_000143785.1                  NaN      NaN   \n",
       "\n",
       "                                                          feature  \\\n",
       "assembly_accession                                                  \n",
       "GCA_000023285.1                                               NaN   \n",
       "GCA_000143785.1     Homo sapiens-associated habitat ENVO:00009003   \n",
       "\n",
       "                                  project_name                  assembly  \\\n",
       "assembly_accession                                                         \n",
       "GCA_000023285.1                            NaN                       NaN   \n",
       "GCA_000143785.1     Chryseobacterium gleum F93  Newbler v. 2.0.0-Eclipse   \n",
       "\n",
       "                   num_replicons isolate depth env_biome altitude  \\\n",
       "assembly_accession                                                  \n",
       "GCA_000023285.1              NaN     NaN   NaN       NaN      NaN   \n",
       "GCA_000143785.1              NaN     NaN   NaN       NaN      NaN   \n",
       "\n",
       "                   ref_biomaterial package geographic location  \\\n",
       "assembly_accession                                               \n",
       "GCA_000023285.1                NaN     NaN                 NaN   \n",
       "GCA_000143785.1                NaN     NaN                 NaN   \n",
       "\n",
       "                   ArrayExpress-StrainOrLine ArrayExpress-Phenotype  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1                          NaN                    NaN   \n",
       "GCA_000143785.1                          NaN                    NaN   \n",
       "\n",
       "                   ArrayExpress-SPECIES INSDC center alias geo-loc-name  \\\n",
       "assembly_accession                                                        \n",
       "GCA_000023285.1                     NaN                NaN          NaN   \n",
       "GCA_000143785.1                     NaN                NaN          NaN   \n",
       "\n",
       "                   environment host_sex elev PublicAccession  \\\n",
       "assembly_accession                                             \n",
       "GCA_000023285.1            NaN      NaN  NaN             NaN   \n",
       "GCA_000143785.1            NaN      NaN  NaN             NaN   \n",
       "\n",
       "                   completeness_estimated  \\\n",
       "assembly_accession                          \n",
       "GCA_000023285.1                       NaN   \n",
       "GCA_000143785.1                       NaN   \n",
       "\n",
       "                   geographic location (country and/or sea)  \\\n",
       "assembly_accession                                            \n",
       "GCA_000023285.1                                         NaN   \n",
       "GCA_000143785.1                                         NaN   \n",
       "\n",
       "                   latitude and longitude specific_host ArrayExpress-Species  \\\n",
       "assembly_accession                                                             \n",
       "GCA_000023285.1                       NaN           NaN                  NaN   \n",
       "GCA_000143785.1                       NaN           NaN                  NaN   \n",
       "\n",
       "                   ENA-FIRST-PUBLIC INSDC center name specific host  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1                 NaN               NaN           NaN   \n",
       "GCA_000143785.1                 NaN               NaN  Homo sapiens   \n",
       "\n",
       "                   Body Sample Site   pH                            biome  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                 NaN  NaN                              NaN   \n",
       "GCA_000143785.1                 NaN  NaN  terrestrial biome ENVO:00000446   \n",
       "\n",
       "                   biomaterial_provider isolate_name_alias  \\\n",
       "assembly_accession                                           \n",
       "GCA_000023285.1                     NaN                NaN   \n",
       "GCA_000143785.1                     NaN                NaN   \n",
       "\n",
       "                   contamination_estimated host scientific name  \\\n",
       "assembly_accession                                                \n",
       "GCA_000023285.1                        NaN                  NaN   \n",
       "GCA_000143785.1                        NaN                  NaN   \n",
       "\n",
       "                   anonymized_name ENA-LAST-UPDATE INSDC first public  \\\n",
       "assembly_accession                                                      \n",
       "GCA_000023285.1                NaN             NaN                NaN   \n",
       "GCA_000143785.1                NaN             NaN                NaN   \n",
       "\n",
       "                   Cell Shape BioSampleModel environmental_sample  \\\n",
       "assembly_accession                                                  \n",
       "GCA_000023285.1           NaN            NaN                  NaN   \n",
       "GCA_000143785.1           NaN        MIGS.ba                  NaN   \n",
       "\n",
       "                                            material identified_by  \\\n",
       "assembly_accession                                                   \n",
       "GCA_000023285.1                                  NaN           NaN   \n",
       "GCA_000143785.1     biological product ENVO:02000043           NaN   \n",
       "\n",
       "                   host health state INSDC last update lat lon Diseases  \\\n",
       "assembly_accession                                                        \n",
       "GCA_000023285.1                  NaN               NaN     NaN      NaN   \n",
       "GCA_000143785.1                  NaN               NaN     NaN      NaN   \n",
       "\n",
       "                   strain_name_alias genome_coverage host_status INSDC status  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1                  NaN             NaN         NaN          NaN   \n",
       "GCA_000143785.1                  NaN             NaN         NaN          NaN   \n",
       "\n",
       "                   host taxid isolation source Gram Staining  \\\n",
       "assembly_accession                                             \n",
       "GCA_000023285.1           NaN              NaN           NaN   \n",
       "GCA_000143785.1          9606              NaN           NaN   \n",
       "\n",
       "                   Temperature Range Sporulation sequencing_meth  \\\n",
       "assembly_accession                                                 \n",
       "GCA_000023285.1                  NaN         NaN             NaN   \n",
       "GCA_000143785.1                  NaN         NaN             NaN   \n",
       "\n",
       "                   Historical Monobactam Produced identification method  \\\n",
       "assembly_accession                                                        \n",
       "GCA_000023285.1                               NaN                   NaN   \n",
       "GCA_000143785.1                               NaN                   NaN   \n",
       "\n",
       "                   supplier_name SRA accession nucleic acid extraction  \\\n",
       "assembly_accession                                                       \n",
       "GCA_000023285.1              NaN           NaN                     NaN   \n",
       "GCA_000143785.1              NaN           NaN                     NaN   \n",
       "\n",
       "                   host_disease_outcome Unique Monobactam Clusters Identified  \\\n",
       "assembly_accession                                                              \n",
       "GCA_000023285.1                     NaN                                   NaN   \n",
       "GCA_000143785.1                     NaN                                   NaN   \n",
       "\n",
       "                   ProjectAccession Sample Name Type Strain Phenotypes  \\\n",
       "assembly_accession                                                       \n",
       "GCA_000023285.1                 NaN         NaN         NaN        NaN   \n",
       "GCA_000143785.1                 NaN         NaN         NaN        NaN   \n",
       "\n",
       "                   Gene Calling Method temp Title investigation type Host  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                    NaN  NaN   NaN                NaN  NaN   \n",
       "GCA_000143785.1                    NaN  NaN   NaN                NaN  NaN   \n",
       "\n",
       "                   rel_to_oxygen host_age Species mapping_method serovar  \\\n",
       "assembly_accession                                                         \n",
       "GCA_000023285.1              NaN      NaN     NaN            NaN     NaN   \n",
       "GCA_000143785.1              NaN      NaN     NaN            NaN     NaN   \n",
       "\n",
       "                                                                  sop  \\\n",
       "assembly_accession                                                      \n",
       "GCA_000023285.1                                                   NaN   \n",
       "GCA_000143785.1     http://hmpdacc.org/doc/CommonGeneAnnotation_SO...   \n",
       "\n",
       "                   Isolation Comments finishing_strategy Genus  \\\n",
       "assembly_accession                                               \n",
       "GCA_000023285.1                   NaN                NaN   NaN   \n",
       "GCA_000143785.1                   NaN                NaN   NaN   \n",
       "\n",
       "                   mapping_method_version environment (biome)  \\\n",
       "assembly_accession                                              \n",
       "GCA_000023285.1                       NaN                 NaN   \n",
       "GCA_000143785.1                       NaN                 NaN   \n",
       "\n",
       "                   attribute_package metagenome_source environment (feature)  \\\n",
       "assembly_accession                                                             \n",
       "GCA_000023285.1                  NaN               NaN                   NaN   \n",
       "GCA_000143785.1                  NaN               NaN                   NaN   \n",
       "\n",
       "                   trophic_level metagenomic environment (material)  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1              NaN         NaN                    NaN   \n",
       "GCA_000143785.1              NaN         NaN                    NaN   \n",
       "\n",
       "                   Funding Program quality_assessment_method  \\\n",
       "assembly_accession                                             \n",
       "GCA_000023285.1                NaN                       NaN   \n",
       "GCA_000143785.1                NaN                       NaN   \n",
       "\n",
       "                   host_health_state quality_assessment_method_version  \\\n",
       "assembly_accession                                                       \n",
       "GCA_000023285.1                  NaN                               NaN   \n",
       "GCA_000143785.1                  NaN                               NaN   \n",
       "\n",
       "                   geographic location (latitude) source_mat_id  \\\n",
       "assembly_accession                                                \n",
       "GCA_000023285.1                               NaN           NaN   \n",
       "GCA_000143785.1                               NaN    ATCC 35910   \n",
       "\n",
       "                   host_tissue_sampled host_description  \\\n",
       "assembly_accession                                        \n",
       "GCA_000023285.1                    NaN              NaN   \n",
       "GCA_000143785.1                    NaN              NaN   \n",
       "\n",
       "                   geographic location (longitude) Host Name  \\\n",
       "assembly_accession                                             \n",
       "GCA_000023285.1                                NaN       NaN   \n",
       "GCA_000143785.1                                NaN       NaN   \n",
       "\n",
       "                   host_disease_stage subsrc_note Host Health value  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1                   NaN         NaN         NaN   NaN   \n",
       "GCA_000143785.1                   NaN         NaN         NaN   NaN   \n",
       "\n",
       "                   Laboratory Host plant-associated environmental package  \\\n",
       "assembly_accession                                                          \n",
       "GCA_000023285.1                NaN                                    NaN   \n",
       "GCA_000143785.1                NaN                                    NaN   \n",
       "\n",
       "                   Extraction Method project name sequencing method  \\\n",
       "assembly_accession                                                    \n",
       "GCA_000023285.1                  NaN          NaN               NaN   \n",
       "GCA_000143785.1                  NaN          NaN        454-GS-FLX   \n",
       "\n",
       "                   Body Sample SubSite passage_history Extraction Date  \\\n",
       "assembly_accession                                                       \n",
       "GCA_000023285.1                    NaN             NaN             NaN   \n",
       "GCA_000143785.1                    NaN             NaN             NaN   \n",
       "\n",
       "                   sample type environmental package Passage Date  \n",
       "assembly_accession                                                 \n",
       "GCA_000023285.1            NaN                   NaN          NaN  \n",
       "GCA_000143785.1            NaN                   NaN          NaN  "
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.replace(\"'\", \"\", regex=True, inplace=True)\n",
    "df.replace(to_replace=[\"missing\", \"Missing\", \"Unclassified\", \"NA\", \"not determined\", \"Not collected\", \n",
    "                                          \"unspecified\", \"not applicable\", \"Unknown\", \"Not applicable\", \"'Missing'\",\n",
    "                                          \"Not Applicable\", \"not collected\"], value= None, regex=True, inplace=True)\n",
    "df.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "null_cols = df.isnull().all()\n",
    "df = df[df.columns[~null_cols]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "#To remove white space at the beginning of string:\n",
    "df.columns = df.columns.str.lstrip()\n",
    "\n",
    "#To remove white space at the end of string:\n",
    "df.columns = df.columns.str.rstrip()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "cols = df.columns.tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "colstoremove =[]\n",
    "for x in cols:\n",
    "    a = str(x)\n",
    "    if (604 - len(df[df[a].isnull()])) <= 2:\n",
    "        colstoremove.append(x)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[]\n"
     ]
    }
   ],
   "source": [
    "print(colstoremove)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "df.drop(colstoremove, axis = 1, inplace=True) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Index: 536 entries, GCA_000023285.1 to GCA_902804485.1\n",
      "Columns: 148 entries, strain to Body Sample SubSite\n",
      "dtypes: object(148)\n",
      "memory usage: 623.9+ KB\n"
     ]
    }
   ],
   "source": [
    "df.info()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4 Unify SRA metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>run_accession</th>\n",
       "      <th>sra_ID</th>\n",
       "      <th>run_ID</th>\n",
       "      <th>run_alias</th>\n",
       "      <th>run_date</th>\n",
       "      <th>updated_date</th>\n",
       "      <th>spots</th>\n",
       "      <th>bases</th>\n",
       "      <th>run_center</th>\n",
       "      <th>experiment_name</th>\n",
       "      <th>run_attribute</th>\n",
       "      <th>experiment_ID</th>\n",
       "      <th>experiment_alias</th>\n",
       "      <th>experiment_accession</th>\n",
       "      <th>experiment_title</th>\n",
       "      <th>study_name</th>\n",
       "      <th>sample_name</th>\n",
       "      <th>design_description</th>\n",
       "      <th>library_name</th>\n",
       "      <th>library_strategy</th>\n",
       "      <th>library_source</th>\n",
       "      <th>library_selection</th>\n",
       "      <th>library_layout</th>\n",
       "      <th>library_construction_protocol</th>\n",
       "      <th>read_spec</th>\n",
       "      <th>platform</th>\n",
       "      <th>instrument_model</th>\n",
       "      <th>platform_parameters</th>\n",
       "      <th>experiment_attribute</th>\n",
       "      <th>sample_ID</th>\n",
       "      <th>sample_alias</th>\n",
       "      <th>sample_accession</th>\n",
       "      <th>taxon_id</th>\n",
       "      <th>common_name</th>\n",
       "      <th>description</th>\n",
       "      <th>sample_url_link</th>\n",
       "      <th>sample_attribute</th>\n",
       "      <th>study_ID</th>\n",
       "      <th>study_alias</th>\n",
       "      <th>study_accession</th>\n",
       "      <th>study_title</th>\n",
       "      <th>study_type</th>\n",
       "      <th>study_abstract</th>\n",
       "      <th>center_project_name</th>\n",
       "      <th>study_description</th>\n",
       "      <th>study_url_link</th>\n",
       "      <th>study_attribute</th>\n",
       "      <th>submission_ID</th>\n",
       "      <th>submission_accession</th>\n",
       "      <th>submission_comment</th>\n",
       "      <th>submission_center</th>\n",
       "      <th>submission_lab</th>\n",
       "      <th>submission_date</th>\n",
       "      <th>sradb_updated</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>SRR013476</td>\n",
       "      <td>0</td>\n",
       "      <td>1093152</td>\n",
       "      <td>FWCBa</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-11-28</td>\n",
       "      <td>9047.0</td>\n",
       "      <td>2866262.0</td>\n",
       "      <td>JGI</td>\n",
       "      <td>FWCBa</td>\n",
       "      <td>flow_count: 110 || flow_sequence: TACG || TCAG...</td>\n",
       "      <td>1051231</td>\n",
       "      <td>FWCBa</td>\n",
       "      <td>SRX002589</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>none provided</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>WGS</td>\n",
       "      <td>GENOMIC</td>\n",
       "      <td>RANDOM</td>\n",
       "      <td>SINGLE -</td>\n",
       "      <td>NaN</td>\n",
       "      <td>READ_INDEX: 0; READ_CLASS: Technical Read; REA...</td>\n",
       "      <td>LS454</td>\n",
       "      <td>454 GS FLX</td>\n",
       "      <td>INSTRUMENT_MODEL: 454 GS FLX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>780997.0</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>SRS002078</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain: DSM 7271 || type-material: type strain...</td>\n",
       "      <td>16818.0</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>SRP000584</td>\n",
       "      <td>Causes periodontitis</td>\n",
       "      <td>Whole Genome Sequencing</td>\n",
       "      <td>&lt;P&gt;&lt;B&gt;&lt;I&gt;Capnocytophaga ochracea&lt;/I&gt; DSM 7271&lt;...</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>parent_bioproject: PRJNA30815</td>\n",
       "      <td>310018</td>\n",
       "      <td>SRA008146</td>\n",
       "      <td>FTP</td>\n",
       "      <td>JGI</td>\n",
       "      <td>PGF</td>\n",
       "      <td>2009-02-19</td>\n",
       "      <td>2020-09-03 11:01:35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000143785.1</td>\n",
       "      <td>SRR004314</td>\n",
       "      <td>0</td>\n",
       "      <td>1083352</td>\n",
       "      <td>2007-08-02.1147-FLX01070128</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-11-28</td>\n",
       "      <td>158268.0</td>\n",
       "      <td>44181818.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4WG_HMP0003.CGLE.01_000sA</td>\n",
       "      <td>run_center: BCM</td>\n",
       "      <td>1048730</td>\n",
       "      <td>4WG_HMP0003.CGLE.01_000sA</td>\n",
       "      <td>SRX001228</td>\n",
       "      <td>454 sequencing of Chryseobacterium gleum ATCC ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FLX fragment</td>\n",
       "      <td>4WG_HMP0003.CGLE.01_000sA</td>\n",
       "      <td>WGS</td>\n",
       "      <td>GENOMIC</td>\n",
       "      <td>RANDOM</td>\n",
       "      <td>SINGLE -</td>\n",
       "      <td>NaN</td>\n",
       "      <td>READ_INDEX: 0; READ_CLASS: Technical Read; REA...</td>\n",
       "      <td>LS454</td>\n",
       "      <td>454 GS FLX</td>\n",
       "      <td>INSTRUMENT_MODEL: 454 GS FLX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>779023.0</td>\n",
       "      <td>BCM-HMP0003</td>\n",
       "      <td>SRS000860</td>\n",
       "      <td>525257.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Chryseobacterium gleum ATCC 35910</td>\n",
       "      <td>BCM HMP Project: http://www.hgsc.bcm.edu/micro...</td>\n",
       "      <td>strain: ATCC 35910 || misc_param: HMP supersit...</td>\n",
       "      <td>16258.0</td>\n",
       "      <td>WG_HMP0003.CGLE.01_000</td>\n",
       "      <td>SRP000258</td>\n",
       "      <td>Reference genome for the Human Microbiome Project</td>\n",
       "      <td>Whole Genome Sequencing</td>\n",
       "      <td>&lt;P&gt; &lt;I&gt;&lt;B&gt;Chryseobacterium gleum&lt;/I&gt; ATCC 3591...</td>\n",
       "      <td>Chryseobacterium gleum ATCC 35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>parent_bioproject: PRJNA28331</td>\n",
       "      <td>304555</td>\n",
       "      <td>SRA002043</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BCM</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2008-09-11</td>\n",
       "      <td>2020-09-03 11:01:35</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  assembly_accession run_accession  sra_ID   run_ID  \\\n",
       "0    GCA_000023285.1     SRR013476       0  1093152   \n",
       "1    GCA_000143785.1     SRR004314       0  1083352   \n",
       "\n",
       "                     run_alias run_date updated_date     spots       bases  \\\n",
       "0                        FWCBa      NaN   2019-11-28    9047.0   2866262.0   \n",
       "1  2007-08-02.1147-FLX01070128      NaN   2019-11-28  158268.0  44181818.0   \n",
       "\n",
       "  run_center            experiment_name  \\\n",
       "0        JGI                      FWCBa   \n",
       "1        NaN  4WG_HMP0003.CGLE.01_000sA   \n",
       "\n",
       "                                       run_attribute  experiment_ID  \\\n",
       "0  flow_count: 110 || flow_sequence: TACG || TCAG...        1051231   \n",
       "1                                    run_center: BCM        1048730   \n",
       "\n",
       "            experiment_alias experiment_accession  \\\n",
       "0                      FWCBa            SRX002589   \n",
       "1  4WG_HMP0003.CGLE.01_000sA            SRX001228   \n",
       "\n",
       "                                    experiment_title study_name sample_name  \\\n",
       "0                                                NaN       FWCB        FWCB   \n",
       "1  454 sequencing of Chryseobacterium gleum ATCC ...        NaN         NaN   \n",
       "\n",
       "                  design_description               library_name  \\\n",
       "0             none provided                                FWCB   \n",
       "1                       FLX fragment  4WG_HMP0003.CGLE.01_000sA   \n",
       "\n",
       "  library_strategy library_source library_selection library_layout  \\\n",
       "0              WGS        GENOMIC            RANDOM      SINGLE -    \n",
       "1              WGS        GENOMIC            RANDOM      SINGLE -    \n",
       "\n",
       "  library_construction_protocol  \\\n",
       "0                           NaN   \n",
       "1                           NaN   \n",
       "\n",
       "                                           read_spec platform  \\\n",
       "0  READ_INDEX: 0; READ_CLASS: Technical Read; REA...    LS454   \n",
       "1  READ_INDEX: 0; READ_CLASS: Technical Read; REA...    LS454   \n",
       "\n",
       "  instrument_model           platform_parameters experiment_attribute  \\\n",
       "0       454 GS FLX  INSTRUMENT_MODEL: 454 GS FLX                  NaN   \n",
       "1       454 GS FLX  INSTRUMENT_MODEL: 454 GS FLX                  NaN   \n",
       "\n",
       "   sample_ID sample_alias sample_accession  taxon_id common_name  \\\n",
       "0   780997.0         FWCB        SRS002078  521097.0         NaN   \n",
       "1   779023.0  BCM-HMP0003        SRS000860  525257.0         NaN   \n",
       "\n",
       "                         description  \\\n",
       "0   Capnocytophaga ochracea DSM 7271   \n",
       "1  Chryseobacterium gleum ATCC 35910   \n",
       "\n",
       "                                     sample_url_link  \\\n",
       "0                                                NaN   \n",
       "1  BCM HMP Project: http://www.hgsc.bcm.edu/micro...   \n",
       "\n",
       "                                    sample_attribute  study_ID  \\\n",
       "0  strain: DSM 7271 || type-material: type strain...   16818.0   \n",
       "1  strain: ATCC 35910 || misc_param: HMP supersit...   16258.0   \n",
       "\n",
       "              study_alias study_accession  \\\n",
       "0                    FWCB       SRP000584   \n",
       "1  WG_HMP0003.CGLE.01_000       SRP000258   \n",
       "\n",
       "                                         study_title               study_type  \\\n",
       "0                               Causes periodontitis  Whole Genome Sequencing   \n",
       "1  Reference genome for the Human Microbiome Project  Whole Genome Sequencing   \n",
       "\n",
       "                                      study_abstract  \\\n",
       "0  <P><B><I>Capnocytophaga ochracea</I> DSM 7271<...   \n",
       "1  <P> <I><B>Chryseobacterium gleum</I> ATCC 3591...   \n",
       "\n",
       "                 center_project_name study_description study_url_link  \\\n",
       "0   Capnocytophaga ochracea DSM 7271               NaN            NaN   \n",
       "1  Chryseobacterium gleum ATCC 35910               NaN            NaN   \n",
       "\n",
       "                 study_attribute  submission_ID submission_accession  \\\n",
       "0  parent_bioproject: PRJNA30815         310018            SRA008146   \n",
       "1  parent_bioproject: PRJNA28331         304555            SRA002043   \n",
       "\n",
       "  submission_comment submission_center submission_lab submission_date  \\\n",
       "0                FTP               JGI            PGF      2009-02-19   \n",
       "1                NaN               BCM            NaN      2008-09-11   \n",
       "\n",
       "         sradb_updated  \n",
       "0  2020-09-03 11:01:35  \n",
       "1  2020-09-03 11:01:35  "
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "code_sra.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>strain</th>\n",
       "      <th>finishing strategy (depth of coverage)</th>\n",
       "      <th>biotic_relationship</th>\n",
       "      <th>geo_loc_name</th>\n",
       "      <th>host</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>host_disease</th>\n",
       "      <th>isolation_source</th>\n",
       "      <th>sample_name</th>\n",
       "      <th>isolation-source</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>FDA CVM ID</th>\n",
       "      <th>culture_collection</th>\n",
       "      <th>assembly_method</th>\n",
       "      <th>collection_date</th>\n",
       "      <th>env_feature</th>\n",
       "      <th>synonym</th>\n",
       "      <th>Strain</th>\n",
       "      <th>SUBJECT_ID</th>\n",
       "      <th>Alias</th>\n",
       "      <th>type-material</th>\n",
       "      <th>misc_param</th>\n",
       "      <th>collection date</th>\n",
       "      <th>Isolation Site</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>country</th>\n",
       "      <th>note</th>\n",
       "      <th>GOLD Stamp ID</th>\n",
       "      <th>investigation_type</th>\n",
       "      <th>culture-collection</th>\n",
       "      <th>lat_lon</th>\n",
       "      <th>alt_elev</th>\n",
       "      <th>locus_tag_prefix</th>\n",
       "      <th>nat-host</th>\n",
       "      <th>assembly_method_version</th>\n",
       "      <th>env_material</th>\n",
       "      <th>model</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>Sample Description</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>project_type</th>\n",
       "      <th>estimated_size</th>\n",
       "      <th>Temperature Optimum</th>\n",
       "      <th>isol_growth_condt</th>\n",
       "      <th>Motility</th>\n",
       "      <th>feature</th>\n",
       "      <th>project_name</th>\n",
       "      <th>assembly</th>\n",
       "      <th>num_replicons</th>\n",
       "      <th>isolate</th>\n",
       "      <th>depth</th>\n",
       "      <th>env_biome</th>\n",
       "      <th>altitude</th>\n",
       "      <th>ref_biomaterial</th>\n",
       "      <th>package</th>\n",
       "      <th>geographic location</th>\n",
       "      <th>ArrayExpress-StrainOrLine</th>\n",
       "      <th>ArrayExpress-Phenotype</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>geo-loc-name</th>\n",
       "      <th>environment</th>\n",
       "      <th>host_sex</th>\n",
       "      <th>elev</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>completeness_estimated</th>\n",
       "      <th>geographic location (country and/or sea)</th>\n",
       "      <th>latitude and longitude</th>\n",
       "      <th>specific_host</th>\n",
       "      <th>ArrayExpress-Species</th>\n",
       "      <th>ENA-FIRST-PUBLIC</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>specific host</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>pH</th>\n",
       "      <th>biome</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>isolate_name_alias</th>\n",
       "      <th>contamination_estimated</th>\n",
       "      <th>host scientific name</th>\n",
       "      <th>anonymized_name</th>\n",
       "      <th>ENA-LAST-UPDATE</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>BioSampleModel</th>\n",
       "      <th>environmental_sample</th>\n",
       "      <th>material</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>host health state</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>Diseases</th>\n",
       "      <th>strain_name_alias</th>\n",
       "      <th>genome_coverage</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>host taxid</th>\n",
       "      <th>isolation source</th>\n",
       "      <th>Gram Staining</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>sequencing_meth</th>\n",
       "      <th>Historical Monobactam Produced</th>\n",
       "      <th>identification method</th>\n",
       "      <th>supplier_name</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>host_disease_outcome</th>\n",
       "      <th>Unique Monobactam Clusters Identified</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>Sample Name</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Gene Calling Method</th>\n",
       "      <th>temp</th>\n",
       "      <th>Title</th>\n",
       "      <th>investigation type</th>\n",
       "      <th>Host</th>\n",
       "      <th>rel_to_oxygen</th>\n",
       "      <th>host_age</th>\n",
       "      <th>Species</th>\n",
       "      <th>mapping_method</th>\n",
       "      <th>serovar</th>\n",
       "      <th>sop</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>finishing_strategy</th>\n",
       "      <th>Genus</th>\n",
       "      <th>mapping_method_version</th>\n",
       "      <th>environment (biome)</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>metagenome_source</th>\n",
       "      <th>environment (feature)</th>\n",
       "      <th>trophic_level</th>\n",
       "      <th>metagenomic</th>\n",
       "      <th>environment (material)</th>\n",
       "      <th>Funding Program</th>\n",
       "      <th>quality_assessment_method</th>\n",
       "      <th>quality_assessment_method_version</th>\n",
       "      <th>geographic location (latitude)</th>\n",
       "      <th>source_mat_id</th>\n",
       "      <th>host_tissue_sampled</th>\n",
       "      <th>geographic location (longitude)</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>subsrc_note</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>value</th>\n",
       "      <th>plant-associated environmental package</th>\n",
       "      <th>project name</th>\n",
       "      <th>sequencing method</th>\n",
       "      <th>Body Sample SubSite</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000143785.1</td>\n",
       "      <td>ATCC 35910</td>\n",
       "      <td>Level 6, Finished33.84x;7</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Chryseobacterium gleum</td>\n",
       "      <td>HMP body site, Vaginal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC:35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Reference Genome</td>\n",
       "      <td>5562000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens-associated habitat ENVO:00009003</td>\n",
       "      <td>Chryseobacterium gleum F93</td>\n",
       "      <td>Newbler v. 2.0.0-Eclipse</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>terrestrial biome ENVO:00000446</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MIGS.ba</td>\n",
       "      <td>NaN</td>\n",
       "      <td>biological product ENVO:02000043</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9606</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>http://hmpdacc.org/doc/CommonGeneAnnotation_SO...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>454-GS-FLX</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  assembly_accession      strain finishing strategy (depth of coverage)  \\\n",
       "0    GCA_000023285.1    DSM 7271                                    NaN   \n",
       "1    GCA_000143785.1  ATCC 35910              Level 6, Finished33.84x;7   \n",
       "\n",
       "  biotic_relationship geo_loc_name host collected_by host_disease  \\\n",
       "0                 NaN          NaN  NaN          NaN          NaN   \n",
       "1                 NaN          NaN  NaN          NaN          NaN   \n",
       "\n",
       "  isolation_source sample_name isolation-source sample_type FDA CVM ID  \\\n",
       "0              NaN         NaN              NaN         NaN        NaN   \n",
       "1              NaN         NaN              NaN         NaN        NaN   \n",
       "\n",
       "  culture_collection assembly_method collection_date env_feature synonym  \\\n",
       "0                NaN             NaN             NaN         NaN     NaN   \n",
       "1                NaN             NaN             NaN         NaN     NaN   \n",
       "\n",
       "  Strain SUBJECT_ID Alias                           type-material  \\\n",
       "0    NaN        NaN   NaN  type strain of Capnocytophaga ochracea   \n",
       "1    NaN        NaN   NaN   type strain of Chryseobacterium gleum   \n",
       "\n",
       "               misc_param collection date Isolation Site Isolation source  \\\n",
       "0                     NaN             NaN            NaN              NaN   \n",
       "1  HMP body site, Vaginal             NaN            NaN              NaN   \n",
       "\n",
       "  country note GOLD Stamp ID investigation_type culture-collection lat_lon  \\\n",
       "0     NaN  NaN           NaN                NaN                NaN     NaN   \n",
       "1     NaN  NaN           NaN                NaN         ATCC:35910     NaN   \n",
       "\n",
       "  alt_elev locus_tag_prefix nat-host assembly_method_version env_material  \\\n",
       "0      NaN              NaN      NaN                     NaN          NaN   \n",
       "1      NaN              NaN      NaN                     NaN          NaN   \n",
       "\n",
       "  model sample_description Sample Description ArrayExpress-STRAIN_OR_LINE  \\\n",
       "0   NaN                NaN                NaN                         NaN   \n",
       "1   NaN                NaN                NaN                         NaN   \n",
       "\n",
       "  ENA checklist      project_type estimated_size Temperature Optimum  \\\n",
       "0           NaN               NaN            NaN                 NaN   \n",
       "1           NaN  Reference Genome        5562000                 NaN   \n",
       "\n",
       "  isol_growth_condt Motility                                        feature  \\\n",
       "0               NaN      NaN                                            NaN   \n",
       "1               NaN      NaN  Homo sapiens-associated habitat ENVO:00009003   \n",
       "\n",
       "                 project_name                  assembly num_replicons isolate  \\\n",
       "0                         NaN                       NaN           NaN     NaN   \n",
       "1  Chryseobacterium gleum F93  Newbler v. 2.0.0-Eclipse           NaN     NaN   \n",
       "\n",
       "  depth env_biome altitude ref_biomaterial package geographic location  \\\n",
       "0   NaN       NaN      NaN             NaN     NaN                 NaN   \n",
       "1   NaN       NaN      NaN             NaN     NaN                 NaN   \n",
       "\n",
       "  ArrayExpress-StrainOrLine ArrayExpress-Phenotype ArrayExpress-SPECIES  \\\n",
       "0                       NaN                    NaN                  NaN   \n",
       "1                       NaN                    NaN                  NaN   \n",
       "\n",
       "  INSDC center alias geo-loc-name environment host_sex elev PublicAccession  \\\n",
       "0                NaN          NaN         NaN      NaN  NaN             NaN   \n",
       "1                NaN          NaN         NaN      NaN  NaN             NaN   \n",
       "\n",
       "  completeness_estimated geographic location (country and/or sea)  \\\n",
       "0                    NaN                                      NaN   \n",
       "1                    NaN                                      NaN   \n",
       "\n",
       "  latitude and longitude specific_host ArrayExpress-Species ENA-FIRST-PUBLIC  \\\n",
       "0                    NaN           NaN                  NaN              NaN   \n",
       "1                    NaN           NaN                  NaN              NaN   \n",
       "\n",
       "  INSDC center name specific host Body Sample Site   pH  \\\n",
       "0               NaN           NaN              NaN  NaN   \n",
       "1               NaN  Homo sapiens              NaN  NaN   \n",
       "\n",
       "                             biome biomaterial_provider isolate_name_alias  \\\n",
       "0                              NaN                  NaN                NaN   \n",
       "1  terrestrial biome ENVO:00000446                  NaN                NaN   \n",
       "\n",
       "  contamination_estimated host scientific name anonymized_name  \\\n",
       "0                     NaN                  NaN             NaN   \n",
       "1                     NaN                  NaN             NaN   \n",
       "\n",
       "  ENA-LAST-UPDATE INSDC first public Cell Shape BioSampleModel  \\\n",
       "0             NaN                NaN        NaN            NaN   \n",
       "1             NaN                NaN        NaN        MIGS.ba   \n",
       "\n",
       "  environmental_sample                          material identified_by  \\\n",
       "0                  NaN                               NaN           NaN   \n",
       "1                  NaN  biological product ENVO:02000043           NaN   \n",
       "\n",
       "  host health state INSDC last update Diseases strain_name_alias  \\\n",
       "0               NaN               NaN      NaN               NaN   \n",
       "1               NaN               NaN      NaN               NaN   \n",
       "\n",
       "  genome_coverage INSDC status host taxid isolation source Gram Staining  \\\n",
       "0             NaN          NaN        NaN              NaN           NaN   \n",
       "1             NaN          NaN       9606              NaN           NaN   \n",
       "\n",
       "  Temperature Range Sporulation sequencing_meth  \\\n",
       "0               NaN         NaN             NaN   \n",
       "1               NaN         NaN             NaN   \n",
       "\n",
       "  Historical Monobactam Produced identification method supplier_name  \\\n",
       "0                            NaN                   NaN           NaN   \n",
       "1                            NaN                   NaN           NaN   \n",
       "\n",
       "  SRA accession host_disease_outcome Unique Monobactam Clusters Identified  \\\n",
       "0           NaN                  NaN                                   NaN   \n",
       "1           NaN                  NaN                                   NaN   \n",
       "\n",
       "  ProjectAccession Sample Name Type Strain Phenotypes Gene Calling Method  \\\n",
       "0              NaN         NaN         NaN        NaN                 NaN   \n",
       "1              NaN         NaN         NaN        NaN                 NaN   \n",
       "\n",
       "  temp Title investigation type Host rel_to_oxygen host_age Species  \\\n",
       "0  NaN   NaN                NaN  NaN           NaN      NaN     NaN   \n",
       "1  NaN   NaN                NaN  NaN           NaN      NaN     NaN   \n",
       "\n",
       "  mapping_method serovar                                                sop  \\\n",
       "0            NaN     NaN                                                NaN   \n",
       "1            NaN     NaN  http://hmpdacc.org/doc/CommonGeneAnnotation_SO...   \n",
       "\n",
       "  Isolation Comments finishing_strategy Genus mapping_method_version  \\\n",
       "0                NaN                NaN   NaN                    NaN   \n",
       "1                NaN                NaN   NaN                    NaN   \n",
       "\n",
       "  environment (biome) attribute_package metagenome_source  \\\n",
       "0                 NaN               NaN               NaN   \n",
       "1                 NaN               NaN               NaN   \n",
       "\n",
       "  environment (feature) trophic_level metagenomic environment (material)  \\\n",
       "0                   NaN           NaN         NaN                    NaN   \n",
       "1                   NaN           NaN         NaN                    NaN   \n",
       "\n",
       "  Funding Program quality_assessment_method quality_assessment_method_version  \\\n",
       "0             NaN                       NaN                               NaN   \n",
       "1             NaN                       NaN                               NaN   \n",
       "\n",
       "  geographic location (latitude) source_mat_id host_tissue_sampled  \\\n",
       "0                            NaN           NaN                 NaN   \n",
       "1                            NaN    ATCC 35910                 NaN   \n",
       "\n",
       "  geographic location (longitude) Host Name subsrc_note Host Health value  \\\n",
       "0                             NaN       NaN         NaN         NaN   NaN   \n",
       "1                             NaN       NaN         NaN         NaN   NaN   \n",
       "\n",
       "  plant-associated environmental package project name sequencing method  \\\n",
       "0                                    NaN          NaN               NaN   \n",
       "1                                    NaN          NaN        454-GS-FLX   \n",
       "\n",
       "  Body Sample SubSite  \n",
       "0                 NaN  \n",
       "1                 NaN  "
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = df.reset_index()\n",
    "df.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "both          536\n",
       "right_only      0\n",
       "left_only       0\n",
       "Name: merge, dtype: int64"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merged = pd.merge(df, code_sra, how=\"right\", left_on=\"assembly_accession\", right_on=\"assembly_accession\", indicator=\"merge\")\n",
    "merged[\"merge\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>strain</th>\n",
       "      <th>finishing strategy (depth of coverage)</th>\n",
       "      <th>biotic_relationship</th>\n",
       "      <th>geo_loc_name</th>\n",
       "      <th>host</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>host_disease</th>\n",
       "      <th>isolation_source</th>\n",
       "      <th>sample_name_x</th>\n",
       "      <th>isolation-source</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>FDA CVM ID</th>\n",
       "      <th>culture_collection</th>\n",
       "      <th>assembly_method</th>\n",
       "      <th>collection_date</th>\n",
       "      <th>env_feature</th>\n",
       "      <th>synonym</th>\n",
       "      <th>Strain</th>\n",
       "      <th>SUBJECT_ID</th>\n",
       "      <th>Alias</th>\n",
       "      <th>type-material</th>\n",
       "      <th>misc_param</th>\n",
       "      <th>collection date</th>\n",
       "      <th>Isolation Site</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>country</th>\n",
       "      <th>note</th>\n",
       "      <th>GOLD Stamp ID</th>\n",
       "      <th>investigation_type</th>\n",
       "      <th>culture-collection</th>\n",
       "      <th>lat_lon</th>\n",
       "      <th>alt_elev</th>\n",
       "      <th>locus_tag_prefix</th>\n",
       "      <th>nat-host</th>\n",
       "      <th>assembly_method_version</th>\n",
       "      <th>env_material</th>\n",
       "      <th>model</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>Sample Description</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>project_type</th>\n",
       "      <th>estimated_size</th>\n",
       "      <th>Temperature Optimum</th>\n",
       "      <th>isol_growth_condt</th>\n",
       "      <th>Motility</th>\n",
       "      <th>feature</th>\n",
       "      <th>project_name</th>\n",
       "      <th>assembly</th>\n",
       "      <th>num_replicons</th>\n",
       "      <th>isolate</th>\n",
       "      <th>depth</th>\n",
       "      <th>env_biome</th>\n",
       "      <th>altitude</th>\n",
       "      <th>ref_biomaterial</th>\n",
       "      <th>package</th>\n",
       "      <th>geographic location</th>\n",
       "      <th>ArrayExpress-StrainOrLine</th>\n",
       "      <th>ArrayExpress-Phenotype</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>geo-loc-name</th>\n",
       "      <th>environment</th>\n",
       "      <th>host_sex</th>\n",
       "      <th>elev</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>completeness_estimated</th>\n",
       "      <th>geographic location (country and/or sea)</th>\n",
       "      <th>latitude and longitude</th>\n",
       "      <th>specific_host</th>\n",
       "      <th>ArrayExpress-Species</th>\n",
       "      <th>ENA-FIRST-PUBLIC</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>specific host</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>pH</th>\n",
       "      <th>biome</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>isolate_name_alias</th>\n",
       "      <th>contamination_estimated</th>\n",
       "      <th>host scientific name</th>\n",
       "      <th>anonymized_name</th>\n",
       "      <th>ENA-LAST-UPDATE</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>BioSampleModel</th>\n",
       "      <th>environmental_sample</th>\n",
       "      <th>material</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>host health state</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>Diseases</th>\n",
       "      <th>strain_name_alias</th>\n",
       "      <th>genome_coverage</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>host taxid</th>\n",
       "      <th>isolation source</th>\n",
       "      <th>Gram Staining</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>...</th>\n",
       "      <th>Historical Monobactam Produced</th>\n",
       "      <th>identification method</th>\n",
       "      <th>supplier_name</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>host_disease_outcome</th>\n",
       "      <th>Unique Monobactam Clusters Identified</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>Sample Name</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Gene Calling Method</th>\n",
       "      <th>temp</th>\n",
       "      <th>Title</th>\n",
       "      <th>investigation type</th>\n",
       "      <th>Host</th>\n",
       "      <th>rel_to_oxygen</th>\n",
       "      <th>host_age</th>\n",
       "      <th>Species</th>\n",
       "      <th>mapping_method</th>\n",
       "      <th>serovar</th>\n",
       "      <th>sop</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>finishing_strategy</th>\n",
       "      <th>Genus</th>\n",
       "      <th>mapping_method_version</th>\n",
       "      <th>environment (biome)</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>metagenome_source</th>\n",
       "      <th>environment (feature)</th>\n",
       "      <th>trophic_level</th>\n",
       "      <th>metagenomic</th>\n",
       "      <th>environment (material)</th>\n",
       "      <th>Funding Program</th>\n",
       "      <th>quality_assessment_method</th>\n",
       "      <th>quality_assessment_method_version</th>\n",
       "      <th>geographic location (latitude)</th>\n",
       "      <th>source_mat_id</th>\n",
       "      <th>host_tissue_sampled</th>\n",
       "      <th>geographic location (longitude)</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>subsrc_note</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>value</th>\n",
       "      <th>plant-associated environmental package</th>\n",
       "      <th>project name</th>\n",
       "      <th>sequencing method</th>\n",
       "      <th>Body Sample SubSite</th>\n",
       "      <th>run_accession</th>\n",
       "      <th>sra_ID</th>\n",
       "      <th>run_ID</th>\n",
       "      <th>run_alias</th>\n",
       "      <th>run_date</th>\n",
       "      <th>updated_date</th>\n",
       "      <th>spots</th>\n",
       "      <th>bases</th>\n",
       "      <th>run_center</th>\n",
       "      <th>experiment_name</th>\n",
       "      <th>run_attribute</th>\n",
       "      <th>experiment_ID</th>\n",
       "      <th>experiment_alias</th>\n",
       "      <th>experiment_accession</th>\n",
       "      <th>experiment_title</th>\n",
       "      <th>study_name</th>\n",
       "      <th>sample_name_y</th>\n",
       "      <th>design_description</th>\n",
       "      <th>library_name</th>\n",
       "      <th>library_strategy</th>\n",
       "      <th>library_source</th>\n",
       "      <th>library_selection</th>\n",
       "      <th>library_layout</th>\n",
       "      <th>library_construction_protocol</th>\n",
       "      <th>read_spec</th>\n",
       "      <th>platform</th>\n",
       "      <th>instrument_model</th>\n",
       "      <th>platform_parameters</th>\n",
       "      <th>experiment_attribute</th>\n",
       "      <th>sample_ID</th>\n",
       "      <th>sample_alias</th>\n",
       "      <th>sample_accession</th>\n",
       "      <th>taxon_id</th>\n",
       "      <th>common_name</th>\n",
       "      <th>description</th>\n",
       "      <th>sample_url_link</th>\n",
       "      <th>study_ID</th>\n",
       "      <th>study_alias</th>\n",
       "      <th>study_accession</th>\n",
       "      <th>study_title</th>\n",
       "      <th>study_type</th>\n",
       "      <th>study_abstract</th>\n",
       "      <th>center_project_name</th>\n",
       "      <th>study_description</th>\n",
       "      <th>study_url_link</th>\n",
       "      <th>study_attribute</th>\n",
       "      <th>submission_ID</th>\n",
       "      <th>submission_accession</th>\n",
       "      <th>submission_comment</th>\n",
       "      <th>submission_center</th>\n",
       "      <th>submission_lab</th>\n",
       "      <th>submission_date</th>\n",
       "      <th>sradb_updated</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SRR013476</td>\n",
       "      <td>0</td>\n",
       "      <td>1093152</td>\n",
       "      <td>FWCBa</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-11-28</td>\n",
       "      <td>9047.0</td>\n",
       "      <td>2866262.0</td>\n",
       "      <td>JGI</td>\n",
       "      <td>FWCBa</td>\n",
       "      <td>flow_count: 110 || flow_sequence: TACG || TCAG...</td>\n",
       "      <td>1051231</td>\n",
       "      <td>FWCBa</td>\n",
       "      <td>SRX002589</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>none provided</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>WGS</td>\n",
       "      <td>GENOMIC</td>\n",
       "      <td>RANDOM</td>\n",
       "      <td>SINGLE -</td>\n",
       "      <td>NaN</td>\n",
       "      <td>READ_INDEX: 0; READ_CLASS: Technical Read; REA...</td>\n",
       "      <td>LS454</td>\n",
       "      <td>454 GS FLX</td>\n",
       "      <td>INSTRUMENT_MODEL: 454 GS FLX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>780997.0</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>SRS002078</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>16818.0</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>SRP000584</td>\n",
       "      <td>Causes periodontitis</td>\n",
       "      <td>Whole Genome Sequencing</td>\n",
       "      <td>&lt;P&gt;&lt;B&gt;&lt;I&gt;Capnocytophaga ochracea&lt;/I&gt; DSM 7271&lt;...</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>parent_bioproject: PRJNA30815</td>\n",
       "      <td>310018</td>\n",
       "      <td>SRA008146</td>\n",
       "      <td>FTP</td>\n",
       "      <td>JGI</td>\n",
       "      <td>PGF</td>\n",
       "      <td>2009-02-19</td>\n",
       "      <td>2020-09-03 11:01:35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000143785.1</td>\n",
       "      <td>ATCC 35910</td>\n",
       "      <td>Level 6, Finished33.84x;7</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Chryseobacterium gleum</td>\n",
       "      <td>HMP body site, Vaginal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC:35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Reference Genome</td>\n",
       "      <td>5562000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens-associated habitat ENVO:00009003</td>\n",
       "      <td>Chryseobacterium gleum F93</td>\n",
       "      <td>Newbler v. 2.0.0-Eclipse</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>terrestrial biome ENVO:00000446</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MIGS.ba</td>\n",
       "      <td>NaN</td>\n",
       "      <td>biological product ENVO:02000043</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9606</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>http://hmpdacc.org/doc/CommonGeneAnnotation_SO...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>454-GS-FLX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SRR004314</td>\n",
       "      <td>0</td>\n",
       "      <td>1083352</td>\n",
       "      <td>2007-08-02.1147-FLX01070128</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2019-11-28</td>\n",
       "      <td>158268.0</td>\n",
       "      <td>44181818.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4WG_HMP0003.CGLE.01_000sA</td>\n",
       "      <td>run_center: BCM</td>\n",
       "      <td>1048730</td>\n",
       "      <td>4WG_HMP0003.CGLE.01_000sA</td>\n",
       "      <td>SRX001228</td>\n",
       "      <td>454 sequencing of Chryseobacterium gleum ATCC ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FLX fragment</td>\n",
       "      <td>4WG_HMP0003.CGLE.01_000sA</td>\n",
       "      <td>WGS</td>\n",
       "      <td>GENOMIC</td>\n",
       "      <td>RANDOM</td>\n",
       "      <td>SINGLE -</td>\n",
       "      <td>NaN</td>\n",
       "      <td>READ_INDEX: 0; READ_CLASS: Technical Read; REA...</td>\n",
       "      <td>LS454</td>\n",
       "      <td>454 GS FLX</td>\n",
       "      <td>INSTRUMENT_MODEL: 454 GS FLX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>779023.0</td>\n",
       "      <td>BCM-HMP0003</td>\n",
       "      <td>SRS000860</td>\n",
       "      <td>525257.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Chryseobacterium gleum ATCC 35910</td>\n",
       "      <td>BCM HMP Project: http://www.hgsc.bcm.edu/micro...</td>\n",
       "      <td>16258.0</td>\n",
       "      <td>WG_HMP0003.CGLE.01_000</td>\n",
       "      <td>SRP000258</td>\n",
       "      <td>Reference genome for the Human Microbiome Project</td>\n",
       "      <td>Whole Genome Sequencing</td>\n",
       "      <td>&lt;P&gt; &lt;I&gt;&lt;B&gt;Chryseobacterium gleum&lt;/I&gt; ATCC 3591...</td>\n",
       "      <td>Chryseobacterium gleum ATCC 35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>parent_bioproject: PRJNA28331</td>\n",
       "      <td>304555</td>\n",
       "      <td>SRA002043</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BCM</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2008-09-11</td>\n",
       "      <td>2020-09-03 11:01:35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GCA_000173675.1</td>\n",
       "      <td>Capno</td>\n",
       "      <td>Level 2, High-Quality Draft19x;65</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HMP supersite, oral</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Reference Genome</td>\n",
       "      <td>2998000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>human-associated habitat ENVO:00009003</td>\n",
       "      <td>Capnocytophaga sputigena Capno</td>\n",
       "      <td>Newbler v. 1.1.03.24-64-7</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>terrestrial biome ENVO:00000446</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MIGS.ba</td>\n",
       "      <td>NaN</td>\n",
       "      <td>biological product ENVO:02000043</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9606</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>http://hmpdacc.org/doc/CommonGeneAnnotation_SO...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 33612</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>454-GS-FLX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SRR005028</td>\n",
       "      <td>0</td>\n",
       "      <td>1084559</td>\n",
       "      <td>FCRNIZW</td>\n",
       "      <td>2008-09-30</td>\n",
       "      <td>2020-06-16</td>\n",
       "      <td>229445.0</td>\n",
       "      <td>64077935.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HMP015-454 Experiment</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1049060</td>\n",
       "      <td>HMP015-454 Experiment</td>\n",
       "      <td>SRX001307</td>\n",
       "      <td>454 sequencing of Capnocytophaga sputigena Cap...</td>\n",
       "      <td>HMP015-454 Study</td>\n",
       "      <td>HMP015-454 Sample</td>\n",
       "      <td>Sequencing and analysis of the genome of Capno...</td>\n",
       "      <td>HMP015-G-02-697</td>\n",
       "      <td>WGS</td>\n",
       "      <td>GENOMIC</td>\n",
       "      <td>RANDOM</td>\n",
       "      <td>SINGLE -</td>\n",
       "      <td>NaN</td>\n",
       "      <td>READ_INDEX: 0; READ_CLASS: Technical Read; REA...</td>\n",
       "      <td>LS454</td>\n",
       "      <td>454 GS FLX</td>\n",
       "      <td>INSTRUMENT_MODEL: 454 GS FLX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>779245.0</td>\n",
       "      <td>HMP015-454 Sample</td>\n",
       "      <td>SRS000995</td>\n",
       "      <td>553177.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga sputigena is a gram-negative, r...</td>\n",
       "      <td>JCVI HMP Page: http://hmp.jcvi.org/</td>\n",
       "      <td>16339.0</td>\n",
       "      <td>HMP015-454 Study</td>\n",
       "      <td>SRP000317</td>\n",
       "      <td>reference genome for the Human Microbiome Project</td>\n",
       "      <td>Whole Genome Sequencing</td>\n",
       "      <td>&lt;P&gt; &lt;I&gt;&lt;B&gt;Capnocytophaga sputigena&lt;/I&gt; Capno&lt;/...</td>\n",
       "      <td>Capnocytophaga sputigena ATCC 33612</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>parent_bioproject: PRJNA28331</td>\n",
       "      <td>305210</td>\n",
       "      <td>SRA002772</td>\n",
       "      <td>JCVI Human Microbiome Reference - HMP015 seque...</td>\n",
       "      <td>JCVI</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2008-09-30</td>\n",
       "      <td>2020-09-03 11:01:35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GCA_000174755.1</td>\n",
       "      <td>ATCC 33624</td>\n",
       "      <td>Level 2, High-Quality Draft37x;37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga gingivalis</td>\n",
       "      <td>HMP supersite, oral</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC:33624</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Reference Genome</td>\n",
       "      <td>2667000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>human-associated habitat ENVO:00009003</td>\n",
       "      <td>Capnocytophaga gingivalis ATCC 33624</td>\n",
       "      <td>Newbler v. 1.1.03.24-64-7</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>terrestrial biome ENVO:00000446</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MIGS.ba</td>\n",
       "      <td>NaN</td>\n",
       "      <td>biological product ENVO:02000043</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9606</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>http://hmpdacc.org/doc/CommonGeneAnnotation_SO...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 33624</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>454</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SRR013509</td>\n",
       "      <td>0</td>\n",
       "      <td>1093169</td>\n",
       "      <td>FJ8IWNQ</td>\n",
       "      <td>2009-02-24</td>\n",
       "      <td>2020-06-17</td>\n",
       "      <td>289003.0</td>\n",
       "      <td>79866027.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HMP016-454 Experiment</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1051243</td>\n",
       "      <td>HMP016-454 Experiment</td>\n",
       "      <td>SRX002609</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HMP016-454 Study</td>\n",
       "      <td>HMP016-454 Sample</td>\n",
       "      <td>Sequencing and analysis of the genome of Capno...</td>\n",
       "      <td>HMP016-PL-01-680</td>\n",
       "      <td>WGS</td>\n",
       "      <td>GENOMIC</td>\n",
       "      <td>RANDOM</td>\n",
       "      <td>PAIRED - NOMINAL_LENGTH: 3000; NOMINAL_SDEV: 3...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>READ_INDEX: 0; READ_CLASS: Technical Read; REA...</td>\n",
       "      <td>LS454</td>\n",
       "      <td>454 GS FLX</td>\n",
       "      <td>INSTRUMENT_MODEL: 454 GS FLX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>781003.0</td>\n",
       "      <td>HMP016-454 Sample</td>\n",
       "      <td>SRS002089</td>\n",
       "      <td>553178.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga gingivalis is a gram-negative, ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>16824.0</td>\n",
       "      <td>HMP016-454 Study</td>\n",
       "      <td>SRP000590</td>\n",
       "      <td>Reference genome for the Human Microbiome Project</td>\n",
       "      <td>Whole Genome Sequencing</td>\n",
       "      <td>&lt;b&gt;&lt;i&gt;Capnocytophaga gingivalis&lt;/i&gt; ATCC 33624...</td>\n",
       "      <td>Capnocytophaga gingivalis ATCC 33624</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>parent_bioproject: PRJNA28331</td>\n",
       "      <td>310024</td>\n",
       "      <td>SRA008153</td>\n",
       "      <td>JCVI Human Microbiome Reference - HMP016 seque...</td>\n",
       "      <td>JCVI</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2009-02-24</td>\n",
       "      <td>2020-09-03 11:01:35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GCA_000183155.1</td>\n",
       "      <td>DSM 15868</td>\n",
       "      <td>NaN</td>\n",
       "      <td>symbiont</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>blood</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1954</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Duck blood</td>\n",
       "      <td>NaN</td>\n",
       "      <td>USA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Gi04613</td>\n",
       "      <td>bacteria_archaea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>37C</td>\n",
       "      <td>8240957</td>\n",
       "      <td>Nonmotile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Riemerella anatipestifer A350/72</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 15868</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Blood</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod-shaped</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Septicemia</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Mesophile</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>microaerophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>isolated by Bruner and Fabricant</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DOE-GEBA 2007</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Anas platyrhynchos (duck)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Septicemic disease in waterfowls. Septicemia a...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SRR074211</td>\n",
       "      <td>0</td>\n",
       "      <td>1156157</td>\n",
       "      <td>GGZPa</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2020-06-14</td>\n",
       "      <td>239200.0</td>\n",
       "      <td>137869032.0</td>\n",
       "      <td>JGI</td>\n",
       "      <td>GGZPa</td>\n",
       "      <td>flow_count: 200 || flow_sequence: TACG || KEYS...</td>\n",
       "      <td>1081394</td>\n",
       "      <td>GGZPa</td>\n",
       "      <td>SRX032303</td>\n",
       "      <td>454 sequencing of Riemerella anatipestifer DSM...</td>\n",
       "      <td>387</td>\n",
       "      <td>4087355</td>\n",
       "      <td>Titanium</td>\n",
       "      <td>GGZP</td>\n",
       "      <td>WGS</td>\n",
       "      <td>GENOMIC</td>\n",
       "      <td>RANDOM</td>\n",
       "      <td>SINGLE -</td>\n",
       "      <td>NaN</td>\n",
       "      <td>READ_INDEX: 0; READ_CLASS: Technical Read; REA...</td>\n",
       "      <td>LS454</td>\n",
       "      <td>454 GS FLX Titanium</td>\n",
       "      <td>INSTRUMENT_MODEL: 454 GS FLX Titanium</td>\n",
       "      <td>NaN</td>\n",
       "      <td>883204.0</td>\n",
       "      <td>4087355</td>\n",
       "      <td>SRS139719</td>\n",
       "      <td>693978.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Keywords: GSC:MIGS:2.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>18897.0</td>\n",
       "      <td>387</td>\n",
       "      <td>SRP002772</td>\n",
       "      <td>Riemerella anatipestifer DSM 15868 genome sequ...</td>\n",
       "      <td>Whole Genome Sequencing</td>\n",
       "      <td>&lt;b&gt;&lt;i&gt;Riemerella anatipestifer&lt;/i&gt; DSM 15868&lt;/...</td>\n",
       "      <td>Riemerella anatipestifer ATCC 11845 = DSM 15868</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>parent_bioproject: PRJNA30815</td>\n",
       "      <td>324771</td>\n",
       "      <td>SRA026712</td>\n",
       "      <td>FTP</td>\n",
       "      <td>JGI</td>\n",
       "      <td>PGF</td>\n",
       "      <td>2010-11-29</td>\n",
       "      <td>2020-09-03 11:01:35</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 202 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  assembly_accession      strain finishing strategy (depth of coverage)  \\\n",
       "0    GCA_000023285.1    DSM 7271                                    NaN   \n",
       "1    GCA_000143785.1  ATCC 35910              Level 6, Finished33.84x;7   \n",
       "2    GCA_000173675.1       Capno      Level 2, High-Quality Draft19x;65   \n",
       "3    GCA_000174755.1  ATCC 33624      Level 2, High-Quality Draft37x;37   \n",
       "4    GCA_000183155.1   DSM 15868                                    NaN   \n",
       "\n",
       "  biotic_relationship geo_loc_name host collected_by host_disease  \\\n",
       "0                 NaN          NaN  NaN          NaN          NaN   \n",
       "1                 NaN          NaN  NaN          NaN          NaN   \n",
       "2                 NaN          NaN  NaN          NaN          NaN   \n",
       "3                 NaN          NaN  NaN          NaN          NaN   \n",
       "4            symbiont          NaN  NaN          NaN          NaN   \n",
       "\n",
       "  isolation_source sample_name_x isolation-source sample_type FDA CVM ID  \\\n",
       "0              NaN           NaN              NaN         NaN        NaN   \n",
       "1              NaN           NaN              NaN         NaN        NaN   \n",
       "2              NaN           NaN              NaN         NaN        NaN   \n",
       "3              NaN           NaN              NaN         NaN        NaN   \n",
       "4              NaN           NaN            blood         NaN        NaN   \n",
       "\n",
       "  culture_collection assembly_method collection_date env_feature synonym  \\\n",
       "0                NaN             NaN             NaN         NaN     NaN   \n",
       "1                NaN             NaN             NaN         NaN     NaN   \n",
       "2                NaN             NaN             NaN         NaN     NaN   \n",
       "3                NaN             NaN             NaN         NaN     NaN   \n",
       "4                NaN             NaN            1954         NaN     NaN   \n",
       "\n",
       "  Strain SUBJECT_ID Alias                             type-material  \\\n",
       "0    NaN        NaN   NaN    type strain of Capnocytophaga ochracea   \n",
       "1    NaN        NaN   NaN     type strain of Chryseobacterium gleum   \n",
       "2    NaN        NaN   NaN                                       NaN   \n",
       "3    NaN        NaN   NaN  type strain of Capnocytophaga gingivalis   \n",
       "4    NaN        NaN   NaN                                       NaN   \n",
       "\n",
       "               misc_param collection date Isolation Site Isolation source  \\\n",
       "0                     NaN             NaN            NaN              NaN   \n",
       "1  HMP body site, Vaginal             NaN            NaN              NaN   \n",
       "2     HMP supersite, oral             NaN            NaN              NaN   \n",
       "3     HMP supersite, oral             NaN            NaN              NaN   \n",
       "4                     NaN             NaN     Duck blood              NaN   \n",
       "\n",
       "  country note GOLD Stamp ID investigation_type culture-collection lat_lon  \\\n",
       "0     NaN  NaN           NaN                NaN                NaN     NaN   \n",
       "1     NaN  NaN           NaN                NaN         ATCC:35910     NaN   \n",
       "2     NaN  NaN           NaN                NaN                NaN     NaN   \n",
       "3     NaN  NaN           NaN                NaN         ATCC:33624     NaN   \n",
       "4     USA  NaN       Gi04613   bacteria_archaea                NaN     NaN   \n",
       "\n",
       "  alt_elev locus_tag_prefix nat-host assembly_method_version env_material  \\\n",
       "0      NaN              NaN      NaN                     NaN          NaN   \n",
       "1      NaN              NaN      NaN                     NaN          NaN   \n",
       "2      NaN              NaN      NaN                     NaN          NaN   \n",
       "3      NaN              NaN      NaN                     NaN          NaN   \n",
       "4      NaN              NaN      NaN                     NaN          NaN   \n",
       "\n",
       "  model sample_description Sample Description ArrayExpress-STRAIN_OR_LINE  \\\n",
       "0   NaN                NaN                NaN                         NaN   \n",
       "1   NaN                NaN                NaN                         NaN   \n",
       "2   NaN                NaN                NaN                         NaN   \n",
       "3   NaN                NaN                NaN                         NaN   \n",
       "4   NaN                NaN                NaN                         NaN   \n",
       "\n",
       "  ENA checklist      project_type estimated_size Temperature Optimum  \\\n",
       "0           NaN               NaN            NaN                 NaN   \n",
       "1           NaN  Reference Genome        5562000                 NaN   \n",
       "2           NaN  Reference Genome        2998000                 NaN   \n",
       "3           NaN  Reference Genome        2667000                 NaN   \n",
       "4           NaN               NaN            NaN                 37C   \n",
       "\n",
       "  isol_growth_condt   Motility                                        feature  \\\n",
       "0               NaN        NaN                                            NaN   \n",
       "1               NaN        NaN  Homo sapiens-associated habitat ENVO:00009003   \n",
       "2               NaN        NaN         human-associated habitat ENVO:00009003   \n",
       "3               NaN        NaN         human-associated habitat ENVO:00009003   \n",
       "4           8240957  Nonmotile                                            NaN   \n",
       "\n",
       "                           project_name                   assembly  \\\n",
       "0                                   NaN                        NaN   \n",
       "1            Chryseobacterium gleum F93   Newbler v. 2.0.0-Eclipse   \n",
       "2        Capnocytophaga sputigena Capno  Newbler v. 1.1.03.24-64-7   \n",
       "3  Capnocytophaga gingivalis ATCC 33624  Newbler v. 1.1.03.24-64-7   \n",
       "4      Riemerella anatipestifer A350/72                        NaN   \n",
       "\n",
       "  num_replicons isolate depth env_biome altitude ref_biomaterial package  \\\n",
       "0           NaN     NaN   NaN       NaN      NaN             NaN     NaN   \n",
       "1           NaN     NaN   NaN       NaN      NaN             NaN     NaN   \n",
       "2           NaN     NaN   NaN       NaN      NaN             NaN     NaN   \n",
       "3           NaN     NaN   NaN       NaN      NaN             NaN     NaN   \n",
       "4             1     NaN   NaN       NaN      NaN       DSM 15868     NaN   \n",
       "\n",
       "  geographic location ArrayExpress-StrainOrLine ArrayExpress-Phenotype  \\\n",
       "0                 NaN                       NaN                    NaN   \n",
       "1                 NaN                       NaN                    NaN   \n",
       "2                 NaN                       NaN                    NaN   \n",
       "3                 NaN                       NaN                    NaN   \n",
       "4                 NaN                       NaN                    NaN   \n",
       "\n",
       "  ArrayExpress-SPECIES INSDC center alias geo-loc-name environment host_sex  \\\n",
       "0                  NaN                NaN          NaN         NaN      NaN   \n",
       "1                  NaN                NaN          NaN         NaN      NaN   \n",
       "2                  NaN                NaN          NaN         NaN      NaN   \n",
       "3                  NaN                NaN          NaN         NaN      NaN   \n",
       "4                  NaN                NaN          NaN         NaN      NaN   \n",
       "\n",
       "  elev PublicAccession completeness_estimated  \\\n",
       "0  NaN             NaN                    NaN   \n",
       "1  NaN             NaN                    NaN   \n",
       "2  NaN             NaN                    NaN   \n",
       "3  NaN             NaN                    NaN   \n",
       "4  NaN             NaN                    NaN   \n",
       "\n",
       "  geographic location (country and/or sea) latitude and longitude  \\\n",
       "0                                      NaN                    NaN   \n",
       "1                                      NaN                    NaN   \n",
       "2                                      NaN                    NaN   \n",
       "3                                      NaN                    NaN   \n",
       "4                                      NaN                    NaN   \n",
       "\n",
       "  specific_host ArrayExpress-Species ENA-FIRST-PUBLIC INSDC center name  \\\n",
       "0           NaN                  NaN              NaN               NaN   \n",
       "1           NaN                  NaN              NaN               NaN   \n",
       "2           NaN                  NaN              NaN               NaN   \n",
       "3           NaN                  NaN              NaN               NaN   \n",
       "4           NaN                  NaN              NaN               NaN   \n",
       "\n",
       "  specific host Body Sample Site   pH                            biome  \\\n",
       "0           NaN              NaN  NaN                              NaN   \n",
       "1  Homo sapiens              NaN  NaN  terrestrial biome ENVO:00000446   \n",
       "2  Homo sapiens              NaN  NaN  terrestrial biome ENVO:00000446   \n",
       "3  Homo sapiens              NaN  NaN  terrestrial biome ENVO:00000446   \n",
       "4           NaN            Blood  NaN                              NaN   \n",
       "\n",
       "  biomaterial_provider isolate_name_alias contamination_estimated  \\\n",
       "0                  NaN                NaN                     NaN   \n",
       "1                  NaN                NaN                     NaN   \n",
       "2                  NaN                NaN                     NaN   \n",
       "3                  NaN                NaN                     NaN   \n",
       "4                  NaN                NaN                     NaN   \n",
       "\n",
       "  host scientific name anonymized_name ENA-LAST-UPDATE INSDC first public  \\\n",
       "0                  NaN             NaN             NaN                NaN   \n",
       "1                  NaN             NaN             NaN                NaN   \n",
       "2                  NaN             NaN             NaN                NaN   \n",
       "3                  NaN             NaN             NaN                NaN   \n",
       "4                  NaN             NaN             NaN                NaN   \n",
       "\n",
       "   Cell Shape BioSampleModel environmental_sample  \\\n",
       "0         NaN            NaN                  NaN   \n",
       "1         NaN        MIGS.ba                  NaN   \n",
       "2         NaN        MIGS.ba                  NaN   \n",
       "3         NaN        MIGS.ba                  NaN   \n",
       "4  Rod-shaped            NaN                  NaN   \n",
       "\n",
       "                           material identified_by host health state  \\\n",
       "0                               NaN           NaN               NaN   \n",
       "1  biological product ENVO:02000043           NaN               NaN   \n",
       "2  biological product ENVO:02000043           NaN               NaN   \n",
       "3  biological product ENVO:02000043           NaN               NaN   \n",
       "4                               NaN           NaN               NaN   \n",
       "\n",
       "  INSDC last update    Diseases strain_name_alias genome_coverage  \\\n",
       "0               NaN         NaN               NaN             NaN   \n",
       "1               NaN         NaN               NaN             NaN   \n",
       "2               NaN         NaN               NaN             NaN   \n",
       "3               NaN         NaN               NaN             NaN   \n",
       "4               NaN  Septicemia               NaN             NaN   \n",
       "\n",
       "  INSDC status host taxid isolation source Gram Staining Temperature Range  \\\n",
       "0          NaN        NaN              NaN           NaN               NaN   \n",
       "1          NaN       9606              NaN           NaN               NaN   \n",
       "2          NaN       9606              NaN           NaN               NaN   \n",
       "3          NaN       9606              NaN           NaN               NaN   \n",
       "4          NaN        NaN              NaN           NaN         Mesophile   \n",
       "\n",
       "   ... Historical Monobactam Produced identification method supplier_name  \\\n",
       "0  ...                            NaN                   NaN           NaN   \n",
       "1  ...                            NaN                   NaN           NaN   \n",
       "2  ...                            NaN                   NaN           NaN   \n",
       "3  ...                            NaN                   NaN           NaN   \n",
       "4  ...                            NaN                   NaN           NaN   \n",
       "\n",
       "  SRA accession host_disease_outcome Unique Monobactam Clusters Identified  \\\n",
       "0           NaN                  NaN                                   NaN   \n",
       "1           NaN                  NaN                                   NaN   \n",
       "2           NaN                  NaN                                   NaN   \n",
       "3           NaN                  NaN                                   NaN   \n",
       "4           NaN                  NaN                                   NaN   \n",
       "\n",
       "  ProjectAccession Sample Name Type Strain Phenotypes Gene Calling Method  \\\n",
       "0              NaN         NaN         NaN        NaN                 NaN   \n",
       "1              NaN         NaN         NaN        NaN                 NaN   \n",
       "2              NaN         NaN         NaN        NaN                 NaN   \n",
       "3              NaN         NaN         NaN        NaN                 NaN   \n",
       "4              NaN         NaN         Yes        NaN                 NaN   \n",
       "\n",
       "  temp Title investigation type Host    rel_to_oxygen host_age Species  \\\n",
       "0  NaN   NaN                NaN  NaN              NaN      NaN     NaN   \n",
       "1  NaN   NaN                NaN  NaN              NaN      NaN     NaN   \n",
       "2  NaN   NaN                NaN  NaN              NaN      NaN     NaN   \n",
       "3  NaN   NaN                NaN  NaN              NaN      NaN     NaN   \n",
       "4  NaN   NaN                NaN  NaN  microaerophilic      NaN     NaN   \n",
       "\n",
       "  mapping_method serovar                                                sop  \\\n",
       "0            NaN     NaN                                                NaN   \n",
       "1            NaN     NaN  http://hmpdacc.org/doc/CommonGeneAnnotation_SO...   \n",
       "2            NaN     NaN  http://hmpdacc.org/doc/CommonGeneAnnotation_SO...   \n",
       "3            NaN     NaN  http://hmpdacc.org/doc/CommonGeneAnnotation_SO...   \n",
       "4            NaN     NaN                                                NaN   \n",
       "\n",
       "                 Isolation Comments finishing_strategy Genus  \\\n",
       "0                               NaN                NaN   NaN   \n",
       "1                               NaN                NaN   NaN   \n",
       "2                               NaN                NaN   NaN   \n",
       "3                               NaN                NaN   NaN   \n",
       "4  isolated by Bruner and Fabricant                NaN   NaN   \n",
       "\n",
       "  mapping_method_version environment (biome) attribute_package  \\\n",
       "0                    NaN                 NaN               NaN   \n",
       "1                    NaN                 NaN               NaN   \n",
       "2                    NaN                 NaN               NaN   \n",
       "3                    NaN                 NaN               NaN   \n",
       "4                    NaN                 NaN               NaN   \n",
       "\n",
       "  metagenome_source environment (feature) trophic_level metagenomic  \\\n",
       "0               NaN                   NaN           NaN         NaN   \n",
       "1               NaN                   NaN           NaN         NaN   \n",
       "2               NaN                   NaN           NaN         NaN   \n",
       "3               NaN                   NaN           NaN         NaN   \n",
       "4               NaN                   NaN           NaN         NaN   \n",
       "\n",
       "  environment (material) Funding Program quality_assessment_method  \\\n",
       "0                    NaN             NaN                       NaN   \n",
       "1                    NaN             NaN                       NaN   \n",
       "2                    NaN             NaN                       NaN   \n",
       "3                    NaN             NaN                       NaN   \n",
       "4                    NaN   DOE-GEBA 2007                       NaN   \n",
       "\n",
       "  quality_assessment_method_version geographic location (latitude)  \\\n",
       "0                               NaN                            NaN   \n",
       "1                               NaN                            NaN   \n",
       "2                               NaN                            NaN   \n",
       "3                               NaN                            NaN   \n",
       "4                               NaN                            NaN   \n",
       "\n",
       "  source_mat_id host_tissue_sampled geographic location (longitude)  \\\n",
       "0           NaN                 NaN                             NaN   \n",
       "1    ATCC 35910                 NaN                             NaN   \n",
       "2    ATCC 33612                 NaN                             NaN   \n",
       "3    ATCC 33624                 NaN                             NaN   \n",
       "4           NaN                 NaN                             NaN   \n",
       "\n",
       "                   Host Name subsrc_note  \\\n",
       "0                        NaN         NaN   \n",
       "1                        NaN         NaN   \n",
       "2                        NaN         NaN   \n",
       "3                        NaN         NaN   \n",
       "4  Anas platyrhynchos (duck)         NaN   \n",
       "\n",
       "                                         Host Health value  \\\n",
       "0                                                NaN   NaN   \n",
       "1                                                NaN   NaN   \n",
       "2                                                NaN   NaN   \n",
       "3                                                NaN   NaN   \n",
       "4  Septicemic disease in waterfowls. Septicemia a...   NaN   \n",
       "\n",
       "  plant-associated environmental package project name sequencing method  \\\n",
       "0                                    NaN          NaN               NaN   \n",
       "1                                    NaN          NaN        454-GS-FLX   \n",
       "2                                    NaN          NaN        454-GS-FLX   \n",
       "3                                    NaN          NaN               454   \n",
       "4                                    NaN          NaN               NaN   \n",
       "\n",
       "  Body Sample SubSite run_accession sra_ID   run_ID  \\\n",
       "0                 NaN     SRR013476      0  1093152   \n",
       "1                 NaN     SRR004314      0  1083352   \n",
       "2                 NaN     SRR005028      0  1084559   \n",
       "3                 NaN     SRR013509      0  1093169   \n",
       "4                 NaN     SRR074211      0  1156157   \n",
       "\n",
       "                     run_alias    run_date updated_date     spots  \\\n",
       "0                        FWCBa         NaN   2019-11-28    9047.0   \n",
       "1  2007-08-02.1147-FLX01070128         NaN   2019-11-28  158268.0   \n",
       "2                      FCRNIZW  2008-09-30   2020-06-16  229445.0   \n",
       "3                      FJ8IWNQ  2009-02-24   2020-06-17  289003.0   \n",
       "4                        GGZPa         NaN   2020-06-14  239200.0   \n",
       "\n",
       "         bases  run_center            experiment_name  \\\n",
       "0    2866262.0         JGI                      FWCBa   \n",
       "1   44181818.0         NaN  4WG_HMP0003.CGLE.01_000sA   \n",
       "2   64077935.0         NaN      HMP015-454 Experiment   \n",
       "3   79866027.0         NaN      HMP016-454 Experiment   \n",
       "4  137869032.0         JGI                      GGZPa   \n",
       "\n",
       "                                       run_attribute experiment_ID  \\\n",
       "0  flow_count: 110 || flow_sequence: TACG || TCAG...       1051231   \n",
       "1                                    run_center: BCM       1048730   \n",
       "2                                                NaN       1049060   \n",
       "3                                                NaN       1051243   \n",
       "4  flow_count: 200 || flow_sequence: TACG || KEYS...       1081394   \n",
       "\n",
       "            experiment_alias  experiment_accession  \\\n",
       "0                      FWCBa             SRX002589   \n",
       "1  4WG_HMP0003.CGLE.01_000sA             SRX001228   \n",
       "2      HMP015-454 Experiment             SRX001307   \n",
       "3      HMP016-454 Experiment             SRX002609   \n",
       "4                      GGZPa             SRX032303   \n",
       "\n",
       "                                    experiment_title        study_name  \\\n",
       "0                                                NaN              FWCB   \n",
       "1  454 sequencing of Chryseobacterium gleum ATCC ...               NaN   \n",
       "2  454 sequencing of Capnocytophaga sputigena Cap...  HMP015-454 Study   \n",
       "3                                                NaN  HMP016-454 Study   \n",
       "4  454 sequencing of Riemerella anatipestifer DSM...               387   \n",
       "\n",
       "       sample_name_y                                 design_description  \\\n",
       "0               FWCB                             none provided            \n",
       "1                NaN                                       FLX fragment   \n",
       "2  HMP015-454 Sample  Sequencing and analysis of the genome of Capno...   \n",
       "3  HMP016-454 Sample  Sequencing and analysis of the genome of Capno...   \n",
       "4            4087355                                  Titanium            \n",
       "\n",
       "                library_name library_strategy library_source  \\\n",
       "0                       FWCB              WGS        GENOMIC   \n",
       "1  4WG_HMP0003.CGLE.01_000sA              WGS        GENOMIC   \n",
       "2            HMP015-G-02-697              WGS        GENOMIC   \n",
       "3           HMP016-PL-01-680              WGS        GENOMIC   \n",
       "4                       GGZP              WGS        GENOMIC   \n",
       "\n",
       "  library_selection                                     library_layout  \\\n",
       "0            RANDOM                                          SINGLE -    \n",
       "1            RANDOM                                          SINGLE -    \n",
       "2            RANDOM                                          SINGLE -    \n",
       "3            RANDOM  PAIRED - NOMINAL_LENGTH: 3000; NOMINAL_SDEV: 3...   \n",
       "4            RANDOM                                          SINGLE -    \n",
       "\n",
       "  library_construction_protocol  \\\n",
       "0                           NaN   \n",
       "1                           NaN   \n",
       "2                           NaN   \n",
       "3                           NaN   \n",
       "4                           NaN   \n",
       "\n",
       "                                           read_spec platform  \\\n",
       "0  READ_INDEX: 0; READ_CLASS: Technical Read; REA...    LS454   \n",
       "1  READ_INDEX: 0; READ_CLASS: Technical Read; REA...    LS454   \n",
       "2  READ_INDEX: 0; READ_CLASS: Technical Read; REA...    LS454   \n",
       "3  READ_INDEX: 0; READ_CLASS: Technical Read; REA...    LS454   \n",
       "4  READ_INDEX: 0; READ_CLASS: Technical Read; REA...    LS454   \n",
       "\n",
       "      instrument_model                    platform_parameters  \\\n",
       "0           454 GS FLX           INSTRUMENT_MODEL: 454 GS FLX   \n",
       "1           454 GS FLX           INSTRUMENT_MODEL: 454 GS FLX   \n",
       "2           454 GS FLX           INSTRUMENT_MODEL: 454 GS FLX   \n",
       "3           454 GS FLX           INSTRUMENT_MODEL: 454 GS FLX   \n",
       "4  454 GS FLX Titanium  INSTRUMENT_MODEL: 454 GS FLX Titanium   \n",
       "\n",
       "  experiment_attribute sample_ID       sample_alias  sample_accession  \\\n",
       "0                  NaN  780997.0               FWCB         SRS002078   \n",
       "1                  NaN  779023.0        BCM-HMP0003         SRS000860   \n",
       "2                  NaN  779245.0  HMP015-454 Sample         SRS000995   \n",
       "3                  NaN  781003.0  HMP016-454 Sample         SRS002089   \n",
       "4                  NaN  883204.0            4087355         SRS139719   \n",
       "\n",
       "   taxon_id common_name                                        description  \\\n",
       "0  521097.0         NaN                   Capnocytophaga ochracea DSM 7271   \n",
       "1  525257.0         NaN                  Chryseobacterium gleum ATCC 35910   \n",
       "2  553177.0         NaN  Capnocytophaga sputigena is a gram-negative, r...   \n",
       "3  553178.0         NaN  Capnocytophaga gingivalis is a gram-negative, ...   \n",
       "4  693978.0         NaN                             Keywords: GSC:MIGS:2.1   \n",
       "\n",
       "                                     sample_url_link study_ID  \\\n",
       "0                                                NaN  16818.0   \n",
       "1  BCM HMP Project: http://www.hgsc.bcm.edu/micro...  16258.0   \n",
       "2                JCVI HMP Page: http://hmp.jcvi.org/  16339.0   \n",
       "3                                                NaN  16824.0   \n",
       "4                                                NaN  18897.0   \n",
       "\n",
       "              study_alias  study_accession  \\\n",
       "0                    FWCB        SRP000584   \n",
       "1  WG_HMP0003.CGLE.01_000        SRP000258   \n",
       "2        HMP015-454 Study        SRP000317   \n",
       "3        HMP016-454 Study        SRP000590   \n",
       "4                     387        SRP002772   \n",
       "\n",
       "                                         study_title               study_type  \\\n",
       "0                               Causes periodontitis  Whole Genome Sequencing   \n",
       "1  Reference genome for the Human Microbiome Project  Whole Genome Sequencing   \n",
       "2  reference genome for the Human Microbiome Project  Whole Genome Sequencing   \n",
       "3  Reference genome for the Human Microbiome Project  Whole Genome Sequencing   \n",
       "4  Riemerella anatipestifer DSM 15868 genome sequ...  Whole Genome Sequencing   \n",
       "\n",
       "                                      study_abstract  \\\n",
       "0  <P><B><I>Capnocytophaga ochracea</I> DSM 7271<...   \n",
       "1  <P> <I><B>Chryseobacterium gleum</I> ATCC 3591...   \n",
       "2  <P> <I><B>Capnocytophaga sputigena</I> Capno</...   \n",
       "3  <b><i>Capnocytophaga gingivalis</i> ATCC 33624...   \n",
       "4  <b><i>Riemerella anatipestifer</i> DSM 15868</...   \n",
       "\n",
       "                               center_project_name study_description  \\\n",
       "0                 Capnocytophaga ochracea DSM 7271               NaN   \n",
       "1                Chryseobacterium gleum ATCC 35910               NaN   \n",
       "2              Capnocytophaga sputigena ATCC 33612               NaN   \n",
       "3             Capnocytophaga gingivalis ATCC 33624               NaN   \n",
       "4  Riemerella anatipestifer ATCC 11845 = DSM 15868               NaN   \n",
       "\n",
       "  study_url_link                study_attribute submission_ID  \\\n",
       "0            NaN  parent_bioproject: PRJNA30815        310018   \n",
       "1            NaN  parent_bioproject: PRJNA28331        304555   \n",
       "2            NaN  parent_bioproject: PRJNA28331        305210   \n",
       "3            NaN  parent_bioproject: PRJNA28331        310024   \n",
       "4            NaN  parent_bioproject: PRJNA30815        324771   \n",
       "\n",
       "  submission_accession                                 submission_comment  \\\n",
       "0            SRA008146                                                FTP   \n",
       "1            SRA002043                                                NaN   \n",
       "2            SRA002772  JCVI Human Microbiome Reference - HMP015 seque...   \n",
       "3            SRA008153  JCVI Human Microbiome Reference - HMP016 seque...   \n",
       "4            SRA026712                                                FTP   \n",
       "\n",
       "  submission_center submission_lab submission_date        sradb_updated  \n",
       "0               JGI            PGF      2009-02-19  2020-09-03 11:01:35  \n",
       "1               BCM            NaN      2008-09-11  2020-09-03 11:01:35  \n",
       "2              JCVI            NaN      2008-09-30  2020-09-03 11:01:35  \n",
       "3              JCVI            NaN      2009-02-24  2020-09-03 11:01:35  \n",
       "4               JGI            PGF      2010-11-29  2020-09-03 11:01:35  \n",
       "\n",
       "[5 rows x 202 columns]"
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merged.drop(columns=[\"merge\", \"sample_attribute\"], inplace=True)\n",
    "\n",
    "null_cols = merged.isnull().all() #REMOVE empty columns\n",
    "merged = merged[merged.columns[~null_cols]]\n",
    "\n",
    "merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "df.to_csv(\"SRA_metadata.csv\", index=False)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
