{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2.1 Metadata"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Author: Sandra Godinho Silva \\\n",
    "Creation date: 07/09/2020 \\\n",
    "Version: 0.1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#import libraries\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import seaborn as sns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.set_option('display.max_columns', 1000)\n",
    "pd.set_option('display.max_rows', 10)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Import dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genome_ID</th>\n",
       "      <th>Bin_Id</th>\n",
       "      <th>Marker lineage</th>\n",
       "      <th>Completeness</th>\n",
       "      <th>Contamination</th>\n",
       "      <th>Strain_heterogeneity</th>\n",
       "      <th>Quality_score</th>\n",
       "      <th>classification</th>\n",
       "      <th>Domain</th>\n",
       "      <th>Phyla</th>\n",
       "      <th>...</th>\n",
       "      <th>scaf_L90</th>\n",
       "      <th>ctg_N90</th>\n",
       "      <th>ctg_L90</th>\n",
       "      <th>scaf_max</th>\n",
       "      <th>ctg_max</th>\n",
       "      <th>scaf_n_gt50K</th>\n",
       "      <th>scaf_pct_gt50K</th>\n",
       "      <th>gc_avg</th>\n",
       "      <th>gc_std</th>\n",
       "      <th>Classification_quality</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>GCA_000016645.1_ASM1664v1_genomic</td>\n",
       "      <td>f__Flavobacteriaceae (UID2817)</td>\n",
       "      <td>99.65</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.0</td>\n",
       "      <td>98.95</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>...</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>100.0</td>\n",
       "      <td>0.34113</td>\n",
       "      <td>0.0</td>\n",
       "      <td>Medium</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>GCA_000023285.1_ASM2328v1_genomic</td>\n",
       "      <td>p__Bacteroidetes (UID2605)</td>\n",
       "      <td>100.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>100.00</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>...</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>100.0</td>\n",
       "      <td>0.39588</td>\n",
       "      <td>0.0</td>\n",
       "      <td>Medium</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 35 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         Genome_ID                             Bin_Id  \\\n",
       "0  GCA_000016645.1  GCA_000016645.1_ASM1664v1_genomic   \n",
       "1  GCA_000023285.1  GCA_000023285.1_ASM2328v1_genomic   \n",
       "\n",
       "                   Marker lineage  Completeness  Contamination  \\\n",
       "0  f__Flavobacteriaceae (UID2817)         99.65           0.14   \n",
       "1      p__Bacteroidetes (UID2605)        100.00           0.00   \n",
       "\n",
       "   Strain_heterogeneity  Quality_score  \\\n",
       "0                   0.0          98.95   \n",
       "1                   0.0         100.00   \n",
       "\n",
       "                                      classification    Domain         Phyla  \\\n",
       "0  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "1  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "\n",
       "   ... scaf_L90 ctg_N90  ctg_L90 scaf_max  ctg_max  scaf_n_gt50K  \\\n",
       "0  ...  6096872       1  6096872  6096872  6096872             1   \n",
       "1  ...  2612925       1  2612925  2612925  2612925             1   \n",
       "\n",
       "   scaf_pct_gt50K   gc_avg  gc_std  Classification_quality  \n",
       "0           100.0  0.34113     0.0                  Medium  \n",
       "1           100.0  0.39588     0.0                  Medium  \n",
       "\n",
       "[2 rows x 35 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset = pd.read_csv(\"../1_Dataset_creation/Dataset.csv\")\n",
    "dataset.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "genomes = dataset[\"Genome_ID\"]\n",
    "genomes.to_csv(\"assembly_ids.csv\", index=False,  header=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2680"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(genomes)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.1 Import metadata coupled with GenBank"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "***On the command line:*** \\\n",
    "wget ftp://ftp.ncbi.nlm.nih.gov/genomes/ASSEMBLY_REPORTS/assembly_summary_genbank.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "genbank_metadata = pd.read_csv(\"assembly_summary_genbank.txt\", sep=\"\\t\", skiprows=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th># assembly_accession</th>\n",
       "      <th>bioproject</th>\n",
       "      <th>biosample</th>\n",
       "      <th>wgs_master</th>\n",
       "      <th>refseq_category</th>\n",
       "      <th>taxid</th>\n",
       "      <th>species_taxid</th>\n",
       "      <th>organism_name</th>\n",
       "      <th>infraspecific_name</th>\n",
       "      <th>isolate</th>\n",
       "      <th>version_status</th>\n",
       "      <th>assembly_level</th>\n",
       "      <th>release_type</th>\n",
       "      <th>genome_rep</th>\n",
       "      <th>seq_rel_date</th>\n",
       "      <th>asm_name</th>\n",
       "      <th>submitter</th>\n",
       "      <th>gbrs_paired_asm</th>\n",
       "      <th>paired_asm_comp</th>\n",
       "      <th>ftp_path</th>\n",
       "      <th>excluded_from_refseq</th>\n",
       "      <th>relation_to_type_material</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000001215.4</td>\n",
       "      <td>PRJNA13812</td>\n",
       "      <td>SAMN02803731</td>\n",
       "      <td>NaN</td>\n",
       "      <td>reference genome</td>\n",
       "      <td>7227</td>\n",
       "      <td>7227</td>\n",
       "      <td>Drosophila melanogaster</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>Major</td>\n",
       "      <td>Full</td>\n",
       "      <td>2014/08/01</td>\n",
       "      <td>Release 6 plus ISO1 MT</td>\n",
       "      <td>The FlyBase Consortium/Berkeley Drosophila Gen...</td>\n",
       "      <td>GCF_000001215.4</td>\n",
       "      <td>identical</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000001405.28</td>\n",
       "      <td>PRJNA31257</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>reference genome</td>\n",
       "      <td>9606</td>\n",
       "      <td>9606</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>Chromosome</td>\n",
       "      <td>Patch</td>\n",
       "      <td>Full</td>\n",
       "      <td>2019/02/28</td>\n",
       "      <td>GRCh38.p13</td>\n",
       "      <td>Genome Reference Consortium</td>\n",
       "      <td>GCF_000001405.39</td>\n",
       "      <td>different</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  # assembly_accession  bioproject     biosample wgs_master   refseq_category  \\\n",
       "0      GCA_000001215.4  PRJNA13812  SAMN02803731        NaN  reference genome   \n",
       "1     GCA_000001405.28  PRJNA31257           NaN        NaN  reference genome   \n",
       "\n",
       "   taxid  species_taxid            organism_name infraspecific_name isolate  \\\n",
       "0   7227           7227  Drosophila melanogaster                NaN     NaN   \n",
       "1   9606           9606             Homo sapiens                NaN     NaN   \n",
       "\n",
       "  version_status assembly_level release_type genome_rep seq_rel_date  \\\n",
       "0         latest     Chromosome        Major       Full   2014/08/01   \n",
       "1         latest     Chromosome        Patch       Full   2019/02/28   \n",
       "\n",
       "                 asm_name                                          submitter  \\\n",
       "0  Release 6 plus ISO1 MT  The FlyBase Consortium/Berkeley Drosophila Gen...   \n",
       "1              GRCh38.p13                        Genome Reference Consortium   \n",
       "\n",
       "    gbrs_paired_asm paired_asm_comp  \\\n",
       "0   GCF_000001215.4       identical   \n",
       "1  GCF_000001405.39       different   \n",
       "\n",
       "                                            ftp_path excluded_from_refseq  \\\n",
       "0  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...                  NaN   \n",
       "1  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...                  NaN   \n",
       "\n",
       "  relation_to_type_material  \n",
       "0                       NaN  \n",
       "1                       NaN  "
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "genbank_metadata.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "both          2679\n",
       "left_only        1\n",
       "right_only       0\n",
       "Name: merge, dtype: int64"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "genbank_filtered = pd.merge(genomes, genbank_metadata, how=\"left\", left_on=\"Genome_ID\", right_on=\"# assembly_accession\", indicator=\"merge\")\n",
    "genbank_filtered[\"merge\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genome_ID</th>\n",
       "      <th># assembly_accession</th>\n",
       "      <th>bioproject</th>\n",
       "      <th>biosample</th>\n",
       "      <th>wgs_master</th>\n",
       "      <th>refseq_category</th>\n",
       "      <th>taxid</th>\n",
       "      <th>species_taxid</th>\n",
       "      <th>organism_name</th>\n",
       "      <th>infraspecific_name</th>\n",
       "      <th>isolate</th>\n",
       "      <th>version_status</th>\n",
       "      <th>assembly_level</th>\n",
       "      <th>release_type</th>\n",
       "      <th>genome_rep</th>\n",
       "      <th>seq_rel_date</th>\n",
       "      <th>asm_name</th>\n",
       "      <th>submitter</th>\n",
       "      <th>gbrs_paired_asm</th>\n",
       "      <th>paired_asm_comp</th>\n",
       "      <th>ftp_path</th>\n",
       "      <th>excluded_from_refseq</th>\n",
       "      <th>relation_to_type_material</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>NaN</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>986.0</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>strain=UW101; ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>Major</td>\n",
       "      <td>Full</td>\n",
       "      <td>2007/05/01</td>\n",
       "      <td>ASM1664v1</td>\n",
       "      <td>US DOE Joint Genome Institute</td>\n",
       "      <td>GCF_000016645.1</td>\n",
       "      <td>identical</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>1018.0</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>strain=DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>Major</td>\n",
       "      <td>Full</td>\n",
       "      <td>2009/08/26</td>\n",
       "      <td>ASM2328v1</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)</td>\n",
       "      <td>GCF_000023285.1</td>\n",
       "      <td>identical</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Genome_ID # assembly_accession  bioproject     biosample wgs_master  \\\n",
       "0  GCA_000016645.1      GCA_000016645.1  PRJNA16082  SAMN02598357        NaN   \n",
       "1  GCA_000023285.1      GCA_000023285.1  PRJNA29403  SAMN00001911        NaN   \n",
       "\n",
       "         refseq_category     taxid  species_taxid  \\\n",
       "0  representative genome  376686.0          986.0   \n",
       "1  representative genome  521097.0         1018.0   \n",
       "\n",
       "                      organism_name        infraspecific_name isolate  \\\n",
       "0   Flavobacterium johnsoniae UW101  strain=UW101; ATCC 17061     NaN   \n",
       "1  Capnocytophaga ochracea DSM 7271           strain=DSM 7271     NaN   \n",
       "\n",
       "  version_status   assembly_level release_type genome_rep seq_rel_date  \\\n",
       "0         latest  Complete Genome        Major       Full   2007/05/01   \n",
       "1         latest  Complete Genome        Major       Full   2009/08/26   \n",
       "\n",
       "    asm_name                                submitter  gbrs_paired_asm  \\\n",
       "0  ASM1664v1            US DOE Joint Genome Institute  GCF_000016645.1   \n",
       "1  ASM2328v1  US DOE Joint Genome Institute (JGI-PGF)  GCF_000023285.1   \n",
       "\n",
       "  paired_asm_comp                                           ftp_path  \\\n",
       "0       identical  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "1       identical  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "\n",
       "  excluded_from_refseq    relation_to_type_material  \n",
       "0                  NaN  assembly from type material  \n",
       "1                  NaN  assembly from type material  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "genbank_filtered.drop(columns=[\"merge\"], inplace=True)\n",
    "genbank_filtered.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "#REMOVE empty columns\n",
    "null_cols = genbank_filtered.isnull().all()\n",
    "genbank_filtered = genbank_filtered[genbank_filtered.columns[~null_cols]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "#genbank_filtered.info()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.2 Import Metadata from PATRIC"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Download Genome features for the taxon Flavobacteriia on 7 September 2020"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "patric = pd.read_csv(\"PATRIC_genome_Flavobacteria.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Genome ID', 'Genome Name', 'Organism Name', 'NCBI Taxon ID', 'Genome Status', 'Strain', 'Serovar', 'Biovar', 'Pathovar', 'MLST', 'Other Typing', 'Culture Collection', 'Type Strain', 'Completion Date', 'Publication', 'BioProject Accession', 'BioSample Accession', 'Assembly Accession', 'SRA Accession', 'GenBank Accessions', 'RefSeq Accessions', 'Sequencing Centers', 'Sequencing Status', 'Sequencing Platform', 'Sequencing Depth', 'Assembly Method', 'Chromosomes', 'Plasmids', 'Contigs', 'Genome Length', 'GC Content', 'PATRIC CDS', 'RefSeq CDS', 'Isolation Site', 'Isolation Source', 'Isolation Comments', 'Collection Date', 'Isolation Country', 'Geographic Location', 'Latitude', 'Longitude', 'Altitude', 'Depth', 'Other Environmental', 'Host Name', 'Host Gender', 'Host Age', 'Host Health', 'Body Sample Site', 'Body Sample Subsite', 'Other Clinical', 'AntiMicrobial Resistance', 'AntiMicrobial Resistance Evidence', 'Gram Stain', 'Cell Shape', 'Motility', 'Sporulation', 'Temperature Range', 'Optimal Temperature', 'Salinity', 'Oxygen Requirement', 'Habitat', 'Disease', 'Comments', 'Additional Metadata', 'Coarse Consistency', 'Fine Consistency', 'Checkm Completeness', 'Checkm Contamination', 'Genome Quality Flags', 'Genome Quality', 'Date Inserted', 'Date Modified']\n"
     ]
    }
   ],
   "source": [
    "l = patric.columns.to_list()\n",
    "print(l)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "#REMOVE uninteresting columns\n",
    "patric = patric.drop(columns=['Genome ID','PATRIC CDS','Coarse Consistency', 'Fine Consistency', 'Checkm Completeness', 'Checkm Contamination','Genome Quality','Date Inserted', 'Date Modified'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "#REMOVE empty columns\n",
    "null_cols = patric.isnull().all()\n",
    "patric = patric[patric.columns[~null_cols]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genome Name</th>\n",
       "      <th>Organism Name</th>\n",
       "      <th>NCBI Taxon ID</th>\n",
       "      <th>Genome Status</th>\n",
       "      <th>Strain</th>\n",
       "      <th>Serovar</th>\n",
       "      <th>Pathovar</th>\n",
       "      <th>MLST</th>\n",
       "      <th>Other Typing</th>\n",
       "      <th>Culture Collection</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Completion Date</th>\n",
       "      <th>Publication</th>\n",
       "      <th>BioProject Accession</th>\n",
       "      <th>BioSample Accession</th>\n",
       "      <th>Assembly Accession</th>\n",
       "      <th>SRA Accession</th>\n",
       "      <th>GenBank Accessions</th>\n",
       "      <th>RefSeq Accessions</th>\n",
       "      <th>Sequencing Centers</th>\n",
       "      <th>Sequencing Status</th>\n",
       "      <th>Sequencing Platform</th>\n",
       "      <th>Sequencing Depth</th>\n",
       "      <th>Assembly Method</th>\n",
       "      <th>Chromosomes</th>\n",
       "      <th>Plasmids</th>\n",
       "      <th>Contigs</th>\n",
       "      <th>Genome Length</th>\n",
       "      <th>GC Content</th>\n",
       "      <th>RefSeq CDS</th>\n",
       "      <th>Isolation Source</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>Collection Date</th>\n",
       "      <th>Isolation Country</th>\n",
       "      <th>Geographic Location</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Altitude</th>\n",
       "      <th>Depth</th>\n",
       "      <th>Other Environmental</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>Host Gender</th>\n",
       "      <th>Host Age</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>Other Clinical</th>\n",
       "      <th>Gram Stain</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>Motility</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Optimal Temperature</th>\n",
       "      <th>Salinity</th>\n",
       "      <th>Oxygen Requirement</th>\n",
       "      <th>Habitat</th>\n",
       "      <th>Disease</th>\n",
       "      <th>Comments</th>\n",
       "      <th>Additional Metadata</th>\n",
       "      <th>Genome Quality Flags</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Muricauda taeanensis strain JCM 17757</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1005926</td>\n",
       "      <td>WGS</td>\n",
       "      <td>JCM 17757</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2018-09-20T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA437398</td>\n",
       "      <td>SAMN09939642</td>\n",
       "      <td>GCA_003584105.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>QXFO01000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Second Institute of Oceanography, State Oceani...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Illumina HiSeq X-Ten</td>\n",
       "      <td>308x</td>\n",
       "      <td>ABySS v. 1.9.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>28</td>\n",
       "      <td>4101660.0</td>\n",
       "      <td>45.60536</td>\n",
       "      <td>3644.0</td>\n",
       "      <td>tidal mudflat</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>South Korea</td>\n",
       "      <td>South Korea: Taean coast of the Yellow Sea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5m</td>\n",
       "      <td>env_biome:marine biome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>This project was used for calculating ANI valu...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Myroides injenensis M09-0166</td>\n",
       "      <td>Myroides injenensis M09-0166</td>\n",
       "      <td>1008457</td>\n",
       "      <td>WGS</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td></td>\n",
       "      <td>NaN</td>\n",
       "      <td>2012-02-04T00:00:00Z</td>\n",
       "      <td>22535932</td>\n",
       "      <td>PRJDA171989</td>\n",
       "      <td>SAMD00036610</td>\n",
       "      <td>GCA_000246945.2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BAEX00000000</td>\n",
       "      <td>-</td>\n",
       "      <td>Korea Research Institute of Bioscience and Bio...</td>\n",
       "      <td>WGS</td>\n",
       "      <td>454</td>\n",
       "      <td>36x</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>226</td>\n",
       "      <td>3458160.0</td>\n",
       "      <td>31.35000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>urine of a patient with fever</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The Myroides strains are not normal human micr...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                             Genome Name                 Organism Name  \\\n",
       "0  Muricauda taeanensis strain JCM 17757                           NaN   \n",
       "1           Myroides injenensis M09-0166  Myroides injenensis M09-0166   \n",
       "\n",
       "   NCBI Taxon ID Genome Status     Strain Serovar Pathovar MLST Other Typing  \\\n",
       "0        1005926           WGS  JCM 17757     NaN      NaN  NaN          NaN   \n",
       "1        1008457           WGS        NaN     NaN      NaN  NaN          NaN   \n",
       "\n",
       "  Culture Collection Type Strain       Completion Date Publication  \\\n",
       "0                NaN         NaN  2018-09-20T00:00:00Z         NaN   \n",
       "1                            NaN  2012-02-04T00:00:00Z    22535932   \n",
       "\n",
       "  BioProject Accession BioSample Accession Assembly Accession SRA Accession  \\\n",
       "0          PRJNA437398        SAMN09939642    GCA_003584105.1           NaN   \n",
       "1          PRJDA171989        SAMD00036610    GCA_000246945.2           NaN   \n",
       "\n",
       "  GenBank Accessions RefSeq Accessions  \\\n",
       "0       QXFO01000000               NaN   \n",
       "1       BAEX00000000                 -   \n",
       "\n",
       "                                  Sequencing Centers Sequencing Status  \\\n",
       "0  Second Institute of Oceanography, State Oceani...               NaN   \n",
       "1  Korea Research Institute of Bioscience and Bio...               WGS   \n",
       "\n",
       "    Sequencing Platform Sequencing Depth Assembly Method  Chromosomes  \\\n",
       "0  Illumina HiSeq X-Ten             308x  ABySS v. 1.9.0          NaN   \n",
       "1                   454              36x             NaN          NaN   \n",
       "\n",
       "   Plasmids  Contigs  Genome Length  GC Content  RefSeq CDS  \\\n",
       "0       NaN       28      4101660.0    45.60536      3644.0   \n",
       "1       NaN      226      3458160.0    31.35000         NaN   \n",
       "\n",
       "                Isolation Source Isolation Comments Collection Date  \\\n",
       "0                  tidal mudflat                NaN             NaN   \n",
       "1  urine of a patient with fever                NaN             NaN   \n",
       "\n",
       "  Isolation Country                         Geographic Location  Latitude  \\\n",
       "0       South Korea  South Korea: Taean coast of the Yellow Sea       NaN   \n",
       "1               NaN                                         NaN       NaN   \n",
       "\n",
       "   Longitude Altitude Depth     Other Environmental            Host Name  \\\n",
       "0        NaN      NaN    5m  env_biome:marine biome                  NaN   \n",
       "1        NaN      NaN   NaN                     NaN  Human, Homo sapiens   \n",
       "\n",
       "  Host Gender Host Age Host Health Body Sample Site Other Clinical Gram Stain  \\\n",
       "0         NaN      NaN         NaN              NaN            NaN        NaN   \n",
       "1         NaN      NaN         NaN              NaN            NaN        NaN   \n",
       "\n",
       "  Cell Shape Motility Sporulation Temperature Range Optimal Temperature  \\\n",
       "0        NaN      NaN         NaN               NaN                 NaN   \n",
       "1        NaN      NaN         NaN               NaN                 NaN   \n",
       "\n",
       "  Salinity Oxygen Requirement Habitat Disease  \\\n",
       "0      NaN                NaN     NaN     NaN   \n",
       "1      NaN                NaN     NaN     NaN   \n",
       "\n",
       "                                            Comments Additional Metadata  \\\n",
       "0  This project was used for calculating ANI valu...                 NaN   \n",
       "1  The Myroides strains are not normal human micr...                 NaN   \n",
       "\n",
       "  Genome Quality Flags  \n",
       "0                  NaN  \n",
       "1                  NaN  "
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "patric.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "both          2081\n",
       "left_only      605\n",
       "right_only       0\n",
       "Name: merge, dtype: int64"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "patric_filtered = pd.merge(genomes, patric, how=\"left\", left_on=\"Genome_ID\", right_on=\"Assembly Accession\", indicator=\"merge\")\n",
    "patric_filtered[\"merge\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genome_ID</th>\n",
       "      <th>Genome Name</th>\n",
       "      <th>Organism Name</th>\n",
       "      <th>NCBI Taxon ID</th>\n",
       "      <th>Genome Status</th>\n",
       "      <th>Strain</th>\n",
       "      <th>Serovar</th>\n",
       "      <th>Pathovar</th>\n",
       "      <th>MLST</th>\n",
       "      <th>Other Typing</th>\n",
       "      <th>Culture Collection</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Completion Date</th>\n",
       "      <th>Publication</th>\n",
       "      <th>BioProject Accession</th>\n",
       "      <th>BioSample Accession</th>\n",
       "      <th>Assembly Accession</th>\n",
       "      <th>SRA Accession</th>\n",
       "      <th>GenBank Accessions</th>\n",
       "      <th>RefSeq Accessions</th>\n",
       "      <th>Sequencing Centers</th>\n",
       "      <th>Sequencing Status</th>\n",
       "      <th>Sequencing Platform</th>\n",
       "      <th>Sequencing Depth</th>\n",
       "      <th>Assembly Method</th>\n",
       "      <th>Chromosomes</th>\n",
       "      <th>Plasmids</th>\n",
       "      <th>Contigs</th>\n",
       "      <th>Genome Length</th>\n",
       "      <th>GC Content</th>\n",
       "      <th>RefSeq CDS</th>\n",
       "      <th>Isolation Source</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>Collection Date</th>\n",
       "      <th>Isolation Country</th>\n",
       "      <th>Geographic Location</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Altitude</th>\n",
       "      <th>Depth</th>\n",
       "      <th>Other Environmental</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>Host Gender</th>\n",
       "      <th>Host Age</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>Other Clinical</th>\n",
       "      <th>Gram Stain</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>Motility</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Optimal Temperature</th>\n",
       "      <th>Salinity</th>\n",
       "      <th>Oxygen Requirement</th>\n",
       "      <th>Habitat</th>\n",
       "      <th>Disease</th>\n",
       "      <th>Comments</th>\n",
       "      <th>Additional Metadata</th>\n",
       "      <th>Genome Quality Flags</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>Complete</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>Yes</td>\n",
       "      <td>2007-05-01T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NC_009441</td>\n",
       "      <td>DOE Joint Genome Institute</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>6096872.0</td>\n",
       "      <td>34.1</td>\n",
       "      <td>5017.0</td>\n",
       "      <td>soil</td>\n",
       "      <td>isolated from soil in England and is the type ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>United Kingdom: England</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-</td>\n",
       "      <td>Rod</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>20-30</td>\n",
       "      <td>Non-halophilic</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>Multiple</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae ATCC 17061. Flavobac...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>Complete</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>Yes</td>\n",
       "      <td>2009-04-30T00:00:00Z</td>\n",
       "      <td>21304645</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>SRR013476,SRR013477</td>\n",
       "      <td>CP001632</td>\n",
       "      <td>NC_013162</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2612925.0</td>\n",
       "      <td>39.6</td>\n",
       "      <td>2171.0</td>\n",
       "      <td>the human oral cavity</td>\n",
       "      <td>isolated from the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-</td>\n",
       "      <td>Rod</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>35-37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Facultative</td>\n",
       "      <td>Host-associated</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271.  Capnocytoph...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Genome_ID                       Genome Name  \\\n",
       "0  GCA_000016645.1   Flavobacterium johnsoniae UW101   \n",
       "1  GCA_000023285.1  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "                      Organism Name  NCBI Taxon ID Genome Status      Strain  \\\n",
       "0   Flavobacterium johnsoniae UW101       376686.0      Complete  ATCC 17061   \n",
       "1  Capnocytophaga ochracea DSM 7271       521097.0      Complete    DSM 7271   \n",
       "\n",
       "  Serovar Pathovar MLST Other Typing Culture Collection Type Strain  \\\n",
       "0     NaN      NaN  NaN          NaN         ATCC 17061         Yes   \n",
       "1     NaN      NaN  NaN          NaN           DSM 7271         Yes   \n",
       "\n",
       "        Completion Date Publication BioProject Accession BioSample Accession  \\\n",
       "0  2007-05-01T00:00:00Z         NaN           PRJNA16082        SAMN02598357   \n",
       "1  2009-04-30T00:00:00Z    21304645           PRJNA29403        SAMN00001911   \n",
       "\n",
       "  Assembly Accession        SRA Accession GenBank Accessions  \\\n",
       "0    GCA_000016645.1                  NaN           CP000685   \n",
       "1    GCA_000023285.1  SRR013476,SRR013477           CP001632   \n",
       "\n",
       "  RefSeq Accessions                                 Sequencing Centers  \\\n",
       "0         NC_009441                         DOE Joint Genome Institute   \n",
       "1         NC_013162  US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...   \n",
       "\n",
       "  Sequencing Status Sequencing Platform Sequencing Depth Assembly Method  \\\n",
       "0          complete                 NaN              NaN             NaN   \n",
       "1          complete                 NaN              NaN             NaN   \n",
       "\n",
       "   Chromosomes  Plasmids  Contigs  Genome Length  GC Content  RefSeq CDS  \\\n",
       "0          1.0       NaN      1.0      6096872.0        34.1      5017.0   \n",
       "1          1.0       NaN      1.0      2612925.0        39.6      2171.0   \n",
       "\n",
       "        Isolation Source                                 Isolation Comments  \\\n",
       "0                   soil  isolated from soil in England and is the type ...   \n",
       "1  the human oral cavity                isolated from the human oral cavity   \n",
       "\n",
       "  Collection Date Isolation Country      Geographic Location  Latitude  \\\n",
       "0             NaN    United Kingdom  United Kingdom: England       NaN   \n",
       "1             NaN               NaN                      NaN       NaN   \n",
       "\n",
       "   Longitude Altitude Depth Other Environmental            Host Name  \\\n",
       "0        NaN      NaN   NaN                 NaN                  NaN   \n",
       "1        NaN      NaN   NaN                 NaN  Human, Homo sapiens   \n",
       "\n",
       "  Host Gender Host Age Host Health Body Sample Site Other Clinical Gram Stain  \\\n",
       "0         NaN      NaN         NaN              NaN            NaN          -   \n",
       "1         NaN      NaN         NaN              NaN            NaN          -   \n",
       "\n",
       "  Cell Shape Motility Sporulation Temperature Range Optimal Temperature  \\\n",
       "0        Rod      Yes          No        Mesophilic               20-30   \n",
       "1        Rod      Yes          No        Mesophilic               35-37   \n",
       "\n",
       "         Salinity Oxygen Requirement          Habitat Disease  \\\n",
       "0  Non-halophilic            Aerobic         Multiple     NaN   \n",
       "1             NaN        Facultative  Host-associated     NaN   \n",
       "\n",
       "                                            Comments Additional Metadata  \\\n",
       "0  Flavobacterium johnsoniae ATCC 17061. Flavobac...                 NaN   \n",
       "1  Capnocytophaga ochracea DSM 7271.  Capnocytoph...                 NaN   \n",
       "\n",
       "  Genome Quality Flags  \n",
       "0                  NaN  \n",
       "1                  NaN  "
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "patric_filtered[\"Additional Metadata\"] = patric_filtered[\"Additional Metadata\"].str.replace(\"ï¿½\",\"\")\n",
    "patric_filtered.drop(columns=[\"merge\"], inplace=True)\n",
    "patric_filtered.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "#REMOVE empty columns\n",
    "null_cols = patric_filtered.isnull().all()\n",
    "patric_filtered = patric_filtered[patric_filtered.columns[~null_cols]]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2.2.1 Split Patric column: Additional Metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "assembly_accession = patric_filtered[\"Genome_ID\"].to_list()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "splitted = patric_filtered[\"Additional Metadata\"].str.split(\";\", expand=True)\n",
    "splitted = splitted.rename(index=patric_filtered[\"Genome_ID\"])\n",
    "splitted = splitted.where(pd.notnull(splitted), None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['collected_by', 'sample_type', 'biomaterial_provider', 'identified_by', 'passage_history', 'isolate', 'specimen_voucher', '', 'lab_host', 'mating_type']\n"
     ]
    }
   ],
   "source": [
    "col_list = []\n",
    "for i, cell in splitted.iteritems():\n",
    "    for x in cell:\n",
    "        if x is None:   #while y == nan:\n",
    "            continue\n",
    "        else:\n",
    "            key = x.split(\":\")[0]\n",
    "            value = x.split(\":\")[1:]\n",
    "            if key not in col_list:\n",
    "                col_list.append(key)\n",
    "print(col_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "patric_addit_metadata = pd.DataFrame(columns=col_list, index=patric_filtered[\"Genome_ID\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i, cell in splitted.iterrows():\n",
    "    for x in cell:\n",
    "        cell = x\n",
    "        if x is None:\n",
    "            continue\n",
    "        else:\n",
    "            if \":\" in x:\n",
    "                value = x.split(\":\")[1]\n",
    "                key = x.split(\":\")[0]\n",
    "                if key in col_list:\n",
    "                    patric_addit_metadata.loc[i,key] = str(value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genome_ID</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>passage_history</th>\n",
       "      <th>isolate</th>\n",
       "      <th>specimen_voucher</th>\n",
       "      <th></th>\n",
       "      <th>lab_host</th>\n",
       "      <th>mating_type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>2684</th>\n",
       "      <td>GCA_903819445.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2685</th>\n",
       "      <td>GCA_903969135.1</td>\n",
       "      <td>Yvan Rahb</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "            Genome_ID collected_by sample_type biomaterial_provider  \\\n",
       "2684  GCA_903819445.1          NaN         NaN                  NaN   \n",
       "2685  GCA_903969135.1    Yvan Rahb         NaN                  NaN   \n",
       "\n",
       "     identified_by passage_history isolate specimen_voucher      lab_host  \\\n",
       "2684           NaN             NaN     NaN              NaN  NaN      NaN   \n",
       "2685           NaN             NaN     NaN              NaN  NaN      NaN   \n",
       "\n",
       "     mating_type  \n",
       "2684         NaN  \n",
       "2685         NaN  "
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "patric_addit_metadata = patric_addit_metadata.reset_index()\n",
    "patric_addit_metadata = patric_addit_metadata.drop_duplicates(subset=['Genome_ID'])\n",
    "patric_addit_metadata.tail(2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.3 Biosamples"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "BioSamples stores and supplies descriptions and metadata about biological samples used in research and development by academia and industry. Samples are either 'reference' samples (e.g. from 1000 Genomes, HipSci, FAANG) or have been used in an assay database such as the European Nucleotide Archive (ENA) or ArrayExpress."
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "First: 8 May 2020 \n",
    "download xml file from Biosamples after searching for Flavobacteriales (function is now deprecated on Biosamples website)\n",
    "\n",
    "Second: convert to readable format:\n",
    "https://github.com/transcript/BettaMetaData/tree/master/data/ncbi-biosample\n",
    "\n",
    "sed 's/<BioSample /\\n<BioSample /g' biosample_result.xml | grep '^<BioSample ' > biosample-bacteria-fixed.xml\n",
    "\n",
    "python3 -u ncbi-xml-to-json.py biosample-bacteria-fixed.xml --biosample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "biosamples = pd.read_csv(\"Biosamples_db.csv\", low_memory=False)\n",
    "l = biosamples.columns.to_list()\n",
    "#print(l)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>biosample_title</th>\n",
       "      <th>contact_email</th>\n",
       "      <th>first_name</th>\n",
       "      <th>growth phase</th>\n",
       "      <th>last_name</th>\n",
       "      <th>name</th>\n",
       "      <th>organism</th>\n",
       "      <th>publication_date</th>\n",
       "      <th>sample_accession</th>\n",
       "      <th>source_name</th>\n",
       "      <th>strain</th>\n",
       "      <th>submission_model</th>\n",
       "      <th>submission_package</th>\n",
       "      <th>submission_package_name</th>\n",
       "      <th>submitter</th>\n",
       "      <th>taxonomy_id</th>\n",
       "      <th>treatment</th>\n",
       "      <th>collection_date</th>\n",
       "      <th>culture_collection</th>\n",
       "      <th>geo_loc_name</th>\n",
       "      <th>isolation_source</th>\n",
       "      <th>sample_name</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>type-material</th>\n",
       "      <th>comment_paragraph</th>\n",
       "      <th>depth</th>\n",
       "      <th>derived-from</th>\n",
       "      <th>env_broad_scale</th>\n",
       "      <th>env_local_scale</th>\n",
       "      <th>env_medium</th>\n",
       "      <th>environmental-sample</th>\n",
       "      <th>isolate</th>\n",
       "      <th>lat_lon</th>\n",
       "      <th>metagenome-source</th>\n",
       "      <th>metagenomic</th>\n",
       "      <th>sample-type</th>\n",
       "      <th>elev</th>\n",
       "      <th>environmetal-sample</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>host</th>\n",
       "      <th>host_tissue_sampled</th>\n",
       "      <th>temp</th>\n",
       "      <th>env_biome</th>\n",
       "      <th>env_feature</th>\n",
       "      <th>env_material</th>\n",
       "      <th>isol_growth_condt</th>\n",
       "      <th>locus_tag_prefix</th>\n",
       "      <th>num_replicons</th>\n",
       "      <th>project_name</th>\n",
       "      <th>ref_biomaterial</th>\n",
       "      <th>16S recovered</th>\n",
       "      <th>16S recovery software</th>\n",
       "      <th>analysis project type</th>\n",
       "      <th>assembly quality</th>\n",
       "      <th>assembly software</th>\n",
       "      <th>bin parameters</th>\n",
       "      <th>binning software</th>\n",
       "      <th>completeness score</th>\n",
       "      <th>completeness software</th>\n",
       "      <th>contamination score</th>\n",
       "      <th>contig L50</th>\n",
       "      <th>decontamination software</th>\n",
       "      <th>note</th>\n",
       "      <th>number of contigs</th>\n",
       "      <th>reassembly post binning</th>\n",
       "      <th>relative coverage Anaerobic (SRR10097246)</th>\n",
       "      <th>relative coverage Anoxic (SRR10375098)</th>\n",
       "      <th>relative coverage Infiltration (SRR10097247)</th>\n",
       "      <th>relative coverage Low pH (SRR10097245)</th>\n",
       "      <th>relative coverage Oxic (SRR10375099)</th>\n",
       "      <th>relative coverage Reference (SRR10375097)</th>\n",
       "      <th>taxa id</th>\n",
       "      <th>total assembly size</th>\n",
       "      <th>nat_host</th>\n",
       "      <th>assembly_method_and_version</th>\n",
       "      <th>completeness_estimated</th>\n",
       "      <th>contamination_estimated</th>\n",
       "      <th>genome_coverage</th>\n",
       "      <th>mapping_method_and_version</th>\n",
       "      <th>quality_assessment_method_and_version</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>ENA-FIRST-PUBLIC</th>\n",
       "      <th>ENA-LAST-UPDATE</th>\n",
       "      <th>SUBJECT_ID</th>\n",
       "      <th>contact_lab</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>isolate-name-alias</th>\n",
       "      <th>environmental_sample</th>\n",
       "      <th>metagenome_source</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>culture media</th>\n",
       "      <th>replicate</th>\n",
       "      <th>biome</th>\n",
       "      <th>feature</th>\n",
       "      <th>material</th>\n",
       "      <th>alternate_ID</th>\n",
       "      <th>culture-collection</th>\n",
       "      <th>altitude</th>\n",
       "      <th>Genus</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>Species</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>isolate_name_alias</th>\n",
       "      <th>host_disease</th>\n",
       "      <th>rel_to_oxygen</th>\n",
       "      <th>Alias</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>Sample Name</th>\n",
       "      <th>Title</th>\n",
       "      <th>serotype</th>\n",
       "      <th>subsrc_note</th>\n",
       "      <th>annotation_method</th>\n",
       "      <th>assembly_method</th>\n",
       "      <th>biosamplemodel</th>\n",
       "      <th>geographic location (country and/or atlantic ocean)</th>\n",
       "      <th>geographic location (region and locality)</th>\n",
       "      <th>mapping_method</th>\n",
       "      <th>quality_assessment_method</th>\n",
       "      <th>refinement_method</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>Funding Program</th>\n",
       "      <th>GOLD Stamp ID</th>\n",
       "      <th>Gene Calling Method</th>\n",
       "      <th>Gram Staining</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>Isolation Site</th>\n",
       "      <th>Motility</th>\n",
       "      <th>Temperature Optimum</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>alt_elev</th>\n",
       "      <th>biotic_relationship</th>\n",
       "      <th>country</th>\n",
       "      <th>environment</th>\n",
       "      <th>investigation_type</th>\n",
       "      <th>isolation-source</th>\n",
       "      <th>trophic_level</th>\n",
       "      <th>pH</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>Body Sample SubSite</th>\n",
       "      <th>Diseases</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>experimental_factor</th>\n",
       "      <th>breeding_method</th>\n",
       "      <th>lat-lon</th>\n",
       "      <th>ENA-CHECKLIST</th>\n",
       "      <th>External Id</th>\n",
       "      <th>Submitter Id</th>\n",
       "      <th>subsrc-note</th>\n",
       "      <th>collection date</th>\n",
       "      <th>environment (biome)</th>\n",
       "      <th>environment (feature)</th>\n",
       "      <th>environment (material)</th>\n",
       "      <th>geographic location (country and/or sea)</th>\n",
       "      <th>geographic location (latitude)</th>\n",
       "      <th>geographic location (longitude)</th>\n",
       "      <th>host common name</th>\n",
       "      <th>host-associated environmental package</th>\n",
       "      <th>investigation type</th>\n",
       "      <th>project name</th>\n",
       "      <th>sequencing method</th>\n",
       "      <th>STRAIN</th>\n",
       "      <th>co_assembly_bin</th>\n",
       "      <th>derived from</th>\n",
       "      <th>description</th>\n",
       "      <th>Sequence processing</th>\n",
       "      <th>completeness</th>\n",
       "      <th>contamination</th>\n",
       "      <th>host_age</th>\n",
       "      <th>host_sex</th>\n",
       "      <th>oxy_stat_samp</th>\n",
       "      <th>samp_store_dur</th>\n",
       "      <th>samp_store_temp</th>\n",
       "      <th>FDA CVM ID</th>\n",
       "      <th>bio_material</th>\n",
       "      <th>common name</th>\n",
       "      <th>host scientific name</th>\n",
       "      <th>samp_mat_process</th>\n",
       "      <th>estimated_size</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>Tax ID</th>\n",
       "      <th>Collaborator ID</th>\n",
       "      <th>host_description</th>\n",
       "      <th>host_disease_outcome</th>\n",
       "      <th>host_disease_stage</th>\n",
       "      <th>host_health_state</th>\n",
       "      <th>identification method</th>\n",
       "      <th>strain_name_alias</th>\n",
       "      <th>WGA amplification approach</th>\n",
       "      <th>metagenomic source</th>\n",
       "      <th>sample derived from</th>\n",
       "      <th>single cell or viral particle lysis approach</th>\n",
       "      <th>sorting technology</th>\n",
       "      <th>taxonomic identity marker</th>\n",
       "      <th>passage_history</th>\n",
       "      <th>samp_size</th>\n",
       "      <th>timepoint</th>\n",
       "      <th>Description</th>\n",
       "      <th>subsource_note</th>\n",
       "      <th>lab_host</th>\n",
       "      <th>collection_room</th>\n",
       "      <th>relative_week</th>\n",
       "      <th>assembly_tool</th>\n",
       "      <th>bin_id</th>\n",
       "      <th>Broker name</th>\n",
       "      <th>number_of_identified_antimicrobial_resistance_genes</th>\n",
       "      <th>MAG_number</th>\n",
       "      <th>derived_from</th>\n",
       "      <th>alkalinity (carbonate/bicarbonate)</th>\n",
       "      <th>salinity</th>\n",
       "      <th>original-organism-name</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>env_package</th>\n",
       "      <th>label</th>\n",
       "      <th>specific_host</th>\n",
       "      <th>binning parameters</th>\n",
       "      <th>plant-associated environmental package</th>\n",
       "      <th>culture collection</th>\n",
       "      <th>repository</th>\n",
       "      <th>risk group</th>\n",
       "      <th>type status</th>\n",
       "      <th>Assembly Method</th>\n",
       "      <th>Genome Coverage</th>\n",
       "      <th>Sequencing Technology</th>\n",
       "      <th>nat-host</th>\n",
       "      <th>collection-date</th>\n",
       "      <th>Historical Monobactam Produced</th>\n",
       "      <th>Unique Monobactam Clusters Identified</th>\n",
       "      <th>extrachrom_elements</th>\n",
       "      <th>pathogenicity</th>\n",
       "      <th>source_material_id</th>\n",
       "      <th>genotype</th>\n",
       "      <th>mating_type</th>\n",
       "      <th>serovar</th>\n",
       "      <th>specimen_voucher</th>\n",
       "      <th>Salinity (PSU)</th>\n",
       "      <th>collected-by</th>\n",
       "      <th>orgmod_note</th>\n",
       "      <th>assembly</th>\n",
       "      <th>sequencing_meth</th>\n",
       "      <th>finishing_strategy</th>\n",
       "      <th>geo-loc-name</th>\n",
       "      <th>finishing strategy (depth of coverage)</th>\n",
       "      <th>host taxid</th>\n",
       "      <th>misc_param: HMP body site</th>\n",
       "      <th>misc_param: HMP supersite</th>\n",
       "      <th>nucleic acid extraction</th>\n",
       "      <th>project_type</th>\n",
       "      <th>sop</th>\n",
       "      <th>source_mat_id</th>\n",
       "      <th>Salinity</th>\n",
       "      <th>age</th>\n",
       "      <th>body_site</th>\n",
       "      <th>disease</th>\n",
       "      <th>health_state</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>Host</th>\n",
       "      <th>Sample name</th>\n",
       "      <th>geographic location (country and/or sea,region)</th>\n",
       "      <th>isolation source</th>\n",
       "      <th>environmental package</th>\n",
       "      <th>specific host</th>\n",
       "      <th>lat lon</th>\n",
       "      <th>host_subject_id</th>\n",
       "      <th>pathotype</th>\n",
       "      <th>subgroup</th>\n",
       "      <th>subtype</th>\n",
       "      <th>isolate name alias</th>\n",
       "      <th>zoonotic agent</th>\n",
       "      <th>Extraction Date</th>\n",
       "      <th>Extraction Method</th>\n",
       "      <th>Laboratory Host</th>\n",
       "      <th>Passage Date</th>\n",
       "      <th>sample type</th>\n",
       "      <th>HOST TAXON NAME</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>host health state</th>\n",
       "      <th>geographic location (elevation)</th>\n",
       "      <th>sediment environmental package</th>\n",
       "      <th>anonymized_name</th>\n",
       "      <th>host_status</th>\n",
       "      <th>supplier_name</th>\n",
       "      <th>plant product</th>\n",
       "      <th>ecotype</th>\n",
       "      <th>growth condition</th>\n",
       "      <th>abs_air_humidity</th>\n",
       "      <th>air_temp</th>\n",
       "      <th>build_occup_type</th>\n",
       "      <th>building_setting</th>\n",
       "      <th>carb_dioxide</th>\n",
       "      <th>filter_type</th>\n",
       "      <th>heat_cool_type</th>\n",
       "      <th>indoor_space</th>\n",
       "      <th>light_type</th>\n",
       "      <th>occup_samp</th>\n",
       "      <th>occupant_dens_samp</th>\n",
       "      <th>organism_count</th>\n",
       "      <th>rel_air_humidity</th>\n",
       "      <th>space_typ_state</th>\n",
       "      <th>typ_occupant_dens</th>\n",
       "      <th>ventilation_type</th>\n",
       "      <th>Type-material</th>\n",
       "      <th>samp_collect_device</th>\n",
       "      <th>Depth</th>\n",
       "      <th>Size fraction</th>\n",
       "      <th>genomovar</th>\n",
       "      <th>geographic location (altitude)</th>\n",
       "      <th>env_packatge</th>\n",
       "      <th>lib_reads_seqd</th>\n",
       "      <th>metagenomic_source</th>\n",
       "      <th>env material</th>\n",
       "      <th>other identifiers</th>\n",
       "      <th>assembly_method_version</th>\n",
       "      <th>mapping_method_version</th>\n",
       "      <th>quality_assessment_method_version</th>\n",
       "      <th>value</th>\n",
       "      <th>state</th>\n",
       "      <th>time</th>\n",
       "      <th>method</th>\n",
       "      <th>dna treatment</th>\n",
       "      <th>pacbio sequencing chemistry</th>\n",
       "      <th>samp_size (Library conc., pM)</th>\n",
       "      <th>cyanobacterial_culture</th>\n",
       "      <th>missing</th>\n",
       "      <th>samp_store_loc</th>\n",
       "      <th>sample_alias</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>vector</th>\n",
       "      <th>host_growth_cond</th>\n",
       "      <th>host_life_stage</th>\n",
       "      <th>c-source</th>\n",
       "      <th>host_body_habitat</th>\n",
       "      <th>host_taxid</th>\n",
       "      <th>misc_param</th>\n",
       "      <th>lab-host</th>\n",
       "      <th>v_type</th>\n",
       "      <th>Replicate</th>\n",
       "      <th>DIP (uM)</th>\n",
       "      <th>NH4 (uM)</th>\n",
       "      <th>NO2 (uM)</th>\n",
       "      <th>NO3 (uM)</th>\n",
       "      <th>O2 (uM)</th>\n",
       "      <th>Salinity (psu)</th>\n",
       "      <th>ph</th>\n",
       "      <th>beta_lactamase_family</th>\n",
       "      <th>carbapenemase</th>\n",
       "      <th>edta_inhibitor_tested</th>\n",
       "      <th>Sample Number</th>\n",
       "      <th>health_disease_stat</th>\n",
       "      <th>host_spec_range</th>\n",
       "      <th>assembly-note</th>\n",
       "      <th>metagenome-source seawater metagenome</th>\n",
       "      <th>type_status</th>\n",
       "      <th>indoor_surf</th>\n",
       "      <th>samp_sort_meth</th>\n",
       "      <th>samp_vol_we_dna_ext</th>\n",
       "      <th>surf_material</th>\n",
       "      <th>time point</th>\n",
       "      <th>Strain</th>\n",
       "      <th>sample comment</th>\n",
       "      <th>growth_med</th>\n",
       "      <th>host_genotype</th>\n",
       "      <th>plant_body_site</th>\n",
       "      <th>plant_product</th>\n",
       "      <th>BioSampleModel</th>\n",
       "      <th>closest_match</th>\n",
       "      <th>percent_match</th>\n",
       "      <th>Environmental feature</th>\n",
       "      <th>Environmental material</th>\n",
       "      <th>ammonium</th>\n",
       "      <th>biomass</th>\n",
       "      <th>carb_nitro_ratio</th>\n",
       "      <th>chlorophyll</th>\n",
       "      <th>diss_inorg_nitro</th>\n",
       "      <th>diss_inorg_phosp</th>\n",
       "      <th>diss_org_carb</th>\n",
       "      <th>diss_org_nitro</th>\n",
       "      <th>nitrate</th>\n",
       "      <th>nitrite</th>\n",
       "      <th>phosphate</th>\n",
       "      <th>silicate</th>\n",
       "      <th>tidal_stage</th>\n",
       "      <th>tot_diss_nitro</th>\n",
       "      <th>Alistipes</th>\n",
       "      <th>Anaerophaga</th>\n",
       "      <th>Anaerostipes</th>\n",
       "      <th>Bacteroidaceae</th>\n",
       "      <th>Bacteroidales</th>\n",
       "      <th>Bacteroides</th>\n",
       "      <th>Bacteroidetes</th>\n",
       "      <th>Bacteroidia</th>\n",
       "      <th>Barnesiella</th>\n",
       "      <th>Blautia</th>\n",
       "      <th>Butyrivibrio</th>\n",
       "      <th>Campylobacterales</th>\n",
       "      <th>Clostridia</th>\n",
       "      <th>Clostridiales</th>\n",
       "      <th>Deferribacteraceae</th>\n",
       "      <th>Deferribacterales</th>\n",
       "      <th>Deferribacteres</th>\n",
       "      <th>Deltaproteobacteria</th>\n",
       "      <th>Desulfovibrionaceae</th>\n",
       "      <th>Desulfovibrionales</th>\n",
       "      <th>Epsilonproteobacteria</th>\n",
       "      <th>Firmicutes</th>\n",
       "      <th>Flammeovirgaceae</th>\n",
       "      <th>Flavobacteria</th>\n",
       "      <th>Flavobacteriaceae</th>\n",
       "      <th>Flavobacteriales</th>\n",
       "      <th>Flexithrix</th>\n",
       "      <th>Helicobacter</th>\n",
       "      <th>Helicobacteraceae</th>\n",
       "      <th>Hespellia</th>\n",
       "      <th>Incertae_Sedis_XIV</th>\n",
       "      <th>Lachnospiraceae</th>\n",
       "      <th>Lawsonia</th>\n",
       "      <th>Marinilabiaceae</th>\n",
       "      <th>Mollicutes</th>\n",
       "      <th>Mucispirillum</th>\n",
       "      <th>Oscillibacter</th>\n",
       "      <th>Parasporobacterium</th>\n",
       "      <th>Peptococcaceae</th>\n",
       "      <th>Porphyromonadaceae</th>\n",
       "      <th>Proteobacteria</th>\n",
       "      <th>Rikenellaceae</th>\n",
       "      <th>Robinsoniella</th>\n",
       "      <th>Ruminococcaceae</th>\n",
       "      <th>Sandarakinotalea</th>\n",
       "      <th>Sphingobacteria</th>\n",
       "      <th>Sphingobacteriales</th>\n",
       "      <th>Sporobacterium</th>\n",
       "      <th>Tenericutes</th>\n",
       "      <th>X1</th>\n",
       "      <th>X103</th>\n",
       "      <th>X10763</th>\n",
       "      <th>X111</th>\n",
       "      <th>X119</th>\n",
       "      <th>X1291</th>\n",
       "      <th>X13004</th>\n",
       "      <th>X14</th>\n",
       "      <th>X14089</th>\n",
       "      <th>X168</th>\n",
       "      <th>X2</th>\n",
       "      <th>X229</th>\n",
       "      <th>X230</th>\n",
       "      <th>X2660</th>\n",
       "      <th>X28</th>\n",
       "      <th>X321</th>\n",
       "      <th>X3422</th>\n",
       "      <th>X4</th>\n",
       "      <th>X5217</th>\n",
       "      <th>X5802</th>\n",
       "      <th>X6075</th>\n",
       "      <th>X62</th>\n",
       "      <th>X6209</th>\n",
       "      <th>X7</th>\n",
       "      <th>X8</th>\n",
       "      <th>X8841</th>\n",
       "      <th>X9</th>\n",
       "      <th>breed</th>\n",
       "      <th>sex</th>\n",
       "      <th>tissue</th>\n",
       "      <th>locus_tag prefix</th>\n",
       "      <th>sample ID</th>\n",
       "      <th>strains</th>\n",
       "      <th>host sex</th>\n",
       "      <th>cell_type</th>\n",
       "      <th>geographic location (country and/or sea, region)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Iron_minus_A</td>\n",
       "      <td>shicheng@msu.edu</td>\n",
       "      <td>Shicheng</td>\n",
       "      <td>Log</td>\n",
       "      <td>Chen</td>\n",
       "      <td>Microbiology and Molecular Genetics, Michigan ...</td>\n",
       "      <td>Elizabethkingia anophelis</td>\n",
       "      <td>2020-05-07T00:00:00.000</td>\n",
       "      <td>SAMN12086807</td>\n",
       "      <td>bacterial culture_iron_minus</td>\n",
       "      <td>Ag1</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Microbiology and Molecular Genetics, Michigan ...</td>\n",
       "      <td>1117645</td>\n",
       "      <td>Minus iron supplement</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Iron_minus_B</td>\n",
       "      <td>shicheng@msu.edu</td>\n",
       "      <td>Shicheng</td>\n",
       "      <td>Log</td>\n",
       "      <td>Chen</td>\n",
       "      <td>Microbiology and Molecular Genetics, Michigan ...</td>\n",
       "      <td>Elizabethkingia anophelis</td>\n",
       "      <td>2020-05-07T00:00:00.000</td>\n",
       "      <td>SAMN12086806</td>\n",
       "      <td>bacterial culture_iron_minus</td>\n",
       "      <td>Ag1</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Microbiology and Molecular Genetics, Michigan ...</td>\n",
       "      <td>1117645</td>\n",
       "      <td>Minus iron supplement</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  biosample_title     contact_email first_name growth phase last_name  \\\n",
       "0    Iron_minus_A  shicheng@msu.edu   Shicheng          Log      Chen   \n",
       "1    Iron_minus_B  shicheng@msu.edu   Shicheng          Log      Chen   \n",
       "\n",
       "                                                name  \\\n",
       "0  Microbiology and Molecular Genetics, Michigan ...   \n",
       "1  Microbiology and Molecular Genetics, Michigan ...   \n",
       "\n",
       "                    organism         publication_date sample_accession  \\\n",
       "0  Elizabethkingia anophelis  2020-05-07T00:00:00.000     SAMN12086807   \n",
       "1  Elizabethkingia anophelis  2020-05-07T00:00:00.000     SAMN12086806   \n",
       "\n",
       "                    source_name strain submission_model submission_package  \\\n",
       "0  bacterial culture_iron_minus    Ag1          Generic        Generic.1.0   \n",
       "1  bacterial culture_iron_minus    Ag1          Generic        Generic.1.0   \n",
       "\n",
       "  submission_package_name                                          submitter  \\\n",
       "0                 Generic  Microbiology and Molecular Genetics, Michigan ...   \n",
       "1                 Generic  Microbiology and Molecular Genetics, Michigan ...   \n",
       "\n",
       "   taxonomy_id              treatment collection_date culture_collection  \\\n",
       "0      1117645  Minus iron supplement             NaN                NaN   \n",
       "1      1117645  Minus iron supplement             NaN                NaN   \n",
       "\n",
       "  geo_loc_name isolation_source sample_name sample_type type-material  \\\n",
       "0          NaN              NaN         NaN         NaN           NaN   \n",
       "1          NaN              NaN         NaN         NaN           NaN   \n",
       "\n",
       "  comment_paragraph depth derived-from env_broad_scale env_local_scale  \\\n",
       "0               NaN   NaN          NaN             NaN             NaN   \n",
       "1               NaN   NaN          NaN             NaN             NaN   \n",
       "\n",
       "  env_medium environmental-sample isolate lat_lon metagenome-source  \\\n",
       "0        NaN                  NaN     NaN     NaN               NaN   \n",
       "1        NaN                  NaN     NaN     NaN               NaN   \n",
       "\n",
       "  metagenomic sample-type elev environmetal-sample collected_by host  \\\n",
       "0         NaN         NaN  NaN                 NaN          NaN  NaN   \n",
       "1         NaN         NaN  NaN                 NaN          NaN  NaN   \n",
       "\n",
       "  host_tissue_sampled temp env_biome env_feature env_material  \\\n",
       "0                 NaN  NaN       NaN         NaN          NaN   \n",
       "1                 NaN  NaN       NaN         NaN          NaN   \n",
       "\n",
       "  isol_growth_condt locus_tag_prefix num_replicons project_name  \\\n",
       "0               NaN              NaN           NaN          NaN   \n",
       "1               NaN              NaN           NaN          NaN   \n",
       "\n",
       "  ref_biomaterial 16S recovered 16S recovery software analysis project type  \\\n",
       "0             NaN           NaN                   NaN                   NaN   \n",
       "1             NaN           NaN                   NaN                   NaN   \n",
       "\n",
       "  assembly quality assembly software bin parameters binning software  \\\n",
       "0              NaN               NaN            NaN              NaN   \n",
       "1              NaN               NaN            NaN              NaN   \n",
       "\n",
       "   completeness score completeness software  contamination score  contig L50  \\\n",
       "0                 NaN                   NaN                  NaN         NaN   \n",
       "1                 NaN                   NaN                  NaN         NaN   \n",
       "\n",
       "  decontamination software note  number of contigs reassembly post binning  \\\n",
       "0                      NaN  NaN                NaN                     NaN   \n",
       "1                      NaN  NaN                NaN                     NaN   \n",
       "\n",
       "   relative coverage Anaerobic (SRR10097246)  \\\n",
       "0                                        NaN   \n",
       "1                                        NaN   \n",
       "\n",
       "   relative coverage Anoxic (SRR10375098)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Infiltration (SRR10097247)  \\\n",
       "0                                           NaN   \n",
       "1                                           NaN   \n",
       "\n",
       "   relative coverage Low pH (SRR10097245)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Oxic (SRR10375099)  \\\n",
       "0                                   NaN   \n",
       "1                                   NaN   \n",
       "\n",
       "   relative coverage Reference (SRR10375097) taxa id  total assembly size  \\\n",
       "0                                        NaN     NaN                  NaN   \n",
       "1                                        NaN     NaN                  NaN   \n",
       "\n",
       "  nat_host assembly_method_and_version completeness_estimated  \\\n",
       "0      NaN                         NaN                    NaN   \n",
       "1      NaN                         NaN                    NaN   \n",
       "\n",
       "  contamination_estimated genome_coverage mapping_method_and_version  \\\n",
       "0                     NaN             NaN                        NaN   \n",
       "1                     NaN             NaN                        NaN   \n",
       "\n",
       "  quality_assessment_method_and_version ArrayExpress-SPECIES ENA-FIRST-PUBLIC  \\\n",
       "0                                   NaN                  NaN              NaN   \n",
       "1                                   NaN                  NaN              NaN   \n",
       "\n",
       "  ENA-LAST-UPDATE SUBJECT_ID contact_lab identified_by isolate-name-alias  \\\n",
       "0             NaN        NaN         NaN           NaN                NaN   \n",
       "1             NaN        NaN         NaN           NaN                NaN   \n",
       "\n",
       "  environmental_sample metagenome_source biomaterial_provider culture media  \\\n",
       "0                  NaN               NaN                  NaN           NaN   \n",
       "1                  NaN               NaN                  NaN           NaN   \n",
       "\n",
       "  replicate biome feature material alternate_ID culture-collection altitude  \\\n",
       "0       NaN   NaN     NaN      NaN          NaN                NaN      NaN   \n",
       "1       NaN   NaN     NaN      NaN          NaN                NaN      NaN   \n",
       "\n",
       "  Genus ProjectAccession PublicAccession Species attribute_package  \\\n",
       "0   NaN              NaN             NaN     NaN               NaN   \n",
       "1   NaN              NaN             NaN     NaN               NaN   \n",
       "\n",
       "  isolate_name_alias host_disease rel_to_oxygen Alias ENA checklist  \\\n",
       "0                NaN          NaN           NaN   NaN           NaN   \n",
       "1                NaN          NaN           NaN   NaN           NaN   \n",
       "\n",
       "  INSDC center alias INSDC center name INSDC first public INSDC last update  \\\n",
       "0                NaN               NaN                NaN               NaN   \n",
       "1                NaN               NaN                NaN               NaN   \n",
       "\n",
       "  INSDC status SRA accession Sample Name Title serotype subsrc_note  \\\n",
       "0          NaN           NaN         NaN   NaN      NaN         NaN   \n",
       "1          NaN           NaN         NaN   NaN      NaN         NaN   \n",
       "\n",
       "  annotation_method assembly_method biosamplemodel  \\\n",
       "0               NaN             NaN            NaN   \n",
       "1               NaN             NaN            NaN   \n",
       "\n",
       "  geographic location (country and/or atlantic ocean)  \\\n",
       "0                                                NaN    \n",
       "1                                                NaN    \n",
       "\n",
       "  geographic location (region and locality) mapping_method  \\\n",
       "0                                       NaN            NaN   \n",
       "1                                       NaN            NaN   \n",
       "\n",
       "  quality_assessment_method refinement_method Cell Shape Funding Program  \\\n",
       "0                       NaN               NaN        NaN             NaN   \n",
       "1                       NaN               NaN        NaN             NaN   \n",
       "\n",
       "  GOLD Stamp ID Gene Calling Method Gram Staining Isolation Comments  \\\n",
       "0           NaN                 NaN           NaN                NaN   \n",
       "1           NaN                 NaN           NaN                NaN   \n",
       "\n",
       "  Isolation Site Motility Temperature Optimum Temperature Range Type Strain  \\\n",
       "0            NaN      NaN                 NaN               NaN         NaN   \n",
       "1            NaN      NaN                 NaN               NaN         NaN   \n",
       "\n",
       "  alt_elev biotic_relationship country environment investigation_type  \\\n",
       "0      NaN                 NaN     NaN         NaN                NaN   \n",
       "1      NaN                 NaN     NaN         NaN                NaN   \n",
       "\n",
       "  isolation-source trophic_level   pH Body Sample Site Body Sample SubSite  \\\n",
       "0              NaN           NaN  NaN              NaN                 NaN   \n",
       "1              NaN           NaN  NaN              NaN                 NaN   \n",
       "\n",
       "  Diseases Host Name Host Health Sporulation experimental_factor  \\\n",
       "0      NaN       NaN         NaN         NaN                 NaN   \n",
       "1      NaN       NaN         NaN         NaN                 NaN   \n",
       "\n",
       "  breeding_method lat-lon ENA-CHECKLIST External Id Submitter Id subsrc-note  \\\n",
       "0             NaN     NaN           NaN         NaN          NaN         NaN   \n",
       "1             NaN     NaN           NaN         NaN          NaN         NaN   \n",
       "\n",
       "  collection date environment (biome) environment (feature)  \\\n",
       "0             NaN                 NaN                   NaN   \n",
       "1             NaN                 NaN                   NaN   \n",
       "\n",
       "  environment (material) geographic location (country and/or sea)  \\\n",
       "0                    NaN                                      NaN   \n",
       "1                    NaN                                      NaN   \n",
       "\n",
       "  geographic location (latitude) geographic location (longitude)  \\\n",
       "0                            NaN                             NaN   \n",
       "1                            NaN                             NaN   \n",
       "\n",
       "  host common name host-associated environmental package investigation type  \\\n",
       "0              NaN                                   NaN                NaN   \n",
       "1              NaN                                   NaN                NaN   \n",
       "\n",
       "  project name sequencing method STRAIN co_assembly_bin derived from  \\\n",
       "0          NaN               NaN    NaN             NaN          NaN   \n",
       "1          NaN               NaN    NaN             NaN          NaN   \n",
       "\n",
       "  description Sequence processing  completeness  contamination host_age  \\\n",
       "0         NaN                 NaN           NaN            NaN      NaN   \n",
       "1         NaN                 NaN           NaN            NaN      NaN   \n",
       "\n",
       "  host_sex oxy_stat_samp samp_store_dur samp_store_temp FDA CVM ID  \\\n",
       "0      NaN           NaN            NaN             NaN        NaN   \n",
       "1      NaN           NaN            NaN             NaN        NaN   \n",
       "\n",
       "  bio_material common name host scientific name samp_mat_process  \\\n",
       "0          NaN         NaN                  NaN              NaN   \n",
       "1          NaN         NaN                  NaN              NaN   \n",
       "\n",
       "  estimated_size ArrayExpress-STRAIN_OR_LINE  Tax ID Collaborator ID  \\\n",
       "0            NaN                         NaN     NaN             NaN   \n",
       "1            NaN                         NaN     NaN             NaN   \n",
       "\n",
       "  host_description host_disease_outcome host_disease_stage host_health_state  \\\n",
       "0              NaN                  NaN                NaN               NaN   \n",
       "1              NaN                  NaN                NaN               NaN   \n",
       "\n",
       "  identification method strain_name_alias WGA amplification approach  \\\n",
       "0                   NaN               NaN                        NaN   \n",
       "1                   NaN               NaN                        NaN   \n",
       "\n",
       "  metagenomic source sample derived from  \\\n",
       "0                NaN                 NaN   \n",
       "1                NaN                 NaN   \n",
       "\n",
       "  single cell or viral particle lysis approach sorting technology  \\\n",
       "0                                          NaN                NaN   \n",
       "1                                          NaN                NaN   \n",
       "\n",
       "  taxonomic identity marker passage_history samp_size timepoint Description  \\\n",
       "0                       NaN             NaN       NaN       NaN         NaN   \n",
       "1                       NaN             NaN       NaN       NaN         NaN   \n",
       "\n",
       "  subsource_note lab_host collection_room  relative_week assembly_tool bin_id  \\\n",
       "0            NaN      NaN             NaN            NaN           NaN    NaN   \n",
       "1            NaN      NaN             NaN            NaN           NaN    NaN   \n",
       "\n",
       "  Broker name  number_of_identified_antimicrobial_resistance_genes  \\\n",
       "0         NaN                                                NaN     \n",
       "1         NaN                                                NaN     \n",
       "\n",
       "   MAG_number derived_from alkalinity (carbonate/bicarbonate) salinity  \\\n",
       "0         NaN          NaN                                NaN      NaN   \n",
       "1         NaN          NaN                                NaN      NaN   \n",
       "\n",
       "  original-organism-name Phenotypes env_package label specific_host  \\\n",
       "0                    NaN        NaN         NaN   NaN           NaN   \n",
       "1                    NaN        NaN         NaN   NaN           NaN   \n",
       "\n",
       "  binning parameters plant-associated environmental package  \\\n",
       "0                NaN                                    NaN   \n",
       "1                NaN                                    NaN   \n",
       "\n",
       "  culture collection repository risk group type status Assembly Method  \\\n",
       "0                NaN        NaN        NaN         NaN             NaN   \n",
       "1                NaN        NaN        NaN         NaN             NaN   \n",
       "\n",
       "  Genome Coverage  Sequencing Technology nat-host collection-date  \\\n",
       "0             NaN                    NaN      NaN             NaN   \n",
       "1             NaN                    NaN      NaN             NaN   \n",
       "\n",
       "  Historical Monobactam Produced Unique Monobactam Clusters Identified  \\\n",
       "0                            NaN                                   NaN   \n",
       "1                            NaN                                   NaN   \n",
       "\n",
       "   extrachrom_elements pathogenicity source_material_id genotype mating_type  \\\n",
       "0                  NaN           NaN                NaN      NaN         NaN   \n",
       "1                  NaN           NaN                NaN      NaN         NaN   \n",
       "\n",
       "  serovar specimen_voucher  Salinity (PSU) collected-by orgmod_note assembly  \\\n",
       "0     NaN              NaN             NaN          NaN         NaN      NaN   \n",
       "1     NaN              NaN             NaN          NaN         NaN      NaN   \n",
       "\n",
       "  sequencing_meth finishing_strategy geo-loc-name  \\\n",
       "0             NaN                NaN          NaN   \n",
       "1             NaN                NaN          NaN   \n",
       "\n",
       "  finishing strategy (depth of coverage)  host taxid  \\\n",
       "0                                    NaN         NaN   \n",
       "1                                    NaN         NaN   \n",
       "\n",
       "  misc_param: HMP body site misc_param: HMP supersite nucleic acid extraction  \\\n",
       "0                       NaN                       NaN                     NaN   \n",
       "1                       NaN                       NaN                     NaN   \n",
       "\n",
       "  project_type  sop source_mat_id Salinity  age body_site disease  \\\n",
       "0          NaN  NaN           NaN      NaN  NaN       NaN     NaN   \n",
       "1          NaN  NaN           NaN      NaN  NaN       NaN     NaN   \n",
       "\n",
       "  health_state Isolation source Host Sample name  \\\n",
       "0          NaN              NaN  NaN         NaN   \n",
       "1          NaN              NaN  NaN         NaN   \n",
       "\n",
       "  geographic location (country and/or sea,region) isolation source  \\\n",
       "0                                             NaN              NaN   \n",
       "1                                             NaN              NaN   \n",
       "\n",
       "  environmental package specific host lat lon host_subject_id pathotype  \\\n",
       "0                   NaN           NaN     NaN             NaN       NaN   \n",
       "1                   NaN           NaN     NaN             NaN       NaN   \n",
       "\n",
       "  subgroup subtype isolate name alias zoonotic agent Extraction Date  \\\n",
       "0      NaN     NaN                NaN            NaN             NaN   \n",
       "1      NaN     NaN                NaN            NaN             NaN   \n",
       "\n",
       "  Extraction Method Laboratory Host Passage Date sample type HOST TAXON NAME  \\\n",
       "0               NaN             NaN          NaN         NaN             NaN   \n",
       "1               NaN             NaN          NaN         NaN             NaN   \n",
       "\n",
       "  sample_description host health state  geographic location (elevation)  \\\n",
       "0                NaN               NaN                              NaN   \n",
       "1                NaN               NaN                              NaN   \n",
       "\n",
       "  sediment environmental package anonymized_name host_status supplier_name  \\\n",
       "0                            NaN             NaN         NaN           NaN   \n",
       "1                            NaN             NaN         NaN           NaN   \n",
       "\n",
       "  plant product ecotype growth condition abs_air_humidity  air_temp  \\\n",
       "0           NaN     NaN              NaN              NaN       NaN   \n",
       "1           NaN     NaN              NaN              NaN       NaN   \n",
       "\n",
       "  build_occup_type building_setting carb_dioxide filter_type heat_cool_type  \\\n",
       "0              NaN              NaN          NaN         NaN            NaN   \n",
       "1              NaN              NaN          NaN         NaN            NaN   \n",
       "\n",
       "  indoor_space light_type occup_samp occupant_dens_samp organism_count  \\\n",
       "0          NaN        NaN        NaN                NaN            NaN   \n",
       "1          NaN        NaN        NaN                NaN            NaN   \n",
       "\n",
       "  rel_air_humidity space_typ_state typ_occupant_dens ventilation_type  \\\n",
       "0              NaN             NaN               NaN              NaN   \n",
       "1              NaN             NaN               NaN              NaN   \n",
       "\n",
       "  Type-material samp_collect_device Depth Size fraction genomovar  \\\n",
       "0           NaN                 NaN   NaN           NaN       NaN   \n",
       "1           NaN                 NaN   NaN           NaN       NaN   \n",
       "\n",
       "  geographic location (altitude) env_packatge lib_reads_seqd  \\\n",
       "0                            NaN          NaN            NaN   \n",
       "1                            NaN          NaN            NaN   \n",
       "\n",
       "  metagenomic_source env material other identifiers assembly_method_version  \\\n",
       "0                NaN          NaN               NaN                     NaN   \n",
       "1                NaN          NaN               NaN                     NaN   \n",
       "\n",
       "  mapping_method_version quality_assessment_method_version value state time  \\\n",
       "0                    NaN                               NaN   NaN   NaN  NaN   \n",
       "1                    NaN                               NaN   NaN   NaN  NaN   \n",
       "\n",
       "  method dna treatment pacbio sequencing chemistry  \\\n",
       "0    NaN           NaN                         NaN   \n",
       "1    NaN           NaN                         NaN   \n",
       "\n",
       "   samp_size (Library conc., pM) cyanobacterial_culture missing  \\\n",
       "0                            NaN                    NaN     NaN   \n",
       "1                            NaN                    NaN     NaN   \n",
       "\n",
       "  samp_store_loc sample_alias ethnicity vector host_growth_cond  \\\n",
       "0            NaN          NaN       NaN    NaN              NaN   \n",
       "1            NaN          NaN       NaN    NaN              NaN   \n",
       "\n",
       "  host_life_stage c-source host_body_habitat  host_taxid misc_param lab-host  \\\n",
       "0             NaN      NaN               NaN         NaN        NaN      NaN   \n",
       "1             NaN      NaN               NaN         NaN        NaN      NaN   \n",
       "\n",
       "  v_type Replicate  DIP (uM)  NH4 (uM)  NO2 (uM)  NO3 (uM)  O2 (uM)  \\\n",
       "0    NaN       NaN       NaN       NaN       NaN       NaN      NaN   \n",
       "1    NaN       NaN       NaN       NaN       NaN       NaN      NaN   \n",
       "\n",
       "   Salinity (psu)  ph beta_lactamase_family carbapenemase  \\\n",
       "0             NaN NaN                   NaN           NaN   \n",
       "1             NaN NaN                   NaN           NaN   \n",
       "\n",
       "  edta_inhibitor_tested Sample Number health_disease_stat  host_spec_range  \\\n",
       "0                   NaN           NaN                 NaN              NaN   \n",
       "1                   NaN           NaN                 NaN              NaN   \n",
       "\n",
       "  assembly-note metagenome-source seawater metagenome type_status indoor_surf  \\\n",
       "0           NaN                                   NaN         NaN         NaN   \n",
       "1           NaN                                   NaN         NaN         NaN   \n",
       "\n",
       "  samp_sort_meth samp_vol_we_dna_ext surf_material  time point Strain  \\\n",
       "0            NaN                 NaN           NaN         NaN    NaN   \n",
       "1            NaN                 NaN           NaN         NaN    NaN   \n",
       "\n",
       "  sample comment growth_med host_genotype plant_body_site plant_product  \\\n",
       "0            NaN        NaN           NaN             NaN           NaN   \n",
       "1            NaN        NaN           NaN             NaN           NaN   \n",
       "\n",
       "  BioSampleModel closest_match  percent_match Environmental feature  \\\n",
       "0            NaN           NaN            NaN                   NaN   \n",
       "1            NaN           NaN            NaN                   NaN   \n",
       "\n",
       "  Environmental material ammonium biomass  carb_nitro_ratio chlorophyll  \\\n",
       "0                    NaN      NaN     NaN               NaN         NaN   \n",
       "1                    NaN      NaN     NaN               NaN         NaN   \n",
       "\n",
       "  diss_inorg_nitro diss_inorg_phosp diss_org_carb diss_org_nitro nitrate  \\\n",
       "0              NaN              NaN           NaN            NaN     NaN   \n",
       "1              NaN              NaN           NaN            NaN     NaN   \n",
       "\n",
       "  nitrite phosphate silicate tidal_stage tot_diss_nitro  Alistipes  \\\n",
       "0     NaN       NaN      NaN         NaN            NaN        NaN   \n",
       "1     NaN       NaN      NaN         NaN            NaN        NaN   \n",
       "\n",
       "   Anaerophaga  Anaerostipes  Bacteroidaceae  Bacteroidales  Bacteroides  \\\n",
       "0          NaN           NaN             NaN            NaN          NaN   \n",
       "1          NaN           NaN             NaN            NaN          NaN   \n",
       "\n",
       "   Bacteroidetes  Bacteroidia  Barnesiella  Blautia  Butyrivibrio  \\\n",
       "0            NaN          NaN          NaN      NaN           NaN   \n",
       "1            NaN          NaN          NaN      NaN           NaN   \n",
       "\n",
       "   Campylobacterales  Clostridia  Clostridiales  Deferribacteraceae  \\\n",
       "0                NaN         NaN            NaN                 NaN   \n",
       "1                NaN         NaN            NaN                 NaN   \n",
       "\n",
       "   Deferribacterales  Deferribacteres  Deltaproteobacteria  \\\n",
       "0                NaN              NaN                  NaN   \n",
       "1                NaN              NaN                  NaN   \n",
       "\n",
       "   Desulfovibrionaceae  Desulfovibrionales  Epsilonproteobacteria  Firmicutes  \\\n",
       "0                  NaN                 NaN                    NaN         NaN   \n",
       "1                  NaN                 NaN                    NaN         NaN   \n",
       "\n",
       "   Flammeovirgaceae  Flavobacteria  Flavobacteriaceae  Flavobacteriales  \\\n",
       "0               NaN            NaN                NaN               NaN   \n",
       "1               NaN            NaN                NaN               NaN   \n",
       "\n",
       "   Flexithrix  Helicobacter  Helicobacteraceae  Hespellia  Incertae_Sedis_XIV  \\\n",
       "0         NaN           NaN                NaN        NaN                 NaN   \n",
       "1         NaN           NaN                NaN        NaN                 NaN   \n",
       "\n",
       "   Lachnospiraceae  Lawsonia  Marinilabiaceae  Mollicutes  Mucispirillum  \\\n",
       "0              NaN       NaN              NaN         NaN            NaN   \n",
       "1              NaN       NaN              NaN         NaN            NaN   \n",
       "\n",
       "   Oscillibacter  Parasporobacterium  Peptococcaceae  Porphyromonadaceae  \\\n",
       "0            NaN                 NaN             NaN                 NaN   \n",
       "1            NaN                 NaN             NaN                 NaN   \n",
       "\n",
       "   Proteobacteria  Rikenellaceae  Robinsoniella  Ruminococcaceae  \\\n",
       "0             NaN            NaN            NaN              NaN   \n",
       "1             NaN            NaN            NaN              NaN   \n",
       "\n",
       "   Sandarakinotalea  Sphingobacteria  Sphingobacteriales  Sporobacterium  \\\n",
       "0               NaN              NaN                 NaN             NaN   \n",
       "1               NaN              NaN                 NaN             NaN   \n",
       "\n",
       "   Tenericutes  X1  X103  X10763  X111  X119  X1291  X13004  X14  X14089  \\\n",
       "0          NaN NaN   NaN     NaN   NaN   NaN    NaN     NaN  NaN     NaN   \n",
       "1          NaN NaN   NaN     NaN   NaN   NaN    NaN     NaN  NaN     NaN   \n",
       "\n",
       "   X168  X2  X229  X230  X2660  X28  X321  X3422  X4  X5217  X5802  X6075  \\\n",
       "0   NaN NaN   NaN   NaN    NaN  NaN   NaN    NaN NaN    NaN    NaN    NaN   \n",
       "1   NaN NaN   NaN   NaN    NaN  NaN   NaN    NaN NaN    NaN    NaN    NaN   \n",
       "\n",
       "   X62  X6209  X7  X8  X8841  X9 breed  sex tissue locus_tag prefix sample ID  \\\n",
       "0  NaN    NaN NaN NaN    NaN NaN   NaN  NaN    NaN              NaN       NaN   \n",
       "1  NaN    NaN NaN NaN    NaN NaN   NaN  NaN    NaN              NaN       NaN   \n",
       "\n",
       "  strains host sex cell_type geographic location (country and/or sea, region)  \n",
       "0     NaN      NaN       NaN                                              NaN  \n",
       "1     NaN      NaN       NaN                                              NaN  "
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "biosamples.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "both          1973\n",
       "left_only      713\n",
       "right_only       0\n",
       "Name: merge, dtype: int64"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "biosamples_filtered = pd.merge(patric_filtered, biosamples, how=\"left\", left_on=\"BioSample Accession\", right_on=\"sample_accession\", indicator=\"merge\")\n",
    "biosamples_filtered[\"merge\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genome_ID</th>\n",
       "      <th>Genome Name</th>\n",
       "      <th>Organism Name</th>\n",
       "      <th>NCBI Taxon ID</th>\n",
       "      <th>Genome Status</th>\n",
       "      <th>Strain_x</th>\n",
       "      <th>Serovar</th>\n",
       "      <th>Pathovar</th>\n",
       "      <th>MLST</th>\n",
       "      <th>Other Typing</th>\n",
       "      <th>Culture Collection</th>\n",
       "      <th>Type Strain_x</th>\n",
       "      <th>Completion Date</th>\n",
       "      <th>Publication</th>\n",
       "      <th>BioProject Accession</th>\n",
       "      <th>BioSample Accession</th>\n",
       "      <th>Assembly Accession</th>\n",
       "      <th>SRA Accession</th>\n",
       "      <th>GenBank Accessions</th>\n",
       "      <th>RefSeq Accessions</th>\n",
       "      <th>Sequencing Centers</th>\n",
       "      <th>Sequencing Status</th>\n",
       "      <th>Sequencing Platform</th>\n",
       "      <th>Sequencing Depth</th>\n",
       "      <th>Assembly Method_x</th>\n",
       "      <th>Chromosomes</th>\n",
       "      <th>Plasmids</th>\n",
       "      <th>Contigs</th>\n",
       "      <th>Genome Length</th>\n",
       "      <th>GC Content</th>\n",
       "      <th>RefSeq CDS</th>\n",
       "      <th>Isolation Source</th>\n",
       "      <th>Isolation Comments_x</th>\n",
       "      <th>Collection Date</th>\n",
       "      <th>Isolation Country</th>\n",
       "      <th>Geographic Location</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Altitude</th>\n",
       "      <th>Depth_x</th>\n",
       "      <th>Other Environmental</th>\n",
       "      <th>Host Name_x</th>\n",
       "      <th>Host Gender</th>\n",
       "      <th>Host Age</th>\n",
       "      <th>Host Health_x</th>\n",
       "      <th>Body Sample Site_x</th>\n",
       "      <th>Other Clinical</th>\n",
       "      <th>Gram Stain</th>\n",
       "      <th>Cell Shape_x</th>\n",
       "      <th>Motility_x</th>\n",
       "      <th>Sporulation_x</th>\n",
       "      <th>Temperature Range_x</th>\n",
       "      <th>Optimal Temperature</th>\n",
       "      <th>Salinity_x</th>\n",
       "      <th>Oxygen Requirement</th>\n",
       "      <th>Habitat</th>\n",
       "      <th>Disease</th>\n",
       "      <th>Comments</th>\n",
       "      <th>Additional Metadata</th>\n",
       "      <th>Genome Quality Flags</th>\n",
       "      <th>biosample_title</th>\n",
       "      <th>contact_email</th>\n",
       "      <th>first_name</th>\n",
       "      <th>growth phase</th>\n",
       "      <th>last_name</th>\n",
       "      <th>name</th>\n",
       "      <th>organism</th>\n",
       "      <th>publication_date</th>\n",
       "      <th>sample_accession</th>\n",
       "      <th>source_name</th>\n",
       "      <th>strain</th>\n",
       "      <th>submission_model</th>\n",
       "      <th>submission_package</th>\n",
       "      <th>submission_package_name</th>\n",
       "      <th>submitter</th>\n",
       "      <th>taxonomy_id</th>\n",
       "      <th>treatment</th>\n",
       "      <th>collection_date</th>\n",
       "      <th>culture_collection</th>\n",
       "      <th>geo_loc_name</th>\n",
       "      <th>isolation_source</th>\n",
       "      <th>sample_name</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>type-material</th>\n",
       "      <th>comment_paragraph</th>\n",
       "      <th>depth</th>\n",
       "      <th>derived-from</th>\n",
       "      <th>env_broad_scale</th>\n",
       "      <th>env_local_scale</th>\n",
       "      <th>env_medium</th>\n",
       "      <th>environmental-sample</th>\n",
       "      <th>isolate</th>\n",
       "      <th>lat_lon</th>\n",
       "      <th>metagenome-source</th>\n",
       "      <th>metagenomic</th>\n",
       "      <th>sample-type</th>\n",
       "      <th>elev</th>\n",
       "      <th>environmetal-sample</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>host</th>\n",
       "      <th>host_tissue_sampled</th>\n",
       "      <th>temp</th>\n",
       "      <th>env_biome</th>\n",
       "      <th>env_feature</th>\n",
       "      <th>env_material</th>\n",
       "      <th>isol_growth_condt</th>\n",
       "      <th>locus_tag_prefix</th>\n",
       "      <th>num_replicons</th>\n",
       "      <th>project_name</th>\n",
       "      <th>ref_biomaterial</th>\n",
       "      <th>16S recovered</th>\n",
       "      <th>16S recovery software</th>\n",
       "      <th>analysis project type</th>\n",
       "      <th>assembly quality</th>\n",
       "      <th>assembly software</th>\n",
       "      <th>bin parameters</th>\n",
       "      <th>binning software</th>\n",
       "      <th>completeness score</th>\n",
       "      <th>completeness software</th>\n",
       "      <th>contamination score</th>\n",
       "      <th>contig L50</th>\n",
       "      <th>decontamination software</th>\n",
       "      <th>note</th>\n",
       "      <th>number of contigs</th>\n",
       "      <th>reassembly post binning</th>\n",
       "      <th>relative coverage Anaerobic (SRR10097246)</th>\n",
       "      <th>relative coverage Anoxic (SRR10375098)</th>\n",
       "      <th>relative coverage Infiltration (SRR10097247)</th>\n",
       "      <th>relative coverage Low pH (SRR10097245)</th>\n",
       "      <th>relative coverage Oxic (SRR10375099)</th>\n",
       "      <th>relative coverage Reference (SRR10375097)</th>\n",
       "      <th>taxa id</th>\n",
       "      <th>total assembly size</th>\n",
       "      <th>nat_host</th>\n",
       "      <th>assembly_method_and_version</th>\n",
       "      <th>completeness_estimated</th>\n",
       "      <th>contamination_estimated</th>\n",
       "      <th>genome_coverage</th>\n",
       "      <th>mapping_method_and_version</th>\n",
       "      <th>quality_assessment_method_and_version</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>ENA-FIRST-PUBLIC</th>\n",
       "      <th>ENA-LAST-UPDATE</th>\n",
       "      <th>SUBJECT_ID</th>\n",
       "      <th>contact_lab</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>isolate-name-alias</th>\n",
       "      <th>environmental_sample</th>\n",
       "      <th>metagenome_source</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>culture media</th>\n",
       "      <th>replicate</th>\n",
       "      <th>biome</th>\n",
       "      <th>feature</th>\n",
       "      <th>material</th>\n",
       "      <th>alternate_ID</th>\n",
       "      <th>culture-collection</th>\n",
       "      <th>altitude</th>\n",
       "      <th>Genus</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>Species</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>isolate_name_alias</th>\n",
       "      <th>host_disease</th>\n",
       "      <th>rel_to_oxygen</th>\n",
       "      <th>Alias</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>Sample Name</th>\n",
       "      <th>Title</th>\n",
       "      <th>serotype</th>\n",
       "      <th>subsrc_note</th>\n",
       "      <th>annotation_method</th>\n",
       "      <th>assembly_method</th>\n",
       "      <th>biosamplemodel</th>\n",
       "      <th>geographic location (country and/or atlantic ocean)</th>\n",
       "      <th>geographic location (region and locality)</th>\n",
       "      <th>mapping_method</th>\n",
       "      <th>quality_assessment_method</th>\n",
       "      <th>refinement_method</th>\n",
       "      <th>Cell Shape_y</th>\n",
       "      <th>Funding Program</th>\n",
       "      <th>GOLD Stamp ID</th>\n",
       "      <th>Gene Calling Method</th>\n",
       "      <th>Gram Staining</th>\n",
       "      <th>Isolation Comments_y</th>\n",
       "      <th>Isolation Site</th>\n",
       "      <th>Motility_y</th>\n",
       "      <th>Temperature Optimum</th>\n",
       "      <th>Temperature Range_y</th>\n",
       "      <th>Type Strain_y</th>\n",
       "      <th>alt_elev</th>\n",
       "      <th>biotic_relationship</th>\n",
       "      <th>country</th>\n",
       "      <th>environment</th>\n",
       "      <th>investigation_type</th>\n",
       "      <th>isolation-source</th>\n",
       "      <th>trophic_level</th>\n",
       "      <th>pH</th>\n",
       "      <th>Body Sample Site_y</th>\n",
       "      <th>Body Sample SubSite</th>\n",
       "      <th>Diseases</th>\n",
       "      <th>Host Name_y</th>\n",
       "      <th>Host Health_y</th>\n",
       "      <th>Sporulation_y</th>\n",
       "      <th>experimental_factor</th>\n",
       "      <th>breeding_method</th>\n",
       "      <th>lat-lon</th>\n",
       "      <th>ENA-CHECKLIST</th>\n",
       "      <th>External Id</th>\n",
       "      <th>Submitter Id</th>\n",
       "      <th>subsrc-note</th>\n",
       "      <th>collection date</th>\n",
       "      <th>environment (biome)</th>\n",
       "      <th>environment (feature)</th>\n",
       "      <th>environment (material)</th>\n",
       "      <th>geographic location (country and/or sea)</th>\n",
       "      <th>geographic location (latitude)</th>\n",
       "      <th>geographic location (longitude)</th>\n",
       "      <th>host common name</th>\n",
       "      <th>host-associated environmental package</th>\n",
       "      <th>investigation type</th>\n",
       "      <th>project name</th>\n",
       "      <th>sequencing method</th>\n",
       "      <th>STRAIN</th>\n",
       "      <th>co_assembly_bin</th>\n",
       "      <th>derived from</th>\n",
       "      <th>description</th>\n",
       "      <th>Sequence processing</th>\n",
       "      <th>completeness</th>\n",
       "      <th>contamination</th>\n",
       "      <th>host_age</th>\n",
       "      <th>host_sex</th>\n",
       "      <th>oxy_stat_samp</th>\n",
       "      <th>samp_store_dur</th>\n",
       "      <th>samp_store_temp</th>\n",
       "      <th>FDA CVM ID</th>\n",
       "      <th>bio_material</th>\n",
       "      <th>common name</th>\n",
       "      <th>host scientific name</th>\n",
       "      <th>samp_mat_process</th>\n",
       "      <th>estimated_size</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>Tax ID</th>\n",
       "      <th>Collaborator ID</th>\n",
       "      <th>host_description</th>\n",
       "      <th>host_disease_outcome</th>\n",
       "      <th>host_disease_stage</th>\n",
       "      <th>host_health_state</th>\n",
       "      <th>identification method</th>\n",
       "      <th>strain_name_alias</th>\n",
       "      <th>WGA amplification approach</th>\n",
       "      <th>metagenomic source</th>\n",
       "      <th>sample derived from</th>\n",
       "      <th>single cell or viral particle lysis approach</th>\n",
       "      <th>sorting technology</th>\n",
       "      <th>taxonomic identity marker</th>\n",
       "      <th>passage_history</th>\n",
       "      <th>samp_size</th>\n",
       "      <th>timepoint</th>\n",
       "      <th>Description</th>\n",
       "      <th>subsource_note</th>\n",
       "      <th>lab_host</th>\n",
       "      <th>collection_room</th>\n",
       "      <th>relative_week</th>\n",
       "      <th>assembly_tool</th>\n",
       "      <th>bin_id</th>\n",
       "      <th>Broker name</th>\n",
       "      <th>number_of_identified_antimicrobial_resistance_genes</th>\n",
       "      <th>MAG_number</th>\n",
       "      <th>derived_from</th>\n",
       "      <th>alkalinity (carbonate/bicarbonate)</th>\n",
       "      <th>salinity</th>\n",
       "      <th>original-organism-name</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>env_package</th>\n",
       "      <th>label</th>\n",
       "      <th>specific_host</th>\n",
       "      <th>binning parameters</th>\n",
       "      <th>plant-associated environmental package</th>\n",
       "      <th>culture collection</th>\n",
       "      <th>repository</th>\n",
       "      <th>risk group</th>\n",
       "      <th>type status</th>\n",
       "      <th>Assembly Method_y</th>\n",
       "      <th>Genome Coverage</th>\n",
       "      <th>Sequencing Technology</th>\n",
       "      <th>nat-host</th>\n",
       "      <th>collection-date</th>\n",
       "      <th>Historical Monobactam Produced</th>\n",
       "      <th>Unique Monobactam Clusters Identified</th>\n",
       "      <th>extrachrom_elements</th>\n",
       "      <th>pathogenicity</th>\n",
       "      <th>source_material_id</th>\n",
       "      <th>genotype</th>\n",
       "      <th>mating_type</th>\n",
       "      <th>serovar</th>\n",
       "      <th>specimen_voucher</th>\n",
       "      <th>Salinity (PSU)</th>\n",
       "      <th>collected-by</th>\n",
       "      <th>orgmod_note</th>\n",
       "      <th>assembly</th>\n",
       "      <th>sequencing_meth</th>\n",
       "      <th>finishing_strategy</th>\n",
       "      <th>geo-loc-name</th>\n",
       "      <th>finishing strategy (depth of coverage)</th>\n",
       "      <th>host taxid</th>\n",
       "      <th>misc_param: HMP body site</th>\n",
       "      <th>misc_param: HMP supersite</th>\n",
       "      <th>nucleic acid extraction</th>\n",
       "      <th>project_type</th>\n",
       "      <th>sop</th>\n",
       "      <th>source_mat_id</th>\n",
       "      <th>Salinity_y</th>\n",
       "      <th>age</th>\n",
       "      <th>body_site</th>\n",
       "      <th>disease</th>\n",
       "      <th>health_state</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>Host</th>\n",
       "      <th>Sample name</th>\n",
       "      <th>geographic location (country and/or sea,region)</th>\n",
       "      <th>isolation source</th>\n",
       "      <th>environmental package</th>\n",
       "      <th>specific host</th>\n",
       "      <th>lat lon</th>\n",
       "      <th>host_subject_id</th>\n",
       "      <th>pathotype</th>\n",
       "      <th>subgroup</th>\n",
       "      <th>subtype</th>\n",
       "      <th>isolate name alias</th>\n",
       "      <th>zoonotic agent</th>\n",
       "      <th>Extraction Date</th>\n",
       "      <th>Extraction Method</th>\n",
       "      <th>Laboratory Host</th>\n",
       "      <th>Passage Date</th>\n",
       "      <th>sample type</th>\n",
       "      <th>HOST TAXON NAME</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>host health state</th>\n",
       "      <th>geographic location (elevation)</th>\n",
       "      <th>sediment environmental package</th>\n",
       "      <th>anonymized_name</th>\n",
       "      <th>host_status</th>\n",
       "      <th>supplier_name</th>\n",
       "      <th>plant product</th>\n",
       "      <th>ecotype</th>\n",
       "      <th>growth condition</th>\n",
       "      <th>abs_air_humidity</th>\n",
       "      <th>air_temp</th>\n",
       "      <th>build_occup_type</th>\n",
       "      <th>building_setting</th>\n",
       "      <th>carb_dioxide</th>\n",
       "      <th>filter_type</th>\n",
       "      <th>heat_cool_type</th>\n",
       "      <th>indoor_space</th>\n",
       "      <th>light_type</th>\n",
       "      <th>occup_samp</th>\n",
       "      <th>occupant_dens_samp</th>\n",
       "      <th>organism_count</th>\n",
       "      <th>rel_air_humidity</th>\n",
       "      <th>space_typ_state</th>\n",
       "      <th>typ_occupant_dens</th>\n",
       "      <th>ventilation_type</th>\n",
       "      <th>Type-material</th>\n",
       "      <th>samp_collect_device</th>\n",
       "      <th>Depth_y</th>\n",
       "      <th>Size fraction</th>\n",
       "      <th>genomovar</th>\n",
       "      <th>geographic location (altitude)</th>\n",
       "      <th>env_packatge</th>\n",
       "      <th>lib_reads_seqd</th>\n",
       "      <th>metagenomic_source</th>\n",
       "      <th>env material</th>\n",
       "      <th>other identifiers</th>\n",
       "      <th>assembly_method_version</th>\n",
       "      <th>mapping_method_version</th>\n",
       "      <th>quality_assessment_method_version</th>\n",
       "      <th>value</th>\n",
       "      <th>state</th>\n",
       "      <th>time</th>\n",
       "      <th>method</th>\n",
       "      <th>dna treatment</th>\n",
       "      <th>pacbio sequencing chemistry</th>\n",
       "      <th>samp_size (Library conc., pM)</th>\n",
       "      <th>cyanobacterial_culture</th>\n",
       "      <th>missing</th>\n",
       "      <th>samp_store_loc</th>\n",
       "      <th>sample_alias</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>vector</th>\n",
       "      <th>host_growth_cond</th>\n",
       "      <th>host_life_stage</th>\n",
       "      <th>c-source</th>\n",
       "      <th>host_body_habitat</th>\n",
       "      <th>host_taxid</th>\n",
       "      <th>misc_param</th>\n",
       "      <th>lab-host</th>\n",
       "      <th>v_type</th>\n",
       "      <th>Replicate</th>\n",
       "      <th>DIP (uM)</th>\n",
       "      <th>NH4 (uM)</th>\n",
       "      <th>NO2 (uM)</th>\n",
       "      <th>NO3 (uM)</th>\n",
       "      <th>O2 (uM)</th>\n",
       "      <th>Salinity (psu)</th>\n",
       "      <th>ph</th>\n",
       "      <th>beta_lactamase_family</th>\n",
       "      <th>carbapenemase</th>\n",
       "      <th>edta_inhibitor_tested</th>\n",
       "      <th>Sample Number</th>\n",
       "      <th>health_disease_stat</th>\n",
       "      <th>host_spec_range</th>\n",
       "      <th>assembly-note</th>\n",
       "      <th>metagenome-source seawater metagenome</th>\n",
       "      <th>type_status</th>\n",
       "      <th>indoor_surf</th>\n",
       "      <th>samp_sort_meth</th>\n",
       "      <th>samp_vol_we_dna_ext</th>\n",
       "      <th>surf_material</th>\n",
       "      <th>time point</th>\n",
       "      <th>Strain_y</th>\n",
       "      <th>sample comment</th>\n",
       "      <th>growth_med</th>\n",
       "      <th>host_genotype</th>\n",
       "      <th>plant_body_site</th>\n",
       "      <th>plant_product</th>\n",
       "      <th>BioSampleModel</th>\n",
       "      <th>closest_match</th>\n",
       "      <th>percent_match</th>\n",
       "      <th>Environmental feature</th>\n",
       "      <th>Environmental material</th>\n",
       "      <th>ammonium</th>\n",
       "      <th>biomass</th>\n",
       "      <th>carb_nitro_ratio</th>\n",
       "      <th>chlorophyll</th>\n",
       "      <th>diss_inorg_nitro</th>\n",
       "      <th>diss_inorg_phosp</th>\n",
       "      <th>diss_org_carb</th>\n",
       "      <th>diss_org_nitro</th>\n",
       "      <th>nitrate</th>\n",
       "      <th>nitrite</th>\n",
       "      <th>phosphate</th>\n",
       "      <th>silicate</th>\n",
       "      <th>tidal_stage</th>\n",
       "      <th>tot_diss_nitro</th>\n",
       "      <th>Alistipes</th>\n",
       "      <th>Anaerophaga</th>\n",
       "      <th>Anaerostipes</th>\n",
       "      <th>Bacteroidaceae</th>\n",
       "      <th>Bacteroidales</th>\n",
       "      <th>Bacteroides</th>\n",
       "      <th>Bacteroidetes</th>\n",
       "      <th>Bacteroidia</th>\n",
       "      <th>Barnesiella</th>\n",
       "      <th>Blautia</th>\n",
       "      <th>Butyrivibrio</th>\n",
       "      <th>Campylobacterales</th>\n",
       "      <th>Clostridia</th>\n",
       "      <th>Clostridiales</th>\n",
       "      <th>Deferribacteraceae</th>\n",
       "      <th>Deferribacterales</th>\n",
       "      <th>Deferribacteres</th>\n",
       "      <th>Deltaproteobacteria</th>\n",
       "      <th>Desulfovibrionaceae</th>\n",
       "      <th>Desulfovibrionales</th>\n",
       "      <th>Epsilonproteobacteria</th>\n",
       "      <th>Firmicutes</th>\n",
       "      <th>Flammeovirgaceae</th>\n",
       "      <th>Flavobacteria</th>\n",
       "      <th>Flavobacteriaceae</th>\n",
       "      <th>Flavobacteriales</th>\n",
       "      <th>Flexithrix</th>\n",
       "      <th>Helicobacter</th>\n",
       "      <th>Helicobacteraceae</th>\n",
       "      <th>Hespellia</th>\n",
       "      <th>Incertae_Sedis_XIV</th>\n",
       "      <th>Lachnospiraceae</th>\n",
       "      <th>Lawsonia</th>\n",
       "      <th>Marinilabiaceae</th>\n",
       "      <th>Mollicutes</th>\n",
       "      <th>Mucispirillum</th>\n",
       "      <th>Oscillibacter</th>\n",
       "      <th>Parasporobacterium</th>\n",
       "      <th>Peptococcaceae</th>\n",
       "      <th>Porphyromonadaceae</th>\n",
       "      <th>Proteobacteria</th>\n",
       "      <th>Rikenellaceae</th>\n",
       "      <th>Robinsoniella</th>\n",
       "      <th>Ruminococcaceae</th>\n",
       "      <th>Sandarakinotalea</th>\n",
       "      <th>Sphingobacteria</th>\n",
       "      <th>Sphingobacteriales</th>\n",
       "      <th>Sporobacterium</th>\n",
       "      <th>Tenericutes</th>\n",
       "      <th>X1</th>\n",
       "      <th>X103</th>\n",
       "      <th>X10763</th>\n",
       "      <th>X111</th>\n",
       "      <th>X119</th>\n",
       "      <th>X1291</th>\n",
       "      <th>X13004</th>\n",
       "      <th>X14</th>\n",
       "      <th>X14089</th>\n",
       "      <th>X168</th>\n",
       "      <th>X2</th>\n",
       "      <th>X229</th>\n",
       "      <th>X230</th>\n",
       "      <th>X2660</th>\n",
       "      <th>X28</th>\n",
       "      <th>X321</th>\n",
       "      <th>X3422</th>\n",
       "      <th>X4</th>\n",
       "      <th>X5217</th>\n",
       "      <th>X5802</th>\n",
       "      <th>X6075</th>\n",
       "      <th>X62</th>\n",
       "      <th>X6209</th>\n",
       "      <th>X7</th>\n",
       "      <th>X8</th>\n",
       "      <th>X8841</th>\n",
       "      <th>X9</th>\n",
       "      <th>breed</th>\n",
       "      <th>sex</th>\n",
       "      <th>tissue</th>\n",
       "      <th>locus_tag prefix</th>\n",
       "      <th>sample ID</th>\n",
       "      <th>strains</th>\n",
       "      <th>host sex</th>\n",
       "      <th>cell_type</th>\n",
       "      <th>geographic location (country and/or sea, region)</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>Complete</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>Yes</td>\n",
       "      <td>2007-05-01T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NC_009441</td>\n",
       "      <td>DOE Joint Genome Institute</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>6096872.0</td>\n",
       "      <td>34.1</td>\n",
       "      <td>5017.0</td>\n",
       "      <td>soil</td>\n",
       "      <td>isolated from soil in England and is the type ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>United Kingdom: England</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-</td>\n",
       "      <td>Rod</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>20-30</td>\n",
       "      <td>Non-halophilic</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>Multiple</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae ATCC 17061. Flavobac...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Sample from Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>2014-01-28T00:00:00.000</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>NaN</td>\n",
       "      <td>UW101; ATCC 17061</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Flavobacterium johnsoniae</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>Complete</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>Yes</td>\n",
       "      <td>2009-04-30T00:00:00Z</td>\n",
       "      <td>21304645</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>SRR013476,SRR013477</td>\n",
       "      <td>CP001632</td>\n",
       "      <td>NC_013162</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2612925.0</td>\n",
       "      <td>39.6</td>\n",
       "      <td>2171.0</td>\n",
       "      <td>the human oral cavity</td>\n",
       "      <td>isolated from the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-</td>\n",
       "      <td>Rod</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>35-37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Facultative</td>\n",
       "      <td>Host-associated</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271.  Capnocytoph...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic sample from Capnocytophaga ochracea DS...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>2009-02-19T00:00:00.000</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Genome_ID                       Genome Name  \\\n",
       "0  GCA_000016645.1   Flavobacterium johnsoniae UW101   \n",
       "1  GCA_000023285.1  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "                      Organism Name  NCBI Taxon ID Genome Status    Strain_x  \\\n",
       "0   Flavobacterium johnsoniae UW101       376686.0      Complete  ATCC 17061   \n",
       "1  Capnocytophaga ochracea DSM 7271       521097.0      Complete    DSM 7271   \n",
       "\n",
       "  Serovar Pathovar MLST Other Typing Culture Collection Type Strain_x  \\\n",
       "0     NaN      NaN  NaN          NaN         ATCC 17061           Yes   \n",
       "1     NaN      NaN  NaN          NaN           DSM 7271           Yes   \n",
       "\n",
       "        Completion Date Publication BioProject Accession BioSample Accession  \\\n",
       "0  2007-05-01T00:00:00Z         NaN           PRJNA16082        SAMN02598357   \n",
       "1  2009-04-30T00:00:00Z    21304645           PRJNA29403        SAMN00001911   \n",
       "\n",
       "  Assembly Accession        SRA Accession GenBank Accessions  \\\n",
       "0    GCA_000016645.1                  NaN           CP000685   \n",
       "1    GCA_000023285.1  SRR013476,SRR013477           CP001632   \n",
       "\n",
       "  RefSeq Accessions                                 Sequencing Centers  \\\n",
       "0         NC_009441                         DOE Joint Genome Institute   \n",
       "1         NC_013162  US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...   \n",
       "\n",
       "  Sequencing Status Sequencing Platform Sequencing Depth Assembly Method_x  \\\n",
       "0          complete                 NaN              NaN               NaN   \n",
       "1          complete                 NaN              NaN               NaN   \n",
       "\n",
       "   Chromosomes  Plasmids  Contigs  Genome Length  GC Content  RefSeq CDS  \\\n",
       "0          1.0       NaN      1.0      6096872.0        34.1      5017.0   \n",
       "1          1.0       NaN      1.0      2612925.0        39.6      2171.0   \n",
       "\n",
       "        Isolation Source                               Isolation Comments_x  \\\n",
       "0                   soil  isolated from soil in England and is the type ...   \n",
       "1  the human oral cavity                isolated from the human oral cavity   \n",
       "\n",
       "  Collection Date Isolation Country      Geographic Location  Latitude  \\\n",
       "0             NaN    United Kingdom  United Kingdom: England       NaN   \n",
       "1             NaN               NaN                      NaN       NaN   \n",
       "\n",
       "   Longitude Altitude Depth_x Other Environmental          Host Name_x  \\\n",
       "0        NaN      NaN     NaN                 NaN                  NaN   \n",
       "1        NaN      NaN     NaN                 NaN  Human, Homo sapiens   \n",
       "\n",
       "  Host Gender Host Age Host Health_x Body Sample Site_x Other Clinical  \\\n",
       "0         NaN      NaN           NaN                NaN            NaN   \n",
       "1         NaN      NaN           NaN                NaN            NaN   \n",
       "\n",
       "  Gram Stain Cell Shape_x Motility_x Sporulation_x Temperature Range_x  \\\n",
       "0          -          Rod        Yes            No          Mesophilic   \n",
       "1          -          Rod        Yes            No          Mesophilic   \n",
       "\n",
       "  Optimal Temperature      Salinity_x Oxygen Requirement          Habitat  \\\n",
       "0               20-30  Non-halophilic            Aerobic         Multiple   \n",
       "1               35-37             NaN        Facultative  Host-associated   \n",
       "\n",
       "  Disease                                           Comments  \\\n",
       "0     NaN  Flavobacterium johnsoniae ATCC 17061. Flavobac...   \n",
       "1     NaN  Capnocytophaga ochracea DSM 7271.  Capnocytoph...   \n",
       "\n",
       "  Additional Metadata Genome Quality Flags  \\\n",
       "0                 NaN                  NaN   \n",
       "1                 NaN                  NaN   \n",
       "\n",
       "                                     biosample_title contact_email first_name  \\\n",
       "0        Sample from Flavobacterium johnsoniae UW101           NaN        NaN   \n",
       "1  Generic sample from Capnocytophaga ochracea DS...           NaN        NaN   \n",
       "\n",
       "  growth phase last_name                                               name  \\\n",
       "0          NaN       NaN                                               NCBI   \n",
       "1          NaN       NaN  Joint Genome Institute, U.S. Department of Energy   \n",
       "\n",
       "                           organism         publication_date sample_accession  \\\n",
       "0   Flavobacterium johnsoniae UW101  2014-01-28T00:00:00.000     SAMN02598357   \n",
       "1  Capnocytophaga ochracea DSM 7271  2009-02-19T00:00:00.000     SAMN00001911   \n",
       "\n",
       "  source_name             strain submission_model submission_package  \\\n",
       "0         NaN  UW101; ATCC 17061          Generic        Generic.1.0   \n",
       "1         NaN           DSM 7271          Generic        Generic.1.0   \n",
       "\n",
       "  submission_package_name                                          submitter  \\\n",
       "0                 Generic                                               NCBI   \n",
       "1                 Generic  Joint Genome Institute, U.S. Department of Energy   \n",
       "\n",
       "   taxonomy_id treatment collection_date culture_collection geo_loc_name  \\\n",
       "0     376686.0       NaN             NaN                NaN          NaN   \n",
       "1     521097.0       NaN             NaN                NaN          NaN   \n",
       "\n",
       "  isolation_source sample_name sample_type  \\\n",
       "0              NaN    CP000685         NaN   \n",
       "1              NaN        FWCB         NaN   \n",
       "\n",
       "                              type-material                 comment_paragraph  \\\n",
       "0  type strain of Flavobacterium johnsoniae                               NaN   \n",
       "1    type strain of Capnocytophaga ochracea  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "  depth derived-from env_broad_scale env_local_scale env_medium  \\\n",
       "0   NaN          NaN             NaN             NaN        NaN   \n",
       "1   NaN          NaN             NaN             NaN        NaN   \n",
       "\n",
       "  environmental-sample isolate lat_lon metagenome-source metagenomic  \\\n",
       "0                  NaN     NaN     NaN               NaN         NaN   \n",
       "1                  NaN     NaN     NaN               NaN         NaN   \n",
       "\n",
       "  sample-type elev environmetal-sample collected_by host host_tissue_sampled  \\\n",
       "0         NaN  NaN                 NaN          NaN  NaN                 NaN   \n",
       "1         NaN  NaN                 NaN          NaN  NaN                 NaN   \n",
       "\n",
       "  temp env_biome env_feature env_material isol_growth_condt locus_tag_prefix  \\\n",
       "0  NaN       NaN         NaN          NaN               NaN              NaN   \n",
       "1  NaN       NaN         NaN          NaN               NaN              NaN   \n",
       "\n",
       "  num_replicons project_name ref_biomaterial 16S recovered  \\\n",
       "0           NaN          NaN             NaN           NaN   \n",
       "1           NaN          NaN             NaN           NaN   \n",
       "\n",
       "  16S recovery software analysis project type assembly quality  \\\n",
       "0                   NaN                   NaN              NaN   \n",
       "1                   NaN                   NaN              NaN   \n",
       "\n",
       "  assembly software bin parameters binning software  completeness score  \\\n",
       "0               NaN            NaN              NaN                 NaN   \n",
       "1               NaN            NaN              NaN                 NaN   \n",
       "\n",
       "  completeness software  contamination score  contig L50  \\\n",
       "0                   NaN                  NaN         NaN   \n",
       "1                   NaN                  NaN         NaN   \n",
       "\n",
       "  decontamination software note  number of contigs reassembly post binning  \\\n",
       "0                      NaN  NaN                NaN                     NaN   \n",
       "1                      NaN  NaN                NaN                     NaN   \n",
       "\n",
       "   relative coverage Anaerobic (SRR10097246)  \\\n",
       "0                                        NaN   \n",
       "1                                        NaN   \n",
       "\n",
       "   relative coverage Anoxic (SRR10375098)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Infiltration (SRR10097247)  \\\n",
       "0                                           NaN   \n",
       "1                                           NaN   \n",
       "\n",
       "   relative coverage Low pH (SRR10097245)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Oxic (SRR10375099)  \\\n",
       "0                                   NaN   \n",
       "1                                   NaN   \n",
       "\n",
       "   relative coverage Reference (SRR10375097) taxa id  total assembly size  \\\n",
       "0                                        NaN     NaN                  NaN   \n",
       "1                                        NaN     NaN                  NaN   \n",
       "\n",
       "  nat_host assembly_method_and_version completeness_estimated  \\\n",
       "0      NaN                         NaN                    NaN   \n",
       "1      NaN                         NaN                    NaN   \n",
       "\n",
       "  contamination_estimated genome_coverage mapping_method_and_version  \\\n",
       "0                     NaN             NaN                        NaN   \n",
       "1                     NaN             NaN                        NaN   \n",
       "\n",
       "  quality_assessment_method_and_version ArrayExpress-SPECIES ENA-FIRST-PUBLIC  \\\n",
       "0                                   NaN                  NaN              NaN   \n",
       "1                                   NaN                  NaN              NaN   \n",
       "\n",
       "  ENA-LAST-UPDATE SUBJECT_ID contact_lab identified_by isolate-name-alias  \\\n",
       "0             NaN        NaN         NaN           NaN                NaN   \n",
       "1             NaN        NaN         NaN           NaN                NaN   \n",
       "\n",
       "  environmental_sample metagenome_source biomaterial_provider culture media  \\\n",
       "0                  NaN               NaN                  NaN           NaN   \n",
       "1                  NaN               NaN                  NaN           NaN   \n",
       "\n",
       "  replicate biome feature material alternate_ID culture-collection altitude  \\\n",
       "0       NaN   NaN     NaN      NaN          NaN                NaN      NaN   \n",
       "1       NaN   NaN     NaN      NaN          NaN                NaN      NaN   \n",
       "\n",
       "  Genus ProjectAccession PublicAccession Species attribute_package  \\\n",
       "0   NaN              NaN             NaN     NaN               NaN   \n",
       "1   NaN              NaN             NaN     NaN               NaN   \n",
       "\n",
       "  isolate_name_alias host_disease rel_to_oxygen Alias ENA checklist  \\\n",
       "0                NaN          NaN           NaN   NaN           NaN   \n",
       "1                NaN          NaN           NaN   NaN           NaN   \n",
       "\n",
       "  INSDC center alias INSDC center name INSDC first public INSDC last update  \\\n",
       "0                NaN               NaN                NaN               NaN   \n",
       "1                NaN               NaN                NaN               NaN   \n",
       "\n",
       "  INSDC status SRA accession Sample Name Title serotype subsrc_note  \\\n",
       "0          NaN           NaN         NaN   NaN      NaN         NaN   \n",
       "1          NaN           NaN         NaN   NaN      NaN         NaN   \n",
       "\n",
       "  annotation_method assembly_method biosamplemodel  \\\n",
       "0               NaN             NaN            NaN   \n",
       "1               NaN             NaN            NaN   \n",
       "\n",
       "  geographic location (country and/or atlantic ocean)  \\\n",
       "0                                                NaN    \n",
       "1                                                NaN    \n",
       "\n",
       "  geographic location (region and locality) mapping_method  \\\n",
       "0                                       NaN            NaN   \n",
       "1                                       NaN            NaN   \n",
       "\n",
       "  quality_assessment_method refinement_method Cell Shape_y Funding Program  \\\n",
       "0                       NaN               NaN          NaN             NaN   \n",
       "1                       NaN               NaN          NaN             NaN   \n",
       "\n",
       "  GOLD Stamp ID Gene Calling Method Gram Staining Isolation Comments_y  \\\n",
       "0           NaN                 NaN           NaN                  NaN   \n",
       "1           NaN                 NaN           NaN                  NaN   \n",
       "\n",
       "  Isolation Site Motility_y Temperature Optimum Temperature Range_y  \\\n",
       "0            NaN        NaN                 NaN                 NaN   \n",
       "1            NaN        NaN                 NaN                 NaN   \n",
       "\n",
       "  Type Strain_y alt_elev biotic_relationship country environment  \\\n",
       "0           NaN      NaN                 NaN     NaN         NaN   \n",
       "1           NaN      NaN                 NaN     NaN         NaN   \n",
       "\n",
       "  investigation_type isolation-source trophic_level   pH Body Sample Site_y  \\\n",
       "0                NaN              NaN           NaN  NaN                NaN   \n",
       "1                NaN              NaN           NaN  NaN                NaN   \n",
       "\n",
       "  Body Sample SubSite Diseases Host Name_y Host Health_y Sporulation_y  \\\n",
       "0                 NaN      NaN         NaN           NaN           NaN   \n",
       "1                 NaN      NaN         NaN           NaN           NaN   \n",
       "\n",
       "  experimental_factor breeding_method lat-lon ENA-CHECKLIST External Id  \\\n",
       "0                 NaN             NaN     NaN           NaN         NaN   \n",
       "1                 NaN             NaN     NaN           NaN         NaN   \n",
       "\n",
       "  Submitter Id subsrc-note collection date environment (biome)  \\\n",
       "0          NaN         NaN             NaN                 NaN   \n",
       "1          NaN         NaN             NaN                 NaN   \n",
       "\n",
       "  environment (feature) environment (material)  \\\n",
       "0                   NaN                    NaN   \n",
       "1                   NaN                    NaN   \n",
       "\n",
       "  geographic location (country and/or sea) geographic location (latitude)  \\\n",
       "0                                      NaN                            NaN   \n",
       "1                                      NaN                            NaN   \n",
       "\n",
       "  geographic location (longitude) host common name  \\\n",
       "0                             NaN              NaN   \n",
       "1                             NaN              NaN   \n",
       "\n",
       "  host-associated environmental package investigation type project name  \\\n",
       "0                                   NaN                NaN          NaN   \n",
       "1                                   NaN                NaN          NaN   \n",
       "\n",
       "  sequencing method STRAIN co_assembly_bin derived from description  \\\n",
       "0               NaN    NaN             NaN          NaN         NaN   \n",
       "1               NaN    NaN             NaN          NaN         NaN   \n",
       "\n",
       "  Sequence processing  completeness  contamination host_age host_sex  \\\n",
       "0                 NaN           NaN            NaN      NaN      NaN   \n",
       "1                 NaN           NaN            NaN      NaN      NaN   \n",
       "\n",
       "  oxy_stat_samp samp_store_dur samp_store_temp FDA CVM ID bio_material  \\\n",
       "0           NaN            NaN             NaN        NaN          NaN   \n",
       "1           NaN            NaN             NaN        NaN          NaN   \n",
       "\n",
       "  common name host scientific name samp_mat_process estimated_size  \\\n",
       "0         NaN                  NaN              NaN            NaN   \n",
       "1         NaN                  NaN              NaN            NaN   \n",
       "\n",
       "  ArrayExpress-STRAIN_OR_LINE  Tax ID Collaborator ID host_description  \\\n",
       "0                         NaN     NaN             NaN              NaN   \n",
       "1                         NaN     NaN             NaN              NaN   \n",
       "\n",
       "  host_disease_outcome host_disease_stage host_health_state  \\\n",
       "0                  NaN                NaN               NaN   \n",
       "1                  NaN                NaN               NaN   \n",
       "\n",
       "  identification method strain_name_alias WGA amplification approach  \\\n",
       "0                   NaN               NaN                        NaN   \n",
       "1                   NaN               NaN                        NaN   \n",
       "\n",
       "  metagenomic source sample derived from  \\\n",
       "0                NaN                 NaN   \n",
       "1                NaN                 NaN   \n",
       "\n",
       "  single cell or viral particle lysis approach sorting technology  \\\n",
       "0                                          NaN                NaN   \n",
       "1                                          NaN                NaN   \n",
       "\n",
       "  taxonomic identity marker passage_history samp_size timepoint Description  \\\n",
       "0                       NaN             NaN       NaN       NaN         NaN   \n",
       "1                       NaN             NaN       NaN       NaN         NaN   \n",
       "\n",
       "  subsource_note lab_host collection_room  relative_week assembly_tool bin_id  \\\n",
       "0            NaN      NaN             NaN            NaN           NaN    NaN   \n",
       "1            NaN      NaN             NaN            NaN           NaN    NaN   \n",
       "\n",
       "  Broker name  number_of_identified_antimicrobial_resistance_genes  \\\n",
       "0         NaN                                                NaN     \n",
       "1         NaN                                                NaN     \n",
       "\n",
       "   MAG_number derived_from alkalinity (carbonate/bicarbonate) salinity  \\\n",
       "0         NaN          NaN                                NaN      NaN   \n",
       "1         NaN          NaN                                NaN      NaN   \n",
       "\n",
       "  original-organism-name Phenotypes env_package label specific_host  \\\n",
       "0                    NaN        NaN         NaN   NaN           NaN   \n",
       "1                    NaN        NaN         NaN   NaN           NaN   \n",
       "\n",
       "  binning parameters plant-associated environmental package  \\\n",
       "0                NaN                                    NaN   \n",
       "1                NaN                                    NaN   \n",
       "\n",
       "  culture collection repository risk group type status Assembly Method_y  \\\n",
       "0                NaN        NaN        NaN         NaN               NaN   \n",
       "1                NaN        NaN        NaN         NaN               NaN   \n",
       "\n",
       "  Genome Coverage  Sequencing Technology nat-host collection-date  \\\n",
       "0             NaN                    NaN      NaN             NaN   \n",
       "1             NaN                    NaN      NaN             NaN   \n",
       "\n",
       "  Historical Monobactam Produced Unique Monobactam Clusters Identified  \\\n",
       "0                            NaN                                   NaN   \n",
       "1                            NaN                                   NaN   \n",
       "\n",
       "   extrachrom_elements pathogenicity source_material_id genotype mating_type  \\\n",
       "0                  NaN           NaN                NaN      NaN         NaN   \n",
       "1                  NaN           NaN                NaN      NaN         NaN   \n",
       "\n",
       "  serovar specimen_voucher  Salinity (PSU) collected-by orgmod_note assembly  \\\n",
       "0     NaN              NaN             NaN          NaN         NaN      NaN   \n",
       "1     NaN              NaN             NaN          NaN         NaN      NaN   \n",
       "\n",
       "  sequencing_meth finishing_strategy geo-loc-name  \\\n",
       "0             NaN                NaN          NaN   \n",
       "1             NaN                NaN          NaN   \n",
       "\n",
       "  finishing strategy (depth of coverage)  host taxid  \\\n",
       "0                                    NaN         NaN   \n",
       "1                                    NaN         NaN   \n",
       "\n",
       "  misc_param: HMP body site misc_param: HMP supersite nucleic acid extraction  \\\n",
       "0                       NaN                       NaN                     NaN   \n",
       "1                       NaN                       NaN                     NaN   \n",
       "\n",
       "  project_type  sop source_mat_id Salinity_y  age body_site disease  \\\n",
       "0          NaN  NaN           NaN        NaN  NaN       NaN     NaN   \n",
       "1          NaN  NaN           NaN        NaN  NaN       NaN     NaN   \n",
       "\n",
       "  health_state Isolation source Host Sample name  \\\n",
       "0          NaN              NaN  NaN         NaN   \n",
       "1          NaN              NaN  NaN         NaN   \n",
       "\n",
       "  geographic location (country and/or sea,region) isolation source  \\\n",
       "0                                             NaN              NaN   \n",
       "1                                             NaN              NaN   \n",
       "\n",
       "  environmental package specific host lat lon host_subject_id pathotype  \\\n",
       "0                   NaN           NaN     NaN             NaN       NaN   \n",
       "1                   NaN           NaN     NaN             NaN       NaN   \n",
       "\n",
       "  subgroup subtype isolate name alias zoonotic agent Extraction Date  \\\n",
       "0      NaN     NaN                NaN            NaN             NaN   \n",
       "1      NaN     NaN                NaN            NaN             NaN   \n",
       "\n",
       "  Extraction Method Laboratory Host Passage Date sample type HOST TAXON NAME  \\\n",
       "0               NaN             NaN          NaN         NaN             NaN   \n",
       "1               NaN             NaN          NaN         NaN             NaN   \n",
       "\n",
       "  sample_description host health state  geographic location (elevation)  \\\n",
       "0                NaN               NaN                              NaN   \n",
       "1                NaN               NaN                              NaN   \n",
       "\n",
       "  sediment environmental package anonymized_name host_status supplier_name  \\\n",
       "0                            NaN             NaN         NaN           NaN   \n",
       "1                            NaN             NaN         NaN           NaN   \n",
       "\n",
       "  plant product ecotype growth condition abs_air_humidity  air_temp  \\\n",
       "0           NaN     NaN              NaN              NaN       NaN   \n",
       "1           NaN     NaN              NaN              NaN       NaN   \n",
       "\n",
       "  build_occup_type building_setting carb_dioxide filter_type heat_cool_type  \\\n",
       "0              NaN              NaN          NaN         NaN            NaN   \n",
       "1              NaN              NaN          NaN         NaN            NaN   \n",
       "\n",
       "  indoor_space light_type occup_samp occupant_dens_samp organism_count  \\\n",
       "0          NaN        NaN        NaN                NaN            NaN   \n",
       "1          NaN        NaN        NaN                NaN            NaN   \n",
       "\n",
       "  rel_air_humidity space_typ_state typ_occupant_dens ventilation_type  \\\n",
       "0              NaN             NaN               NaN              NaN   \n",
       "1              NaN             NaN               NaN              NaN   \n",
       "\n",
       "  Type-material samp_collect_device Depth_y Size fraction genomovar  \\\n",
       "0           NaN                 NaN     NaN           NaN       NaN   \n",
       "1           NaN                 NaN     NaN           NaN       NaN   \n",
       "\n",
       "  geographic location (altitude) env_packatge lib_reads_seqd  \\\n",
       "0                            NaN          NaN            NaN   \n",
       "1                            NaN          NaN            NaN   \n",
       "\n",
       "  metagenomic_source env material other identifiers assembly_method_version  \\\n",
       "0                NaN          NaN               NaN                     NaN   \n",
       "1                NaN          NaN               NaN                     NaN   \n",
       "\n",
       "  mapping_method_version quality_assessment_method_version value state time  \\\n",
       "0                    NaN                               NaN   NaN   NaN  NaN   \n",
       "1                    NaN                               NaN   NaN   NaN  NaN   \n",
       "\n",
       "  method dna treatment pacbio sequencing chemistry  \\\n",
       "0    NaN           NaN                         NaN   \n",
       "1    NaN           NaN                         NaN   \n",
       "\n",
       "   samp_size (Library conc., pM) cyanobacterial_culture missing  \\\n",
       "0                            NaN                    NaN     NaN   \n",
       "1                            NaN                    NaN     NaN   \n",
       "\n",
       "  samp_store_loc sample_alias ethnicity vector host_growth_cond  \\\n",
       "0            NaN          NaN       NaN    NaN              NaN   \n",
       "1            NaN          NaN       NaN    NaN              NaN   \n",
       "\n",
       "  host_life_stage c-source host_body_habitat  host_taxid misc_param lab-host  \\\n",
       "0             NaN      NaN               NaN         NaN        NaN      NaN   \n",
       "1             NaN      NaN               NaN         NaN        NaN      NaN   \n",
       "\n",
       "  v_type Replicate  DIP (uM)  NH4 (uM)  NO2 (uM)  NO3 (uM)  O2 (uM)  \\\n",
       "0    NaN       NaN       NaN       NaN       NaN       NaN      NaN   \n",
       "1    NaN       NaN       NaN       NaN       NaN       NaN      NaN   \n",
       "\n",
       "   Salinity (psu)  ph beta_lactamase_family carbapenemase  \\\n",
       "0             NaN NaN                   NaN           NaN   \n",
       "1             NaN NaN                   NaN           NaN   \n",
       "\n",
       "  edta_inhibitor_tested Sample Number health_disease_stat  host_spec_range  \\\n",
       "0                   NaN           NaN                 NaN              NaN   \n",
       "1                   NaN           NaN                 NaN              NaN   \n",
       "\n",
       "  assembly-note metagenome-source seawater metagenome type_status indoor_surf  \\\n",
       "0           NaN                                   NaN         NaN         NaN   \n",
       "1           NaN                                   NaN         NaN         NaN   \n",
       "\n",
       "  samp_sort_meth samp_vol_we_dna_ext surf_material  time point Strain_y  \\\n",
       "0            NaN                 NaN           NaN         NaN      NaN   \n",
       "1            NaN                 NaN           NaN         NaN      NaN   \n",
       "\n",
       "  sample comment growth_med host_genotype plant_body_site plant_product  \\\n",
       "0            NaN        NaN           NaN             NaN           NaN   \n",
       "1            NaN        NaN           NaN             NaN           NaN   \n",
       "\n",
       "  BioSampleModel closest_match  percent_match Environmental feature  \\\n",
       "0            NaN           NaN            NaN                   NaN   \n",
       "1            NaN           NaN            NaN                   NaN   \n",
       "\n",
       "  Environmental material ammonium biomass  carb_nitro_ratio chlorophyll  \\\n",
       "0                    NaN      NaN     NaN               NaN         NaN   \n",
       "1                    NaN      NaN     NaN               NaN         NaN   \n",
       "\n",
       "  diss_inorg_nitro diss_inorg_phosp diss_org_carb diss_org_nitro nitrate  \\\n",
       "0              NaN              NaN           NaN            NaN     NaN   \n",
       "1              NaN              NaN           NaN            NaN     NaN   \n",
       "\n",
       "  nitrite phosphate silicate tidal_stage tot_diss_nitro  Alistipes  \\\n",
       "0     NaN       NaN      NaN         NaN            NaN        NaN   \n",
       "1     NaN       NaN      NaN         NaN            NaN        NaN   \n",
       "\n",
       "   Anaerophaga  Anaerostipes  Bacteroidaceae  Bacteroidales  Bacteroides  \\\n",
       "0          NaN           NaN             NaN            NaN          NaN   \n",
       "1          NaN           NaN             NaN            NaN          NaN   \n",
       "\n",
       "   Bacteroidetes  Bacteroidia  Barnesiella  Blautia  Butyrivibrio  \\\n",
       "0            NaN          NaN          NaN      NaN           NaN   \n",
       "1            NaN          NaN          NaN      NaN           NaN   \n",
       "\n",
       "   Campylobacterales  Clostridia  Clostridiales  Deferribacteraceae  \\\n",
       "0                NaN         NaN            NaN                 NaN   \n",
       "1                NaN         NaN            NaN                 NaN   \n",
       "\n",
       "   Deferribacterales  Deferribacteres  Deltaproteobacteria  \\\n",
       "0                NaN              NaN                  NaN   \n",
       "1                NaN              NaN                  NaN   \n",
       "\n",
       "   Desulfovibrionaceae  Desulfovibrionales  Epsilonproteobacteria  Firmicutes  \\\n",
       "0                  NaN                 NaN                    NaN         NaN   \n",
       "1                  NaN                 NaN                    NaN         NaN   \n",
       "\n",
       "   Flammeovirgaceae  Flavobacteria  Flavobacteriaceae  Flavobacteriales  \\\n",
       "0               NaN            NaN                NaN               NaN   \n",
       "1               NaN            NaN                NaN               NaN   \n",
       "\n",
       "   Flexithrix  Helicobacter  Helicobacteraceae  Hespellia  Incertae_Sedis_XIV  \\\n",
       "0         NaN           NaN                NaN        NaN                 NaN   \n",
       "1         NaN           NaN                NaN        NaN                 NaN   \n",
       "\n",
       "   Lachnospiraceae  Lawsonia  Marinilabiaceae  Mollicutes  Mucispirillum  \\\n",
       "0              NaN       NaN              NaN         NaN            NaN   \n",
       "1              NaN       NaN              NaN         NaN            NaN   \n",
       "\n",
       "   Oscillibacter  Parasporobacterium  Peptococcaceae  Porphyromonadaceae  \\\n",
       "0            NaN                 NaN             NaN                 NaN   \n",
       "1            NaN                 NaN             NaN                 NaN   \n",
       "\n",
       "   Proteobacteria  Rikenellaceae  Robinsoniella  Ruminococcaceae  \\\n",
       "0             NaN            NaN            NaN              NaN   \n",
       "1             NaN            NaN            NaN              NaN   \n",
       "\n",
       "   Sandarakinotalea  Sphingobacteria  Sphingobacteriales  Sporobacterium  \\\n",
       "0               NaN              NaN                 NaN             NaN   \n",
       "1               NaN              NaN                 NaN             NaN   \n",
       "\n",
       "   Tenericutes  X1  X103  X10763  X111  X119  X1291  X13004  X14  X14089  \\\n",
       "0          NaN NaN   NaN     NaN   NaN   NaN    NaN     NaN  NaN     NaN   \n",
       "1          NaN NaN   NaN     NaN   NaN   NaN    NaN     NaN  NaN     NaN   \n",
       "\n",
       "   X168  X2  X229  X230  X2660  X28  X321  X3422  X4  X5217  X5802  X6075  \\\n",
       "0   NaN NaN   NaN   NaN    NaN  NaN   NaN    NaN NaN    NaN    NaN    NaN   \n",
       "1   NaN NaN   NaN   NaN    NaN  NaN   NaN    NaN NaN    NaN    NaN    NaN   \n",
       "\n",
       "   X62  X6209  X7  X8  X8841  X9 breed  sex tissue locus_tag prefix sample ID  \\\n",
       "0  NaN    NaN NaN NaN    NaN NaN   NaN  NaN    NaN              NaN       NaN   \n",
       "1  NaN    NaN NaN NaN    NaN NaN   NaN  NaN    NaN              NaN       NaN   \n",
       "\n",
       "  strains host sex cell_type geographic location (country and/or sea, region)  \n",
       "0     NaN      NaN       NaN                                              NaN  \n",
       "1     NaN      NaN       NaN                                              NaN  "
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "biosamples_filtered.drop(columns=[\"merge\"], inplace=True)\n",
    "biosamples_filtered.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "#REMOVE empty columns\n",
    "null_cols = biosamples_filtered.isnull().all()\n",
    "biosamples_filtered = biosamples_filtered[biosamples_filtered.columns[~null_cols]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Int64Index: 2686 entries, 0 to 2685\n",
      "Columns: 377 entries, Genome_ID to locus_tag prefix\n",
      "dtypes: float64(34), object(343)\n",
      "memory usage: 7.7+ MB\n"
     ]
    }
   ],
   "source": [
    "biosamples_filtered.info()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2.4 SRA"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**SRA data manipulation was done in jupyter notebook:** 2.1.1_SRA_attribute"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>strain</th>\n",
       "      <th>finishing strategy (depth of coverage)</th>\n",
       "      <th>biotic_relationship</th>\n",
       "      <th>geo_loc_name</th>\n",
       "      <th>host</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>host_disease</th>\n",
       "      <th>isolation_source</th>\n",
       "      <th>sample_name</th>\n",
       "      <th>isolation-source</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>FDA CVM ID</th>\n",
       "      <th>culture_collection</th>\n",
       "      <th>assembly_method</th>\n",
       "      <th>collection_date</th>\n",
       "      <th>env_feature</th>\n",
       "      <th>synonym</th>\n",
       "      <th>Strain</th>\n",
       "      <th>SUBJECT_ID</th>\n",
       "      <th>Alias</th>\n",
       "      <th>type-material</th>\n",
       "      <th>misc_param</th>\n",
       "      <th>collection date</th>\n",
       "      <th>Isolation Site</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>country</th>\n",
       "      <th>note</th>\n",
       "      <th>GOLD Stamp ID</th>\n",
       "      <th>investigation_type</th>\n",
       "      <th>culture-collection</th>\n",
       "      <th>lat_lon</th>\n",
       "      <th>alt_elev</th>\n",
       "      <th>locus_tag_prefix</th>\n",
       "      <th>nat-host</th>\n",
       "      <th>assembly_method_version</th>\n",
       "      <th>env_material</th>\n",
       "      <th>model</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>Sample Description</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>project_type</th>\n",
       "      <th>estimated_size</th>\n",
       "      <th>Temperature Optimum</th>\n",
       "      <th>isol_growth_condt</th>\n",
       "      <th>Motility</th>\n",
       "      <th>feature</th>\n",
       "      <th>project_name</th>\n",
       "      <th>assembly</th>\n",
       "      <th>num_replicons</th>\n",
       "      <th>isolate</th>\n",
       "      <th>depth</th>\n",
       "      <th>env_biome</th>\n",
       "      <th>altitude</th>\n",
       "      <th>ref_biomaterial</th>\n",
       "      <th>package</th>\n",
       "      <th>geographic location</th>\n",
       "      <th>ArrayExpress-StrainOrLine</th>\n",
       "      <th>ArrayExpress-Phenotype</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>geo-loc-name</th>\n",
       "      <th>environment</th>\n",
       "      <th>host_sex</th>\n",
       "      <th>elev</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>completeness_estimated</th>\n",
       "      <th>geographic location (country and/or sea)</th>\n",
       "      <th>latitude and longitude</th>\n",
       "      <th>specific_host</th>\n",
       "      <th>ArrayExpress-Species</th>\n",
       "      <th>ENA-FIRST-PUBLIC</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>specific host</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>pH</th>\n",
       "      <th>biome</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>isolate_name_alias</th>\n",
       "      <th>contamination_estimated</th>\n",
       "      <th>host scientific name</th>\n",
       "      <th>anonymized_name</th>\n",
       "      <th>ENA-LAST-UPDATE</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>BioSampleModel</th>\n",
       "      <th>environmental_sample</th>\n",
       "      <th>material</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>host health state</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>Diseases</th>\n",
       "      <th>strain_name_alias</th>\n",
       "      <th>genome_coverage</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>host taxid</th>\n",
       "      <th>isolation source</th>\n",
       "      <th>Gram Staining</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>sequencing_meth</th>\n",
       "      <th>Historical Monobactam Produced</th>\n",
       "      <th>identification method</th>\n",
       "      <th>supplier_name</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>host_disease_outcome</th>\n",
       "      <th>Unique Monobactam Clusters Identified</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>Sample Name</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Gene Calling Method</th>\n",
       "      <th>temp</th>\n",
       "      <th>Title</th>\n",
       "      <th>investigation type</th>\n",
       "      <th>Host</th>\n",
       "      <th>rel_to_oxygen</th>\n",
       "      <th>host_age</th>\n",
       "      <th>Species</th>\n",
       "      <th>mapping_method</th>\n",
       "      <th>serovar</th>\n",
       "      <th>sop</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>finishing_strategy</th>\n",
       "      <th>Genus</th>\n",
       "      <th>mapping_method_version</th>\n",
       "      <th>environment (biome)</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>metagenome_source</th>\n",
       "      <th>environment (feature)</th>\n",
       "      <th>trophic_level</th>\n",
       "      <th>metagenomic</th>\n",
       "      <th>environment (material)</th>\n",
       "      <th>Funding Program</th>\n",
       "      <th>quality_assessment_method</th>\n",
       "      <th>quality_assessment_method_version</th>\n",
       "      <th>geographic location (latitude)</th>\n",
       "      <th>source_mat_id</th>\n",
       "      <th>host_tissue_sampled</th>\n",
       "      <th>geographic location (longitude)</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>subsrc_note</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>value</th>\n",
       "      <th>plant-associated environmental package</th>\n",
       "      <th>project name</th>\n",
       "      <th>sequencing method</th>\n",
       "      <th>Body Sample SubSite</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000143785.1</td>\n",
       "      <td>ATCC 35910</td>\n",
       "      <td>Level 6, Finished33.84x;7</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Chryseobacterium gleum</td>\n",
       "      <td>HMP body site, Vaginal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC:35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Reference Genome</td>\n",
       "      <td>5562000.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens-associated habitat ENVO:00009003</td>\n",
       "      <td>Chryseobacterium gleum F93</td>\n",
       "      <td>Newbler v. 2.0.0-Eclipse</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>terrestrial biome ENVO:00000446</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>MIGS.ba</td>\n",
       "      <td>NaN</td>\n",
       "      <td>biological product ENVO:02000043</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9606.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>http://hmpdacc.org/doc/CommonGeneAnnotation_SO...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 35910</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>454-GS-FLX</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  assembly_accession      strain finishing strategy (depth of coverage)  \\\n",
       "0    GCA_000023285.1    DSM 7271                                    NaN   \n",
       "1    GCA_000143785.1  ATCC 35910              Level 6, Finished33.84x;7   \n",
       "\n",
       "  biotic_relationship geo_loc_name host collected_by host_disease  \\\n",
       "0                 NaN          NaN  NaN          NaN          NaN   \n",
       "1                 NaN          NaN  NaN          NaN          NaN   \n",
       "\n",
       "  isolation_source sample_name isolation-source sample_type FDA CVM ID  \\\n",
       "0              NaN         NaN              NaN         NaN        NaN   \n",
       "1              NaN         NaN              NaN         NaN        NaN   \n",
       "\n",
       "  culture_collection assembly_method collection_date env_feature synonym  \\\n",
       "0                NaN             NaN             NaN         NaN     NaN   \n",
       "1                NaN             NaN             NaN         NaN     NaN   \n",
       "\n",
       "  Strain SUBJECT_ID Alias                           type-material  \\\n",
       "0    NaN        NaN   NaN  type strain of Capnocytophaga ochracea   \n",
       "1    NaN        NaN   NaN   type strain of Chryseobacterium gleum   \n",
       "\n",
       "               misc_param collection date Isolation Site Isolation source  \\\n",
       "0                     NaN             NaN            NaN              NaN   \n",
       "1  HMP body site, Vaginal             NaN            NaN              NaN   \n",
       "\n",
       "  country note GOLD Stamp ID investigation_type culture-collection lat_lon  \\\n",
       "0     NaN  NaN           NaN                NaN                NaN     NaN   \n",
       "1     NaN  NaN           NaN                NaN         ATCC:35910     NaN   \n",
       "\n",
       "  alt_elev locus_tag_prefix nat-host assembly_method_version env_material  \\\n",
       "0      NaN              NaN      NaN                     NaN          NaN   \n",
       "1      NaN              NaN      NaN                     NaN          NaN   \n",
       "\n",
       "  model  sample_description  Sample Description ArrayExpress-STRAIN_OR_LINE  \\\n",
       "0   NaN                 NaN                 NaN                         NaN   \n",
       "1   NaN                 NaN                 NaN                         NaN   \n",
       "\n",
       "  ENA checklist      project_type  estimated_size Temperature Optimum  \\\n",
       "0           NaN               NaN             NaN                 NaN   \n",
       "1           NaN  Reference Genome       5562000.0                 NaN   \n",
       "\n",
       "   isol_growth_condt Motility                                        feature  \\\n",
       "0                NaN      NaN                                            NaN   \n",
       "1                NaN      NaN  Homo sapiens-associated habitat ENVO:00009003   \n",
       "\n",
       "                 project_name                  assembly  num_replicons  \\\n",
       "0                         NaN                       NaN            NaN   \n",
       "1  Chryseobacterium gleum F93  Newbler v. 2.0.0-Eclipse            NaN   \n",
       "\n",
       "  isolate depth env_biome altitude ref_biomaterial package  \\\n",
       "0     NaN   NaN       NaN      NaN             NaN     NaN   \n",
       "1     NaN   NaN       NaN      NaN             NaN     NaN   \n",
       "\n",
       "  geographic location ArrayExpress-StrainOrLine  ArrayExpress-Phenotype  \\\n",
       "0                 NaN                       NaN                     NaN   \n",
       "1                 NaN                       NaN                     NaN   \n",
       "\n",
       "  ArrayExpress-SPECIES INSDC center alias geo-loc-name environment host_sex  \\\n",
       "0                  NaN                NaN          NaN         NaN      NaN   \n",
       "1                  NaN                NaN          NaN         NaN      NaN   \n",
       "\n",
       "   elev PublicAccession completeness_estimated  \\\n",
       "0   NaN             NaN                    NaN   \n",
       "1   NaN             NaN                    NaN   \n",
       "\n",
       "  geographic location (country and/or sea) latitude and longitude  \\\n",
       "0                                      NaN                    NaN   \n",
       "1                                      NaN                    NaN   \n",
       "\n",
       "  specific_host ArrayExpress-Species ENA-FIRST-PUBLIC INSDC center name  \\\n",
       "0           NaN                  NaN              NaN               NaN   \n",
       "1           NaN                  NaN              NaN               NaN   \n",
       "\n",
       "  specific host Body Sample Site   pH                            biome  \\\n",
       "0           NaN              NaN  NaN                              NaN   \n",
       "1  Homo sapiens              NaN  NaN  terrestrial biome ENVO:00000446   \n",
       "\n",
       "  biomaterial_provider isolate_name_alias contamination_estimated  \\\n",
       "0                  NaN                NaN                     NaN   \n",
       "1                  NaN                NaN                     NaN   \n",
       "\n",
       "  host scientific name  anonymized_name ENA-LAST-UPDATE INSDC first public  \\\n",
       "0                  NaN              NaN             NaN                NaN   \n",
       "1                  NaN              NaN             NaN                NaN   \n",
       "\n",
       "  Cell Shape BioSampleModel environmental_sample  \\\n",
       "0        NaN            NaN                  NaN   \n",
       "1        NaN        MIGS.ba                  NaN   \n",
       "\n",
       "                           material identified_by host health state  \\\n",
       "0                               NaN           NaN               NaN   \n",
       "1  biological product ENVO:02000043           NaN               NaN   \n",
       "\n",
       "  INSDC last update Diseases strain_name_alias  genome_coverage INSDC status  \\\n",
       "0               NaN      NaN               NaN              NaN          NaN   \n",
       "1               NaN      NaN               NaN              NaN          NaN   \n",
       "\n",
       "   host taxid isolation source Gram Staining Temperature Range Sporulation  \\\n",
       "0         NaN              NaN           NaN               NaN         NaN   \n",
       "1      9606.0              NaN           NaN               NaN         NaN   \n",
       "\n",
       "  sequencing_meth Historical Monobactam Produced identification method  \\\n",
       "0             NaN                            NaN                   NaN   \n",
       "1             NaN                            NaN                   NaN   \n",
       "\n",
       "  supplier_name SRA accession host_disease_outcome  \\\n",
       "0           NaN           NaN                  NaN   \n",
       "1           NaN           NaN                  NaN   \n",
       "\n",
       "  Unique Monobactam Clusters Identified ProjectAccession Sample Name  \\\n",
       "0                                   NaN              NaN         NaN   \n",
       "1                                   NaN              NaN         NaN   \n",
       "\n",
       "  Type Strain Phenotypes Gene Calling Method temp Title investigation type  \\\n",
       "0         NaN        NaN                 NaN  NaN   NaN                NaN   \n",
       "1         NaN        NaN                 NaN  NaN   NaN                NaN   \n",
       "\n",
       "  Host rel_to_oxygen host_age Species mapping_method  serovar  \\\n",
       "0  NaN           NaN      NaN     NaN            NaN      NaN   \n",
       "1  NaN           NaN      NaN     NaN            NaN      NaN   \n",
       "\n",
       "                                                 sop Isolation Comments  \\\n",
       "0                                                NaN                NaN   \n",
       "1  http://hmpdacc.org/doc/CommonGeneAnnotation_SO...                NaN   \n",
       "\n",
       "  finishing_strategy Genus mapping_method_version environment (biome)  \\\n",
       "0                NaN   NaN                    NaN                 NaN   \n",
       "1                NaN   NaN                    NaN                 NaN   \n",
       "\n",
       "  attribute_package metagenome_source environment (feature) trophic_level  \\\n",
       "0               NaN               NaN                   NaN           NaN   \n",
       "1               NaN               NaN                   NaN           NaN   \n",
       "\n",
       "  metagenomic environment (material) Funding Program  \\\n",
       "0         NaN                    NaN             NaN   \n",
       "1         NaN                    NaN             NaN   \n",
       "\n",
       "  quality_assessment_method quality_assessment_method_version  \\\n",
       "0                       NaN                               NaN   \n",
       "1                       NaN                               NaN   \n",
       "\n",
       "   geographic location (latitude) source_mat_id host_tissue_sampled  \\\n",
       "0                             NaN           NaN                 NaN   \n",
       "1                             NaN    ATCC 35910                 NaN   \n",
       "\n",
       "   geographic location (longitude) Host Name subsrc_note Host Health value  \\\n",
       "0                              NaN       NaN         NaN         NaN   NaN   \n",
       "1                              NaN       NaN         NaN         NaN   NaN   \n",
       "\n",
       "  plant-associated environmental package project name sequencing method  \\\n",
       "0                                    NaN          NaN               NaN   \n",
       "1                                    NaN          NaN        454-GS-FLX   \n",
       "\n",
       "  Body Sample SubSite  \n",
       "0                 NaN  \n",
       "1                 NaN  "
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sra = pd.read_csv(\"SRA_metadata.csv\")\n",
    "sra.head(2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "----"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2.5 MERGE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "both          2686\n",
       "right_only       0\n",
       "left_only        0\n",
       "Name: merge, dtype: int64"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merge1 = pd.merge(genbank_filtered, biosamples_filtered, how=\"left\", left_on=\"Genome_ID\", right_on=\"Genome_ID\", indicator=\"merge\")\n",
    "merge1[\"merge\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "left_only     2149\n",
       "both           537\n",
       "right_only       0\n",
       "Name: merge2, dtype: int64"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merge2 = pd.merge(merge1, sra, how=\"left\", left_on=\"Genome_ID\", right_on=\"assembly_accession\", indicator=\"merge2\")\n",
    "merge2[\"merge2\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genome_ID</th>\n",
       "      <th># assembly_accession</th>\n",
       "      <th>bioproject</th>\n",
       "      <th>biosample</th>\n",
       "      <th>wgs_master</th>\n",
       "      <th>refseq_category</th>\n",
       "      <th>taxid</th>\n",
       "      <th>species_taxid</th>\n",
       "      <th>organism_name</th>\n",
       "      <th>infraspecific_name</th>\n",
       "      <th>isolate_x</th>\n",
       "      <th>version_status</th>\n",
       "      <th>assembly_level</th>\n",
       "      <th>release_type</th>\n",
       "      <th>genome_rep</th>\n",
       "      <th>seq_rel_date</th>\n",
       "      <th>asm_name</th>\n",
       "      <th>submitter_x</th>\n",
       "      <th>gbrs_paired_asm</th>\n",
       "      <th>paired_asm_comp</th>\n",
       "      <th>ftp_path</th>\n",
       "      <th>excluded_from_refseq</th>\n",
       "      <th>relation_to_type_material</th>\n",
       "      <th>Genome Name</th>\n",
       "      <th>Organism Name</th>\n",
       "      <th>NCBI Taxon ID</th>\n",
       "      <th>Genome Status</th>\n",
       "      <th>Strain_x</th>\n",
       "      <th>Serovar</th>\n",
       "      <th>Pathovar</th>\n",
       "      <th>MLST</th>\n",
       "      <th>Other Typing</th>\n",
       "      <th>Culture Collection</th>\n",
       "      <th>Type Strain_x</th>\n",
       "      <th>Completion Date</th>\n",
       "      <th>Publication</th>\n",
       "      <th>BioProject Accession</th>\n",
       "      <th>BioSample Accession</th>\n",
       "      <th>Assembly Accession</th>\n",
       "      <th>SRA Accession</th>\n",
       "      <th>GenBank Accessions</th>\n",
       "      <th>RefSeq Accessions</th>\n",
       "      <th>Sequencing Centers</th>\n",
       "      <th>Sequencing Status</th>\n",
       "      <th>Sequencing Platform</th>\n",
       "      <th>Sequencing Depth</th>\n",
       "      <th>Assembly Method_x</th>\n",
       "      <th>Chromosomes</th>\n",
       "      <th>Plasmids</th>\n",
       "      <th>Contigs</th>\n",
       "      <th>Genome Length</th>\n",
       "      <th>GC Content</th>\n",
       "      <th>RefSeq CDS</th>\n",
       "      <th>Isolation Source</th>\n",
       "      <th>Isolation Comments_x</th>\n",
       "      <th>Collection Date</th>\n",
       "      <th>Isolation Country</th>\n",
       "      <th>Geographic Location</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Altitude</th>\n",
       "      <th>Depth_x</th>\n",
       "      <th>Other Environmental</th>\n",
       "      <th>Host Name_x</th>\n",
       "      <th>Host Gender</th>\n",
       "      <th>Host Age</th>\n",
       "      <th>Host Health_x</th>\n",
       "      <th>Body Sample Site_x</th>\n",
       "      <th>Other Clinical</th>\n",
       "      <th>Gram Stain</th>\n",
       "      <th>Cell Shape_x</th>\n",
       "      <th>Motility_x</th>\n",
       "      <th>Sporulation_x</th>\n",
       "      <th>Temperature Range_x</th>\n",
       "      <th>Optimal Temperature</th>\n",
       "      <th>Salinity_x</th>\n",
       "      <th>Oxygen Requirement</th>\n",
       "      <th>Habitat</th>\n",
       "      <th>Disease</th>\n",
       "      <th>Comments</th>\n",
       "      <th>Additional Metadata</th>\n",
       "      <th>Genome Quality Flags</th>\n",
       "      <th>biosample_title</th>\n",
       "      <th>contact_email</th>\n",
       "      <th>first_name</th>\n",
       "      <th>last_name</th>\n",
       "      <th>name</th>\n",
       "      <th>organism</th>\n",
       "      <th>publication_date</th>\n",
       "      <th>sample_accession</th>\n",
       "      <th>strain_x</th>\n",
       "      <th>submission_model</th>\n",
       "      <th>submission_package</th>\n",
       "      <th>submission_package_name</th>\n",
       "      <th>submitter_y</th>\n",
       "      <th>taxonomy_id</th>\n",
       "      <th>collection_date_x</th>\n",
       "      <th>culture_collection_x</th>\n",
       "      <th>geo_loc_name_x</th>\n",
       "      <th>isolation_source_x</th>\n",
       "      <th>sample_name_x</th>\n",
       "      <th>sample_type_x</th>\n",
       "      <th>type-material_x</th>\n",
       "      <th>comment_paragraph</th>\n",
       "      <th>depth_x</th>\n",
       "      <th>derived-from</th>\n",
       "      <th>env_broad_scale</th>\n",
       "      <th>env_local_scale</th>\n",
       "      <th>env_medium</th>\n",
       "      <th>environmental-sample</th>\n",
       "      <th>isolate_y</th>\n",
       "      <th>lat_lon_x</th>\n",
       "      <th>metagenome-source</th>\n",
       "      <th>metagenomic_x</th>\n",
       "      <th>sample-type</th>\n",
       "      <th>elev_x</th>\n",
       "      <th>environmetal-sample</th>\n",
       "      <th>collected_by_x</th>\n",
       "      <th>host_x</th>\n",
       "      <th>host_tissue_sampled_x</th>\n",
       "      <th>temp_x</th>\n",
       "      <th>env_biome_x</th>\n",
       "      <th>env_feature_x</th>\n",
       "      <th>env_material_x</th>\n",
       "      <th>isol_growth_condt_x</th>\n",
       "      <th>locus_tag_prefix_x</th>\n",
       "      <th>num_replicons_x</th>\n",
       "      <th>project_name_x</th>\n",
       "      <th>ref_biomaterial_x</th>\n",
       "      <th>16S recovered</th>\n",
       "      <th>16S recovery software</th>\n",
       "      <th>analysis project type</th>\n",
       "      <th>assembly quality</th>\n",
       "      <th>assembly software</th>\n",
       "      <th>bin parameters</th>\n",
       "      <th>binning software</th>\n",
       "      <th>completeness score</th>\n",
       "      <th>completeness software</th>\n",
       "      <th>contamination score</th>\n",
       "      <th>contig L50</th>\n",
       "      <th>decontamination software</th>\n",
       "      <th>note_x</th>\n",
       "      <th>number of contigs</th>\n",
       "      <th>reassembly post binning</th>\n",
       "      <th>relative coverage Anaerobic (SRR10097246)</th>\n",
       "      <th>relative coverage Anoxic (SRR10375098)</th>\n",
       "      <th>relative coverage Infiltration (SRR10097247)</th>\n",
       "      <th>relative coverage Low pH (SRR10097245)</th>\n",
       "      <th>relative coverage Oxic (SRR10375099)</th>\n",
       "      <th>relative coverage Reference (SRR10375097)</th>\n",
       "      <th>taxa id</th>\n",
       "      <th>total assembly size</th>\n",
       "      <th>nat_host</th>\n",
       "      <th>assembly_method_and_version</th>\n",
       "      <th>completeness_estimated_x</th>\n",
       "      <th>contamination_estimated_x</th>\n",
       "      <th>genome_coverage_x</th>\n",
       "      <th>mapping_method_and_version</th>\n",
       "      <th>quality_assessment_method_and_version</th>\n",
       "      <th>ENA-FIRST-PUBLIC_x</th>\n",
       "      <th>ENA-LAST-UPDATE_x</th>\n",
       "      <th>SUBJECT_ID_x</th>\n",
       "      <th>contact_lab</th>\n",
       "      <th>identified_by_x</th>\n",
       "      <th>environmental_sample_x</th>\n",
       "      <th>metagenome_source_x</th>\n",
       "      <th>biomaterial_provider_x</th>\n",
       "      <th>biome_x</th>\n",
       "      <th>feature_x</th>\n",
       "      <th>material_x</th>\n",
       "      <th>culture-collection_x</th>\n",
       "      <th>altitude_x</th>\n",
       "      <th>isolate_name_alias_x</th>\n",
       "      <th>host_disease_x</th>\n",
       "      <th>rel_to_oxygen_x</th>\n",
       "      <th>Alias_x</th>\n",
       "      <th>ENA checklist_x</th>\n",
       "      <th>INSDC center alias_x</th>\n",
       "      <th>INSDC center name_x</th>\n",
       "      <th>INSDC first public_x</th>\n",
       "      <th>INSDC last update_x</th>\n",
       "      <th>INSDC status_x</th>\n",
       "      <th>SRA accession_x</th>\n",
       "      <th>Sample Name_x</th>\n",
       "      <th>Title_x</th>\n",
       "      <th>serotype</th>\n",
       "      <th>subsrc_note_x</th>\n",
       "      <th>annotation_method</th>\n",
       "      <th>assembly_method_x</th>\n",
       "      <th>biosamplemodel</th>\n",
       "      <th>geographic location (country and/or atlantic ocean)</th>\n",
       "      <th>geographic location (region and locality)</th>\n",
       "      <th>mapping_method_x</th>\n",
       "      <th>quality_assessment_method_x</th>\n",
       "      <th>refinement_method</th>\n",
       "      <th>Cell Shape_y</th>\n",
       "      <th>Funding Program_x</th>\n",
       "      <th>GOLD Stamp ID_x</th>\n",
       "      <th>Gene Calling Method_x</th>\n",
       "      <th>Gram Staining_x</th>\n",
       "      <th>Isolation Comments_y</th>\n",
       "      <th>Isolation Site_x</th>\n",
       "      <th>Motility_y</th>\n",
       "      <th>Temperature Optimum_x</th>\n",
       "      <th>Temperature Range_y</th>\n",
       "      <th>Type Strain_y</th>\n",
       "      <th>alt_elev_x</th>\n",
       "      <th>biotic_relationship_x</th>\n",
       "      <th>country_x</th>\n",
       "      <th>environment_x</th>\n",
       "      <th>investigation_type_x</th>\n",
       "      <th>isolation-source_x</th>\n",
       "      <th>trophic_level_x</th>\n",
       "      <th>pH_x</th>\n",
       "      <th>Body Sample Site_y</th>\n",
       "      <th>Body Sample SubSite_x</th>\n",
       "      <th>Diseases_x</th>\n",
       "      <th>Host Name_y</th>\n",
       "      <th>Host Health_y</th>\n",
       "      <th>Sporulation_y</th>\n",
       "      <th>experimental_factor</th>\n",
       "      <th>lat-lon</th>\n",
       "      <th>ENA-CHECKLIST</th>\n",
       "      <th>External Id</th>\n",
       "      <th>Submitter Id</th>\n",
       "      <th>subsrc-note</th>\n",
       "      <th>collection date_x</th>\n",
       "      <th>environment (biome)_x</th>\n",
       "      <th>environment (feature)_x</th>\n",
       "      <th>environment (material)_x</th>\n",
       "      <th>geographic location (country and/or sea)_x</th>\n",
       "      <th>geographic location (latitude)_x</th>\n",
       "      <th>geographic location (longitude)_x</th>\n",
       "      <th>host common name</th>\n",
       "      <th>host-associated environmental package</th>\n",
       "      <th>investigation type_x</th>\n",
       "      <th>project name_x</th>\n",
       "      <th>sequencing method_x</th>\n",
       "      <th>STRAIN</th>\n",
       "      <th>description</th>\n",
       "      <th>completeness</th>\n",
       "      <th>contamination</th>\n",
       "      <th>host_age_x</th>\n",
       "      <th>host_sex_x</th>\n",
       "      <th>oxy_stat_samp</th>\n",
       "      <th>samp_store_dur</th>\n",
       "      <th>samp_store_temp</th>\n",
       "      <th>FDA CVM ID_x</th>\n",
       "      <th>bio_material</th>\n",
       "      <th>common name</th>\n",
       "      <th>host scientific name_x</th>\n",
       "      <th>samp_mat_process</th>\n",
       "      <th>estimated_size_x</th>\n",
       "      <th>Tax ID</th>\n",
       "      <th>Collaborator ID</th>\n",
       "      <th>host_description</th>\n",
       "      <th>host_disease_outcome_x</th>\n",
       "      <th>host_disease_stage</th>\n",
       "      <th>host_health_state</th>\n",
       "      <th>identification method_x</th>\n",
       "      <th>strain_name_alias_x</th>\n",
       "      <th>WGA amplification approach</th>\n",
       "      <th>metagenomic source</th>\n",
       "      <th>sample derived from</th>\n",
       "      <th>single cell or viral particle lysis approach</th>\n",
       "      <th>sorting technology</th>\n",
       "      <th>taxonomic identity marker</th>\n",
       "      <th>passage_history</th>\n",
       "      <th>samp_size</th>\n",
       "      <th>Description</th>\n",
       "      <th>subsource_note</th>\n",
       "      <th>lab_host</th>\n",
       "      <th>collection_room</th>\n",
       "      <th>relative_week</th>\n",
       "      <th>assembly_tool</th>\n",
       "      <th>bin_id</th>\n",
       "      <th>Broker name</th>\n",
       "      <th>MAG_number</th>\n",
       "      <th>derived_from</th>\n",
       "      <th>alkalinity (carbonate/bicarbonate)</th>\n",
       "      <th>salinity</th>\n",
       "      <th>Phenotypes_x</th>\n",
       "      <th>env_package</th>\n",
       "      <th>specific_host_x</th>\n",
       "      <th>binning parameters</th>\n",
       "      <th>plant-associated environmental package_x</th>\n",
       "      <th>culture collection</th>\n",
       "      <th>repository</th>\n",
       "      <th>risk group</th>\n",
       "      <th>type status</th>\n",
       "      <th>Assembly Method_y</th>\n",
       "      <th>Genome Coverage</th>\n",
       "      <th>Sequencing Technology</th>\n",
       "      <th>nat-host_x</th>\n",
       "      <th>collection-date</th>\n",
       "      <th>Historical Monobactam Produced_x</th>\n",
       "      <th>Unique Monobactam Clusters Identified_x</th>\n",
       "      <th>extrachrom_elements</th>\n",
       "      <th>pathogenicity</th>\n",
       "      <th>source_material_id</th>\n",
       "      <th>genotype</th>\n",
       "      <th>mating_type</th>\n",
       "      <th>serovar_x</th>\n",
       "      <th>specimen_voucher</th>\n",
       "      <th>Salinity (PSU)</th>\n",
       "      <th>collected-by</th>\n",
       "      <th>orgmod_note</th>\n",
       "      <th>assembly_x</th>\n",
       "      <th>sequencing_meth_x</th>\n",
       "      <th>finishing_strategy_x</th>\n",
       "      <th>geo-loc-name_x</th>\n",
       "      <th>finishing strategy (depth of coverage)_x</th>\n",
       "      <th>host taxid_x</th>\n",
       "      <th>misc_param: HMP body site</th>\n",
       "      <th>misc_param: HMP supersite</th>\n",
       "      <th>nucleic acid extraction</th>\n",
       "      <th>project_type_x</th>\n",
       "      <th>sop_x</th>\n",
       "      <th>source_mat_id_x</th>\n",
       "      <th>Salinity_y</th>\n",
       "      <th>body_site</th>\n",
       "      <th>health_state</th>\n",
       "      <th>Isolation source_x</th>\n",
       "      <th>Host_x</th>\n",
       "      <th>isolation source_x</th>\n",
       "      <th>environmental package</th>\n",
       "      <th>specific host_x</th>\n",
       "      <th>lat lon</th>\n",
       "      <th>host_subject_id</th>\n",
       "      <th>pathotype</th>\n",
       "      <th>subgroup</th>\n",
       "      <th>subtype</th>\n",
       "      <th>Extraction Date</th>\n",
       "      <th>Extraction Method</th>\n",
       "      <th>Laboratory Host</th>\n",
       "      <th>Passage Date</th>\n",
       "      <th>sample type</th>\n",
       "      <th>host health state_x</th>\n",
       "      <th>anonymized_name_x</th>\n",
       "      <th>host_status</th>\n",
       "      <th>supplier_name_x</th>\n",
       "      <th>abs_air_humidity</th>\n",
       "      <th>air_temp</th>\n",
       "      <th>build_occup_type</th>\n",
       "      <th>building_setting</th>\n",
       "      <th>carb_dioxide</th>\n",
       "      <th>filter_type</th>\n",
       "      <th>heat_cool_type</th>\n",
       "      <th>indoor_space</th>\n",
       "      <th>light_type</th>\n",
       "      <th>occup_samp</th>\n",
       "      <th>occupant_dens_samp</th>\n",
       "      <th>organism_count</th>\n",
       "      <th>rel_air_humidity</th>\n",
       "      <th>space_typ_state</th>\n",
       "      <th>typ_occupant_dens</th>\n",
       "      <th>ventilation_type</th>\n",
       "      <th>Type-material</th>\n",
       "      <th>samp_collect_device</th>\n",
       "      <th>geographic location (altitude)</th>\n",
       "      <th>env_packatge</th>\n",
       "      <th>lib_reads_seqd</th>\n",
       "      <th>metagenomic_source</th>\n",
       "      <th>env material</th>\n",
       "      <th>other identifiers</th>\n",
       "      <th>assembly_method_version_x</th>\n",
       "      <th>mapping_method_version_x</th>\n",
       "      <th>quality_assessment_method_version_x</th>\n",
       "      <th>value_x</th>\n",
       "      <th>cyanobacterial_culture</th>\n",
       "      <th>missing</th>\n",
       "      <th>samp_store_loc</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>host_body_habitat</th>\n",
       "      <th>host_taxid</th>\n",
       "      <th>misc_param_x</th>\n",
       "      <th>ph</th>\n",
       "      <th>metagenome-source seawater metagenome</th>\n",
       "      <th>type_status</th>\n",
       "      <th>indoor_surf</th>\n",
       "      <th>samp_sort_meth</th>\n",
       "      <th>samp_vol_we_dna_ext</th>\n",
       "      <th>surf_material</th>\n",
       "      <th>BioSampleModel_x</th>\n",
       "      <th>ammonium</th>\n",
       "      <th>biomass</th>\n",
       "      <th>carb_nitro_ratio</th>\n",
       "      <th>chlorophyll</th>\n",
       "      <th>diss_inorg_nitro</th>\n",
       "      <th>diss_inorg_phosp</th>\n",
       "      <th>diss_org_carb</th>\n",
       "      <th>diss_org_nitro</th>\n",
       "      <th>nitrate</th>\n",
       "      <th>nitrite</th>\n",
       "      <th>phosphate</th>\n",
       "      <th>silicate</th>\n",
       "      <th>tidal_stage</th>\n",
       "      <th>tot_diss_nitro</th>\n",
       "      <th>locus_tag prefix</th>\n",
       "      <th>merge</th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>strain_y</th>\n",
       "      <th>finishing strategy (depth of coverage)_y</th>\n",
       "      <th>biotic_relationship_y</th>\n",
       "      <th>geo_loc_name_y</th>\n",
       "      <th>host_y</th>\n",
       "      <th>collected_by_y</th>\n",
       "      <th>host_disease_y</th>\n",
       "      <th>isolation_source_y</th>\n",
       "      <th>sample_name_y</th>\n",
       "      <th>isolation-source_y</th>\n",
       "      <th>sample_type_y</th>\n",
       "      <th>FDA CVM ID_y</th>\n",
       "      <th>culture_collection_y</th>\n",
       "      <th>assembly_method_y</th>\n",
       "      <th>collection_date_y</th>\n",
       "      <th>env_feature_y</th>\n",
       "      <th>synonym</th>\n",
       "      <th>Strain</th>\n",
       "      <th>SUBJECT_ID_y</th>\n",
       "      <th>Alias_y</th>\n",
       "      <th>type-material_y</th>\n",
       "      <th>misc_param_y</th>\n",
       "      <th>collection date_y</th>\n",
       "      <th>Isolation Site_y</th>\n",
       "      <th>Isolation source_y</th>\n",
       "      <th>country_y</th>\n",
       "      <th>note_y</th>\n",
       "      <th>GOLD Stamp ID_y</th>\n",
       "      <th>investigation_type_y</th>\n",
       "      <th>culture-collection_y</th>\n",
       "      <th>lat_lon_y</th>\n",
       "      <th>alt_elev_y</th>\n",
       "      <th>locus_tag_prefix_y</th>\n",
       "      <th>nat-host_y</th>\n",
       "      <th>assembly_method_version_y</th>\n",
       "      <th>env_material_y</th>\n",
       "      <th>model</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>Sample Description</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>ENA checklist_y</th>\n",
       "      <th>project_type_y</th>\n",
       "      <th>estimated_size_y</th>\n",
       "      <th>Temperature Optimum_y</th>\n",
       "      <th>isol_growth_condt_y</th>\n",
       "      <th>Motility</th>\n",
       "      <th>feature_y</th>\n",
       "      <th>project_name_y</th>\n",
       "      <th>assembly_y</th>\n",
       "      <th>num_replicons_y</th>\n",
       "      <th>isolate</th>\n",
       "      <th>depth_y</th>\n",
       "      <th>env_biome_y</th>\n",
       "      <th>altitude_y</th>\n",
       "      <th>ref_biomaterial_y</th>\n",
       "      <th>package</th>\n",
       "      <th>geographic location</th>\n",
       "      <th>ArrayExpress-StrainOrLine</th>\n",
       "      <th>ArrayExpress-Phenotype</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>INSDC center alias_y</th>\n",
       "      <th>geo-loc-name_y</th>\n",
       "      <th>environment_y</th>\n",
       "      <th>host_sex_y</th>\n",
       "      <th>elev_y</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>completeness_estimated_y</th>\n",
       "      <th>geographic location (country and/or sea)_y</th>\n",
       "      <th>latitude and longitude</th>\n",
       "      <th>specific_host_y</th>\n",
       "      <th>ArrayExpress-Species</th>\n",
       "      <th>ENA-FIRST-PUBLIC_y</th>\n",
       "      <th>INSDC center name_y</th>\n",
       "      <th>specific host_y</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>pH_y</th>\n",
       "      <th>biome_y</th>\n",
       "      <th>biomaterial_provider_y</th>\n",
       "      <th>isolate_name_alias_y</th>\n",
       "      <th>contamination_estimated_y</th>\n",
       "      <th>host scientific name_y</th>\n",
       "      <th>anonymized_name_y</th>\n",
       "      <th>ENA-LAST-UPDATE_y</th>\n",
       "      <th>INSDC first public_y</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>BioSampleModel_y</th>\n",
       "      <th>environmental_sample_y</th>\n",
       "      <th>material_y</th>\n",
       "      <th>identified_by_y</th>\n",
       "      <th>host health state_y</th>\n",
       "      <th>INSDC last update_y</th>\n",
       "      <th>Diseases_y</th>\n",
       "      <th>strain_name_alias_y</th>\n",
       "      <th>genome_coverage_y</th>\n",
       "      <th>INSDC status_y</th>\n",
       "      <th>host taxid_y</th>\n",
       "      <th>isolation source_y</th>\n",
       "      <th>Gram Staining_y</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>sequencing_meth_y</th>\n",
       "      <th>Historical Monobactam Produced_y</th>\n",
       "      <th>identification method_y</th>\n",
       "      <th>supplier_name_y</th>\n",
       "      <th>SRA accession_y</th>\n",
       "      <th>host_disease_outcome_y</th>\n",
       "      <th>Unique Monobactam Clusters Identified_y</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>Sample Name_y</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Phenotypes_y</th>\n",
       "      <th>Gene Calling Method_y</th>\n",
       "      <th>temp_y</th>\n",
       "      <th>Title_y</th>\n",
       "      <th>investigation type_y</th>\n",
       "      <th>Host_y</th>\n",
       "      <th>rel_to_oxygen_y</th>\n",
       "      <th>host_age_y</th>\n",
       "      <th>Species</th>\n",
       "      <th>mapping_method_y</th>\n",
       "      <th>serovar_y</th>\n",
       "      <th>sop_y</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>finishing_strategy_y</th>\n",
       "      <th>Genus</th>\n",
       "      <th>mapping_method_version_y</th>\n",
       "      <th>environment (biome)_y</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>metagenome_source_y</th>\n",
       "      <th>environment (feature)_y</th>\n",
       "      <th>trophic_level_y</th>\n",
       "      <th>metagenomic_y</th>\n",
       "      <th>environment (material)_y</th>\n",
       "      <th>Funding Program_y</th>\n",
       "      <th>quality_assessment_method_y</th>\n",
       "      <th>quality_assessment_method_version_y</th>\n",
       "      <th>geographic location (latitude)_y</th>\n",
       "      <th>source_mat_id_y</th>\n",
       "      <th>host_tissue_sampled_y</th>\n",
       "      <th>geographic location (longitude)_y</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>subsrc_note_y</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>value_y</th>\n",
       "      <th>plant-associated environmental package_y</th>\n",
       "      <th>project name_y</th>\n",
       "      <th>sequencing method_y</th>\n",
       "      <th>Body Sample SubSite_y</th>\n",
       "      <th>merge2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>NaN</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>986.0</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>strain=UW101; ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>Major</td>\n",
       "      <td>Full</td>\n",
       "      <td>2007/05/01</td>\n",
       "      <td>ASM1664v1</td>\n",
       "      <td>US DOE Joint Genome Institute</td>\n",
       "      <td>GCF_000016645.1</td>\n",
       "      <td>identical</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>Complete</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>Yes</td>\n",
       "      <td>2007-05-01T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NC_009441</td>\n",
       "      <td>DOE Joint Genome Institute</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>6096872.0</td>\n",
       "      <td>34.1</td>\n",
       "      <td>5017.0</td>\n",
       "      <td>soil</td>\n",
       "      <td>isolated from soil in England and is the type ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>United Kingdom: England</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-</td>\n",
       "      <td>Rod</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>20-30</td>\n",
       "      <td>Non-halophilic</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>Multiple</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae ATCC 17061. Flavobac...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Sample from Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>2014-01-28T00:00:00.000</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>UW101; ATCC 17061</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Flavobacterium johnsoniae</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>both</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>left_only</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>1018.0</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>strain=DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>Major</td>\n",
       "      <td>Full</td>\n",
       "      <td>2009/08/26</td>\n",
       "      <td>ASM2328v1</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)</td>\n",
       "      <td>GCF_000023285.1</td>\n",
       "      <td>identical</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>Complete</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>Yes</td>\n",
       "      <td>2009-04-30T00:00:00Z</td>\n",
       "      <td>21304645</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>SRR013476,SRR013477</td>\n",
       "      <td>CP001632</td>\n",
       "      <td>NC_013162</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2612925.0</td>\n",
       "      <td>39.6</td>\n",
       "      <td>2171.0</td>\n",
       "      <td>the human oral cavity</td>\n",
       "      <td>isolated from the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>-</td>\n",
       "      <td>Rod</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>35-37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Facultative</td>\n",
       "      <td>Host-associated</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271.  Capnocytoph...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic sample from Capnocytophaga ochracea DS...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>2009-02-19T00:00:00.000</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>both</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Genome_ID # assembly_accession  bioproject     biosample wgs_master  \\\n",
       "0  GCA_000016645.1      GCA_000016645.1  PRJNA16082  SAMN02598357        NaN   \n",
       "1  GCA_000023285.1      GCA_000023285.1  PRJNA29403  SAMN00001911        NaN   \n",
       "\n",
       "         refseq_category     taxid  species_taxid  \\\n",
       "0  representative genome  376686.0          986.0   \n",
       "1  representative genome  521097.0         1018.0   \n",
       "\n",
       "                      organism_name        infraspecific_name isolate_x  \\\n",
       "0   Flavobacterium johnsoniae UW101  strain=UW101; ATCC 17061       NaN   \n",
       "1  Capnocytophaga ochracea DSM 7271           strain=DSM 7271       NaN   \n",
       "\n",
       "  version_status   assembly_level release_type genome_rep seq_rel_date  \\\n",
       "0         latest  Complete Genome        Major       Full   2007/05/01   \n",
       "1         latest  Complete Genome        Major       Full   2009/08/26   \n",
       "\n",
       "    asm_name                              submitter_x  gbrs_paired_asm  \\\n",
       "0  ASM1664v1            US DOE Joint Genome Institute  GCF_000016645.1   \n",
       "1  ASM2328v1  US DOE Joint Genome Institute (JGI-PGF)  GCF_000023285.1   \n",
       "\n",
       "  paired_asm_comp                                           ftp_path  \\\n",
       "0       identical  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "1       identical  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "\n",
       "  excluded_from_refseq    relation_to_type_material  \\\n",
       "0                  NaN  assembly from type material   \n",
       "1                  NaN  assembly from type material   \n",
       "\n",
       "                        Genome Name                     Organism Name  \\\n",
       "0   Flavobacterium johnsoniae UW101   Flavobacterium johnsoniae UW101   \n",
       "1  Capnocytophaga ochracea DSM 7271  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "   NCBI Taxon ID Genome Status    Strain_x Serovar Pathovar MLST Other Typing  \\\n",
       "0       376686.0      Complete  ATCC 17061     NaN      NaN  NaN          NaN   \n",
       "1       521097.0      Complete    DSM 7271     NaN      NaN  NaN          NaN   \n",
       "\n",
       "  Culture Collection Type Strain_x       Completion Date Publication  \\\n",
       "0         ATCC 17061           Yes  2007-05-01T00:00:00Z         NaN   \n",
       "1           DSM 7271           Yes  2009-04-30T00:00:00Z    21304645   \n",
       "\n",
       "  BioProject Accession BioSample Accession Assembly Accession  \\\n",
       "0           PRJNA16082        SAMN02598357    GCA_000016645.1   \n",
       "1           PRJNA29403        SAMN00001911    GCA_000023285.1   \n",
       "\n",
       "         SRA Accession GenBank Accessions RefSeq Accessions  \\\n",
       "0                  NaN           CP000685         NC_009441   \n",
       "1  SRR013476,SRR013477           CP001632         NC_013162   \n",
       "\n",
       "                                  Sequencing Centers Sequencing Status  \\\n",
       "0                         DOE Joint Genome Institute          complete   \n",
       "1  US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...          complete   \n",
       "\n",
       "  Sequencing Platform Sequencing Depth Assembly Method_x  Chromosomes  \\\n",
       "0                 NaN              NaN               NaN          1.0   \n",
       "1                 NaN              NaN               NaN          1.0   \n",
       "\n",
       "   Plasmids  Contigs  Genome Length  GC Content  RefSeq CDS  \\\n",
       "0       NaN      1.0      6096872.0        34.1      5017.0   \n",
       "1       NaN      1.0      2612925.0        39.6      2171.0   \n",
       "\n",
       "        Isolation Source                               Isolation Comments_x  \\\n",
       "0                   soil  isolated from soil in England and is the type ...   \n",
       "1  the human oral cavity                isolated from the human oral cavity   \n",
       "\n",
       "  Collection Date Isolation Country      Geographic Location  Latitude  \\\n",
       "0             NaN    United Kingdom  United Kingdom: England       NaN   \n",
       "1             NaN               NaN                      NaN       NaN   \n",
       "\n",
       "   Longitude Altitude Depth_x Other Environmental          Host Name_x  \\\n",
       "0        NaN      NaN     NaN                 NaN                  NaN   \n",
       "1        NaN      NaN     NaN                 NaN  Human, Homo sapiens   \n",
       "\n",
       "  Host Gender Host Age Host Health_x Body Sample Site_x Other Clinical  \\\n",
       "0         NaN      NaN           NaN                NaN            NaN   \n",
       "1         NaN      NaN           NaN                NaN            NaN   \n",
       "\n",
       "  Gram Stain Cell Shape_x Motility_x Sporulation_x Temperature Range_x  \\\n",
       "0          -          Rod        Yes            No          Mesophilic   \n",
       "1          -          Rod        Yes            No          Mesophilic   \n",
       "\n",
       "  Optimal Temperature      Salinity_x Oxygen Requirement          Habitat  \\\n",
       "0               20-30  Non-halophilic            Aerobic         Multiple   \n",
       "1               35-37             NaN        Facultative  Host-associated   \n",
       "\n",
       "  Disease                                           Comments  \\\n",
       "0     NaN  Flavobacterium johnsoniae ATCC 17061. Flavobac...   \n",
       "1     NaN  Capnocytophaga ochracea DSM 7271.  Capnocytoph...   \n",
       "\n",
       "  Additional Metadata Genome Quality Flags  \\\n",
       "0                 NaN                  NaN   \n",
       "1                 NaN                  NaN   \n",
       "\n",
       "                                     biosample_title contact_email first_name  \\\n",
       "0        Sample from Flavobacterium johnsoniae UW101           NaN        NaN   \n",
       "1  Generic sample from Capnocytophaga ochracea DS...           NaN        NaN   \n",
       "\n",
       "  last_name                                               name  \\\n",
       "0       NaN                                               NCBI   \n",
       "1       NaN  Joint Genome Institute, U.S. Department of Energy   \n",
       "\n",
       "                           organism         publication_date sample_accession  \\\n",
       "0   Flavobacterium johnsoniae UW101  2014-01-28T00:00:00.000     SAMN02598357   \n",
       "1  Capnocytophaga ochracea DSM 7271  2009-02-19T00:00:00.000     SAMN00001911   \n",
       "\n",
       "            strain_x submission_model submission_package  \\\n",
       "0  UW101; ATCC 17061          Generic        Generic.1.0   \n",
       "1           DSM 7271          Generic        Generic.1.0   \n",
       "\n",
       "  submission_package_name                                        submitter_y  \\\n",
       "0                 Generic                                               NCBI   \n",
       "1                 Generic  Joint Genome Institute, U.S. Department of Energy   \n",
       "\n",
       "   taxonomy_id collection_date_x culture_collection_x geo_loc_name_x  \\\n",
       "0     376686.0               NaN                  NaN            NaN   \n",
       "1     521097.0               NaN                  NaN            NaN   \n",
       "\n",
       "  isolation_source_x sample_name_x sample_type_x  \\\n",
       "0                NaN      CP000685           NaN   \n",
       "1                NaN          FWCB           NaN   \n",
       "\n",
       "                            type-material_x                 comment_paragraph  \\\n",
       "0  type strain of Flavobacterium johnsoniae                               NaN   \n",
       "1    type strain of Capnocytophaga ochracea  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "  depth_x derived-from env_broad_scale env_local_scale env_medium  \\\n",
       "0     NaN          NaN             NaN             NaN        NaN   \n",
       "1     NaN          NaN             NaN             NaN        NaN   \n",
       "\n",
       "  environmental-sample isolate_y lat_lon_x metagenome-source metagenomic_x  \\\n",
       "0                  NaN       NaN       NaN               NaN           NaN   \n",
       "1                  NaN       NaN       NaN               NaN           NaN   \n",
       "\n",
       "  sample-type elev_x environmetal-sample collected_by_x host_x  \\\n",
       "0         NaN    NaN                 NaN            NaN    NaN   \n",
       "1         NaN    NaN                 NaN            NaN    NaN   \n",
       "\n",
       "  host_tissue_sampled_x temp_x env_biome_x env_feature_x env_material_x  \\\n",
       "0                   NaN    NaN         NaN           NaN            NaN   \n",
       "1                   NaN    NaN         NaN           NaN            NaN   \n",
       "\n",
       "  isol_growth_condt_x locus_tag_prefix_x num_replicons_x project_name_x  \\\n",
       "0                 NaN                NaN             NaN            NaN   \n",
       "1                 NaN                NaN             NaN            NaN   \n",
       "\n",
       "  ref_biomaterial_x 16S recovered 16S recovery software analysis project type  \\\n",
       "0               NaN           NaN                   NaN                   NaN   \n",
       "1               NaN           NaN                   NaN                   NaN   \n",
       "\n",
       "  assembly quality assembly software bin parameters binning software  \\\n",
       "0              NaN               NaN            NaN              NaN   \n",
       "1              NaN               NaN            NaN              NaN   \n",
       "\n",
       "   completeness score completeness software  contamination score  contig L50  \\\n",
       "0                 NaN                   NaN                  NaN         NaN   \n",
       "1                 NaN                   NaN                  NaN         NaN   \n",
       "\n",
       "  decontamination software note_x  number of contigs reassembly post binning  \\\n",
       "0                      NaN    NaN                NaN                     NaN   \n",
       "1                      NaN    NaN                NaN                     NaN   \n",
       "\n",
       "   relative coverage Anaerobic (SRR10097246)  \\\n",
       "0                                        NaN   \n",
       "1                                        NaN   \n",
       "\n",
       "   relative coverage Anoxic (SRR10375098)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Infiltration (SRR10097247)  \\\n",
       "0                                           NaN   \n",
       "1                                           NaN   \n",
       "\n",
       "   relative coverage Low pH (SRR10097245)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Oxic (SRR10375099)  \\\n",
       "0                                   NaN   \n",
       "1                                   NaN   \n",
       "\n",
       "   relative coverage Reference (SRR10375097) taxa id  total assembly size  \\\n",
       "0                                        NaN     NaN                  NaN   \n",
       "1                                        NaN     NaN                  NaN   \n",
       "\n",
       "  nat_host assembly_method_and_version completeness_estimated_x  \\\n",
       "0      NaN                         NaN                      NaN   \n",
       "1      NaN                         NaN                      NaN   \n",
       "\n",
       "  contamination_estimated_x genome_coverage_x mapping_method_and_version  \\\n",
       "0                       NaN               NaN                        NaN   \n",
       "1                       NaN               NaN                        NaN   \n",
       "\n",
       "  quality_assessment_method_and_version ENA-FIRST-PUBLIC_x ENA-LAST-UPDATE_x  \\\n",
       "0                                   NaN                NaN               NaN   \n",
       "1                                   NaN                NaN               NaN   \n",
       "\n",
       "  SUBJECT_ID_x contact_lab identified_by_x environmental_sample_x  \\\n",
       "0          NaN         NaN             NaN                    NaN   \n",
       "1          NaN         NaN             NaN                    NaN   \n",
       "\n",
       "  metagenome_source_x biomaterial_provider_x biome_x feature_x material_x  \\\n",
       "0                 NaN                    NaN     NaN       NaN        NaN   \n",
       "1                 NaN                    NaN     NaN       NaN        NaN   \n",
       "\n",
       "  culture-collection_x altitude_x isolate_name_alias_x host_disease_x  \\\n",
       "0                  NaN        NaN                  NaN            NaN   \n",
       "1                  NaN        NaN                  NaN            NaN   \n",
       "\n",
       "  rel_to_oxygen_x Alias_x ENA checklist_x INSDC center alias_x  \\\n",
       "0             NaN     NaN             NaN                  NaN   \n",
       "1             NaN     NaN             NaN                  NaN   \n",
       "\n",
       "  INSDC center name_x INSDC first public_x INSDC last update_x INSDC status_x  \\\n",
       "0                 NaN                  NaN                 NaN            NaN   \n",
       "1                 NaN                  NaN                 NaN            NaN   \n",
       "\n",
       "  SRA accession_x Sample Name_x Title_x serotype subsrc_note_x  \\\n",
       "0             NaN           NaN     NaN      NaN           NaN   \n",
       "1             NaN           NaN     NaN      NaN           NaN   \n",
       "\n",
       "  annotation_method assembly_method_x biosamplemodel  \\\n",
       "0               NaN               NaN            NaN   \n",
       "1               NaN               NaN            NaN   \n",
       "\n",
       "  geographic location (country and/or atlantic ocean)  \\\n",
       "0                                                NaN    \n",
       "1                                                NaN    \n",
       "\n",
       "  geographic location (region and locality) mapping_method_x  \\\n",
       "0                                       NaN              NaN   \n",
       "1                                       NaN              NaN   \n",
       "\n",
       "  quality_assessment_method_x refinement_method Cell Shape_y  \\\n",
       "0                         NaN               NaN          NaN   \n",
       "1                         NaN               NaN          NaN   \n",
       "\n",
       "  Funding Program_x GOLD Stamp ID_x Gene Calling Method_x Gram Staining_x  \\\n",
       "0               NaN             NaN                   NaN             NaN   \n",
       "1               NaN             NaN                   NaN             NaN   \n",
       "\n",
       "  Isolation Comments_y Isolation Site_x Motility_y Temperature Optimum_x  \\\n",
       "0                  NaN              NaN        NaN                   NaN   \n",
       "1                  NaN              NaN        NaN                   NaN   \n",
       "\n",
       "  Temperature Range_y Type Strain_y alt_elev_x biotic_relationship_x  \\\n",
       "0                 NaN           NaN        NaN                   NaN   \n",
       "1                 NaN           NaN        NaN                   NaN   \n",
       "\n",
       "  country_x environment_x investigation_type_x isolation-source_x  \\\n",
       "0       NaN           NaN                  NaN                NaN   \n",
       "1       NaN           NaN                  NaN                NaN   \n",
       "\n",
       "  trophic_level_x pH_x Body Sample Site_y Body Sample SubSite_x Diseases_x  \\\n",
       "0             NaN  NaN                NaN                   NaN        NaN   \n",
       "1             NaN  NaN                NaN                   NaN        NaN   \n",
       "\n",
       "  Host Name_y Host Health_y Sporulation_y experimental_factor lat-lon  \\\n",
       "0         NaN           NaN           NaN                 NaN     NaN   \n",
       "1         NaN           NaN           NaN                 NaN     NaN   \n",
       "\n",
       "  ENA-CHECKLIST External Id Submitter Id subsrc-note collection date_x  \\\n",
       "0           NaN         NaN          NaN         NaN               NaN   \n",
       "1           NaN         NaN          NaN         NaN               NaN   \n",
       "\n",
       "  environment (biome)_x environment (feature)_x environment (material)_x  \\\n",
       "0                   NaN                     NaN                      NaN   \n",
       "1                   NaN                     NaN                      NaN   \n",
       "\n",
       "  geographic location (country and/or sea)_x geographic location (latitude)_x  \\\n",
       "0                                        NaN                              NaN   \n",
       "1                                        NaN                              NaN   \n",
       "\n",
       "  geographic location (longitude)_x host common name  \\\n",
       "0                               NaN              NaN   \n",
       "1                               NaN              NaN   \n",
       "\n",
       "  host-associated environmental package investigation type_x project name_x  \\\n",
       "0                                   NaN                  NaN            NaN   \n",
       "1                                   NaN                  NaN            NaN   \n",
       "\n",
       "  sequencing method_x STRAIN description  completeness  contamination  \\\n",
       "0                 NaN    NaN         NaN           NaN            NaN   \n",
       "1                 NaN    NaN         NaN           NaN            NaN   \n",
       "\n",
       "  host_age_x host_sex_x oxy_stat_samp samp_store_dur samp_store_temp  \\\n",
       "0        NaN        NaN           NaN            NaN             NaN   \n",
       "1        NaN        NaN           NaN            NaN             NaN   \n",
       "\n",
       "  FDA CVM ID_x bio_material common name host scientific name_x  \\\n",
       "0          NaN          NaN         NaN                    NaN   \n",
       "1          NaN          NaN         NaN                    NaN   \n",
       "\n",
       "  samp_mat_process estimated_size_x  Tax ID Collaborator ID host_description  \\\n",
       "0              NaN              NaN     NaN             NaN              NaN   \n",
       "1              NaN              NaN     NaN             NaN              NaN   \n",
       "\n",
       "  host_disease_outcome_x host_disease_stage host_health_state  \\\n",
       "0                    NaN                NaN               NaN   \n",
       "1                    NaN                NaN               NaN   \n",
       "\n",
       "  identification method_x strain_name_alias_x WGA amplification approach  \\\n",
       "0                     NaN                 NaN                        NaN   \n",
       "1                     NaN                 NaN                        NaN   \n",
       "\n",
       "  metagenomic source sample derived from  \\\n",
       "0                NaN                 NaN   \n",
       "1                NaN                 NaN   \n",
       "\n",
       "  single cell or viral particle lysis approach sorting technology  \\\n",
       "0                                          NaN                NaN   \n",
       "1                                          NaN                NaN   \n",
       "\n",
       "  taxonomic identity marker passage_history samp_size Description  \\\n",
       "0                       NaN             NaN       NaN         NaN   \n",
       "1                       NaN             NaN       NaN         NaN   \n",
       "\n",
       "  subsource_note lab_host collection_room  relative_week assembly_tool bin_id  \\\n",
       "0            NaN      NaN             NaN            NaN           NaN    NaN   \n",
       "1            NaN      NaN             NaN            NaN           NaN    NaN   \n",
       "\n",
       "  Broker name  MAG_number derived_from alkalinity (carbonate/bicarbonate)  \\\n",
       "0         NaN         NaN          NaN                                NaN   \n",
       "1         NaN         NaN          NaN                                NaN   \n",
       "\n",
       "  salinity Phenotypes_x env_package specific_host_x binning parameters  \\\n",
       "0      NaN          NaN         NaN             NaN                NaN   \n",
       "1      NaN          NaN         NaN             NaN                NaN   \n",
       "\n",
       "  plant-associated environmental package_x culture collection repository  \\\n",
       "0                                      NaN                NaN        NaN   \n",
       "1                                      NaN                NaN        NaN   \n",
       "\n",
       "  risk group type status Assembly Method_y Genome Coverage  \\\n",
       "0        NaN         NaN               NaN             NaN   \n",
       "1        NaN         NaN               NaN             NaN   \n",
       "\n",
       "   Sequencing Technology nat-host_x collection-date  \\\n",
       "0                    NaN        NaN             NaN   \n",
       "1                    NaN        NaN             NaN   \n",
       "\n",
       "  Historical Monobactam Produced_x Unique Monobactam Clusters Identified_x  \\\n",
       "0                              NaN                                     NaN   \n",
       "1                              NaN                                     NaN   \n",
       "\n",
       "   extrachrom_elements pathogenicity source_material_id genotype mating_type  \\\n",
       "0                  NaN           NaN                NaN      NaN         NaN   \n",
       "1                  NaN           NaN                NaN      NaN         NaN   \n",
       "\n",
       "  serovar_x specimen_voucher  Salinity (PSU) collected-by orgmod_note  \\\n",
       "0       NaN              NaN             NaN          NaN         NaN   \n",
       "1       NaN              NaN             NaN          NaN         NaN   \n",
       "\n",
       "  assembly_x sequencing_meth_x finishing_strategy_x geo-loc-name_x  \\\n",
       "0        NaN               NaN                  NaN            NaN   \n",
       "1        NaN               NaN                  NaN            NaN   \n",
       "\n",
       "  finishing strategy (depth of coverage)_x  host taxid_x  \\\n",
       "0                                      NaN           NaN   \n",
       "1                                      NaN           NaN   \n",
       "\n",
       "  misc_param: HMP body site misc_param: HMP supersite nucleic acid extraction  \\\n",
       "0                       NaN                       NaN                     NaN   \n",
       "1                       NaN                       NaN                     NaN   \n",
       "\n",
       "  project_type_x sop_x source_mat_id_x Salinity_y body_site health_state  \\\n",
       "0            NaN   NaN             NaN        NaN       NaN          NaN   \n",
       "1            NaN   NaN             NaN        NaN       NaN          NaN   \n",
       "\n",
       "  Isolation source_x Host_x isolation source_x environmental package  \\\n",
       "0                NaN    NaN                NaN                   NaN   \n",
       "1                NaN    NaN                NaN                   NaN   \n",
       "\n",
       "  specific host_x lat lon host_subject_id pathotype subgroup subtype  \\\n",
       "0             NaN     NaN             NaN       NaN      NaN     NaN   \n",
       "1             NaN     NaN             NaN       NaN      NaN     NaN   \n",
       "\n",
       "  Extraction Date Extraction Method Laboratory Host Passage Date sample type  \\\n",
       "0             NaN               NaN             NaN          NaN         NaN   \n",
       "1             NaN               NaN             NaN          NaN         NaN   \n",
       "\n",
       "  host health state_x anonymized_name_x host_status supplier_name_x  \\\n",
       "0                 NaN               NaN         NaN             NaN   \n",
       "1                 NaN               NaN         NaN             NaN   \n",
       "\n",
       "  abs_air_humidity  air_temp build_occup_type building_setting carb_dioxide  \\\n",
       "0              NaN       NaN              NaN              NaN          NaN   \n",
       "1              NaN       NaN              NaN              NaN          NaN   \n",
       "\n",
       "  filter_type heat_cool_type indoor_space light_type occup_samp  \\\n",
       "0         NaN            NaN          NaN        NaN        NaN   \n",
       "1         NaN            NaN          NaN        NaN        NaN   \n",
       "\n",
       "  occupant_dens_samp organism_count rel_air_humidity space_typ_state  \\\n",
       "0                NaN            NaN              NaN             NaN   \n",
       "1                NaN            NaN              NaN             NaN   \n",
       "\n",
       "  typ_occupant_dens ventilation_type Type-material samp_collect_device  \\\n",
       "0               NaN              NaN           NaN                 NaN   \n",
       "1               NaN              NaN           NaN                 NaN   \n",
       "\n",
       "  geographic location (altitude) env_packatge lib_reads_seqd  \\\n",
       "0                            NaN          NaN            NaN   \n",
       "1                            NaN          NaN            NaN   \n",
       "\n",
       "  metagenomic_source env material other identifiers assembly_method_version_x  \\\n",
       "0                NaN          NaN               NaN                       NaN   \n",
       "1                NaN          NaN               NaN                       NaN   \n",
       "\n",
       "  mapping_method_version_x quality_assessment_method_version_x value_x  \\\n",
       "0                      NaN                                 NaN     NaN   \n",
       "1                      NaN                                 NaN     NaN   \n",
       "\n",
       "  cyanobacterial_culture missing samp_store_loc ethnicity host_body_habitat  \\\n",
       "0                    NaN     NaN            NaN       NaN               NaN   \n",
       "1                    NaN     NaN            NaN       NaN               NaN   \n",
       "\n",
       "   host_taxid misc_param_x  ph metagenome-source seawater metagenome  \\\n",
       "0         NaN          NaN NaN                                   NaN   \n",
       "1         NaN          NaN NaN                                   NaN   \n",
       "\n",
       "  type_status indoor_surf samp_sort_meth samp_vol_we_dna_ext surf_material  \\\n",
       "0         NaN         NaN            NaN                 NaN           NaN   \n",
       "1         NaN         NaN            NaN                 NaN           NaN   \n",
       "\n",
       "  BioSampleModel_x ammonium biomass  carb_nitro_ratio chlorophyll  \\\n",
       "0              NaN      NaN     NaN               NaN         NaN   \n",
       "1              NaN      NaN     NaN               NaN         NaN   \n",
       "\n",
       "  diss_inorg_nitro diss_inorg_phosp diss_org_carb diss_org_nitro nitrate  \\\n",
       "0              NaN              NaN           NaN            NaN     NaN   \n",
       "1              NaN              NaN           NaN            NaN     NaN   \n",
       "\n",
       "  nitrite phosphate silicate tidal_stage tot_diss_nitro locus_tag prefix  \\\n",
       "0     NaN       NaN      NaN         NaN            NaN              NaN   \n",
       "1     NaN       NaN      NaN         NaN            NaN              NaN   \n",
       "\n",
       "  merge assembly_accession  strain_y finishing strategy (depth of coverage)_y  \\\n",
       "0  both                NaN       NaN                                      NaN   \n",
       "1  both    GCA_000023285.1  DSM 7271                                      NaN   \n",
       "\n",
       "  biotic_relationship_y geo_loc_name_y host_y collected_by_y host_disease_y  \\\n",
       "0                   NaN            NaN    NaN            NaN            NaN   \n",
       "1                   NaN            NaN    NaN            NaN            NaN   \n",
       "\n",
       "  isolation_source_y sample_name_y isolation-source_y sample_type_y  \\\n",
       "0                NaN           NaN                NaN           NaN   \n",
       "1                NaN           NaN                NaN           NaN   \n",
       "\n",
       "  FDA CVM ID_y culture_collection_y assembly_method_y collection_date_y  \\\n",
       "0          NaN                  NaN               NaN               NaN   \n",
       "1          NaN                  NaN               NaN               NaN   \n",
       "\n",
       "  env_feature_y synonym Strain SUBJECT_ID_y Alias_y  \\\n",
       "0           NaN     NaN    NaN          NaN     NaN   \n",
       "1           NaN     NaN    NaN          NaN     NaN   \n",
       "\n",
       "                          type-material_y misc_param_y collection date_y  \\\n",
       "0                                     NaN          NaN               NaN   \n",
       "1  type strain of Capnocytophaga ochracea          NaN               NaN   \n",
       "\n",
       "  Isolation Site_y Isolation source_y country_y note_y GOLD Stamp ID_y  \\\n",
       "0              NaN                NaN       NaN    NaN             NaN   \n",
       "1              NaN                NaN       NaN    NaN             NaN   \n",
       "\n",
       "  investigation_type_y culture-collection_y lat_lon_y alt_elev_y  \\\n",
       "0                  NaN                  NaN       NaN        NaN   \n",
       "1                  NaN                  NaN       NaN        NaN   \n",
       "\n",
       "  locus_tag_prefix_y nat-host_y assembly_method_version_y env_material_y  \\\n",
       "0                NaN        NaN                       NaN            NaN   \n",
       "1                NaN        NaN                       NaN            NaN   \n",
       "\n",
       "  model  sample_description  Sample Description ArrayExpress-STRAIN_OR_LINE  \\\n",
       "0   NaN                 NaN                 NaN                         NaN   \n",
       "1   NaN                 NaN                 NaN                         NaN   \n",
       "\n",
       "  ENA checklist_y project_type_y  estimated_size_y Temperature Optimum_y  \\\n",
       "0             NaN            NaN               NaN                   NaN   \n",
       "1             NaN            NaN               NaN                   NaN   \n",
       "\n",
       "   isol_growth_condt_y Motility feature_y project_name_y assembly_y  \\\n",
       "0                  NaN      NaN       NaN            NaN        NaN   \n",
       "1                  NaN      NaN       NaN            NaN        NaN   \n",
       "\n",
       "   num_replicons_y isolate depth_y env_biome_y altitude_y ref_biomaterial_y  \\\n",
       "0              NaN     NaN     NaN         NaN        NaN               NaN   \n",
       "1              NaN     NaN     NaN         NaN        NaN               NaN   \n",
       "\n",
       "  package geographic location ArrayExpress-StrainOrLine  \\\n",
       "0     NaN                 NaN                       NaN   \n",
       "1     NaN                 NaN                       NaN   \n",
       "\n",
       "   ArrayExpress-Phenotype ArrayExpress-SPECIES INSDC center alias_y  \\\n",
       "0                     NaN                  NaN                  NaN   \n",
       "1                     NaN                  NaN                  NaN   \n",
       "\n",
       "  geo-loc-name_y environment_y host_sex_y  elev_y PublicAccession  \\\n",
       "0            NaN           NaN        NaN     NaN             NaN   \n",
       "1            NaN           NaN        NaN     NaN             NaN   \n",
       "\n",
       "  completeness_estimated_y geographic location (country and/or sea)_y  \\\n",
       "0                      NaN                                        NaN   \n",
       "1                      NaN                                        NaN   \n",
       "\n",
       "  latitude and longitude specific_host_y ArrayExpress-Species  \\\n",
       "0                    NaN             NaN                  NaN   \n",
       "1                    NaN             NaN                  NaN   \n",
       "\n",
       "  ENA-FIRST-PUBLIC_y INSDC center name_y specific host_y Body Sample Site  \\\n",
       "0                NaN                 NaN             NaN              NaN   \n",
       "1                NaN                 NaN             NaN              NaN   \n",
       "\n",
       "  pH_y biome_y biomaterial_provider_y isolate_name_alias_y  \\\n",
       "0  NaN     NaN                    NaN                  NaN   \n",
       "1  NaN     NaN                    NaN                  NaN   \n",
       "\n",
       "  contamination_estimated_y host scientific name_y  anonymized_name_y  \\\n",
       "0                       NaN                    NaN                NaN   \n",
       "1                       NaN                    NaN                NaN   \n",
       "\n",
       "  ENA-LAST-UPDATE_y INSDC first public_y Cell Shape BioSampleModel_y  \\\n",
       "0               NaN                  NaN        NaN              NaN   \n",
       "1               NaN                  NaN        NaN              NaN   \n",
       "\n",
       "  environmental_sample_y material_y identified_by_y host health state_y  \\\n",
       "0                    NaN        NaN             NaN                 NaN   \n",
       "1                    NaN        NaN             NaN                 NaN   \n",
       "\n",
       "  INSDC last update_y Diseases_y strain_name_alias_y  genome_coverage_y  \\\n",
       "0                 NaN        NaN                 NaN                NaN   \n",
       "1                 NaN        NaN                 NaN                NaN   \n",
       "\n",
       "  INSDC status_y  host taxid_y isolation source_y Gram Staining_y  \\\n",
       "0            NaN           NaN                NaN             NaN   \n",
       "1            NaN           NaN                NaN             NaN   \n",
       "\n",
       "  Temperature Range Sporulation sequencing_meth_y  \\\n",
       "0               NaN         NaN               NaN   \n",
       "1               NaN         NaN               NaN   \n",
       "\n",
       "  Historical Monobactam Produced_y identification method_y supplier_name_y  \\\n",
       "0                              NaN                     NaN             NaN   \n",
       "1                              NaN                     NaN             NaN   \n",
       "\n",
       "  SRA accession_y host_disease_outcome_y  \\\n",
       "0             NaN                    NaN   \n",
       "1             NaN                    NaN   \n",
       "\n",
       "  Unique Monobactam Clusters Identified_y ProjectAccession Sample Name_y  \\\n",
       "0                                     NaN              NaN           NaN   \n",
       "1                                     NaN              NaN           NaN   \n",
       "\n",
       "  Type Strain Phenotypes_y Gene Calling Method_y temp_y Title_y  \\\n",
       "0         NaN          NaN                   NaN    NaN     NaN   \n",
       "1         NaN          NaN                   NaN    NaN     NaN   \n",
       "\n",
       "  investigation type_y Host_y rel_to_oxygen_y host_age_y Species  \\\n",
       "0                  NaN    NaN             NaN        NaN     NaN   \n",
       "1                  NaN    NaN             NaN        NaN     NaN   \n",
       "\n",
       "  mapping_method_y  serovar_y sop_y Isolation Comments finishing_strategy_y  \\\n",
       "0              NaN        NaN   NaN                NaN                  NaN   \n",
       "1              NaN        NaN   NaN                NaN                  NaN   \n",
       "\n",
       "  Genus mapping_method_version_y environment (biome)_y attribute_package  \\\n",
       "0   NaN                      NaN                   NaN               NaN   \n",
       "1   NaN                      NaN                   NaN               NaN   \n",
       "\n",
       "  metagenome_source_y environment (feature)_y trophic_level_y metagenomic_y  \\\n",
       "0                 NaN                     NaN             NaN           NaN   \n",
       "1                 NaN                     NaN             NaN           NaN   \n",
       "\n",
       "  environment (material)_y Funding Program_y quality_assessment_method_y  \\\n",
       "0                      NaN               NaN                         NaN   \n",
       "1                      NaN               NaN                         NaN   \n",
       "\n",
       "  quality_assessment_method_version_y  geographic location (latitude)_y  \\\n",
       "0                                 NaN                               NaN   \n",
       "1                                 NaN                               NaN   \n",
       "\n",
       "  source_mat_id_y host_tissue_sampled_y  geographic location (longitude)_y  \\\n",
       "0             NaN                   NaN                                NaN   \n",
       "1             NaN                   NaN                                NaN   \n",
       "\n",
       "  Host Name subsrc_note_y Host Health value_y  \\\n",
       "0       NaN           NaN         NaN     NaN   \n",
       "1       NaN           NaN         NaN     NaN   \n",
       "\n",
       "  plant-associated environmental package_y project name_y sequencing method_y  \\\n",
       "0                                      NaN            NaN                 NaN   \n",
       "1                                      NaN            NaN                 NaN   \n",
       "\n",
       "  Body Sample SubSite_y     merge2  \n",
       "0                   NaN  left_only  \n",
       "1                   NaN       both  "
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merge2.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Int64Index: 2686 entries, 0 to 2685\n",
      "Columns: 550 entries, Genome_ID to merge2\n",
      "dtypes: category(2), float64(49), object(499)\n",
      "memory usage: 11.3+ MB\n"
     ]
    }
   ],
   "source": [
    "merge2.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "both          2686\n",
       "right_only       0\n",
       "left_only        0\n",
       "Name: merge3, dtype: int64"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merge3 = pd.merge(merge2, patric_addit_metadata, how=\"left\", left_on=\"Genome_ID\", right_on=\"Genome_ID\", indicator=\"merge3\", suffixes=\"_z\")\n",
    "merge3[\"merge3\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "merge = merge3.drop(columns=[\"merge\", \"merge2\", \"merge3\", \"Additional Metadata\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "to_replace=[r\"^missing$\", r\"^Missing$\",  r\"^'Missing'\", \n",
    "            r\"^Unclassified$\", r\"^unclassified$\",\n",
    "            r\"^NA$\", r\"^na$\", r'^None$',\n",
    "            r\"^not determined$\", r\"^Not determined$\",\n",
    "            r\"^Not collected$\", r\"^not collected$\", r'^Not Collected$'\n",
    "            r'^Not recorded$', r'^not recorded$',r'^Not Recorded$', \n",
    "            r\"^unspecified$\", r\"^Unspecified$\",\n",
    "            r\"^Not Applicable$\",r\"^Not applicable$\", r'^not applicable$',\n",
    "            r\"^not available: to be reported later$\", r\"^not available$\",\n",
    "            r\"^Unknown\", r\"^unknown\", \n",
    "            r'^\"$', r\"^'$\", r'^_$',r'^-$'\n",
    "            ]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [],
   "source": [
    "merge = merge.replace({w: np.nan for w in to_replace}, regex=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genome_ID</th>\n",
       "      <th># assembly_accession</th>\n",
       "      <th>bioproject</th>\n",
       "      <th>biosample</th>\n",
       "      <th>wgs_master</th>\n",
       "      <th>refseq_category</th>\n",
       "      <th>taxid</th>\n",
       "      <th>species_taxid</th>\n",
       "      <th>organism_name</th>\n",
       "      <th>infraspecific_name</th>\n",
       "      <th>isolate_x</th>\n",
       "      <th>version_status</th>\n",
       "      <th>assembly_level</th>\n",
       "      <th>release_type</th>\n",
       "      <th>genome_rep</th>\n",
       "      <th>seq_rel_date</th>\n",
       "      <th>asm_name</th>\n",
       "      <th>submitter_x</th>\n",
       "      <th>gbrs_paired_asm</th>\n",
       "      <th>paired_asm_comp</th>\n",
       "      <th>ftp_path</th>\n",
       "      <th>excluded_from_refseq</th>\n",
       "      <th>relation_to_type_material</th>\n",
       "      <th>Genome Name</th>\n",
       "      <th>Organism Name</th>\n",
       "      <th>NCBI Taxon ID</th>\n",
       "      <th>Genome Status</th>\n",
       "      <th>Strain_x</th>\n",
       "      <th>Serovar</th>\n",
       "      <th>Pathovar</th>\n",
       "      <th>MLST</th>\n",
       "      <th>Other Typing</th>\n",
       "      <th>Culture Collection</th>\n",
       "      <th>Type Strain_x</th>\n",
       "      <th>Completion Date</th>\n",
       "      <th>Publication</th>\n",
       "      <th>BioProject Accession</th>\n",
       "      <th>BioSample Accession</th>\n",
       "      <th>Assembly Accession</th>\n",
       "      <th>SRA Accession</th>\n",
       "      <th>GenBank Accessions</th>\n",
       "      <th>RefSeq Accessions</th>\n",
       "      <th>Sequencing Centers</th>\n",
       "      <th>Sequencing Status</th>\n",
       "      <th>Sequencing Platform</th>\n",
       "      <th>Sequencing Depth</th>\n",
       "      <th>Assembly Method_x</th>\n",
       "      <th>Chromosomes</th>\n",
       "      <th>Plasmids</th>\n",
       "      <th>Contigs</th>\n",
       "      <th>Genome Length</th>\n",
       "      <th>GC Content</th>\n",
       "      <th>RefSeq CDS</th>\n",
       "      <th>Isolation Source</th>\n",
       "      <th>Isolation Comments_x</th>\n",
       "      <th>Collection Date</th>\n",
       "      <th>Isolation Country</th>\n",
       "      <th>Geographic Location</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Altitude</th>\n",
       "      <th>Depth_x</th>\n",
       "      <th>Other Environmental</th>\n",
       "      <th>Host Name_x</th>\n",
       "      <th>Host Gender</th>\n",
       "      <th>Host Age</th>\n",
       "      <th>Host Health_x</th>\n",
       "      <th>Body Sample Site_x</th>\n",
       "      <th>Other Clinical</th>\n",
       "      <th>Gram Stain</th>\n",
       "      <th>Cell Shape_x</th>\n",
       "      <th>Motility_x</th>\n",
       "      <th>Sporulation_x</th>\n",
       "      <th>Temperature Range_x</th>\n",
       "      <th>Optimal Temperature</th>\n",
       "      <th>Salinity_x</th>\n",
       "      <th>Oxygen Requirement</th>\n",
       "      <th>Habitat</th>\n",
       "      <th>Disease</th>\n",
       "      <th>Comments</th>\n",
       "      <th>Genome Quality Flags</th>\n",
       "      <th>biosample_title</th>\n",
       "      <th>contact_email</th>\n",
       "      <th>first_name</th>\n",
       "      <th>last_name</th>\n",
       "      <th>name</th>\n",
       "      <th>organism</th>\n",
       "      <th>publication_date</th>\n",
       "      <th>sample_accession</th>\n",
       "      <th>strain_x</th>\n",
       "      <th>submission_model</th>\n",
       "      <th>submission_package</th>\n",
       "      <th>submission_package_name</th>\n",
       "      <th>submitter_y</th>\n",
       "      <th>taxonomy_id</th>\n",
       "      <th>collection_date_x</th>\n",
       "      <th>culture_collection_x</th>\n",
       "      <th>geo_loc_name_x</th>\n",
       "      <th>isolation_source_x</th>\n",
       "      <th>sample_name_x</th>\n",
       "      <th>sample_type_x</th>\n",
       "      <th>type-material_x</th>\n",
       "      <th>comment_paragraph</th>\n",
       "      <th>depth_x</th>\n",
       "      <th>derived-from</th>\n",
       "      <th>env_broad_scale</th>\n",
       "      <th>env_local_scale</th>\n",
       "      <th>env_medium</th>\n",
       "      <th>environmental-sample</th>\n",
       "      <th>isolate_y</th>\n",
       "      <th>lat_lon_x</th>\n",
       "      <th>metagenome-source</th>\n",
       "      <th>metagenomic_x</th>\n",
       "      <th>sample-type</th>\n",
       "      <th>elev_x</th>\n",
       "      <th>environmetal-sample</th>\n",
       "      <th>collected_by_x</th>\n",
       "      <th>host_x</th>\n",
       "      <th>host_tissue_sampled_x</th>\n",
       "      <th>temp_x</th>\n",
       "      <th>env_biome_x</th>\n",
       "      <th>env_feature_x</th>\n",
       "      <th>env_material_x</th>\n",
       "      <th>isol_growth_condt_x</th>\n",
       "      <th>locus_tag_prefix_x</th>\n",
       "      <th>num_replicons_x</th>\n",
       "      <th>project_name_x</th>\n",
       "      <th>ref_biomaterial_x</th>\n",
       "      <th>16S recovered</th>\n",
       "      <th>16S recovery software</th>\n",
       "      <th>analysis project type</th>\n",
       "      <th>assembly quality</th>\n",
       "      <th>assembly software</th>\n",
       "      <th>bin parameters</th>\n",
       "      <th>binning software</th>\n",
       "      <th>completeness score</th>\n",
       "      <th>completeness software</th>\n",
       "      <th>contamination score</th>\n",
       "      <th>contig L50</th>\n",
       "      <th>decontamination software</th>\n",
       "      <th>note_x</th>\n",
       "      <th>number of contigs</th>\n",
       "      <th>reassembly post binning</th>\n",
       "      <th>relative coverage Anaerobic (SRR10097246)</th>\n",
       "      <th>relative coverage Anoxic (SRR10375098)</th>\n",
       "      <th>relative coverage Infiltration (SRR10097247)</th>\n",
       "      <th>relative coverage Low pH (SRR10097245)</th>\n",
       "      <th>relative coverage Oxic (SRR10375099)</th>\n",
       "      <th>relative coverage Reference (SRR10375097)</th>\n",
       "      <th>taxa id</th>\n",
       "      <th>total assembly size</th>\n",
       "      <th>nat_host</th>\n",
       "      <th>assembly_method_and_version</th>\n",
       "      <th>completeness_estimated_x</th>\n",
       "      <th>contamination_estimated_x</th>\n",
       "      <th>genome_coverage_x</th>\n",
       "      <th>mapping_method_and_version</th>\n",
       "      <th>quality_assessment_method_and_version</th>\n",
       "      <th>ENA-FIRST-PUBLIC_x</th>\n",
       "      <th>ENA-LAST-UPDATE_x</th>\n",
       "      <th>SUBJECT_ID_x</th>\n",
       "      <th>contact_lab</th>\n",
       "      <th>identified_by_x</th>\n",
       "      <th>environmental_sample_x</th>\n",
       "      <th>metagenome_source_x</th>\n",
       "      <th>biomaterial_provider_x</th>\n",
       "      <th>biome_x</th>\n",
       "      <th>feature_x</th>\n",
       "      <th>material_x</th>\n",
       "      <th>culture-collection_x</th>\n",
       "      <th>altitude_x</th>\n",
       "      <th>isolate_name_alias_x</th>\n",
       "      <th>host_disease_x</th>\n",
       "      <th>rel_to_oxygen_x</th>\n",
       "      <th>Alias_x</th>\n",
       "      <th>ENA checklist_x</th>\n",
       "      <th>INSDC center alias_x</th>\n",
       "      <th>INSDC center name_x</th>\n",
       "      <th>INSDC first public_x</th>\n",
       "      <th>INSDC last update_x</th>\n",
       "      <th>INSDC status_x</th>\n",
       "      <th>SRA accession_x</th>\n",
       "      <th>Sample Name_x</th>\n",
       "      <th>Title_x</th>\n",
       "      <th>serotype</th>\n",
       "      <th>subsrc_note_x</th>\n",
       "      <th>annotation_method</th>\n",
       "      <th>assembly_method_x</th>\n",
       "      <th>biosamplemodel</th>\n",
       "      <th>geographic location (country and/or atlantic ocean)</th>\n",
       "      <th>geographic location (region and locality)</th>\n",
       "      <th>mapping_method_x</th>\n",
       "      <th>quality_assessment_method_x</th>\n",
       "      <th>refinement_method</th>\n",
       "      <th>Cell Shape_y</th>\n",
       "      <th>Funding Program_x</th>\n",
       "      <th>GOLD Stamp ID_x</th>\n",
       "      <th>Gene Calling Method_x</th>\n",
       "      <th>Gram Staining_x</th>\n",
       "      <th>Isolation Comments_y</th>\n",
       "      <th>Isolation Site_x</th>\n",
       "      <th>Motility_y</th>\n",
       "      <th>Temperature Optimum_x</th>\n",
       "      <th>Temperature Range_y</th>\n",
       "      <th>Type Strain_y</th>\n",
       "      <th>alt_elev_x</th>\n",
       "      <th>biotic_relationship_x</th>\n",
       "      <th>country_x</th>\n",
       "      <th>environment_x</th>\n",
       "      <th>investigation_type_x</th>\n",
       "      <th>isolation-source_x</th>\n",
       "      <th>trophic_level_x</th>\n",
       "      <th>pH_x</th>\n",
       "      <th>Body Sample Site_y</th>\n",
       "      <th>Body Sample SubSite_x</th>\n",
       "      <th>Diseases_x</th>\n",
       "      <th>Host Name_y</th>\n",
       "      <th>Host Health_y</th>\n",
       "      <th>Sporulation_y</th>\n",
       "      <th>experimental_factor</th>\n",
       "      <th>lat-lon</th>\n",
       "      <th>ENA-CHECKLIST</th>\n",
       "      <th>External Id</th>\n",
       "      <th>Submitter Id</th>\n",
       "      <th>subsrc-note</th>\n",
       "      <th>collection date_x</th>\n",
       "      <th>environment (biome)_x</th>\n",
       "      <th>environment (feature)_x</th>\n",
       "      <th>environment (material)_x</th>\n",
       "      <th>geographic location (country and/or sea)_x</th>\n",
       "      <th>geographic location (latitude)_x</th>\n",
       "      <th>geographic location (longitude)_x</th>\n",
       "      <th>host common name</th>\n",
       "      <th>host-associated environmental package</th>\n",
       "      <th>investigation type_x</th>\n",
       "      <th>project name_x</th>\n",
       "      <th>sequencing method_x</th>\n",
       "      <th>STRAIN</th>\n",
       "      <th>description</th>\n",
       "      <th>completeness</th>\n",
       "      <th>contamination</th>\n",
       "      <th>host_age_x</th>\n",
       "      <th>host_sex_x</th>\n",
       "      <th>oxy_stat_samp</th>\n",
       "      <th>samp_store_dur</th>\n",
       "      <th>samp_store_temp</th>\n",
       "      <th>FDA CVM ID_x</th>\n",
       "      <th>bio_material</th>\n",
       "      <th>common name</th>\n",
       "      <th>host scientific name_x</th>\n",
       "      <th>samp_mat_process</th>\n",
       "      <th>estimated_size_x</th>\n",
       "      <th>Tax ID</th>\n",
       "      <th>Collaborator ID</th>\n",
       "      <th>host_description</th>\n",
       "      <th>host_disease_outcome_x</th>\n",
       "      <th>host_disease_stage</th>\n",
       "      <th>host_health_state</th>\n",
       "      <th>identification method_x</th>\n",
       "      <th>strain_name_alias_x</th>\n",
       "      <th>WGA amplification approach</th>\n",
       "      <th>metagenomic source</th>\n",
       "      <th>sample derived from</th>\n",
       "      <th>single cell or viral particle lysis approach</th>\n",
       "      <th>sorting technology</th>\n",
       "      <th>taxonomic identity marker</th>\n",
       "      <th>passage_history_</th>\n",
       "      <th>samp_size</th>\n",
       "      <th>Description</th>\n",
       "      <th>subsource_note</th>\n",
       "      <th>lab_host_</th>\n",
       "      <th>collection_room</th>\n",
       "      <th>relative_week</th>\n",
       "      <th>assembly_tool</th>\n",
       "      <th>bin_id</th>\n",
       "      <th>Broker name</th>\n",
       "      <th>MAG_number</th>\n",
       "      <th>derived_from</th>\n",
       "      <th>alkalinity (carbonate/bicarbonate)</th>\n",
       "      <th>salinity</th>\n",
       "      <th>Phenotypes_x</th>\n",
       "      <th>env_package</th>\n",
       "      <th>specific_host_x</th>\n",
       "      <th>binning parameters</th>\n",
       "      <th>plant-associated environmental package_x</th>\n",
       "      <th>culture collection</th>\n",
       "      <th>repository</th>\n",
       "      <th>risk group</th>\n",
       "      <th>type status</th>\n",
       "      <th>Assembly Method_y</th>\n",
       "      <th>Genome Coverage</th>\n",
       "      <th>Sequencing Technology</th>\n",
       "      <th>nat-host_x</th>\n",
       "      <th>collection-date</th>\n",
       "      <th>Historical Monobactam Produced_x</th>\n",
       "      <th>Unique Monobactam Clusters Identified_x</th>\n",
       "      <th>extrachrom_elements</th>\n",
       "      <th>pathogenicity</th>\n",
       "      <th>source_material_id</th>\n",
       "      <th>genotype</th>\n",
       "      <th>mating_type_</th>\n",
       "      <th>serovar_x</th>\n",
       "      <th>specimen_voucher_</th>\n",
       "      <th>Salinity (PSU)</th>\n",
       "      <th>collected-by</th>\n",
       "      <th>orgmod_note</th>\n",
       "      <th>assembly_x</th>\n",
       "      <th>sequencing_meth_x</th>\n",
       "      <th>finishing_strategy_x</th>\n",
       "      <th>geo-loc-name_x</th>\n",
       "      <th>finishing strategy (depth of coverage)_x</th>\n",
       "      <th>host taxid_x</th>\n",
       "      <th>misc_param: HMP body site</th>\n",
       "      <th>misc_param: HMP supersite</th>\n",
       "      <th>nucleic acid extraction</th>\n",
       "      <th>project_type_x</th>\n",
       "      <th>sop_x</th>\n",
       "      <th>source_mat_id_x</th>\n",
       "      <th>Salinity_y</th>\n",
       "      <th>body_site</th>\n",
       "      <th>health_state</th>\n",
       "      <th>Isolation source_x</th>\n",
       "      <th>Host_x</th>\n",
       "      <th>isolation source_x</th>\n",
       "      <th>environmental package</th>\n",
       "      <th>specific host_x</th>\n",
       "      <th>lat lon</th>\n",
       "      <th>host_subject_id</th>\n",
       "      <th>pathotype</th>\n",
       "      <th>subgroup</th>\n",
       "      <th>subtype</th>\n",
       "      <th>Extraction Date</th>\n",
       "      <th>Extraction Method</th>\n",
       "      <th>Laboratory Host</th>\n",
       "      <th>Passage Date</th>\n",
       "      <th>sample type</th>\n",
       "      <th>host health state_x</th>\n",
       "      <th>anonymized_name_x</th>\n",
       "      <th>host_status</th>\n",
       "      <th>supplier_name_x</th>\n",
       "      <th>abs_air_humidity</th>\n",
       "      <th>air_temp</th>\n",
       "      <th>build_occup_type</th>\n",
       "      <th>building_setting</th>\n",
       "      <th>carb_dioxide</th>\n",
       "      <th>filter_type</th>\n",
       "      <th>heat_cool_type</th>\n",
       "      <th>indoor_space</th>\n",
       "      <th>light_type</th>\n",
       "      <th>occup_samp</th>\n",
       "      <th>occupant_dens_samp</th>\n",
       "      <th>organism_count</th>\n",
       "      <th>rel_air_humidity</th>\n",
       "      <th>space_typ_state</th>\n",
       "      <th>typ_occupant_dens</th>\n",
       "      <th>ventilation_type</th>\n",
       "      <th>Type-material</th>\n",
       "      <th>samp_collect_device</th>\n",
       "      <th>geographic location (altitude)</th>\n",
       "      <th>env_packatge</th>\n",
       "      <th>lib_reads_seqd</th>\n",
       "      <th>metagenomic_source</th>\n",
       "      <th>env material</th>\n",
       "      <th>other identifiers</th>\n",
       "      <th>assembly_method_version_x</th>\n",
       "      <th>mapping_method_version_x</th>\n",
       "      <th>quality_assessment_method_version_x</th>\n",
       "      <th>value_x</th>\n",
       "      <th>cyanobacterial_culture</th>\n",
       "      <th>missing</th>\n",
       "      <th>samp_store_loc</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>host_body_habitat</th>\n",
       "      <th>host_taxid</th>\n",
       "      <th>misc_param_x</th>\n",
       "      <th>ph</th>\n",
       "      <th>metagenome-source seawater metagenome</th>\n",
       "      <th>type_status</th>\n",
       "      <th>indoor_surf</th>\n",
       "      <th>samp_sort_meth</th>\n",
       "      <th>samp_vol_we_dna_ext</th>\n",
       "      <th>surf_material</th>\n",
       "      <th>BioSampleModel_x</th>\n",
       "      <th>ammonium</th>\n",
       "      <th>biomass</th>\n",
       "      <th>carb_nitro_ratio</th>\n",
       "      <th>chlorophyll</th>\n",
       "      <th>diss_inorg_nitro</th>\n",
       "      <th>diss_inorg_phosp</th>\n",
       "      <th>diss_org_carb</th>\n",
       "      <th>diss_org_nitro</th>\n",
       "      <th>nitrate</th>\n",
       "      <th>nitrite</th>\n",
       "      <th>phosphate</th>\n",
       "      <th>silicate</th>\n",
       "      <th>tidal_stage</th>\n",
       "      <th>tot_diss_nitro</th>\n",
       "      <th>locus_tag prefix</th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>strain_y</th>\n",
       "      <th>finishing strategy (depth of coverage)_y</th>\n",
       "      <th>biotic_relationship_y</th>\n",
       "      <th>geo_loc_name_y</th>\n",
       "      <th>host_y</th>\n",
       "      <th>collected_by_y</th>\n",
       "      <th>host_disease_y</th>\n",
       "      <th>isolation_source_y</th>\n",
       "      <th>sample_name_y</th>\n",
       "      <th>isolation-source_y</th>\n",
       "      <th>sample_type_y</th>\n",
       "      <th>FDA CVM ID_y</th>\n",
       "      <th>culture_collection_y</th>\n",
       "      <th>assembly_method_y</th>\n",
       "      <th>collection_date_y</th>\n",
       "      <th>env_feature_y</th>\n",
       "      <th>synonym</th>\n",
       "      <th>Strain</th>\n",
       "      <th>SUBJECT_ID_y</th>\n",
       "      <th>Alias_y</th>\n",
       "      <th>type-material_y</th>\n",
       "      <th>misc_param_y</th>\n",
       "      <th>collection date_y</th>\n",
       "      <th>Isolation Site_y</th>\n",
       "      <th>Isolation source_y</th>\n",
       "      <th>country_y</th>\n",
       "      <th>note_y</th>\n",
       "      <th>GOLD Stamp ID_y</th>\n",
       "      <th>investigation_type_y</th>\n",
       "      <th>culture-collection_y</th>\n",
       "      <th>lat_lon_y</th>\n",
       "      <th>alt_elev_y</th>\n",
       "      <th>locus_tag_prefix_y</th>\n",
       "      <th>nat-host_y</th>\n",
       "      <th>assembly_method_version_y</th>\n",
       "      <th>env_material_y</th>\n",
       "      <th>model</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>Sample Description</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>ENA checklist_y</th>\n",
       "      <th>project_type_y</th>\n",
       "      <th>estimated_size_y</th>\n",
       "      <th>Temperature Optimum_y</th>\n",
       "      <th>isol_growth_condt_y</th>\n",
       "      <th>Motility</th>\n",
       "      <th>feature_y</th>\n",
       "      <th>project_name_y</th>\n",
       "      <th>assembly_y</th>\n",
       "      <th>num_replicons_y</th>\n",
       "      <th>isolate_</th>\n",
       "      <th>depth_y</th>\n",
       "      <th>env_biome_y</th>\n",
       "      <th>altitude_y</th>\n",
       "      <th>ref_biomaterial_y</th>\n",
       "      <th>package</th>\n",
       "      <th>geographic location</th>\n",
       "      <th>ArrayExpress-StrainOrLine</th>\n",
       "      <th>ArrayExpress-Phenotype</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>INSDC center alias_y</th>\n",
       "      <th>geo-loc-name_y</th>\n",
       "      <th>environment_y</th>\n",
       "      <th>host_sex_y</th>\n",
       "      <th>elev_y</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>completeness_estimated_y</th>\n",
       "      <th>geographic location (country and/or sea)_y</th>\n",
       "      <th>latitude and longitude</th>\n",
       "      <th>specific_host_y</th>\n",
       "      <th>ArrayExpress-Species</th>\n",
       "      <th>ENA-FIRST-PUBLIC_y</th>\n",
       "      <th>INSDC center name_y</th>\n",
       "      <th>specific host_y</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>pH_y</th>\n",
       "      <th>biome_y</th>\n",
       "      <th>biomaterial_provider_y</th>\n",
       "      <th>isolate_name_alias_y</th>\n",
       "      <th>contamination_estimated_y</th>\n",
       "      <th>host scientific name_y</th>\n",
       "      <th>anonymized_name_y</th>\n",
       "      <th>ENA-LAST-UPDATE_y</th>\n",
       "      <th>INSDC first public_y</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>BioSampleModel_y</th>\n",
       "      <th>environmental_sample_y</th>\n",
       "      <th>material_y</th>\n",
       "      <th>identified_by_y</th>\n",
       "      <th>host health state_y</th>\n",
       "      <th>INSDC last update_y</th>\n",
       "      <th>Diseases_y</th>\n",
       "      <th>strain_name_alias_y</th>\n",
       "      <th>genome_coverage_y</th>\n",
       "      <th>INSDC status_y</th>\n",
       "      <th>host taxid_y</th>\n",
       "      <th>isolation source_y</th>\n",
       "      <th>Gram Staining_y</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>sequencing_meth_y</th>\n",
       "      <th>Historical Monobactam Produced_y</th>\n",
       "      <th>identification method_y</th>\n",
       "      <th>supplier_name_y</th>\n",
       "      <th>SRA accession_y</th>\n",
       "      <th>host_disease_outcome_y</th>\n",
       "      <th>Unique Monobactam Clusters Identified_y</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>Sample Name_y</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Phenotypes_y</th>\n",
       "      <th>Gene Calling Method_y</th>\n",
       "      <th>temp_y</th>\n",
       "      <th>Title_y</th>\n",
       "      <th>investigation type_y</th>\n",
       "      <th>Host_y</th>\n",
       "      <th>rel_to_oxygen_y</th>\n",
       "      <th>host_age_y</th>\n",
       "      <th>Species</th>\n",
       "      <th>mapping_method_y</th>\n",
       "      <th>serovar_y</th>\n",
       "      <th>sop_y</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>finishing_strategy_y</th>\n",
       "      <th>Genus</th>\n",
       "      <th>mapping_method_version_y</th>\n",
       "      <th>environment (biome)_y</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>metagenome_source_y</th>\n",
       "      <th>environment (feature)_y</th>\n",
       "      <th>trophic_level_y</th>\n",
       "      <th>metagenomic_y</th>\n",
       "      <th>environment (material)_y</th>\n",
       "      <th>Funding Program_y</th>\n",
       "      <th>quality_assessment_method_y</th>\n",
       "      <th>quality_assessment_method_version_y</th>\n",
       "      <th>geographic location (latitude)_y</th>\n",
       "      <th>source_mat_id_y</th>\n",
       "      <th>host_tissue_sampled_y</th>\n",
       "      <th>geographic location (longitude)_y</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>subsrc_note_y</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>value_y</th>\n",
       "      <th>plant-associated environmental package_y</th>\n",
       "      <th>project name_y</th>\n",
       "      <th>sequencing method_y</th>\n",
       "      <th>Body Sample SubSite_y</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>passage_historyz</th>\n",
       "      <th>isolatez</th>\n",
       "      <th>specimen_voucherz</th>\n",
       "      <th></th>\n",
       "      <th>lab_hostz</th>\n",
       "      <th>mating_typez</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>NaN</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>986.0</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>strain=UW101; ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>Major</td>\n",
       "      <td>Full</td>\n",
       "      <td>2007/05/01</td>\n",
       "      <td>ASM1664v1</td>\n",
       "      <td>US DOE Joint Genome Institute</td>\n",
       "      <td>GCF_000016645.1</td>\n",
       "      <td>identical</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>Complete</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>Yes</td>\n",
       "      <td>2007-05-01T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NC_009441</td>\n",
       "      <td>DOE Joint Genome Institute</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>6096872.0</td>\n",
       "      <td>34.1</td>\n",
       "      <td>5017.0</td>\n",
       "      <td>soil</td>\n",
       "      <td>isolated from soil in England and is the type ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>United Kingdom: England</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>20-30</td>\n",
       "      <td>Non-halophilic</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>Multiple</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae ATCC 17061. Flavobac...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Sample from Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>2014-01-28T00:00:00.000</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>UW101; ATCC 17061</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Flavobacterium johnsoniae</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>1018.0</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>strain=DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>Major</td>\n",
       "      <td>Full</td>\n",
       "      <td>2009/08/26</td>\n",
       "      <td>ASM2328v1</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)</td>\n",
       "      <td>GCF_000023285.1</td>\n",
       "      <td>identical</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>Complete</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>Yes</td>\n",
       "      <td>2009-04-30T00:00:00Z</td>\n",
       "      <td>21304645</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>SRR013476,SRR013477</td>\n",
       "      <td>CP001632</td>\n",
       "      <td>NC_013162</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>2612925.0</td>\n",
       "      <td>39.6</td>\n",
       "      <td>2171.0</td>\n",
       "      <td>the human oral cavity</td>\n",
       "      <td>isolated from the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>Yes</td>\n",
       "      <td>No</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>35-37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Facultative</td>\n",
       "      <td>Host-associated</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271.  Capnocytoph...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic sample from Capnocytophaga ochracea DS...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>2009-02-19T00:00:00.000</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Genome_ID # assembly_accession  bioproject     biosample wgs_master  \\\n",
       "0  GCA_000016645.1      GCA_000016645.1  PRJNA16082  SAMN02598357        NaN   \n",
       "1  GCA_000023285.1      GCA_000023285.1  PRJNA29403  SAMN00001911        NaN   \n",
       "\n",
       "         refseq_category     taxid  species_taxid  \\\n",
       "0  representative genome  376686.0          986.0   \n",
       "1  representative genome  521097.0         1018.0   \n",
       "\n",
       "                      organism_name        infraspecific_name isolate_x  \\\n",
       "0   Flavobacterium johnsoniae UW101  strain=UW101; ATCC 17061       NaN   \n",
       "1  Capnocytophaga ochracea DSM 7271           strain=DSM 7271       NaN   \n",
       "\n",
       "  version_status   assembly_level release_type genome_rep seq_rel_date  \\\n",
       "0         latest  Complete Genome        Major       Full   2007/05/01   \n",
       "1         latest  Complete Genome        Major       Full   2009/08/26   \n",
       "\n",
       "    asm_name                              submitter_x  gbrs_paired_asm  \\\n",
       "0  ASM1664v1            US DOE Joint Genome Institute  GCF_000016645.1   \n",
       "1  ASM2328v1  US DOE Joint Genome Institute (JGI-PGF)  GCF_000023285.1   \n",
       "\n",
       "  paired_asm_comp                                           ftp_path  \\\n",
       "0       identical  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "1       identical  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "\n",
       "  excluded_from_refseq    relation_to_type_material  \\\n",
       "0                  NaN  assembly from type material   \n",
       "1                  NaN  assembly from type material   \n",
       "\n",
       "                        Genome Name                     Organism Name  \\\n",
       "0   Flavobacterium johnsoniae UW101   Flavobacterium johnsoniae UW101   \n",
       "1  Capnocytophaga ochracea DSM 7271  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "   NCBI Taxon ID Genome Status    Strain_x Serovar Pathovar MLST Other Typing  \\\n",
       "0       376686.0      Complete  ATCC 17061     NaN      NaN  NaN          NaN   \n",
       "1       521097.0      Complete    DSM 7271     NaN      NaN  NaN          NaN   \n",
       "\n",
       "  Culture Collection Type Strain_x       Completion Date Publication  \\\n",
       "0         ATCC 17061           Yes  2007-05-01T00:00:00Z         NaN   \n",
       "1           DSM 7271           Yes  2009-04-30T00:00:00Z    21304645   \n",
       "\n",
       "  BioProject Accession BioSample Accession Assembly Accession  \\\n",
       "0           PRJNA16082        SAMN02598357    GCA_000016645.1   \n",
       "1           PRJNA29403        SAMN00001911    GCA_000023285.1   \n",
       "\n",
       "         SRA Accession GenBank Accessions RefSeq Accessions  \\\n",
       "0                  NaN           CP000685         NC_009441   \n",
       "1  SRR013476,SRR013477           CP001632         NC_013162   \n",
       "\n",
       "                                  Sequencing Centers Sequencing Status  \\\n",
       "0                         DOE Joint Genome Institute          complete   \n",
       "1  US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...          complete   \n",
       "\n",
       "  Sequencing Platform Sequencing Depth Assembly Method_x  Chromosomes  \\\n",
       "0                 NaN              NaN               NaN          1.0   \n",
       "1                 NaN              NaN               NaN          1.0   \n",
       "\n",
       "   Plasmids  Contigs  Genome Length  GC Content  RefSeq CDS  \\\n",
       "0       NaN      1.0      6096872.0        34.1      5017.0   \n",
       "1       NaN      1.0      2612925.0        39.6      2171.0   \n",
       "\n",
       "        Isolation Source                               Isolation Comments_x  \\\n",
       "0                   soil  isolated from soil in England and is the type ...   \n",
       "1  the human oral cavity                isolated from the human oral cavity   \n",
       "\n",
       "  Collection Date Isolation Country      Geographic Location  Latitude  \\\n",
       "0             NaN    United Kingdom  United Kingdom: England       NaN   \n",
       "1             NaN               NaN                      NaN       NaN   \n",
       "\n",
       "   Longitude Altitude Depth_x Other Environmental          Host Name_x  \\\n",
       "0        NaN      NaN     NaN                 NaN                  NaN   \n",
       "1        NaN      NaN     NaN                 NaN  Human, Homo sapiens   \n",
       "\n",
       "  Host Gender Host Age Host Health_x Body Sample Site_x Other Clinical  \\\n",
       "0         NaN      NaN           NaN                NaN            NaN   \n",
       "1         NaN      NaN           NaN                NaN            NaN   \n",
       "\n",
       "  Gram Stain Cell Shape_x Motility_x Sporulation_x Temperature Range_x  \\\n",
       "0        NaN          Rod        Yes            No          Mesophilic   \n",
       "1        NaN          Rod        Yes            No          Mesophilic   \n",
       "\n",
       "  Optimal Temperature      Salinity_x Oxygen Requirement          Habitat  \\\n",
       "0               20-30  Non-halophilic            Aerobic         Multiple   \n",
       "1               35-37             NaN        Facultative  Host-associated   \n",
       "\n",
       "  Disease                                           Comments  \\\n",
       "0     NaN  Flavobacterium johnsoniae ATCC 17061. Flavobac...   \n",
       "1     NaN  Capnocytophaga ochracea DSM 7271.  Capnocytoph...   \n",
       "\n",
       "  Genome Quality Flags                                    biosample_title  \\\n",
       "0                  NaN        Sample from Flavobacterium johnsoniae UW101   \n",
       "1                  NaN  Generic sample from Capnocytophaga ochracea DS...   \n",
       "\n",
       "  contact_email first_name last_name  \\\n",
       "0           NaN        NaN       NaN   \n",
       "1           NaN        NaN       NaN   \n",
       "\n",
       "                                                name  \\\n",
       "0                                               NCBI   \n",
       "1  Joint Genome Institute, U.S. Department of Energy   \n",
       "\n",
       "                           organism         publication_date sample_accession  \\\n",
       "0   Flavobacterium johnsoniae UW101  2014-01-28T00:00:00.000     SAMN02598357   \n",
       "1  Capnocytophaga ochracea DSM 7271  2009-02-19T00:00:00.000     SAMN00001911   \n",
       "\n",
       "            strain_x submission_model submission_package  \\\n",
       "0  UW101; ATCC 17061          Generic        Generic.1.0   \n",
       "1           DSM 7271          Generic        Generic.1.0   \n",
       "\n",
       "  submission_package_name                                        submitter_y  \\\n",
       "0                 Generic                                               NCBI   \n",
       "1                 Generic  Joint Genome Institute, U.S. Department of Energy   \n",
       "\n",
       "   taxonomy_id collection_date_x culture_collection_x geo_loc_name_x  \\\n",
       "0     376686.0               NaN                  NaN            NaN   \n",
       "1     521097.0               NaN                  NaN            NaN   \n",
       "\n",
       "  isolation_source_x sample_name_x sample_type_x  \\\n",
       "0                NaN      CP000685           NaN   \n",
       "1                NaN          FWCB           NaN   \n",
       "\n",
       "                            type-material_x                 comment_paragraph  \\\n",
       "0  type strain of Flavobacterium johnsoniae                               NaN   \n",
       "1    type strain of Capnocytophaga ochracea  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "  depth_x derived-from env_broad_scale env_local_scale env_medium  \\\n",
       "0     NaN          NaN             NaN             NaN        NaN   \n",
       "1     NaN          NaN             NaN             NaN        NaN   \n",
       "\n",
       "  environmental-sample isolate_y lat_lon_x metagenome-source metagenomic_x  \\\n",
       "0                  NaN       NaN       NaN               NaN           NaN   \n",
       "1                  NaN       NaN       NaN               NaN           NaN   \n",
       "\n",
       "  sample-type elev_x environmetal-sample collected_by_x host_x  \\\n",
       "0         NaN    NaN                 NaN            NaN    NaN   \n",
       "1         NaN    NaN                 NaN            NaN    NaN   \n",
       "\n",
       "  host_tissue_sampled_x temp_x env_biome_x env_feature_x env_material_x  \\\n",
       "0                   NaN    NaN         NaN           NaN            NaN   \n",
       "1                   NaN    NaN         NaN           NaN            NaN   \n",
       "\n",
       "  isol_growth_condt_x locus_tag_prefix_x num_replicons_x project_name_x  \\\n",
       "0                 NaN                NaN             NaN            NaN   \n",
       "1                 NaN                NaN             NaN            NaN   \n",
       "\n",
       "  ref_biomaterial_x 16S recovered 16S recovery software analysis project type  \\\n",
       "0               NaN           NaN                   NaN                   NaN   \n",
       "1               NaN           NaN                   NaN                   NaN   \n",
       "\n",
       "  assembly quality assembly software bin parameters binning software  \\\n",
       "0              NaN               NaN            NaN              NaN   \n",
       "1              NaN               NaN            NaN              NaN   \n",
       "\n",
       "   completeness score completeness software  contamination score  contig L50  \\\n",
       "0                 NaN                   NaN                  NaN         NaN   \n",
       "1                 NaN                   NaN                  NaN         NaN   \n",
       "\n",
       "  decontamination software note_x  number of contigs reassembly post binning  \\\n",
       "0                      NaN    NaN                NaN                     NaN   \n",
       "1                      NaN    NaN                NaN                     NaN   \n",
       "\n",
       "   relative coverage Anaerobic (SRR10097246)  \\\n",
       "0                                        NaN   \n",
       "1                                        NaN   \n",
       "\n",
       "   relative coverage Anoxic (SRR10375098)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Infiltration (SRR10097247)  \\\n",
       "0                                           NaN   \n",
       "1                                           NaN   \n",
       "\n",
       "   relative coverage Low pH (SRR10097245)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Oxic (SRR10375099)  \\\n",
       "0                                   NaN   \n",
       "1                                   NaN   \n",
       "\n",
       "   relative coverage Reference (SRR10375097) taxa id  total assembly size  \\\n",
       "0                                        NaN     NaN                  NaN   \n",
       "1                                        NaN     NaN                  NaN   \n",
       "\n",
       "  nat_host assembly_method_and_version completeness_estimated_x  \\\n",
       "0      NaN                         NaN                      NaN   \n",
       "1      NaN                         NaN                      NaN   \n",
       "\n",
       "  contamination_estimated_x genome_coverage_x mapping_method_and_version  \\\n",
       "0                       NaN               NaN                        NaN   \n",
       "1                       NaN               NaN                        NaN   \n",
       "\n",
       "  quality_assessment_method_and_version ENA-FIRST-PUBLIC_x ENA-LAST-UPDATE_x  \\\n",
       "0                                   NaN                NaN               NaN   \n",
       "1                                   NaN                NaN               NaN   \n",
       "\n",
       "  SUBJECT_ID_x contact_lab identified_by_x environmental_sample_x  \\\n",
       "0          NaN         NaN             NaN                    NaN   \n",
       "1          NaN         NaN             NaN                    NaN   \n",
       "\n",
       "  metagenome_source_x biomaterial_provider_x biome_x feature_x material_x  \\\n",
       "0                 NaN                    NaN     NaN       NaN        NaN   \n",
       "1                 NaN                    NaN     NaN       NaN        NaN   \n",
       "\n",
       "  culture-collection_x altitude_x isolate_name_alias_x host_disease_x  \\\n",
       "0                  NaN        NaN                  NaN            NaN   \n",
       "1                  NaN        NaN                  NaN            NaN   \n",
       "\n",
       "  rel_to_oxygen_x Alias_x ENA checklist_x INSDC center alias_x  \\\n",
       "0             NaN     NaN             NaN                  NaN   \n",
       "1             NaN     NaN             NaN                  NaN   \n",
       "\n",
       "  INSDC center name_x INSDC first public_x INSDC last update_x INSDC status_x  \\\n",
       "0                 NaN                  NaN                 NaN            NaN   \n",
       "1                 NaN                  NaN                 NaN            NaN   \n",
       "\n",
       "  SRA accession_x Sample Name_x Title_x serotype subsrc_note_x  \\\n",
       "0             NaN           NaN     NaN      NaN           NaN   \n",
       "1             NaN           NaN     NaN      NaN           NaN   \n",
       "\n",
       "  annotation_method assembly_method_x biosamplemodel  \\\n",
       "0               NaN               NaN            NaN   \n",
       "1               NaN               NaN            NaN   \n",
       "\n",
       "  geographic location (country and/or atlantic ocean)  \\\n",
       "0                                                NaN    \n",
       "1                                                NaN    \n",
       "\n",
       "  geographic location (region and locality) mapping_method_x  \\\n",
       "0                                       NaN              NaN   \n",
       "1                                       NaN              NaN   \n",
       "\n",
       "  quality_assessment_method_x refinement_method Cell Shape_y  \\\n",
       "0                         NaN               NaN          NaN   \n",
       "1                         NaN               NaN          NaN   \n",
       "\n",
       "  Funding Program_x GOLD Stamp ID_x Gene Calling Method_x Gram Staining_x  \\\n",
       "0               NaN             NaN                   NaN             NaN   \n",
       "1               NaN             NaN                   NaN             NaN   \n",
       "\n",
       "  Isolation Comments_y Isolation Site_x Motility_y Temperature Optimum_x  \\\n",
       "0                  NaN              NaN        NaN                   NaN   \n",
       "1                  NaN              NaN        NaN                   NaN   \n",
       "\n",
       "  Temperature Range_y Type Strain_y alt_elev_x biotic_relationship_x  \\\n",
       "0                 NaN           NaN        NaN                   NaN   \n",
       "1                 NaN           NaN        NaN                   NaN   \n",
       "\n",
       "  country_x environment_x investigation_type_x isolation-source_x  \\\n",
       "0       NaN           NaN                  NaN                NaN   \n",
       "1       NaN           NaN                  NaN                NaN   \n",
       "\n",
       "  trophic_level_x pH_x Body Sample Site_y Body Sample SubSite_x  Diseases_x  \\\n",
       "0             NaN  NaN                NaN                   NaN         NaN   \n",
       "1             NaN  NaN                NaN                   NaN         NaN   \n",
       "\n",
       "  Host Name_y Host Health_y Sporulation_y experimental_factor lat-lon  \\\n",
       "0         NaN           NaN           NaN                 NaN     NaN   \n",
       "1         NaN           NaN           NaN                 NaN     NaN   \n",
       "\n",
       "  ENA-CHECKLIST External Id Submitter Id subsrc-note collection date_x  \\\n",
       "0           NaN         NaN          NaN         NaN               NaN   \n",
       "1           NaN         NaN          NaN         NaN               NaN   \n",
       "\n",
       "  environment (biome)_x environment (feature)_x environment (material)_x  \\\n",
       "0                   NaN                     NaN                      NaN   \n",
       "1                   NaN                     NaN                      NaN   \n",
       "\n",
       "  geographic location (country and/or sea)_x geographic location (latitude)_x  \\\n",
       "0                                        NaN                              NaN   \n",
       "1                                        NaN                              NaN   \n",
       "\n",
       "  geographic location (longitude)_x host common name  \\\n",
       "0                               NaN              NaN   \n",
       "1                               NaN              NaN   \n",
       "\n",
       "  host-associated environmental package investigation type_x project name_x  \\\n",
       "0                                   NaN                  NaN            NaN   \n",
       "1                                   NaN                  NaN            NaN   \n",
       "\n",
       "  sequencing method_x STRAIN description  completeness  contamination  \\\n",
       "0                 NaN    NaN         NaN           NaN            NaN   \n",
       "1                 NaN    NaN         NaN           NaN            NaN   \n",
       "\n",
       "  host_age_x host_sex_x oxy_stat_samp samp_store_dur samp_store_temp  \\\n",
       "0        NaN        NaN           NaN            NaN             NaN   \n",
       "1        NaN        NaN           NaN            NaN             NaN   \n",
       "\n",
       "  FDA CVM ID_x bio_material common name host scientific name_x  \\\n",
       "0          NaN          NaN         NaN                    NaN   \n",
       "1          NaN          NaN         NaN                    NaN   \n",
       "\n",
       "  samp_mat_process estimated_size_x  Tax ID Collaborator ID host_description  \\\n",
       "0              NaN              NaN     NaN             NaN              NaN   \n",
       "1              NaN              NaN     NaN             NaN              NaN   \n",
       "\n",
       "  host_disease_outcome_x host_disease_stage host_health_state  \\\n",
       "0                    NaN                NaN               NaN   \n",
       "1                    NaN                NaN               NaN   \n",
       "\n",
       "  identification method_x strain_name_alias_x WGA amplification approach  \\\n",
       "0                     NaN                 NaN                        NaN   \n",
       "1                     NaN                 NaN                        NaN   \n",
       "\n",
       "  metagenomic source sample derived from  \\\n",
       "0                NaN                 NaN   \n",
       "1                NaN                 NaN   \n",
       "\n",
       "  single cell or viral particle lysis approach sorting technology  \\\n",
       "0                                          NaN                NaN   \n",
       "1                                          NaN                NaN   \n",
       "\n",
       "  taxonomic identity marker passage_history_ samp_size Description  \\\n",
       "0                       NaN              NaN       NaN         NaN   \n",
       "1                       NaN              NaN       NaN         NaN   \n",
       "\n",
       "  subsource_note lab_host_ collection_room  relative_week assembly_tool  \\\n",
       "0            NaN       NaN             NaN            NaN           NaN   \n",
       "1            NaN       NaN             NaN            NaN           NaN   \n",
       "\n",
       "  bin_id Broker name  MAG_number derived_from  \\\n",
       "0    NaN         NaN         NaN          NaN   \n",
       "1    NaN         NaN         NaN          NaN   \n",
       "\n",
       "  alkalinity (carbonate/bicarbonate) salinity Phenotypes_x  env_package  \\\n",
       "0                                NaN      NaN          NaN          NaN   \n",
       "1                                NaN      NaN          NaN          NaN   \n",
       "\n",
       "  specific_host_x binning parameters plant-associated environmental package_x  \\\n",
       "0             NaN                NaN                                      NaN   \n",
       "1             NaN                NaN                                      NaN   \n",
       "\n",
       "  culture collection repository risk group type status Assembly Method_y  \\\n",
       "0                NaN        NaN        NaN         NaN               NaN   \n",
       "1                NaN        NaN        NaN         NaN               NaN   \n",
       "\n",
       "  Genome Coverage  Sequencing Technology nat-host_x collection-date  \\\n",
       "0             NaN                    NaN        NaN             NaN   \n",
       "1             NaN                    NaN        NaN             NaN   \n",
       "\n",
       "  Historical Monobactam Produced_x Unique Monobactam Clusters Identified_x  \\\n",
       "0                              NaN                                     NaN   \n",
       "1                              NaN                                     NaN   \n",
       "\n",
       "   extrachrom_elements pathogenicity source_material_id genotype  \\\n",
       "0                  NaN           NaN                NaN      NaN   \n",
       "1                  NaN           NaN                NaN      NaN   \n",
       "\n",
       "   mating_type_ serovar_x specimen_voucher_  Salinity (PSU) collected-by  \\\n",
       "0           NaN       NaN               NaN             NaN          NaN   \n",
       "1           NaN       NaN               NaN             NaN          NaN   \n",
       "\n",
       "  orgmod_note assembly_x sequencing_meth_x finishing_strategy_x  \\\n",
       "0         NaN        NaN               NaN                  NaN   \n",
       "1         NaN        NaN               NaN                  NaN   \n",
       "\n",
       "  geo-loc-name_x finishing strategy (depth of coverage)_x  host taxid_x  \\\n",
       "0            NaN                                      NaN           NaN   \n",
       "1            NaN                                      NaN           NaN   \n",
       "\n",
       "  misc_param: HMP body site misc_param: HMP supersite  \\\n",
       "0                       NaN                       NaN   \n",
       "1                       NaN                       NaN   \n",
       "\n",
       "   nucleic acid extraction project_type_x sop_x source_mat_id_x Salinity_y  \\\n",
       "0                      NaN            NaN   NaN             NaN        NaN   \n",
       "1                      NaN            NaN   NaN             NaN        NaN   \n",
       "\n",
       "  body_site health_state Isolation source_x Host_x isolation source_x  \\\n",
       "0       NaN          NaN                NaN    NaN                NaN   \n",
       "1       NaN          NaN                NaN    NaN                NaN   \n",
       "\n",
       "   environmental package specific host_x  lat lon host_subject_id pathotype  \\\n",
       "0                    NaN             NaN      NaN             NaN       NaN   \n",
       "1                    NaN             NaN      NaN             NaN       NaN   \n",
       "\n",
       "   subgroup  subtype  Extraction Date  Extraction Method  Laboratory Host  \\\n",
       "0       NaN      NaN              NaN                NaN              NaN   \n",
       "1       NaN      NaN              NaN                NaN              NaN   \n",
       "\n",
       "   Passage Date  sample type host health state_x anonymized_name_x  \\\n",
       "0           NaN          NaN                 NaN               NaN   \n",
       "1           NaN          NaN                 NaN               NaN   \n",
       "\n",
       "  host_status supplier_name_x  abs_air_humidity  air_temp build_occup_type  \\\n",
       "0         NaN             NaN               NaN       NaN              NaN   \n",
       "1         NaN             NaN               NaN       NaN              NaN   \n",
       "\n",
       "  building_setting carb_dioxide filter_type heat_cool_type indoor_space  \\\n",
       "0              NaN          NaN         NaN            NaN          NaN   \n",
       "1              NaN          NaN         NaN            NaN          NaN   \n",
       "\n",
       "  light_type occup_samp occupant_dens_samp  organism_count rel_air_humidity  \\\n",
       "0        NaN        NaN                NaN             NaN              NaN   \n",
       "1        NaN        NaN                NaN             NaN              NaN   \n",
       "\n",
       "  space_typ_state typ_occupant_dens ventilation_type Type-material  \\\n",
       "0             NaN               NaN              NaN           NaN   \n",
       "1             NaN               NaN              NaN           NaN   \n",
       "\n",
       "  samp_collect_device  geographic location (altitude) env_packatge  \\\n",
       "0                 NaN                             NaN          NaN   \n",
       "1                 NaN                             NaN          NaN   \n",
       "\n",
       "  lib_reads_seqd metagenomic_source env material other identifiers  \\\n",
       "0            NaN                NaN          NaN               NaN   \n",
       "1            NaN                NaN          NaN               NaN   \n",
       "\n",
       "  assembly_method_version_x mapping_method_version_x  \\\n",
       "0                       NaN                      NaN   \n",
       "1                       NaN                      NaN   \n",
       "\n",
       "  quality_assessment_method_version_x value_x  cyanobacterial_culture  \\\n",
       "0                                 NaN     NaN                     NaN   \n",
       "1                                 NaN     NaN                     NaN   \n",
       "\n",
       "   missing samp_store_loc ethnicity host_body_habitat  host_taxid  \\\n",
       "0      NaN            NaN       NaN               NaN         NaN   \n",
       "1      NaN            NaN       NaN               NaN         NaN   \n",
       "\n",
       "  misc_param_x  ph metagenome-source seawater metagenome type_status  \\\n",
       "0          NaN NaN                                   NaN         NaN   \n",
       "1          NaN NaN                                   NaN         NaN   \n",
       "\n",
       "  indoor_surf samp_sort_meth samp_vol_we_dna_ext surf_material  \\\n",
       "0         NaN            NaN                 NaN           NaN   \n",
       "1         NaN            NaN                 NaN           NaN   \n",
       "\n",
       "  BioSampleModel_x ammonium biomass  carb_nitro_ratio chlorophyll  \\\n",
       "0              NaN      NaN     NaN               NaN         NaN   \n",
       "1              NaN      NaN     NaN               NaN         NaN   \n",
       "\n",
       "  diss_inorg_nitro diss_inorg_phosp diss_org_carb diss_org_nitro nitrate  \\\n",
       "0              NaN              NaN           NaN            NaN     NaN   \n",
       "1              NaN              NaN           NaN            NaN     NaN   \n",
       "\n",
       "  nitrite phosphate silicate tidal_stage tot_diss_nitro locus_tag prefix  \\\n",
       "0     NaN       NaN      NaN         NaN            NaN              NaN   \n",
       "1     NaN       NaN      NaN         NaN            NaN              NaN   \n",
       "\n",
       "  assembly_accession  strain_y finishing strategy (depth of coverage)_y  \\\n",
       "0                NaN       NaN                                      NaN   \n",
       "1    GCA_000023285.1  DSM 7271                                      NaN   \n",
       "\n",
       "  biotic_relationship_y geo_loc_name_y host_y collected_by_y host_disease_y  \\\n",
       "0                   NaN            NaN    NaN            NaN            NaN   \n",
       "1                   NaN            NaN    NaN            NaN            NaN   \n",
       "\n",
       "  isolation_source_y sample_name_y isolation-source_y sample_type_y  \\\n",
       "0                NaN           NaN                NaN           NaN   \n",
       "1                NaN           NaN                NaN           NaN   \n",
       "\n",
       "  FDA CVM ID_y culture_collection_y assembly_method_y collection_date_y  \\\n",
       "0          NaN                  NaN               NaN               NaN   \n",
       "1          NaN                  NaN               NaN               NaN   \n",
       "\n",
       "  env_feature_y synonym Strain SUBJECT_ID_y Alias_y  \\\n",
       "0           NaN     NaN    NaN          NaN     NaN   \n",
       "1           NaN     NaN    NaN          NaN     NaN   \n",
       "\n",
       "                          type-material_y misc_param_y collection date_y  \\\n",
       "0                                     NaN          NaN               NaN   \n",
       "1  type strain of Capnocytophaga ochracea          NaN               NaN   \n",
       "\n",
       "  Isolation Site_y Isolation source_y country_y note_y GOLD Stamp ID_y  \\\n",
       "0              NaN                NaN       NaN    NaN             NaN   \n",
       "1              NaN                NaN       NaN    NaN             NaN   \n",
       "\n",
       "  investigation_type_y culture-collection_y lat_lon_y alt_elev_y  \\\n",
       "0                  NaN                  NaN       NaN        NaN   \n",
       "1                  NaN                  NaN       NaN        NaN   \n",
       "\n",
       "  locus_tag_prefix_y nat-host_y assembly_method_version_y env_material_y  \\\n",
       "0                NaN        NaN                       NaN            NaN   \n",
       "1                NaN        NaN                       NaN            NaN   \n",
       "\n",
       "  model  sample_description  Sample Description ArrayExpress-STRAIN_OR_LINE  \\\n",
       "0   NaN                 NaN                 NaN                         NaN   \n",
       "1   NaN                 NaN                 NaN                         NaN   \n",
       "\n",
       "  ENA checklist_y project_type_y  estimated_size_y Temperature Optimum_y  \\\n",
       "0             NaN            NaN               NaN                   NaN   \n",
       "1             NaN            NaN               NaN                   NaN   \n",
       "\n",
       "   isol_growth_condt_y Motility feature_y project_name_y assembly_y  \\\n",
       "0                  NaN      NaN       NaN            NaN        NaN   \n",
       "1                  NaN      NaN       NaN            NaN        NaN   \n",
       "\n",
       "   num_replicons_y isolate_ depth_y env_biome_y altitude_y ref_biomaterial_y  \\\n",
       "0              NaN      NaN     NaN         NaN        NaN               NaN   \n",
       "1              NaN      NaN     NaN         NaN        NaN               NaN   \n",
       "\n",
       "  package geographic location ArrayExpress-StrainOrLine  \\\n",
       "0     NaN                 NaN                       NaN   \n",
       "1     NaN                 NaN                       NaN   \n",
       "\n",
       "   ArrayExpress-Phenotype ArrayExpress-SPECIES INSDC center alias_y  \\\n",
       "0                     NaN                  NaN                  NaN   \n",
       "1                     NaN                  NaN                  NaN   \n",
       "\n",
       "  geo-loc-name_y environment_y host_sex_y  elev_y PublicAccession  \\\n",
       "0            NaN           NaN        NaN     NaN             NaN   \n",
       "1            NaN           NaN        NaN     NaN             NaN   \n",
       "\n",
       "  completeness_estimated_y geographic location (country and/or sea)_y  \\\n",
       "0                      NaN                                        NaN   \n",
       "1                      NaN                                        NaN   \n",
       "\n",
       "  latitude and longitude specific_host_y ArrayExpress-Species  \\\n",
       "0                    NaN             NaN                  NaN   \n",
       "1                    NaN             NaN                  NaN   \n",
       "\n",
       "  ENA-FIRST-PUBLIC_y INSDC center name_y specific host_y Body Sample Site  \\\n",
       "0                NaN                 NaN             NaN              NaN   \n",
       "1                NaN                 NaN             NaN              NaN   \n",
       "\n",
       "  pH_y biome_y biomaterial_provider_y isolate_name_alias_y  \\\n",
       "0  NaN     NaN                    NaN                  NaN   \n",
       "1  NaN     NaN                    NaN                  NaN   \n",
       "\n",
       "  contamination_estimated_y host scientific name_y  anonymized_name_y  \\\n",
       "0                       NaN                    NaN                NaN   \n",
       "1                       NaN                    NaN                NaN   \n",
       "\n",
       "  ENA-LAST-UPDATE_y INSDC first public_y Cell Shape BioSampleModel_y  \\\n",
       "0               NaN                  NaN        NaN              NaN   \n",
       "1               NaN                  NaN        NaN              NaN   \n",
       "\n",
       "  environmental_sample_y material_y identified_by_y host health state_y  \\\n",
       "0                    NaN        NaN             NaN                 NaN   \n",
       "1                    NaN        NaN             NaN                 NaN   \n",
       "\n",
       "  INSDC last update_y Diseases_y strain_name_alias_y  genome_coverage_y  \\\n",
       "0                 NaN        NaN                 NaN                NaN   \n",
       "1                 NaN        NaN                 NaN                NaN   \n",
       "\n",
       "  INSDC status_y  host taxid_y isolation source_y Gram Staining_y  \\\n",
       "0            NaN           NaN                NaN             NaN   \n",
       "1            NaN           NaN                NaN             NaN   \n",
       "\n",
       "  Temperature Range Sporulation sequencing_meth_y  \\\n",
       "0               NaN         NaN               NaN   \n",
       "1               NaN         NaN               NaN   \n",
       "\n",
       "  Historical Monobactam Produced_y identification method_y supplier_name_y  \\\n",
       "0                              NaN                     NaN             NaN   \n",
       "1                              NaN                     NaN             NaN   \n",
       "\n",
       "  SRA accession_y host_disease_outcome_y  \\\n",
       "0             NaN                    NaN   \n",
       "1             NaN                    NaN   \n",
       "\n",
       "  Unique Monobactam Clusters Identified_y ProjectAccession Sample Name_y  \\\n",
       "0                                     NaN              NaN           NaN   \n",
       "1                                     NaN              NaN           NaN   \n",
       "\n",
       "  Type Strain Phenotypes_y Gene Calling Method_y temp_y Title_y  \\\n",
       "0         NaN          NaN                   NaN    NaN     NaN   \n",
       "1         NaN          NaN                   NaN    NaN     NaN   \n",
       "\n",
       "  investigation type_y Host_y rel_to_oxygen_y host_age_y Species  \\\n",
       "0                  NaN    NaN             NaN        NaN     NaN   \n",
       "1                  NaN    NaN             NaN        NaN     NaN   \n",
       "\n",
       "  mapping_method_y  serovar_y sop_y Isolation Comments finishing_strategy_y  \\\n",
       "0              NaN        NaN   NaN                NaN                  NaN   \n",
       "1              NaN        NaN   NaN                NaN                  NaN   \n",
       "\n",
       "  Genus mapping_method_version_y environment (biome)_y attribute_package  \\\n",
       "0   NaN                      NaN                   NaN               NaN   \n",
       "1   NaN                      NaN                   NaN               NaN   \n",
       "\n",
       "  metagenome_source_y environment (feature)_y trophic_level_y metagenomic_y  \\\n",
       "0                 NaN                     NaN             NaN           NaN   \n",
       "1                 NaN                     NaN             NaN           NaN   \n",
       "\n",
       "  environment (material)_y Funding Program_y quality_assessment_method_y  \\\n",
       "0                      NaN               NaN                         NaN   \n",
       "1                      NaN               NaN                         NaN   \n",
       "\n",
       "  quality_assessment_method_version_y  geographic location (latitude)_y  \\\n",
       "0                                 NaN                               NaN   \n",
       "1                                 NaN                               NaN   \n",
       "\n",
       "  source_mat_id_y host_tissue_sampled_y  geographic location (longitude)_y  \\\n",
       "0             NaN                   NaN                                NaN   \n",
       "1             NaN                   NaN                                NaN   \n",
       "\n",
       "  Host Name subsrc_note_y Host Health value_y  \\\n",
       "0       NaN           NaN         NaN     NaN   \n",
       "1       NaN           NaN         NaN     NaN   \n",
       "\n",
       "  plant-associated environmental package_y project name_y sequencing method_y  \\\n",
       "0                                      NaN            NaN                 NaN   \n",
       "1                                      NaN            NaN                 NaN   \n",
       "\n",
       "  Body Sample SubSite_y collected_by sample_type biomaterial_provider  \\\n",
       "0                   NaN          NaN         NaN                  NaN   \n",
       "1                   NaN          NaN         NaN                  NaN   \n",
       "\n",
       "  identified_by passage_historyz isolatez specimen_voucherz     lab_hostz  \\\n",
       "0           NaN              NaN      NaN               NaN NaN       NaN   \n",
       "1           NaN              NaN      NaN               NaN NaN       NaN   \n",
       "\n",
       "   mating_typez  \n",
       "0           NaN  \n",
       "1           NaN  "
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merge.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th># assembly_accession</th>\n",
       "      <th>16S recovered</th>\n",
       "      <th>16S recovery software</th>\n",
       "      <th>Alias_x</th>\n",
       "      <th>Alias_y</th>\n",
       "      <th>Altitude</th>\n",
       "      <th>ArrayExpress-Phenotype</th>\n",
       "      <th>ArrayExpress-SPECIES</th>\n",
       "      <th>ArrayExpress-STRAIN_OR_LINE</th>\n",
       "      <th>ArrayExpress-Species</th>\n",
       "      <th>ArrayExpress-StrainOrLine</th>\n",
       "      <th>Assembly Accession</th>\n",
       "      <th>Assembly Method_x</th>\n",
       "      <th>Assembly Method_y</th>\n",
       "      <th>BioProject Accession</th>\n",
       "      <th>BioSample Accession</th>\n",
       "      <th>BioSampleModel_x</th>\n",
       "      <th>BioSampleModel_y</th>\n",
       "      <th>Body Sample Site</th>\n",
       "      <th>Body Sample Site_x</th>\n",
       "      <th>Body Sample Site_y</th>\n",
       "      <th>Body Sample SubSite_x</th>\n",
       "      <th>Body Sample SubSite_y</th>\n",
       "      <th>Broker name</th>\n",
       "      <th>Cell Shape</th>\n",
       "      <th>Cell Shape_x</th>\n",
       "      <th>Cell Shape_y</th>\n",
       "      <th>Chromosomes</th>\n",
       "      <th>Collaborator ID</th>\n",
       "      <th>Collection Date</th>\n",
       "      <th>Comments</th>\n",
       "      <th>Completion Date</th>\n",
       "      <th>Contigs</th>\n",
       "      <th>Culture Collection</th>\n",
       "      <th>Depth_x</th>\n",
       "      <th>Description</th>\n",
       "      <th>Disease</th>\n",
       "      <th>Diseases_x</th>\n",
       "      <th>Diseases_y</th>\n",
       "      <th>ENA checklist_x</th>\n",
       "      <th>ENA checklist_y</th>\n",
       "      <th>ENA-CHECKLIST</th>\n",
       "      <th>ENA-FIRST-PUBLIC_x</th>\n",
       "      <th>ENA-FIRST-PUBLIC_y</th>\n",
       "      <th>ENA-LAST-UPDATE_x</th>\n",
       "      <th>ENA-LAST-UPDATE_y</th>\n",
       "      <th>External Id</th>\n",
       "      <th>Extraction Date</th>\n",
       "      <th>Extraction Method</th>\n",
       "      <th>FDA CVM ID_x</th>\n",
       "      <th>FDA CVM ID_y</th>\n",
       "      <th>Funding Program_x</th>\n",
       "      <th>Funding Program_y</th>\n",
       "      <th>GC Content</th>\n",
       "      <th>GOLD Stamp ID_x</th>\n",
       "      <th>GOLD Stamp ID_y</th>\n",
       "      <th>GenBank Accessions</th>\n",
       "      <th>Gene Calling Method_x</th>\n",
       "      <th>Gene Calling Method_y</th>\n",
       "      <th>Genome Coverage</th>\n",
       "      <th>Genome Length</th>\n",
       "      <th>Genome Name</th>\n",
       "      <th>Genome Quality Flags</th>\n",
       "      <th>Genome Status</th>\n",
       "      <th>Genome_ID</th>\n",
       "      <th>Genus</th>\n",
       "      <th>Geographic Location</th>\n",
       "      <th>Gram Stain</th>\n",
       "      <th>Gram Staining_x</th>\n",
       "      <th>Gram Staining_y</th>\n",
       "      <th>Habitat</th>\n",
       "      <th>Historical Monobactam Produced_x</th>\n",
       "      <th>Historical Monobactam Produced_y</th>\n",
       "      <th>Host Age</th>\n",
       "      <th>Host Gender</th>\n",
       "      <th>Host Health</th>\n",
       "      <th>Host Health_x</th>\n",
       "      <th>Host Health_y</th>\n",
       "      <th>Host Name</th>\n",
       "      <th>Host Name_x</th>\n",
       "      <th>Host Name_y</th>\n",
       "      <th>Host_x</th>\n",
       "      <th>Host_y</th>\n",
       "      <th>INSDC center alias_x</th>\n",
       "      <th>INSDC center alias_y</th>\n",
       "      <th>INSDC center name_x</th>\n",
       "      <th>INSDC center name_y</th>\n",
       "      <th>INSDC first public_x</th>\n",
       "      <th>INSDC first public_y</th>\n",
       "      <th>INSDC last update_x</th>\n",
       "      <th>INSDC last update_y</th>\n",
       "      <th>INSDC status_x</th>\n",
       "      <th>INSDC status_y</th>\n",
       "      <th>Isolation Comments</th>\n",
       "      <th>Isolation Comments_x</th>\n",
       "      <th>Isolation Comments_y</th>\n",
       "      <th>Isolation Country</th>\n",
       "      <th>Isolation Site_x</th>\n",
       "      <th>Isolation Site_y</th>\n",
       "      <th>Isolation Source</th>\n",
       "      <th>Isolation source_x</th>\n",
       "      <th>Isolation source_y</th>\n",
       "      <th>Laboratory Host</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>MAG_number</th>\n",
       "      <th>MLST</th>\n",
       "      <th>Motility</th>\n",
       "      <th>Motility_x</th>\n",
       "      <th>Motility_y</th>\n",
       "      <th>NCBI Taxon ID</th>\n",
       "      <th>Optimal Temperature</th>\n",
       "      <th>Organism Name</th>\n",
       "      <th>Other Clinical</th>\n",
       "      <th>Other Environmental</th>\n",
       "      <th>Other Typing</th>\n",
       "      <th>Oxygen Requirement</th>\n",
       "      <th>Passage Date</th>\n",
       "      <th>Pathovar</th>\n",
       "      <th>Phenotypes_x</th>\n",
       "      <th>Phenotypes_y</th>\n",
       "      <th>Plasmids</th>\n",
       "      <th>ProjectAccession</th>\n",
       "      <th>PublicAccession</th>\n",
       "      <th>Publication</th>\n",
       "      <th>RefSeq Accessions</th>\n",
       "      <th>RefSeq CDS</th>\n",
       "      <th>SRA Accession</th>\n",
       "      <th>SRA accession_x</th>\n",
       "      <th>SRA accession_y</th>\n",
       "      <th>STRAIN</th>\n",
       "      <th>SUBJECT_ID_x</th>\n",
       "      <th>SUBJECT_ID_y</th>\n",
       "      <th>Salinity (PSU)</th>\n",
       "      <th>Salinity_x</th>\n",
       "      <th>Salinity_y</th>\n",
       "      <th>Sample Description</th>\n",
       "      <th>Sample Name_x</th>\n",
       "      <th>Sample Name_y</th>\n",
       "      <th>Sequencing Centers</th>\n",
       "      <th>Sequencing Depth</th>\n",
       "      <th>Sequencing Platform</th>\n",
       "      <th>Sequencing Status</th>\n",
       "      <th>Sequencing Technology</th>\n",
       "      <th>Serovar</th>\n",
       "      <th>Species</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>Sporulation_x</th>\n",
       "      <th>Sporulation_y</th>\n",
       "      <th>Strain</th>\n",
       "      <th>Strain_x</th>\n",
       "      <th>Submitter Id</th>\n",
       "      <th>Tax ID</th>\n",
       "      <th>Temperature Optimum_x</th>\n",
       "      <th>Temperature Optimum_y</th>\n",
       "      <th>Temperature Range</th>\n",
       "      <th>Temperature Range_x</th>\n",
       "      <th>Temperature Range_y</th>\n",
       "      <th>Title_x</th>\n",
       "      <th>Title_y</th>\n",
       "      <th>Type Strain</th>\n",
       "      <th>Type Strain_x</th>\n",
       "      <th>Type Strain_y</th>\n",
       "      <th>Type-material</th>\n",
       "      <th>Unique Monobactam Clusters Identified_x</th>\n",
       "      <th>Unique Monobactam Clusters Identified_y</th>\n",
       "      <th>WGA amplification approach</th>\n",
       "      <th>abs_air_humidity</th>\n",
       "      <th>air_temp</th>\n",
       "      <th>alkalinity (carbonate/bicarbonate)</th>\n",
       "      <th>alt_elev_x</th>\n",
       "      <th>alt_elev_y</th>\n",
       "      <th>altitude_x</th>\n",
       "      <th>altitude_y</th>\n",
       "      <th>ammonium</th>\n",
       "      <th>analysis project type</th>\n",
       "      <th>annotation_method</th>\n",
       "      <th>anonymized_name_x</th>\n",
       "      <th>anonymized_name_y</th>\n",
       "      <th>asm_name</th>\n",
       "      <th>assembly quality</th>\n",
       "      <th>assembly software</th>\n",
       "      <th>assembly_accession</th>\n",
       "      <th>assembly_level</th>\n",
       "      <th>assembly_method_and_version</th>\n",
       "      <th>assembly_method_version_x</th>\n",
       "      <th>assembly_method_version_y</th>\n",
       "      <th>assembly_method_x</th>\n",
       "      <th>assembly_method_y</th>\n",
       "      <th>assembly_tool</th>\n",
       "      <th>assembly_x</th>\n",
       "      <th>assembly_y</th>\n",
       "      <th>attribute_package</th>\n",
       "      <th>bin parameters</th>\n",
       "      <th>bin_id</th>\n",
       "      <th>binning parameters</th>\n",
       "      <th>binning software</th>\n",
       "      <th>bio_material</th>\n",
       "      <th>biomass</th>\n",
       "      <th>biomaterial_provider</th>\n",
       "      <th>biomaterial_provider_x</th>\n",
       "      <th>biomaterial_provider_y</th>\n",
       "      <th>biome_x</th>\n",
       "      <th>biome_y</th>\n",
       "      <th>bioproject</th>\n",
       "      <th>biosample</th>\n",
       "      <th>biosample_title</th>\n",
       "      <th>biosamplemodel</th>\n",
       "      <th>biotic_relationship_x</th>\n",
       "      <th>biotic_relationship_y</th>\n",
       "      <th>body_site</th>\n",
       "      <th>build_occup_type</th>\n",
       "      <th>building_setting</th>\n",
       "      <th>carb_dioxide</th>\n",
       "      <th>carb_nitro_ratio</th>\n",
       "      <th>chlorophyll</th>\n",
       "      <th>collected-by</th>\n",
       "      <th>collected_by</th>\n",
       "      <th>collected_by_x</th>\n",
       "      <th>collected_by_y</th>\n",
       "      <th>collection date_x</th>\n",
       "      <th>collection date_y</th>\n",
       "      <th>collection-date</th>\n",
       "      <th>collection_date_x</th>\n",
       "      <th>collection_date_y</th>\n",
       "      <th>collection_room</th>\n",
       "      <th>comment_paragraph</th>\n",
       "      <th>common name</th>\n",
       "      <th>completeness</th>\n",
       "      <th>completeness score</th>\n",
       "      <th>completeness software</th>\n",
       "      <th>completeness_estimated_x</th>\n",
       "      <th>completeness_estimated_y</th>\n",
       "      <th>contact_email</th>\n",
       "      <th>contact_lab</th>\n",
       "      <th>contamination</th>\n",
       "      <th>contamination score</th>\n",
       "      <th>contamination_estimated_x</th>\n",
       "      <th>contamination_estimated_y</th>\n",
       "      <th>contig L50</th>\n",
       "      <th>country_x</th>\n",
       "      <th>country_y</th>\n",
       "      <th>culture collection</th>\n",
       "      <th>culture-collection_x</th>\n",
       "      <th>culture-collection_y</th>\n",
       "      <th>culture_collection_x</th>\n",
       "      <th>culture_collection_y</th>\n",
       "      <th>cyanobacterial_culture</th>\n",
       "      <th>decontamination software</th>\n",
       "      <th>depth_x</th>\n",
       "      <th>depth_y</th>\n",
       "      <th>derived-from</th>\n",
       "      <th>derived_from</th>\n",
       "      <th>description</th>\n",
       "      <th>diss_inorg_nitro</th>\n",
       "      <th>diss_inorg_phosp</th>\n",
       "      <th>diss_org_carb</th>\n",
       "      <th>diss_org_nitro</th>\n",
       "      <th>elev_x</th>\n",
       "      <th>elev_y</th>\n",
       "      <th>env material</th>\n",
       "      <th>env_biome_x</th>\n",
       "      <th>env_biome_y</th>\n",
       "      <th>env_broad_scale</th>\n",
       "      <th>env_feature_x</th>\n",
       "      <th>env_feature_y</th>\n",
       "      <th>env_local_scale</th>\n",
       "      <th>env_material_x</th>\n",
       "      <th>env_material_y</th>\n",
       "      <th>env_medium</th>\n",
       "      <th>env_package</th>\n",
       "      <th>env_packatge</th>\n",
       "      <th>environment (biome)_x</th>\n",
       "      <th>environment (biome)_y</th>\n",
       "      <th>environment (feature)_x</th>\n",
       "      <th>environment (feature)_y</th>\n",
       "      <th>environment (material)_x</th>\n",
       "      <th>environment (material)_y</th>\n",
       "      <th>environment_x</th>\n",
       "      <th>environment_y</th>\n",
       "      <th>environmental package</th>\n",
       "      <th>environmental-sample</th>\n",
       "      <th>environmental_sample_x</th>\n",
       "      <th>environmental_sample_y</th>\n",
       "      <th>environmetal-sample</th>\n",
       "      <th>estimated_size_x</th>\n",
       "      <th>estimated_size_y</th>\n",
       "      <th>ethnicity</th>\n",
       "      <th>excluded_from_refseq</th>\n",
       "      <th>experimental_factor</th>\n",
       "      <th>extrachrom_elements</th>\n",
       "      <th>feature_x</th>\n",
       "      <th>feature_y</th>\n",
       "      <th>filter_type</th>\n",
       "      <th>finishing strategy (depth of coverage)_x</th>\n",
       "      <th>finishing strategy (depth of coverage)_y</th>\n",
       "      <th>finishing_strategy_x</th>\n",
       "      <th>finishing_strategy_y</th>\n",
       "      <th>first_name</th>\n",
       "      <th>ftp_path</th>\n",
       "      <th>gbrs_paired_asm</th>\n",
       "      <th>genome_coverage_x</th>\n",
       "      <th>genome_coverage_y</th>\n",
       "      <th>genome_rep</th>\n",
       "      <th>genotype</th>\n",
       "      <th>geo-loc-name_x</th>\n",
       "      <th>geo-loc-name_y</th>\n",
       "      <th>geo_loc_name_x</th>\n",
       "      <th>geo_loc_name_y</th>\n",
       "      <th>geographic location</th>\n",
       "      <th>geographic location (altitude)</th>\n",
       "      <th>geographic location (country and/or atlantic ocean)</th>\n",
       "      <th>geographic location (country and/or sea)_x</th>\n",
       "      <th>geographic location (country and/or sea)_y</th>\n",
       "      <th>geographic location (latitude)_x</th>\n",
       "      <th>geographic location (latitude)_y</th>\n",
       "      <th>geographic location (longitude)_x</th>\n",
       "      <th>geographic location (longitude)_y</th>\n",
       "      <th>geographic location (region and locality)</th>\n",
       "      <th>health_state</th>\n",
       "      <th>heat_cool_type</th>\n",
       "      <th>host common name</th>\n",
       "      <th>host health state_x</th>\n",
       "      <th>host health state_y</th>\n",
       "      <th>host scientific name_x</th>\n",
       "      <th>host scientific name_y</th>\n",
       "      <th>host taxid_x</th>\n",
       "      <th>host taxid_y</th>\n",
       "      <th>host-associated environmental package</th>\n",
       "      <th>host_age_x</th>\n",
       "      <th>host_age_y</th>\n",
       "      <th>host_body_habitat</th>\n",
       "      <th>host_description</th>\n",
       "      <th>host_disease_outcome_x</th>\n",
       "      <th>host_disease_outcome_y</th>\n",
       "      <th>host_disease_stage</th>\n",
       "      <th>host_disease_x</th>\n",
       "      <th>host_disease_y</th>\n",
       "      <th>host_health_state</th>\n",
       "      <th>host_sex_x</th>\n",
       "      <th>host_sex_y</th>\n",
       "      <th>host_status</th>\n",
       "      <th>host_subject_id</th>\n",
       "      <th>host_taxid</th>\n",
       "      <th>host_tissue_sampled_x</th>\n",
       "      <th>host_tissue_sampled_y</th>\n",
       "      <th>host_x</th>\n",
       "      <th>host_y</th>\n",
       "      <th>identification method_x</th>\n",
       "      <th>identification method_y</th>\n",
       "      <th>identified_by</th>\n",
       "      <th>identified_by_x</th>\n",
       "      <th>identified_by_y</th>\n",
       "      <th>indoor_space</th>\n",
       "      <th>indoor_surf</th>\n",
       "      <th>infraspecific_name</th>\n",
       "      <th>investigation type_x</th>\n",
       "      <th>investigation type_y</th>\n",
       "      <th>investigation_type_x</th>\n",
       "      <th>investigation_type_y</th>\n",
       "      <th>isol_growth_condt_x</th>\n",
       "      <th>isol_growth_condt_y</th>\n",
       "      <th>isolate_</th>\n",
       "      <th>isolate_name_alias_x</th>\n",
       "      <th>isolate_name_alias_y</th>\n",
       "      <th>isolate_x</th>\n",
       "      <th>isolate_y</th>\n",
       "      <th>isolatez</th>\n",
       "      <th>isolation source_x</th>\n",
       "      <th>isolation source_y</th>\n",
       "      <th>isolation-source_x</th>\n",
       "      <th>isolation-source_y</th>\n",
       "      <th>isolation_source_x</th>\n",
       "      <th>isolation_source_y</th>\n",
       "      <th>lab_host_</th>\n",
       "      <th>lab_hostz</th>\n",
       "      <th>last_name</th>\n",
       "      <th>lat lon</th>\n",
       "      <th>lat-lon</th>\n",
       "      <th>lat_lon_x</th>\n",
       "      <th>lat_lon_y</th>\n",
       "      <th>latitude and longitude</th>\n",
       "      <th>lib_reads_seqd</th>\n",
       "      <th>light_type</th>\n",
       "      <th>locus_tag prefix</th>\n",
       "      <th>locus_tag_prefix_x</th>\n",
       "      <th>locus_tag_prefix_y</th>\n",
       "      <th>mapping_method_and_version</th>\n",
       "      <th>mapping_method_version_x</th>\n",
       "      <th>mapping_method_version_y</th>\n",
       "      <th>mapping_method_x</th>\n",
       "      <th>mapping_method_y</th>\n",
       "      <th>material_x</th>\n",
       "      <th>material_y</th>\n",
       "      <th>mating_type_</th>\n",
       "      <th>mating_typez</th>\n",
       "      <th>metagenome-source</th>\n",
       "      <th>metagenome-source seawater metagenome</th>\n",
       "      <th>metagenome_source_x</th>\n",
       "      <th>metagenome_source_y</th>\n",
       "      <th>metagenomic source</th>\n",
       "      <th>metagenomic_source</th>\n",
       "      <th>metagenomic_x</th>\n",
       "      <th>metagenomic_y</th>\n",
       "      <th>misc_param: HMP body site</th>\n",
       "      <th>misc_param: HMP supersite</th>\n",
       "      <th>misc_param_x</th>\n",
       "      <th>misc_param_y</th>\n",
       "      <th>missing</th>\n",
       "      <th>model</th>\n",
       "      <th>name</th>\n",
       "      <th>nat-host_x</th>\n",
       "      <th>nat-host_y</th>\n",
       "      <th>nat_host</th>\n",
       "      <th>nitrate</th>\n",
       "      <th>nitrite</th>\n",
       "      <th>note_x</th>\n",
       "      <th>note_y</th>\n",
       "      <th>nucleic acid extraction</th>\n",
       "      <th>num_replicons_x</th>\n",
       "      <th>num_replicons_y</th>\n",
       "      <th>number of contigs</th>\n",
       "      <th>occup_samp</th>\n",
       "      <th>occupant_dens_samp</th>\n",
       "      <th>organism</th>\n",
       "      <th>organism_count</th>\n",
       "      <th>organism_name</th>\n",
       "      <th>orgmod_note</th>\n",
       "      <th>other identifiers</th>\n",
       "      <th>oxy_stat_samp</th>\n",
       "      <th>pH_x</th>\n",
       "      <th>pH_y</th>\n",
       "      <th>package</th>\n",
       "      <th>paired_asm_comp</th>\n",
       "      <th>passage_history_</th>\n",
       "      <th>passage_historyz</th>\n",
       "      <th>pathogenicity</th>\n",
       "      <th>pathotype</th>\n",
       "      <th>ph</th>\n",
       "      <th>phosphate</th>\n",
       "      <th>plant-associated environmental package_x</th>\n",
       "      <th>plant-associated environmental package_y</th>\n",
       "      <th>project name_x</th>\n",
       "      <th>project name_y</th>\n",
       "      <th>project_name_x</th>\n",
       "      <th>project_name_y</th>\n",
       "      <th>project_type_x</th>\n",
       "      <th>project_type_y</th>\n",
       "      <th>publication_date</th>\n",
       "      <th>quality_assessment_method_and_version</th>\n",
       "      <th>quality_assessment_method_version_x</th>\n",
       "      <th>quality_assessment_method_version_y</th>\n",
       "      <th>quality_assessment_method_x</th>\n",
       "      <th>quality_assessment_method_y</th>\n",
       "      <th>reassembly post binning</th>\n",
       "      <th>ref_biomaterial_x</th>\n",
       "      <th>ref_biomaterial_y</th>\n",
       "      <th>refinement_method</th>\n",
       "      <th>refseq_category</th>\n",
       "      <th>rel_air_humidity</th>\n",
       "      <th>rel_to_oxygen_x</th>\n",
       "      <th>rel_to_oxygen_y</th>\n",
       "      <th>relation_to_type_material</th>\n",
       "      <th>relative coverage Anaerobic (SRR10097246)</th>\n",
       "      <th>relative coverage Anoxic (SRR10375098)</th>\n",
       "      <th>relative coverage Infiltration (SRR10097247)</th>\n",
       "      <th>relative coverage Low pH (SRR10097245)</th>\n",
       "      <th>relative coverage Oxic (SRR10375099)</th>\n",
       "      <th>relative coverage Reference (SRR10375097)</th>\n",
       "      <th>relative_week</th>\n",
       "      <th>release_type</th>\n",
       "      <th>repository</th>\n",
       "      <th>risk group</th>\n",
       "      <th>salinity</th>\n",
       "      <th>samp_collect_device</th>\n",
       "      <th>samp_mat_process</th>\n",
       "      <th>samp_size</th>\n",
       "      <th>samp_sort_meth</th>\n",
       "      <th>samp_store_dur</th>\n",
       "      <th>samp_store_loc</th>\n",
       "      <th>samp_store_temp</th>\n",
       "      <th>samp_vol_we_dna_ext</th>\n",
       "      <th>sample derived from</th>\n",
       "      <th>sample type</th>\n",
       "      <th>sample-type</th>\n",
       "      <th>sample_accession</th>\n",
       "      <th>sample_description</th>\n",
       "      <th>sample_name_x</th>\n",
       "      <th>sample_name_y</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>sample_type_x</th>\n",
       "      <th>sample_type_y</th>\n",
       "      <th>seq_rel_date</th>\n",
       "      <th>sequencing method_x</th>\n",
       "      <th>sequencing method_y</th>\n",
       "      <th>sequencing_meth_x</th>\n",
       "      <th>sequencing_meth_y</th>\n",
       "      <th>serotype</th>\n",
       "      <th>serovar_x</th>\n",
       "      <th>serovar_y</th>\n",
       "      <th>silicate</th>\n",
       "      <th>single cell or viral particle lysis approach</th>\n",
       "      <th>sop_x</th>\n",
       "      <th>sop_y</th>\n",
       "      <th>sorting technology</th>\n",
       "      <th>source_mat_id_x</th>\n",
       "      <th>source_mat_id_y</th>\n",
       "      <th>source_material_id</th>\n",
       "      <th>space_typ_state</th>\n",
       "      <th>species_taxid</th>\n",
       "      <th>specific host_x</th>\n",
       "      <th>specific host_y</th>\n",
       "      <th>specific_host_x</th>\n",
       "      <th>specific_host_y</th>\n",
       "      <th>specimen_voucher_</th>\n",
       "      <th>specimen_voucherz</th>\n",
       "      <th>strain_name_alias_x</th>\n",
       "      <th>strain_name_alias_y</th>\n",
       "      <th>strain_x</th>\n",
       "      <th>strain_y</th>\n",
       "      <th>subgroup</th>\n",
       "      <th>submission_model</th>\n",
       "      <th>submission_package</th>\n",
       "      <th>submission_package_name</th>\n",
       "      <th>submitter_x</th>\n",
       "      <th>submitter_y</th>\n",
       "      <th>subsource_note</th>\n",
       "      <th>subsrc-note</th>\n",
       "      <th>subsrc_note_x</th>\n",
       "      <th>subsrc_note_y</th>\n",
       "      <th>subtype</th>\n",
       "      <th>supplier_name_x</th>\n",
       "      <th>supplier_name_y</th>\n",
       "      <th>surf_material</th>\n",
       "      <th>synonym</th>\n",
       "      <th>taxa id</th>\n",
       "      <th>taxid</th>\n",
       "      <th>taxonomic identity marker</th>\n",
       "      <th>taxonomy_id</th>\n",
       "      <th>temp_x</th>\n",
       "      <th>temp_y</th>\n",
       "      <th>tidal_stage</th>\n",
       "      <th>tot_diss_nitro</th>\n",
       "      <th>total assembly size</th>\n",
       "      <th>trophic_level_x</th>\n",
       "      <th>trophic_level_y</th>\n",
       "      <th>typ_occupant_dens</th>\n",
       "      <th>type status</th>\n",
       "      <th>type-material_x</th>\n",
       "      <th>type-material_y</th>\n",
       "      <th>type_status</th>\n",
       "      <th>value_x</th>\n",
       "      <th>value_y</th>\n",
       "      <th>ventilation_type</th>\n",
       "      <th>version_status</th>\n",
       "      <th>wgs_master</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae ATCC 17061. Flavobac...</td>\n",
       "      <td>2007-05-01T00:00:00Z</td>\n",
       "      <td>1.0</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>34.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>6096872.0</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>United Kingdom: England</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Multiple</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>isolated from soil in England and is the type ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>soil</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>20-30</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_009441</td>\n",
       "      <td>5017.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Non-halophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DOE Joint Genome Institute</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM1664v1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>Sample from Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=UW101; ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2014-01-28T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Major</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2007/05/01</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>986.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>UW101; ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>US DOE Joint Genome Institute</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Flavobacterium johnsoniae</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271.  Capnocytoph...</td>\n",
       "      <td>2009-04-30T00:00:00Z</td>\n",
       "      <td>1.0</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>39.6</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CP001632</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2612925.0</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Host-associated</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>isolated from the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>35-37</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Facultative</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>21304645</td>\n",
       "      <td>NC_013162</td>\n",
       "      <td>2171.0</td>\n",
       "      <td>SRR013476,SRR013477</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM2328v1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>Generic sample from Capnocytophaga ochracea DS...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000023285.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2009-02-19T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Major</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2009/08/26</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1018.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>Generic</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      # assembly_accession 16S recovered 16S recovery software Alias_x  \\\n",
       "0 NaN      GCA_000016645.1           NaN                   NaN     NaN   \n",
       "1 NaN      GCA_000023285.1           NaN                   NaN     NaN   \n",
       "\n",
       "  Alias_y Altitude  ArrayExpress-Phenotype ArrayExpress-SPECIES  \\\n",
       "0     NaN      NaN                     NaN                  NaN   \n",
       "1     NaN      NaN                     NaN                  NaN   \n",
       "\n",
       "  ArrayExpress-STRAIN_OR_LINE ArrayExpress-Species ArrayExpress-StrainOrLine  \\\n",
       "0                         NaN                  NaN                       NaN   \n",
       "1                         NaN                  NaN                       NaN   \n",
       "\n",
       "  Assembly Accession Assembly Method_x Assembly Method_y BioProject Accession  \\\n",
       "0    GCA_000016645.1               NaN               NaN           PRJNA16082   \n",
       "1    GCA_000023285.1               NaN               NaN           PRJNA29403   \n",
       "\n",
       "  BioSample Accession BioSampleModel_x BioSampleModel_y Body Sample Site  \\\n",
       "0        SAMN02598357              NaN              NaN              NaN   \n",
       "1        SAMN00001911              NaN              NaN              NaN   \n",
       "\n",
       "  Body Sample Site_x Body Sample Site_y Body Sample SubSite_x  \\\n",
       "0                NaN                NaN                   NaN   \n",
       "1                NaN                NaN                   NaN   \n",
       "\n",
       "  Body Sample SubSite_y Broker name Cell Shape Cell Shape_x Cell Shape_y  \\\n",
       "0                   NaN         NaN        NaN          Rod          NaN   \n",
       "1                   NaN         NaN        NaN          Rod          NaN   \n",
       "\n",
       "   Chromosomes Collaborator ID Collection Date  \\\n",
       "0          1.0             NaN             NaN   \n",
       "1          1.0             NaN             NaN   \n",
       "\n",
       "                                            Comments       Completion Date  \\\n",
       "0  Flavobacterium johnsoniae ATCC 17061. Flavobac...  2007-05-01T00:00:00Z   \n",
       "1  Capnocytophaga ochracea DSM 7271.  Capnocytoph...  2009-04-30T00:00:00Z   \n",
       "\n",
       "   Contigs Culture Collection Depth_x Description Disease  Diseases_x  \\\n",
       "0      1.0         ATCC 17061     NaN         NaN     NaN         NaN   \n",
       "1      1.0           DSM 7271     NaN         NaN     NaN         NaN   \n",
       "\n",
       "  Diseases_y ENA checklist_x ENA checklist_y ENA-CHECKLIST ENA-FIRST-PUBLIC_x  \\\n",
       "0        NaN             NaN             NaN           NaN                NaN   \n",
       "1        NaN             NaN             NaN           NaN                NaN   \n",
       "\n",
       "  ENA-FIRST-PUBLIC_y ENA-LAST-UPDATE_x ENA-LAST-UPDATE_y External Id  \\\n",
       "0                NaN               NaN               NaN         NaN   \n",
       "1                NaN               NaN               NaN         NaN   \n",
       "\n",
       "   Extraction Date  Extraction Method FDA CVM ID_x FDA CVM ID_y  \\\n",
       "0              NaN                NaN          NaN          NaN   \n",
       "1              NaN                NaN          NaN          NaN   \n",
       "\n",
       "  Funding Program_x Funding Program_y  GC Content GOLD Stamp ID_x  \\\n",
       "0               NaN               NaN        34.1             NaN   \n",
       "1               NaN               NaN        39.6             NaN   \n",
       "\n",
       "  GOLD Stamp ID_y GenBank Accessions Gene Calling Method_x  \\\n",
       "0             NaN           CP000685                   NaN   \n",
       "1             NaN           CP001632                   NaN   \n",
       "\n",
       "  Gene Calling Method_y Genome Coverage  Genome Length  \\\n",
       "0                   NaN             NaN      6096872.0   \n",
       "1                   NaN             NaN      2612925.0   \n",
       "\n",
       "                        Genome Name Genome Quality Flags Genome Status  \\\n",
       "0   Flavobacterium johnsoniae UW101                  NaN      Complete   \n",
       "1  Capnocytophaga ochracea DSM 7271                  NaN      Complete   \n",
       "\n",
       "         Genome_ID Genus      Geographic Location Gram Stain Gram Staining_x  \\\n",
       "0  GCA_000016645.1   NaN  United Kingdom: England        NaN             NaN   \n",
       "1  GCA_000023285.1   NaN                      NaN        NaN             NaN   \n",
       "\n",
       "  Gram Staining_y          Habitat Historical Monobactam Produced_x  \\\n",
       "0             NaN         Multiple                              NaN   \n",
       "1             NaN  Host-associated                              NaN   \n",
       "\n",
       "  Historical Monobactam Produced_y Host Age Host Gender Host Health  \\\n",
       "0                              NaN      NaN         NaN         NaN   \n",
       "1                              NaN      NaN         NaN         NaN   \n",
       "\n",
       "  Host Health_x Host Health_y Host Name          Host Name_x Host Name_y  \\\n",
       "0           NaN           NaN       NaN                  NaN         NaN   \n",
       "1           NaN           NaN       NaN  Human, Homo sapiens         NaN   \n",
       "\n",
       "  Host_x Host_y INSDC center alias_x INSDC center alias_y INSDC center name_x  \\\n",
       "0    NaN    NaN                  NaN                  NaN                 NaN   \n",
       "1    NaN    NaN                  NaN                  NaN                 NaN   \n",
       "\n",
       "  INSDC center name_y INSDC first public_x INSDC first public_y  \\\n",
       "0                 NaN                  NaN                  NaN   \n",
       "1                 NaN                  NaN                  NaN   \n",
       "\n",
       "  INSDC last update_x INSDC last update_y INSDC status_x INSDC status_y  \\\n",
       "0                 NaN                 NaN            NaN            NaN   \n",
       "1                 NaN                 NaN            NaN            NaN   \n",
       "\n",
       "  Isolation Comments                               Isolation Comments_x  \\\n",
       "0                NaN  isolated from soil in England and is the type ...   \n",
       "1                NaN                isolated from the human oral cavity   \n",
       "\n",
       "  Isolation Comments_y Isolation Country Isolation Site_x Isolation Site_y  \\\n",
       "0                  NaN    United Kingdom              NaN              NaN   \n",
       "1                  NaN               NaN              NaN              NaN   \n",
       "\n",
       "        Isolation Source Isolation source_x Isolation source_y  \\\n",
       "0                   soil                NaN                NaN   \n",
       "1  the human oral cavity                NaN                NaN   \n",
       "\n",
       "   Laboratory Host  Latitude  Longitude  MAG_number MLST Motility Motility_x  \\\n",
       "0              NaN       NaN        NaN         NaN  NaN      NaN        Yes   \n",
       "1              NaN       NaN        NaN         NaN  NaN      NaN        Yes   \n",
       "\n",
       "  Motility_y  NCBI Taxon ID Optimal Temperature  \\\n",
       "0        NaN       376686.0               20-30   \n",
       "1        NaN       521097.0               35-37   \n",
       "\n",
       "                      Organism Name Other Clinical Other Environmental  \\\n",
       "0   Flavobacterium johnsoniae UW101            NaN                 NaN   \n",
       "1  Capnocytophaga ochracea DSM 7271            NaN                 NaN   \n",
       "\n",
       "  Other Typing Oxygen Requirement  Passage Date Pathovar Phenotypes_x  \\\n",
       "0          NaN            Aerobic           NaN      NaN          NaN   \n",
       "1          NaN        Facultative           NaN      NaN          NaN   \n",
       "\n",
       "  Phenotypes_y  Plasmids ProjectAccession PublicAccession Publication  \\\n",
       "0          NaN       NaN              NaN             NaN         NaN   \n",
       "1          NaN       NaN              NaN             NaN    21304645   \n",
       "\n",
       "  RefSeq Accessions  RefSeq CDS        SRA Accession SRA accession_x  \\\n",
       "0         NC_009441      5017.0                  NaN             NaN   \n",
       "1         NC_013162      2171.0  SRR013476,SRR013477             NaN   \n",
       "\n",
       "  SRA accession_y STRAIN SUBJECT_ID_x SUBJECT_ID_y  Salinity (PSU)  \\\n",
       "0             NaN    NaN          NaN          NaN             NaN   \n",
       "1             NaN    NaN          NaN          NaN             NaN   \n",
       "\n",
       "       Salinity_x Salinity_y  Sample Description Sample Name_x Sample Name_y  \\\n",
       "0  Non-halophilic        NaN                 NaN           NaN           NaN   \n",
       "1             NaN        NaN                 NaN           NaN           NaN   \n",
       "\n",
       "                                  Sequencing Centers Sequencing Depth  \\\n",
       "0                         DOE Joint Genome Institute              NaN   \n",
       "1  US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...              NaN   \n",
       "\n",
       "  Sequencing Platform Sequencing Status  Sequencing Technology Serovar  \\\n",
       "0                 NaN          complete                    NaN     NaN   \n",
       "1                 NaN          complete                    NaN     NaN   \n",
       "\n",
       "  Species Sporulation Sporulation_x Sporulation_y Strain    Strain_x  \\\n",
       "0     NaN         NaN            No           NaN    NaN  ATCC 17061   \n",
       "1     NaN         NaN            No           NaN    NaN    DSM 7271   \n",
       "\n",
       "  Submitter Id  Tax ID Temperature Optimum_x Temperature Optimum_y  \\\n",
       "0          NaN     NaN                   NaN                   NaN   \n",
       "1          NaN     NaN                   NaN                   NaN   \n",
       "\n",
       "  Temperature Range Temperature Range_x Temperature Range_y Title_x Title_y  \\\n",
       "0               NaN          Mesophilic                 NaN     NaN     NaN   \n",
       "1               NaN          Mesophilic                 NaN     NaN     NaN   \n",
       "\n",
       "  Type Strain Type Strain_x Type Strain_y Type-material  \\\n",
       "0         NaN           Yes           NaN           NaN   \n",
       "1         NaN           Yes           NaN           NaN   \n",
       "\n",
       "  Unique Monobactam Clusters Identified_x  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "  Unique Monobactam Clusters Identified_y WGA amplification approach  \\\n",
       "0                                     NaN                        NaN   \n",
       "1                                     NaN                        NaN   \n",
       "\n",
       "   abs_air_humidity  air_temp alkalinity (carbonate/bicarbonate) alt_elev_x  \\\n",
       "0               NaN       NaN                                NaN        NaN   \n",
       "1               NaN       NaN                                NaN        NaN   \n",
       "\n",
       "  alt_elev_y altitude_x altitude_y ammonium analysis project type  \\\n",
       "0        NaN        NaN        NaN      NaN                   NaN   \n",
       "1        NaN        NaN        NaN      NaN                   NaN   \n",
       "\n",
       "  annotation_method anonymized_name_x  anonymized_name_y   asm_name  \\\n",
       "0               NaN               NaN                NaN  ASM1664v1   \n",
       "1               NaN               NaN                NaN  ASM2328v1   \n",
       "\n",
       "  assembly quality assembly software assembly_accession   assembly_level  \\\n",
       "0              NaN               NaN                NaN  Complete Genome   \n",
       "1              NaN               NaN    GCA_000023285.1  Complete Genome   \n",
       "\n",
       "  assembly_method_and_version assembly_method_version_x  \\\n",
       "0                         NaN                       NaN   \n",
       "1                         NaN                       NaN   \n",
       "\n",
       "  assembly_method_version_y assembly_method_x assembly_method_y assembly_tool  \\\n",
       "0                       NaN               NaN               NaN           NaN   \n",
       "1                       NaN               NaN               NaN           NaN   \n",
       "\n",
       "  assembly_x assembly_y attribute_package bin parameters bin_id  \\\n",
       "0        NaN        NaN               NaN            NaN    NaN   \n",
       "1        NaN        NaN               NaN            NaN    NaN   \n",
       "\n",
       "  binning parameters binning software bio_material biomass  \\\n",
       "0                NaN              NaN          NaN     NaN   \n",
       "1                NaN              NaN          NaN     NaN   \n",
       "\n",
       "  biomaterial_provider biomaterial_provider_x biomaterial_provider_y biome_x  \\\n",
       "0                  NaN                    NaN                    NaN     NaN   \n",
       "1                  NaN                    NaN                    NaN     NaN   \n",
       "\n",
       "  biome_y  bioproject     biosample  \\\n",
       "0     NaN  PRJNA16082  SAMN02598357   \n",
       "1     NaN  PRJNA29403  SAMN00001911   \n",
       "\n",
       "                                     biosample_title biosamplemodel  \\\n",
       "0        Sample from Flavobacterium johnsoniae UW101            NaN   \n",
       "1  Generic sample from Capnocytophaga ochracea DS...            NaN   \n",
       "\n",
       "  biotic_relationship_x biotic_relationship_y body_site build_occup_type  \\\n",
       "0                   NaN                   NaN       NaN              NaN   \n",
       "1                   NaN                   NaN       NaN              NaN   \n",
       "\n",
       "  building_setting carb_dioxide  carb_nitro_ratio chlorophyll collected-by  \\\n",
       "0              NaN          NaN               NaN         NaN          NaN   \n",
       "1              NaN          NaN               NaN         NaN          NaN   \n",
       "\n",
       "  collected_by collected_by_x collected_by_y collection date_x  \\\n",
       "0          NaN            NaN            NaN               NaN   \n",
       "1          NaN            NaN            NaN               NaN   \n",
       "\n",
       "  collection date_y collection-date collection_date_x collection_date_y  \\\n",
       "0               NaN             NaN               NaN               NaN   \n",
       "1               NaN             NaN               NaN               NaN   \n",
       "\n",
       "  collection_room                 comment_paragraph common name  completeness  \\\n",
       "0             NaN                               NaN         NaN           NaN   \n",
       "1             NaN  Capnocytophaga ochracea DSM 7271         NaN           NaN   \n",
       "\n",
       "   completeness score completeness software completeness_estimated_x  \\\n",
       "0                 NaN                   NaN                      NaN   \n",
       "1                 NaN                   NaN                      NaN   \n",
       "\n",
       "  completeness_estimated_y contact_email contact_lab  contamination  \\\n",
       "0                      NaN           NaN         NaN            NaN   \n",
       "1                      NaN           NaN         NaN            NaN   \n",
       "\n",
       "   contamination score contamination_estimated_x contamination_estimated_y  \\\n",
       "0                  NaN                       NaN                       NaN   \n",
       "1                  NaN                       NaN                       NaN   \n",
       "\n",
       "   contig L50 country_x country_y culture collection culture-collection_x  \\\n",
       "0         NaN       NaN       NaN                NaN                  NaN   \n",
       "1         NaN       NaN       NaN                NaN                  NaN   \n",
       "\n",
       "  culture-collection_y culture_collection_x culture_collection_y  \\\n",
       "0                  NaN                  NaN                  NaN   \n",
       "1                  NaN                  NaN                  NaN   \n",
       "\n",
       "   cyanobacterial_culture decontamination software depth_x depth_y  \\\n",
       "0                     NaN                      NaN     NaN     NaN   \n",
       "1                     NaN                      NaN     NaN     NaN   \n",
       "\n",
       "  derived-from derived_from description diss_inorg_nitro diss_inorg_phosp  \\\n",
       "0          NaN          NaN         NaN              NaN              NaN   \n",
       "1          NaN          NaN         NaN              NaN              NaN   \n",
       "\n",
       "  diss_org_carb diss_org_nitro elev_x  elev_y env material env_biome_x  \\\n",
       "0           NaN            NaN    NaN     NaN          NaN         NaN   \n",
       "1           NaN            NaN    NaN     NaN          NaN         NaN   \n",
       "\n",
       "  env_biome_y env_broad_scale env_feature_x env_feature_y env_local_scale  \\\n",
       "0         NaN             NaN           NaN           NaN             NaN   \n",
       "1         NaN             NaN           NaN           NaN             NaN   \n",
       "\n",
       "  env_material_x env_material_y env_medium  env_package env_packatge  \\\n",
       "0            NaN            NaN        NaN          NaN          NaN   \n",
       "1            NaN            NaN        NaN          NaN          NaN   \n",
       "\n",
       "  environment (biome)_x environment (biome)_y environment (feature)_x  \\\n",
       "0                   NaN                   NaN                     NaN   \n",
       "1                   NaN                   NaN                     NaN   \n",
       "\n",
       "  environment (feature)_y environment (material)_x environment (material)_y  \\\n",
       "0                     NaN                      NaN                      NaN   \n",
       "1                     NaN                      NaN                      NaN   \n",
       "\n",
       "  environment_x environment_y  environmental package environmental-sample  \\\n",
       "0           NaN           NaN                    NaN                  NaN   \n",
       "1           NaN           NaN                    NaN                  NaN   \n",
       "\n",
       "  environmental_sample_x environmental_sample_y environmetal-sample  \\\n",
       "0                    NaN                    NaN                 NaN   \n",
       "1                    NaN                    NaN                 NaN   \n",
       "\n",
       "  estimated_size_x  estimated_size_y ethnicity excluded_from_refseq  \\\n",
       "0              NaN               NaN       NaN                  NaN   \n",
       "1              NaN               NaN       NaN                  NaN   \n",
       "\n",
       "  experimental_factor  extrachrom_elements feature_x feature_y filter_type  \\\n",
       "0                 NaN                  NaN       NaN       NaN         NaN   \n",
       "1                 NaN                  NaN       NaN       NaN         NaN   \n",
       "\n",
       "  finishing strategy (depth of coverage)_x  \\\n",
       "0                                      NaN   \n",
       "1                                      NaN   \n",
       "\n",
       "  finishing strategy (depth of coverage)_y finishing_strategy_x  \\\n",
       "0                                      NaN                  NaN   \n",
       "1                                      NaN                  NaN   \n",
       "\n",
       "  finishing_strategy_y first_name  \\\n",
       "0                  NaN        NaN   \n",
       "1                  NaN        NaN   \n",
       "\n",
       "                                            ftp_path  gbrs_paired_asm  \\\n",
       "0  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...  GCF_000016645.1   \n",
       "1  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...  GCF_000023285.1   \n",
       "\n",
       "  genome_coverage_x  genome_coverage_y genome_rep genotype geo-loc-name_x  \\\n",
       "0               NaN                NaN       Full      NaN            NaN   \n",
       "1               NaN                NaN       Full      NaN            NaN   \n",
       "\n",
       "  geo-loc-name_y geo_loc_name_x geo_loc_name_y geographic location  \\\n",
       "0            NaN            NaN            NaN                 NaN   \n",
       "1            NaN            NaN            NaN                 NaN   \n",
       "\n",
       "   geographic location (altitude)  \\\n",
       "0                             NaN   \n",
       "1                             NaN   \n",
       "\n",
       "  geographic location (country and/or atlantic ocean)  \\\n",
       "0                                                NaN    \n",
       "1                                                NaN    \n",
       "\n",
       "  geographic location (country and/or sea)_x  \\\n",
       "0                                        NaN   \n",
       "1                                        NaN   \n",
       "\n",
       "  geographic location (country and/or sea)_y geographic location (latitude)_x  \\\n",
       "0                                        NaN                              NaN   \n",
       "1                                        NaN                              NaN   \n",
       "\n",
       "   geographic location (latitude)_y geographic location (longitude)_x  \\\n",
       "0                               NaN                               NaN   \n",
       "1                               NaN                               NaN   \n",
       "\n",
       "   geographic location (longitude)_y  \\\n",
       "0                                NaN   \n",
       "1                                NaN   \n",
       "\n",
       "  geographic location (region and locality) health_state heat_cool_type  \\\n",
       "0                                       NaN          NaN            NaN   \n",
       "1                                       NaN          NaN            NaN   \n",
       "\n",
       "  host common name host health state_x host health state_y  \\\n",
       "0              NaN                 NaN                 NaN   \n",
       "1              NaN                 NaN                 NaN   \n",
       "\n",
       "  host scientific name_x host scientific name_y  host taxid_x  host taxid_y  \\\n",
       "0                    NaN                    NaN           NaN           NaN   \n",
       "1                    NaN                    NaN           NaN           NaN   \n",
       "\n",
       "  host-associated environmental package host_age_x host_age_y  \\\n",
       "0                                   NaN        NaN        NaN   \n",
       "1                                   NaN        NaN        NaN   \n",
       "\n",
       "  host_body_habitat host_description host_disease_outcome_x  \\\n",
       "0               NaN              NaN                    NaN   \n",
       "1               NaN              NaN                    NaN   \n",
       "\n",
       "  host_disease_outcome_y host_disease_stage host_disease_x host_disease_y  \\\n",
       "0                    NaN                NaN            NaN            NaN   \n",
       "1                    NaN                NaN            NaN            NaN   \n",
       "\n",
       "  host_health_state host_sex_x host_sex_y host_status host_subject_id  \\\n",
       "0               NaN        NaN        NaN         NaN             NaN   \n",
       "1               NaN        NaN        NaN         NaN             NaN   \n",
       "\n",
       "   host_taxid host_tissue_sampled_x host_tissue_sampled_y host_x host_y  \\\n",
       "0         NaN                   NaN                   NaN    NaN    NaN   \n",
       "1         NaN                   NaN                   NaN    NaN    NaN   \n",
       "\n",
       "  identification method_x identification method_y identified_by  \\\n",
       "0                     NaN                     NaN           NaN   \n",
       "1                     NaN                     NaN           NaN   \n",
       "\n",
       "  identified_by_x identified_by_y indoor_space indoor_surf  \\\n",
       "0             NaN             NaN          NaN         NaN   \n",
       "1             NaN             NaN          NaN         NaN   \n",
       "\n",
       "         infraspecific_name investigation type_x investigation type_y  \\\n",
       "0  strain=UW101; ATCC 17061                  NaN                  NaN   \n",
       "1           strain=DSM 7271                  NaN                  NaN   \n",
       "\n",
       "  investigation_type_x investigation_type_y isol_growth_condt_x  \\\n",
       "0                  NaN                  NaN                 NaN   \n",
       "1                  NaN                  NaN                 NaN   \n",
       "\n",
       "   isol_growth_condt_y isolate_ isolate_name_alias_x isolate_name_alias_y  \\\n",
       "0                  NaN      NaN                  NaN                  NaN   \n",
       "1                  NaN      NaN                  NaN                  NaN   \n",
       "\n",
       "  isolate_x isolate_y isolatez isolation source_x isolation source_y  \\\n",
       "0       NaN       NaN      NaN                NaN                NaN   \n",
       "1       NaN       NaN      NaN                NaN                NaN   \n",
       "\n",
       "  isolation-source_x isolation-source_y isolation_source_x isolation_source_y  \\\n",
       "0                NaN                NaN                NaN                NaN   \n",
       "1                NaN                NaN                NaN                NaN   \n",
       "\n",
       "  lab_host_ lab_hostz last_name  lat lon lat-lon lat_lon_x lat_lon_y  \\\n",
       "0       NaN       NaN       NaN      NaN     NaN       NaN       NaN   \n",
       "1       NaN       NaN       NaN      NaN     NaN       NaN       NaN   \n",
       "\n",
       "  latitude and longitude lib_reads_seqd light_type locus_tag prefix  \\\n",
       "0                    NaN            NaN        NaN              NaN   \n",
       "1                    NaN            NaN        NaN              NaN   \n",
       "\n",
       "  locus_tag_prefix_x locus_tag_prefix_y mapping_method_and_version  \\\n",
       "0                NaN                NaN                        NaN   \n",
       "1                NaN                NaN                        NaN   \n",
       "\n",
       "  mapping_method_version_x mapping_method_version_y mapping_method_x  \\\n",
       "0                      NaN                      NaN              NaN   \n",
       "1                      NaN                      NaN              NaN   \n",
       "\n",
       "  mapping_method_y material_x material_y  mating_type_  mating_typez  \\\n",
       "0              NaN        NaN        NaN           NaN           NaN   \n",
       "1              NaN        NaN        NaN           NaN           NaN   \n",
       "\n",
       "  metagenome-source metagenome-source seawater metagenome metagenome_source_x  \\\n",
       "0               NaN                                   NaN                 NaN   \n",
       "1               NaN                                   NaN                 NaN   \n",
       "\n",
       "  metagenome_source_y metagenomic source metagenomic_source metagenomic_x  \\\n",
       "0                 NaN                NaN                NaN           NaN   \n",
       "1                 NaN                NaN                NaN           NaN   \n",
       "\n",
       "  metagenomic_y misc_param: HMP body site misc_param: HMP supersite  \\\n",
       "0           NaN                       NaN                       NaN   \n",
       "1           NaN                       NaN                       NaN   \n",
       "\n",
       "  misc_param_x misc_param_y  missing model  \\\n",
       "0          NaN          NaN      NaN   NaN   \n",
       "1          NaN          NaN      NaN   NaN   \n",
       "\n",
       "                                                name nat-host_x nat-host_y  \\\n",
       "0                                               NCBI        NaN        NaN   \n",
       "1  Joint Genome Institute, U.S. Department of Energy        NaN        NaN   \n",
       "\n",
       "  nat_host nitrate nitrite note_x note_y  nucleic acid extraction  \\\n",
       "0      NaN     NaN     NaN    NaN    NaN                      NaN   \n",
       "1      NaN     NaN     NaN    NaN    NaN                      NaN   \n",
       "\n",
       "  num_replicons_x  num_replicons_y  number of contigs occup_samp  \\\n",
       "0             NaN              NaN                NaN        NaN   \n",
       "1             NaN              NaN                NaN        NaN   \n",
       "\n",
       "  occupant_dens_samp                          organism  organism_count  \\\n",
       "0                NaN   Flavobacterium johnsoniae UW101             NaN   \n",
       "1                NaN  Capnocytophaga ochracea DSM 7271             NaN   \n",
       "\n",
       "                      organism_name orgmod_note other identifiers  \\\n",
       "0   Flavobacterium johnsoniae UW101         NaN               NaN   \n",
       "1  Capnocytophaga ochracea DSM 7271         NaN               NaN   \n",
       "\n",
       "  oxy_stat_samp pH_x pH_y package paired_asm_comp passage_history_  \\\n",
       "0           NaN  NaN  NaN     NaN       identical              NaN   \n",
       "1           NaN  NaN  NaN     NaN       identical              NaN   \n",
       "\n",
       "  passage_historyz pathogenicity pathotype  ph phosphate  \\\n",
       "0              NaN           NaN       NaN NaN       NaN   \n",
       "1              NaN           NaN       NaN NaN       NaN   \n",
       "\n",
       "  plant-associated environmental package_x  \\\n",
       "0                                      NaN   \n",
       "1                                      NaN   \n",
       "\n",
       "  plant-associated environmental package_y project name_x project name_y  \\\n",
       "0                                      NaN            NaN            NaN   \n",
       "1                                      NaN            NaN            NaN   \n",
       "\n",
       "  project_name_x project_name_y project_type_x project_type_y  \\\n",
       "0            NaN            NaN            NaN            NaN   \n",
       "1            NaN            NaN            NaN            NaN   \n",
       "\n",
       "          publication_date quality_assessment_method_and_version  \\\n",
       "0  2014-01-28T00:00:00.000                                   NaN   \n",
       "1  2009-02-19T00:00:00.000                                   NaN   \n",
       "\n",
       "  quality_assessment_method_version_x quality_assessment_method_version_y  \\\n",
       "0                                 NaN                                 NaN   \n",
       "1                                 NaN                                 NaN   \n",
       "\n",
       "  quality_assessment_method_x quality_assessment_method_y  \\\n",
       "0                         NaN                         NaN   \n",
       "1                         NaN                         NaN   \n",
       "\n",
       "  reassembly post binning ref_biomaterial_x ref_biomaterial_y  \\\n",
       "0                     NaN               NaN               NaN   \n",
       "1                     NaN               NaN               NaN   \n",
       "\n",
       "  refinement_method        refseq_category rel_air_humidity rel_to_oxygen_x  \\\n",
       "0               NaN  representative genome              NaN             NaN   \n",
       "1               NaN  representative genome              NaN             NaN   \n",
       "\n",
       "  rel_to_oxygen_y    relation_to_type_material  \\\n",
       "0             NaN  assembly from type material   \n",
       "1             NaN  assembly from type material   \n",
       "\n",
       "   relative coverage Anaerobic (SRR10097246)  \\\n",
       "0                                        NaN   \n",
       "1                                        NaN   \n",
       "\n",
       "   relative coverage Anoxic (SRR10375098)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Infiltration (SRR10097247)  \\\n",
       "0                                           NaN   \n",
       "1                                           NaN   \n",
       "\n",
       "   relative coverage Low pH (SRR10097245)  \\\n",
       "0                                     NaN   \n",
       "1                                     NaN   \n",
       "\n",
       "   relative coverage Oxic (SRR10375099)  \\\n",
       "0                                   NaN   \n",
       "1                                   NaN   \n",
       "\n",
       "   relative coverage Reference (SRR10375097)  relative_week release_type  \\\n",
       "0                                        NaN            NaN        Major   \n",
       "1                                        NaN            NaN        Major   \n",
       "\n",
       "  repository risk group salinity samp_collect_device samp_mat_process  \\\n",
       "0        NaN        NaN      NaN                 NaN              NaN   \n",
       "1        NaN        NaN      NaN                 NaN              NaN   \n",
       "\n",
       "  samp_size samp_sort_meth samp_store_dur samp_store_loc samp_store_temp  \\\n",
       "0       NaN            NaN            NaN            NaN             NaN   \n",
       "1       NaN            NaN            NaN            NaN             NaN   \n",
       "\n",
       "  samp_vol_we_dna_ext sample derived from  sample type sample-type  \\\n",
       "0                 NaN                 NaN          NaN         NaN   \n",
       "1                 NaN                 NaN          NaN         NaN   \n",
       "\n",
       "  sample_accession  sample_description sample_name_x sample_name_y  \\\n",
       "0     SAMN02598357                 NaN      CP000685           NaN   \n",
       "1     SAMN00001911                 NaN          FWCB           NaN   \n",
       "\n",
       "  sample_type sample_type_x sample_type_y seq_rel_date sequencing method_x  \\\n",
       "0         NaN           NaN           NaN   2007/05/01                 NaN   \n",
       "1         NaN           NaN           NaN   2009/08/26                 NaN   \n",
       "\n",
       "  sequencing method_y sequencing_meth_x sequencing_meth_y serotype serovar_x  \\\n",
       "0                 NaN               NaN               NaN      NaN       NaN   \n",
       "1                 NaN               NaN               NaN      NaN       NaN   \n",
       "\n",
       "   serovar_y silicate single cell or viral particle lysis approach sop_x  \\\n",
       "0        NaN      NaN                                          NaN   NaN   \n",
       "1        NaN      NaN                                          NaN   NaN   \n",
       "\n",
       "  sop_y sorting technology source_mat_id_x source_mat_id_y source_material_id  \\\n",
       "0   NaN                NaN             NaN             NaN                NaN   \n",
       "1   NaN                NaN             NaN             NaN                NaN   \n",
       "\n",
       "  space_typ_state  species_taxid specific host_x specific host_y  \\\n",
       "0             NaN          986.0             NaN             NaN   \n",
       "1             NaN         1018.0             NaN             NaN   \n",
       "\n",
       "  specific_host_x specific_host_y specimen_voucher_ specimen_voucherz  \\\n",
       "0             NaN             NaN               NaN               NaN   \n",
       "1             NaN             NaN               NaN               NaN   \n",
       "\n",
       "  strain_name_alias_x strain_name_alias_y           strain_x  strain_y  \\\n",
       "0                 NaN                 NaN  UW101; ATCC 17061       NaN   \n",
       "1                 NaN                 NaN           DSM 7271  DSM 7271   \n",
       "\n",
       "   subgroup submission_model submission_package submission_package_name  \\\n",
       "0       NaN          Generic        Generic.1.0                 Generic   \n",
       "1       NaN          Generic        Generic.1.0                 Generic   \n",
       "\n",
       "                               submitter_x  \\\n",
       "0            US DOE Joint Genome Institute   \n",
       "1  US DOE Joint Genome Institute (JGI-PGF)   \n",
       "\n",
       "                                         submitter_y subsource_note  \\\n",
       "0                                               NCBI            NaN   \n",
       "1  Joint Genome Institute, U.S. Department of Energy            NaN   \n",
       "\n",
       "  subsrc-note subsrc_note_x subsrc_note_y  subtype supplier_name_x  \\\n",
       "0         NaN           NaN           NaN      NaN             NaN   \n",
       "1         NaN           NaN           NaN      NaN             NaN   \n",
       "\n",
       "  supplier_name_y surf_material synonym taxa id     taxid  \\\n",
       "0             NaN           NaN     NaN     NaN  376686.0   \n",
       "1             NaN           NaN     NaN     NaN  521097.0   \n",
       "\n",
       "  taxonomic identity marker  taxonomy_id temp_x temp_y tidal_stage  \\\n",
       "0                       NaN     376686.0    NaN    NaN         NaN   \n",
       "1                       NaN     521097.0    NaN    NaN         NaN   \n",
       "\n",
       "  tot_diss_nitro  total assembly size trophic_level_x trophic_level_y  \\\n",
       "0            NaN                  NaN             NaN             NaN   \n",
       "1            NaN                  NaN             NaN             NaN   \n",
       "\n",
       "  typ_occupant_dens type status                           type-material_x  \\\n",
       "0               NaN         NaN  type strain of Flavobacterium johnsoniae   \n",
       "1               NaN         NaN    type strain of Capnocytophaga ochracea   \n",
       "\n",
       "                          type-material_y type_status value_x value_y  \\\n",
       "0                                     NaN         NaN     NaN     NaN   \n",
       "1  type strain of Capnocytophaga ochracea         NaN     NaN     NaN   \n",
       "\n",
       "  ventilation_type version_status wgs_master  \n",
       "0              NaN         latest        NaN  \n",
       "1              NaN         latest        NaN  "
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#merge.columns.duplicated()\n",
    "merge_reordered = merge.reindex(sorted(merge.columns), axis=1)\n",
    "merge_reordered.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "columns = merge_reordered.columns.to_list()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('Attributes.csv', 'w') as f:\n",
    "    for item in columns:\n",
    "        f.write(\"%s\\n\" % item)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Open file that has columns to be joined together:**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"Attributes_reformated.csv\", \"r+\") as f:\n",
    "    lines = f.readlines()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "#create dictionary with info from loaded file\n",
    "d={}\n",
    "for line in lines:\n",
    "    words = line.split(\",\")\n",
    "    words = [x.strip() for x in words if x.strip()]\n",
    "    d[words[0]] = words[1:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [],
   "source": [
    "new_cols=[]\n",
    "to_drop = []\n",
    "dummy = merge_reordered.copy() # copy of the dataframe (for security reasons)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i,a in d.items(): #i - name of new column; a - columns to merge; d : dictionary[i] = a\n",
    "    #print(\"New column: \" + i + \"_merged; combining: \" +  str(a))\n",
    "    #print(i)\n",
    "    to_drop.append(a[0])\n",
    "    dummy[i + \"_merged\"] = dummy[a[0]]\n",
    "    for x in a[1:]:\n",
    "        dummy[i + \"_merged\"] = dummy[i + \"_merged\"].combine_first(dummy[x])\n",
    "        #print(\"added \" +  x)\n",
    "        to_drop.append(x)\n",
    "    new_cols.append(str(i) + \"_merged\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "to_look = dummy.reindex(sorted(dummy.columns), axis=1)\n",
    "to_look.to_csv(\"before_cleaning.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Int64Index: 2686 entries, 0 to 2685\n",
      "Columns: 299 entries,  to WGS master\n",
      "dtypes: float64(44), object(255)\n",
      "memory usage: 6.1+ MB\n"
     ]
    }
   ],
   "source": [
    "# drop columns already used in the joined columns:\n",
    "dummy3 = dummy.drop(columns=to_drop)\n",
    "\n",
    "# rename columns created by the join to the final name:\n",
    "dummy3.columns = dummy3.columns.str.replace('_merged', '')\n",
    "dummy3.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "length = len(dummy3.index)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Index: 2686 entries, 0 to 2685\n",
      "Columns: 169 entries, Alias to WGS master\n",
      "dtypes: float64(9), object(160)\n",
      "memory usage: 3.5+ MB\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/gomes/anaconda3/lib/python3.7/site-packages/pandas/core/frame.py:3997: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  errors=errors,\n"
     ]
    }
   ],
   "source": [
    "dummy3.loc[\"SUM\", :] = dummy3.isnull().sum(axis = 0)\n",
    "#dummy3.to_csv(\"Dummy3.csv\")\n",
    "\n",
    "#remove columns that have less than 10 entries\n",
    "dummy4 = dummy3[dummy3.columns[dummy3.loc[\"SUM\"] <(length - 10)]]\n",
    "dummy4.drop(index=\"SUM\", inplace=True)\n",
    "dummy4.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Alias</th>\n",
       "      <th>Altitude</th>\n",
       "      <th>Anonymized name</th>\n",
       "      <th>Arrayexpress species</th>\n",
       "      <th>Arrayexpress strain or line</th>\n",
       "      <th>ASM name</th>\n",
       "      <th>Assembly accession</th>\n",
       "      <th>Assembly method</th>\n",
       "      <th>Assembly level</th>\n",
       "      <th>Biomaterial provider</th>\n",
       "      <th>Bioproject accession</th>\n",
       "      <th>Biosample accession</th>\n",
       "      <th>Biosample model</th>\n",
       "      <th>Biosample title</th>\n",
       "      <th>Biotic relationship</th>\n",
       "      <th>Body sample site</th>\n",
       "      <th>Cell shape</th>\n",
       "      <th>Chromosomes</th>\n",
       "      <th>Collected by</th>\n",
       "      <th>Collection date</th>\n",
       "      <th>Comment paragraph</th>\n",
       "      <th>Completeness estimated</th>\n",
       "      <th>Completion date</th>\n",
       "      <th>Contact email</th>\n",
       "      <th>Contact lab</th>\n",
       "      <th>Contamination estimated</th>\n",
       "      <th>Contigs</th>\n",
       "      <th>Country</th>\n",
       "      <th>Culture collection</th>\n",
       "      <th>Depth</th>\n",
       "      <th>Derived from</th>\n",
       "      <th>Description</th>\n",
       "      <th>Disease</th>\n",
       "      <th>Elevation</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>ENA-first-public</th>\n",
       "      <th>ENA-last-update</th>\n",
       "      <th>Env biome</th>\n",
       "      <th>Env broad scale</th>\n",
       "      <th>Env feature</th>\n",
       "      <th>Env local scale</th>\n",
       "      <th>Env material</th>\n",
       "      <th>Env medium</th>\n",
       "      <th>Environment</th>\n",
       "      <th>Environmental sample</th>\n",
       "      <th>Estimated size</th>\n",
       "      <th>Excluded from refseq</th>\n",
       "      <th>FDA cvm ID</th>\n",
       "      <th>Finishing strategy (depth of coverage)</th>\n",
       "      <th>First name</th>\n",
       "      <th>Ftp path</th>\n",
       "      <th>Gbrs paired ASM</th>\n",
       "      <th>Gc content</th>\n",
       "      <th>Genbank accessions</th>\n",
       "      <th>Gene calling method</th>\n",
       "      <th>Genome coverage</th>\n",
       "      <th>Genome ID</th>\n",
       "      <th>Genome length</th>\n",
       "      <th>Genome name</th>\n",
       "      <th>Genome quality flags</th>\n",
       "      <th>Genome rep</th>\n",
       "      <th>Genome status</th>\n",
       "      <th>Genotype</th>\n",
       "      <th>Geo loc name</th>\n",
       "      <th>Geographic location</th>\n",
       "      <th>Geographic location (country and/or atlantic ocean)</th>\n",
       "      <th>GOLD stamp ID</th>\n",
       "      <th>Gram stain</th>\n",
       "      <th>Habitat</th>\n",
       "      <th>Host</th>\n",
       "      <th>Host age</th>\n",
       "      <th>Host disease</th>\n",
       "      <th>Host disease outcome</th>\n",
       "      <th>Host gender</th>\n",
       "      <th>Host health</th>\n",
       "      <th>Host name</th>\n",
       "      <th>Host scientific name</th>\n",
       "      <th>Host sex</th>\n",
       "      <th>Host taxid</th>\n",
       "      <th>Host tissue sampled</th>\n",
       "      <th>Identification method</th>\n",
       "      <th>Identified by</th>\n",
       "      <th>Infraspecific name</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>Investigation type</th>\n",
       "      <th>Isol growth condt</th>\n",
       "      <th>Isolate</th>\n",
       "      <th>Isolation comments</th>\n",
       "      <th>Isolation country</th>\n",
       "      <th>Isolation site</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>Last name</th>\n",
       "      <th>lat lon</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Locus tag prefix</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Mapping method and version</th>\n",
       "      <th>Material</th>\n",
       "      <th>Metagenome source</th>\n",
       "      <th>Metagenomic</th>\n",
       "      <th>Misc param</th>\n",
       "      <th>Misc param: hmp supersite</th>\n",
       "      <th>Mlst</th>\n",
       "      <th>Motility</th>\n",
       "      <th>Name</th>\n",
       "      <th>Ncbi taxon ID</th>\n",
       "      <th>Note</th>\n",
       "      <th>Num replicons</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Organism name</th>\n",
       "      <th>Other clinical</th>\n",
       "      <th>Other environmental</th>\n",
       "      <th>Oxygen requirement</th>\n",
       "      <th>Paired ASM comp</th>\n",
       "      <th>pH</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Plasmids</th>\n",
       "      <th>Project name</th>\n",
       "      <th>Project type</th>\n",
       "      <th>Publication</th>\n",
       "      <th>Publication date</th>\n",
       "      <th>Quality assessment method and version</th>\n",
       "      <th>Ref biomaterial</th>\n",
       "      <th>Refseq accessions</th>\n",
       "      <th>Refseq category</th>\n",
       "      <th>Refseq cds</th>\n",
       "      <th>Rel to oxygen</th>\n",
       "      <th>Relation to type material</th>\n",
       "      <th>Release type</th>\n",
       "      <th>Salinity</th>\n",
       "      <th>Samp size</th>\n",
       "      <th>Sample accession</th>\n",
       "      <th>Sample name</th>\n",
       "      <th>Sample type</th>\n",
       "      <th>Seq rel date</th>\n",
       "      <th>Sequencing centers</th>\n",
       "      <th>Sequencing depth</th>\n",
       "      <th>Sequencing method</th>\n",
       "      <th>Sequencing platform</th>\n",
       "      <th>Sequencing status</th>\n",
       "      <th>Sop</th>\n",
       "      <th>Source mat ID</th>\n",
       "      <th>Species taxid</th>\n",
       "      <th>Specific host</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>Strain</th>\n",
       "      <th>Strain name alias</th>\n",
       "      <th>Subject ID</th>\n",
       "      <th>Submission model</th>\n",
       "      <th>Submission package</th>\n",
       "      <th>Submitter</th>\n",
       "      <th>Subsrc note</th>\n",
       "      <th>Supplier name</th>\n",
       "      <th>Taxid</th>\n",
       "      <th>Temperature</th>\n",
       "      <th>Temperature optimum</th>\n",
       "      <th>Temperature range</th>\n",
       "      <th>Title</th>\n",
       "      <th>Trophic level</th>\n",
       "      <th>Type strain</th>\n",
       "      <th>Type material</th>\n",
       "      <th>Value</th>\n",
       "      <th>Version status</th>\n",
       "      <th>WGS master</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM1664v1</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Sample from Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae ATCC 17061. Flavobac...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2007-05-01T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000016645.1</td>\n",
       "      <td>34.1</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>6096872.0</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>Complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>United Kingdom: England</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Multiple</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=UW101; ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>isolated from soil in England and is the type ...</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>NaN</td>\n",
       "      <td>soil</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2014-01-28T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_009441</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>5017.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Major</td>\n",
       "      <td>Non-halophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2007/05/01</td>\n",
       "      <td>DOE Joint Genome Institute</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>986.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>US DOE Joint Genome Institute</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>376686</td>\n",
       "      <td>NaN</td>\n",
       "      <td>20-30</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>type strain of Flavobacterium johnsoniae</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM2328v1</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic sample from Capnocytophaga ochracea DS...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2009-04-30T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000023285.1</td>\n",
       "      <td>39.6</td>\n",
       "      <td>CP001632</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>2612925.0</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>Complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Host-associated</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>isolated from the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Facultative</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>21304645</td>\n",
       "      <td>2009-02-19T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_013162</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>2171.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Major</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2009/08/26</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1018.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>SRR013476,SRR013477</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>521097</td>\n",
       "      <td>NaN</td>\n",
       "      <td>35-37</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Alias Altitude Anonymized name Arrayexpress species  \\\n",
       "0   NaN      NaN             NaN                  NaN   \n",
       "1   NaN      NaN             NaN                  NaN   \n",
       "\n",
       "  Arrayexpress strain or line   ASM name Assembly accession Assembly method  \\\n",
       "0                         NaN  ASM1664v1    GCA_000016645.1             NaN   \n",
       "1                         NaN  ASM2328v1    GCA_000023285.1             NaN   \n",
       "\n",
       "    Assembly level Biomaterial provider Bioproject accession  \\\n",
       "0  Complete Genome                  NaN           PRJNA16082   \n",
       "1  Complete Genome                  NaN           PRJNA29403   \n",
       "\n",
       "  Biosample accession Biosample model  \\\n",
       "0        SAMN02598357             NaN   \n",
       "1        SAMN00001911             NaN   \n",
       "\n",
       "                                     Biosample title Biotic relationship  \\\n",
       "0        Sample from Flavobacterium johnsoniae UW101                 NaN   \n",
       "1  Generic sample from Capnocytophaga ochracea DS...                 NaN   \n",
       "\n",
       "  Body sample site Cell shape  Chromosomes Collected by Collection date  \\\n",
       "0              NaN        Rod          1.0          NaN             NaN   \n",
       "1              NaN        Rod          1.0          NaN             NaN   \n",
       "\n",
       "                                   Comment paragraph Completeness estimated  \\\n",
       "0  Flavobacterium johnsoniae ATCC 17061. Flavobac...                    NaN   \n",
       "1                   Capnocytophaga ochracea DSM 7271                    NaN   \n",
       "\n",
       "        Completion date Contact email Contact lab Contamination estimated  \\\n",
       "0  2007-05-01T00:00:00Z           NaN         NaN                     NaN   \n",
       "1  2009-04-30T00:00:00Z           NaN         NaN                     NaN   \n",
       "\n",
       "   Contigs Country Culture collection Depth Derived from Description Disease  \\\n",
       "0      1.0     NaN         ATCC 17061   NaN          NaN         NaN     NaN   \n",
       "1      1.0     NaN           DSM 7271   NaN          NaN         NaN     NaN   \n",
       "\n",
       "  Elevation ENA checklist ENA-first-public ENA-last-update Env biome  \\\n",
       "0       NaN           NaN              NaN             NaN       NaN   \n",
       "1       NaN           NaN              NaN             NaN       NaN   \n",
       "\n",
       "  Env broad scale Env feature Env local scale Env material Env medium  \\\n",
       "0             NaN         NaN             NaN          NaN        NaN   \n",
       "1             NaN         NaN             NaN          NaN        NaN   \n",
       "\n",
       "  Environment Environmental sample Estimated size Excluded from refseq  \\\n",
       "0         NaN                  NaN            NaN                  NaN   \n",
       "1         NaN                  NaN            NaN                  NaN   \n",
       "\n",
       "  FDA cvm ID Finishing strategy (depth of coverage) First name  \\\n",
       "0        NaN                                    NaN        NaN   \n",
       "1        NaN                                    NaN        NaN   \n",
       "\n",
       "                                            Ftp path  Gbrs paired ASM  \\\n",
       "0  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...  GCF_000016645.1   \n",
       "1  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...  GCF_000023285.1   \n",
       "\n",
       "   Gc content Genbank accessions Gene calling method Genome coverage  \\\n",
       "0        34.1           CP000685                 NaN             NaN   \n",
       "1        39.6           CP001632                 NaN             NaN   \n",
       "\n",
       "         Genome ID  Genome length                       Genome name  \\\n",
       "0  GCA_000016645.1      6096872.0   Flavobacterium johnsoniae UW101   \n",
       "1  GCA_000023285.1      2612925.0  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "  Genome quality flags Genome rep Genome status Genotype Geo loc name  \\\n",
       "0                  NaN       Full      Complete      NaN          NaN   \n",
       "1                  NaN       Full      Complete      NaN          NaN   \n",
       "\n",
       "       Geographic location  \\\n",
       "0  United Kingdom: England   \n",
       "1                      NaN   \n",
       "\n",
       "  Geographic location (country and/or atlantic ocean) GOLD stamp ID  \\\n",
       "0                                                NaN            NaN   \n",
       "1                                                NaN            NaN   \n",
       "\n",
       "  Gram stain          Habitat Host Host age Host disease Host disease outcome  \\\n",
       "0        NaN         Multiple  NaN      NaN          NaN                  NaN   \n",
       "1        NaN  Host-associated  NaN      NaN          NaN                  NaN   \n",
       "\n",
       "  Host gender Host health            Host name Host scientific name Host sex  \\\n",
       "0         NaN         NaN                  NaN                  NaN      NaN   \n",
       "1         NaN         NaN  Human, Homo sapiens                  NaN      NaN   \n",
       "\n",
       "   Host taxid Host tissue sampled Identification method Identified by  \\\n",
       "0         NaN                 NaN                   NaN           NaN   \n",
       "1         NaN                 NaN                   NaN           NaN   \n",
       "\n",
       "         Infraspecific name INSDC center alias INSDC center name  \\\n",
       "0  strain=UW101; ATCC 17061                NaN               NaN   \n",
       "1           strain=DSM 7271                NaN               NaN   \n",
       "\n",
       "  INSDC first public INSDC last update INSDC status Investigation type  \\\n",
       "0                NaN               NaN          NaN                NaN   \n",
       "1                NaN               NaN          NaN                NaN   \n",
       "\n",
       "  Isol growth condt Isolate  \\\n",
       "0               NaN     NaN   \n",
       "1               NaN     NaN   \n",
       "\n",
       "                                  Isolation comments Isolation country  \\\n",
       "0  isolated from soil in England and is the type ...    United Kingdom   \n",
       "1                isolated from the human oral cavity               NaN   \n",
       "\n",
       "  Isolation site       Isolation source Last name lat lon Latitude  \\\n",
       "0            NaN                   soil       NaN     NaN      NaN   \n",
       "1            NaN  the human oral cavity       NaN     NaN      NaN   \n",
       "\n",
       "  Locus tag prefix Longitude Mapping method and version Material  \\\n",
       "0              NaN       NaN                        NaN      NaN   \n",
       "1              NaN       NaN                        NaN      NaN   \n",
       "\n",
       "  Metagenome source Metagenomic Misc param Misc param: hmp supersite Mlst  \\\n",
       "0               NaN         NaN        NaN                       NaN  NaN   \n",
       "1               NaN         NaN        NaN                       NaN  NaN   \n",
       "\n",
       "  Motility                                               Name  Ncbi taxon ID  \\\n",
       "0      Yes                                               NCBI       376686.0   \n",
       "1      Yes  Joint Genome Institute, U.S. Department of Energy       521097.0   \n",
       "\n",
       "  Note Num replicons                          Organism  \\\n",
       "0  NaN           NaN   Flavobacterium johnsoniae UW101   \n",
       "1  NaN           NaN  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "                      Organism name Other clinical Other environmental  \\\n",
       "0   Flavobacterium johnsoniae UW101            NaN                 NaN   \n",
       "1  Capnocytophaga ochracea DSM 7271            NaN                 NaN   \n",
       "\n",
       "  Oxygen requirement Paired ASM comp   pH Phenotypes  Plasmids Project name  \\\n",
       "0            Aerobic       identical  NaN        NaN       NaN          NaN   \n",
       "1        Facultative       identical  NaN        NaN       NaN          NaN   \n",
       "\n",
       "  Project type Publication         Publication date  \\\n",
       "0          NaN         NaN  2014-01-28T00:00:00.000   \n",
       "1          NaN    21304645  2009-02-19T00:00:00.000   \n",
       "\n",
       "  Quality assessment method and version Ref biomaterial Refseq accessions  \\\n",
       "0                                   NaN             NaN         NC_009441   \n",
       "1                                   NaN             NaN         NC_013162   \n",
       "\n",
       "         Refseq category  Refseq cds Rel to oxygen  \\\n",
       "0  representative genome      5017.0           NaN   \n",
       "1  representative genome      2171.0           NaN   \n",
       "\n",
       "     Relation to type material Release type        Salinity Samp size  \\\n",
       "0  assembly from type material        Major  Non-halophilic       NaN   \n",
       "1  assembly from type material        Major             NaN       NaN   \n",
       "\n",
       "  Sample accession Sample name Sample type Seq rel date  \\\n",
       "0     SAMN02598357    CP000685         NaN   2007/05/01   \n",
       "1     SAMN00001911        FWCB         NaN   2009/08/26   \n",
       "\n",
       "                                  Sequencing centers Sequencing depth  \\\n",
       "0                         DOE Joint Genome Institute              NaN   \n",
       "1  US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...              NaN   \n",
       "\n",
       "  Sequencing method Sequencing platform Sequencing status  Sop Source mat ID  \\\n",
       "0               NaN                 NaN          complete  NaN           NaN   \n",
       "1               NaN                 NaN          complete  NaN           NaN   \n",
       "\n",
       "   Species taxid Specific host Sporulation        SRA accession      Strain  \\\n",
       "0          986.0           NaN          No                  NaN  ATCC 17061   \n",
       "1         1018.0           NaN          No  SRR013476,SRR013477    DSM 7271   \n",
       "\n",
       "  Strain name alias Subject ID Submission model Submission package  \\\n",
       "0               NaN        NaN          Generic        Generic.1.0   \n",
       "1               NaN        NaN          Generic        Generic.1.0   \n",
       "\n",
       "                                 Submitter Subsrc note Supplier name   Taxid  \\\n",
       "0            US DOE Joint Genome Institute         NaN           NaN  376686   \n",
       "1  US DOE Joint Genome Institute (JGI-PGF)         NaN           NaN  521097   \n",
       "\n",
       "  Temperature Temperature optimum Temperature range Title Trophic level  \\\n",
       "0         NaN               20-30        Mesophilic   NaN           NaN   \n",
       "1         NaN               35-37        Mesophilic   NaN           NaN   \n",
       "\n",
       "  Type strain                             Type material Value Version status  \\\n",
       "0         Yes  type strain of Flavobacterium johnsoniae   NaN         latest   \n",
       "1         Yes    type strain of Capnocytophaga ochracea   NaN         latest   \n",
       "\n",
       "  WGS master  \n",
       "0        NaN  \n",
       "1        NaN  "
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dummy4.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/gomes/anaconda3/lib/python3.7/site-packages/pandas/core/frame.py:3997: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  errors=errors,\n"
     ]
    }
   ],
   "source": [
    "dummy4.drop(columns=[\"Completeness estimated\",\"Contamination estimated\",\"Gc content\",\"Genome length\",\"Genome coverage\"], inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [],
   "source": [
    "dummy4.to_csv(\"Metadata.csv\", index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.6 Check if sample is marine or not \n",
    "From coordinates (latitude and longitude). \\\n",
    "Python package: global_land_mask"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata = dummy4.copy()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [],
   "source": [
    "pattern = '|'.join([\" N\",\" W\", \" S\", \" E\", \" DD\", \"N\", \"E\"])\n",
    "metadata[\"latitude_longitude\"] = metadata[\"lat lon\"].str.replace(pattern, \"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata[\"Lat\"] = metadata[\"Latitude\"].str.replace(\" DD\", \"\")\n",
    "metadata[\"Long\"] = metadata[\"Longitude\"].str.replace(\" DD\", \"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [],
   "source": [
    "new = metadata[\"latitude_longitude\"].str.split(\" \", expand=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [],
   "source": [
    "# making separate column names\n",
    "metadata[\"lat\"]= new[0] \n",
    "metadata[\"lon\"]= new[1] "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata[\"Latitude\"] = metadata[\"Lat\"].combine_first(metadata[\"lat\"])\n",
    "metadata[\"Longitude\"] = metadata[\"Long\"].combine_first(metadata[\"lon\"])\n",
    "metadata.drop(columns=[\"latitude_longitude\",\"Lat\", \"Long\", \"lat lon\", \"lat\", \"lon\"], inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = metadata[[\"Assembly accession\", \"Latitude\", \"Longitude\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [],
   "source": [
    "from global_land_mask import globe\n",
    "d = {}\n",
    "for i in df.iterrows():\n",
    "    for x in i[1:]:\n",
    "        accession = x[0]\n",
    "        #print(accession)\n",
    "        lat = (x[1])\n",
    "        #print(lat)\n",
    "        lon = (x[2])\n",
    "        #print(lon)\n",
    "        try:\n",
    "            lat = float(lat)\n",
    "            lon = float(lon)\n",
    "            try:\n",
    "                is_on_land = globe.is_land(lat,lon)\n",
    "                d[accession] = is_on_land\n",
    "                #print(\"added to dic\")\n",
    "            except:\n",
    "                d[accession] = None\n",
    "                #print(\"out of bonds\")\n",
    "        except ValueError:\n",
    "            d[accession] = None\n",
    "            #print(\"Not a float\")\n",
    "        #print(\"\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [],
   "source": [
    "is_marine = pd.DataFrame.from_dict(d, orient='index', columns=[\"Marine_coordinates\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "both          2686\n",
       "right_only       0\n",
       "left_only        0\n",
       "Name: merge, dtype: int64"
      ]
     },
     "execution_count": 66,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merge = pd.merge(metadata, is_marine, how=\"left\", left_on=\"Assembly accession\", right_index=True, indicator=\"merge\")\n",
    "merge[\"merge\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True     393\n",
       "False    270\n",
       "Name: Marine_coordinates, dtype: int64"
      ]
     },
     "execution_count": 67,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merge[\"Marine_coordinates\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Index: 2686 entries, 0 to 2685\n",
      "Columns: 164 entries, Alias to Marine_coordinates\n",
      "dtypes: float64(7), object(157)\n",
      "memory usage: 3.4+ MB\n"
     ]
    }
   ],
   "source": [
    "merge = merge.drop(columns=[\"merge\"])\n",
    "merge.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [],
   "source": [
    "merge = merge.set_index(\"Assembly accession\")\n",
    "merge.to_csv(\"Metadata.csv\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.7 Classifier"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "after running classifier_v3.0.py:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Assembly accession</th>\n",
       "      <th>Alias</th>\n",
       "      <th>Altitude</th>\n",
       "      <th>Anonymized name</th>\n",
       "      <th>Arrayexpress species</th>\n",
       "      <th>Arrayexpress strain or line</th>\n",
       "      <th>ASM name</th>\n",
       "      <th>Assembly method</th>\n",
       "      <th>Assembly level</th>\n",
       "      <th>Biomaterial provider</th>\n",
       "      <th>Bioproject accession</th>\n",
       "      <th>Biosample accession</th>\n",
       "      <th>Biosample model</th>\n",
       "      <th>Biosample title</th>\n",
       "      <th>Biotic relationship</th>\n",
       "      <th>Body sample site</th>\n",
       "      <th>Cell shape</th>\n",
       "      <th>Chromosomes</th>\n",
       "      <th>Collected by</th>\n",
       "      <th>Collection date</th>\n",
       "      <th>Comment paragraph</th>\n",
       "      <th>Completion date</th>\n",
       "      <th>Contact email</th>\n",
       "      <th>Contact lab</th>\n",
       "      <th>Contigs</th>\n",
       "      <th>Country</th>\n",
       "      <th>Culture collection</th>\n",
       "      <th>Depth</th>\n",
       "      <th>Derived from</th>\n",
       "      <th>Description</th>\n",
       "      <th>Disease</th>\n",
       "      <th>Elevation</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>ENA-first-public</th>\n",
       "      <th>ENA-last-update</th>\n",
       "      <th>Env biome</th>\n",
       "      <th>Env broad scale</th>\n",
       "      <th>Env feature</th>\n",
       "      <th>Env local scale</th>\n",
       "      <th>Env material</th>\n",
       "      <th>Env medium</th>\n",
       "      <th>Environment</th>\n",
       "      <th>Environmental sample</th>\n",
       "      <th>Estimated size</th>\n",
       "      <th>Excluded from refseq</th>\n",
       "      <th>FDA cvm ID</th>\n",
       "      <th>Finishing strategy (depth of coverage)</th>\n",
       "      <th>First name</th>\n",
       "      <th>Ftp path</th>\n",
       "      <th>Gbrs paired ASM</th>\n",
       "      <th>Genbank accessions</th>\n",
       "      <th>Gene calling method</th>\n",
       "      <th>Genome ID</th>\n",
       "      <th>Genome name</th>\n",
       "      <th>Genome quality flags</th>\n",
       "      <th>Genome rep</th>\n",
       "      <th>Genome status</th>\n",
       "      <th>Genotype</th>\n",
       "      <th>Geo loc name</th>\n",
       "      <th>Geographic location</th>\n",
       "      <th>Geographic location (country and/or atlantic ocean)</th>\n",
       "      <th>GOLD stamp ID</th>\n",
       "      <th>Gram stain</th>\n",
       "      <th>Habitat</th>\n",
       "      <th>Host</th>\n",
       "      <th>Host age</th>\n",
       "      <th>Host disease</th>\n",
       "      <th>Host disease outcome</th>\n",
       "      <th>Host gender</th>\n",
       "      <th>Host health</th>\n",
       "      <th>Host name</th>\n",
       "      <th>Host scientific name</th>\n",
       "      <th>Host sex</th>\n",
       "      <th>Host taxid</th>\n",
       "      <th>Host tissue sampled</th>\n",
       "      <th>Identification method</th>\n",
       "      <th>Identified by</th>\n",
       "      <th>Infraspecific name</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>Investigation type</th>\n",
       "      <th>Isol growth condt</th>\n",
       "      <th>Isolate</th>\n",
       "      <th>Isolation comments</th>\n",
       "      <th>Isolation country</th>\n",
       "      <th>Isolation site</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>Last name</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Locus tag prefix</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Mapping method and version</th>\n",
       "      <th>Material</th>\n",
       "      <th>Metagenome source</th>\n",
       "      <th>Metagenomic</th>\n",
       "      <th>Misc param</th>\n",
       "      <th>Misc param: hmp supersite</th>\n",
       "      <th>Mlst</th>\n",
       "      <th>Motility</th>\n",
       "      <th>Name</th>\n",
       "      <th>Ncbi taxon ID</th>\n",
       "      <th>Note</th>\n",
       "      <th>Num replicons</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Organism name</th>\n",
       "      <th>Other clinical</th>\n",
       "      <th>Other environmental</th>\n",
       "      <th>Oxygen requirement</th>\n",
       "      <th>Paired ASM comp</th>\n",
       "      <th>pH</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Plasmids</th>\n",
       "      <th>Project name</th>\n",
       "      <th>Project type</th>\n",
       "      <th>Publication</th>\n",
       "      <th>Publication date</th>\n",
       "      <th>Quality assessment method and version</th>\n",
       "      <th>Ref biomaterial</th>\n",
       "      <th>Refseq accessions</th>\n",
       "      <th>Refseq category</th>\n",
       "      <th>Refseq cds</th>\n",
       "      <th>Rel to oxygen</th>\n",
       "      <th>Relation to type material</th>\n",
       "      <th>Release type</th>\n",
       "      <th>Salinity</th>\n",
       "      <th>Samp size</th>\n",
       "      <th>Sample accession</th>\n",
       "      <th>Sample name</th>\n",
       "      <th>Sample type</th>\n",
       "      <th>Seq rel date</th>\n",
       "      <th>Sequencing centers</th>\n",
       "      <th>Sequencing depth</th>\n",
       "      <th>Sequencing method</th>\n",
       "      <th>Sequencing platform</th>\n",
       "      <th>Sequencing status</th>\n",
       "      <th>Serovar</th>\n",
       "      <th>Sop</th>\n",
       "      <th>Source mat ID</th>\n",
       "      <th>Species taxid</th>\n",
       "      <th>Specific host</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>Strain</th>\n",
       "      <th>Strain name alias</th>\n",
       "      <th>Subject ID</th>\n",
       "      <th>Submission model</th>\n",
       "      <th>Submission package</th>\n",
       "      <th>Submitter</th>\n",
       "      <th>Subsrc note</th>\n",
       "      <th>Supplier name</th>\n",
       "      <th>Taxid</th>\n",
       "      <th>Temperature</th>\n",
       "      <th>Temperature optimum</th>\n",
       "      <th>Temperature range</th>\n",
       "      <th>Title</th>\n",
       "      <th>Trophic level</th>\n",
       "      <th>Type strain</th>\n",
       "      <th>Type material</th>\n",
       "      <th>Value</th>\n",
       "      <th>Version status</th>\n",
       "      <th>WGS master</th>\n",
       "      <th>Marine_coordinates</th>\n",
       "      <th>MAG_words</th>\n",
       "      <th>MAG</th>\n",
       "      <th>Human-associated_words</th>\n",
       "      <th>Human-associated</th>\n",
       "      <th>Terrestrial_words</th>\n",
       "      <th>Terrestrial</th>\n",
       "      <th>Marine_words</th>\n",
       "      <th>Marine</th>\n",
       "      <th>Freshwater_words</th>\n",
       "      <th>Freshwater</th>\n",
       "      <th>Origin</th>\n",
       "      <th>Conflict_words</th>\n",
       "      <th>Conflict?</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM1664v1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Sample from Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae ATCC 17061. Flavobac...</td>\n",
       "      <td>2007-05-01T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000016645.1</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>Complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>United Kingdom: England</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Multiple</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=UW101; ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>isolated from soil in England and is the type ...</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>NaN</td>\n",
       "      <td>soil</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2014-01-28T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_009441</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>5017.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Major</td>\n",
       "      <td>Non-halophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2007/05/01</td>\n",
       "      <td>DOE Joint Genome Institute</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>986.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>US DOE Joint Genome Institute</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>20-30</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>type strain of Flavobacterium johnsoniae</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>['soil']</td>\n",
       "      <td>Terrestrial</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Terrestrial</td>\n",
       "      <td>Terrestrial</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM2328v1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic sample from Capnocytophaga ochracea DS...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>2009-04-30T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000023285.1</td>\n",
       "      <td>CP001632</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>Complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Host-associated</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>isolated from the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Facultative</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>21304645</td>\n",
       "      <td>2009-02-19T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_013162</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>2171.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Major</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2009/08/26</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1018.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>SRR013476,SRR013477</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>35-37</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>['\"human', 'homo sapiens\"']</td>\n",
       "      <td>Human-associated</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human-associated</td>\n",
       "      <td>Human-associated</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Assembly accession Alias Altitude Anonymized name Arrayexpress species  \\\n",
       "0    GCA_000016645.1   NaN      NaN             NaN                  NaN   \n",
       "1    GCA_000023285.1   NaN      NaN             NaN                  NaN   \n",
       "\n",
       "  Arrayexpress strain or line   ASM name Assembly method   Assembly level  \\\n",
       "0                         NaN  ASM1664v1             NaN  Complete Genome   \n",
       "1                         NaN  ASM2328v1             NaN  Complete Genome   \n",
       "\n",
       "  Biomaterial provider Bioproject accession Biosample accession  \\\n",
       "0                  NaN           PRJNA16082        SAMN02598357   \n",
       "1                  NaN           PRJNA29403        SAMN00001911   \n",
       "\n",
       "  Biosample model                                    Biosample title  \\\n",
       "0             NaN        Sample from Flavobacterium johnsoniae UW101   \n",
       "1             NaN  Generic sample from Capnocytophaga ochracea DS...   \n",
       "\n",
       "  Biotic relationship Body sample site Cell shape  Chromosomes Collected by  \\\n",
       "0                 NaN              NaN        Rod          1.0          NaN   \n",
       "1                 NaN              NaN        Rod          1.0          NaN   \n",
       "\n",
       "  Collection date                                  Comment paragraph  \\\n",
       "0             NaN  Flavobacterium johnsoniae ATCC 17061. Flavobac...   \n",
       "1             NaN                   Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "        Completion date Contact email Contact lab  Contigs Country  \\\n",
       "0  2007-05-01T00:00:00Z           NaN         NaN      1.0     NaN   \n",
       "1  2009-04-30T00:00:00Z           NaN         NaN      1.0     NaN   \n",
       "\n",
       "  Culture collection Depth Derived from Description Disease Elevation  \\\n",
       "0         ATCC 17061   NaN          NaN         NaN     NaN       NaN   \n",
       "1           DSM 7271   NaN          NaN         NaN     NaN       NaN   \n",
       "\n",
       "  ENA checklist ENA-first-public ENA-last-update Env biome Env broad scale  \\\n",
       "0           NaN              NaN             NaN       NaN             NaN   \n",
       "1           NaN              NaN             NaN       NaN             NaN   \n",
       "\n",
       "  Env feature Env local scale Env material Env medium Environment  \\\n",
       "0         NaN             NaN          NaN        NaN         NaN   \n",
       "1         NaN             NaN          NaN        NaN         NaN   \n",
       "\n",
       "  Environmental sample Estimated size Excluded from refseq FDA cvm ID  \\\n",
       "0                  NaN            NaN                  NaN        NaN   \n",
       "1                  NaN            NaN                  NaN        NaN   \n",
       "\n",
       "  Finishing strategy (depth of coverage) First name  \\\n",
       "0                                    NaN        NaN   \n",
       "1                                    NaN        NaN   \n",
       "\n",
       "                                            Ftp path  Gbrs paired ASM  \\\n",
       "0  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...  GCF_000016645.1   \n",
       "1  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...  GCF_000023285.1   \n",
       "\n",
       "  Genbank accessions Gene calling method        Genome ID  \\\n",
       "0           CP000685                 NaN  GCA_000016645.1   \n",
       "1           CP001632                 NaN  GCA_000023285.1   \n",
       "\n",
       "                        Genome name Genome quality flags Genome rep  \\\n",
       "0   Flavobacterium johnsoniae UW101                  NaN       Full   \n",
       "1  Capnocytophaga ochracea DSM 7271                  NaN       Full   \n",
       "\n",
       "  Genome status Genotype Geo loc name      Geographic location  \\\n",
       "0      Complete      NaN          NaN  United Kingdom: England   \n",
       "1      Complete      NaN          NaN                      NaN   \n",
       "\n",
       "  Geographic location (country and/or atlantic ocean) GOLD stamp ID  \\\n",
       "0                                                NaN            NaN   \n",
       "1                                                NaN            NaN   \n",
       "\n",
       "  Gram stain          Habitat Host Host age Host disease Host disease outcome  \\\n",
       "0        NaN         Multiple  NaN      NaN          NaN                  NaN   \n",
       "1        NaN  Host-associated  NaN      NaN          NaN                  NaN   \n",
       "\n",
       "  Host gender Host health            Host name Host scientific name Host sex  \\\n",
       "0         NaN         NaN                  NaN                  NaN      NaN   \n",
       "1         NaN         NaN  Human, Homo sapiens                  NaN      NaN   \n",
       "\n",
       "   Host taxid Host tissue sampled Identification method Identified by  \\\n",
       "0         NaN                 NaN                   NaN           NaN   \n",
       "1         NaN                 NaN                   NaN           NaN   \n",
       "\n",
       "         Infraspecific name INSDC center alias INSDC center name  \\\n",
       "0  strain=UW101; ATCC 17061                NaN               NaN   \n",
       "1           strain=DSM 7271                NaN               NaN   \n",
       "\n",
       "  INSDC first public INSDC last update INSDC status Investigation type  \\\n",
       "0                NaN               NaN          NaN                NaN   \n",
       "1                NaN               NaN          NaN                NaN   \n",
       "\n",
       "  Isol growth condt Isolate  \\\n",
       "0               NaN     NaN   \n",
       "1               NaN     NaN   \n",
       "\n",
       "                                  Isolation comments Isolation country  \\\n",
       "0  isolated from soil in England and is the type ...    United Kingdom   \n",
       "1                isolated from the human oral cavity               NaN   \n",
       "\n",
       "  Isolation site       Isolation source Last name Latitude Locus tag prefix  \\\n",
       "0            NaN                   soil       NaN      NaN              NaN   \n",
       "1            NaN  the human oral cavity       NaN      NaN              NaN   \n",
       "\n",
       "  Longitude Mapping method and version Material Metagenome source Metagenomic  \\\n",
       "0       NaN                        NaN      NaN               NaN         NaN   \n",
       "1       NaN                        NaN      NaN               NaN         NaN   \n",
       "\n",
       "  Misc param Misc param: hmp supersite Mlst Motility  \\\n",
       "0        NaN                       NaN  NaN      Yes   \n",
       "1        NaN                       NaN  NaN      Yes   \n",
       "\n",
       "                                                Name  Ncbi taxon ID Note  \\\n",
       "0                                               NCBI       376686.0  NaN   \n",
       "1  Joint Genome Institute, U.S. Department of Energy       521097.0  NaN   \n",
       "\n",
       "  Num replicons                          Organism  \\\n",
       "0           NaN   Flavobacterium johnsoniae UW101   \n",
       "1           NaN  Capnocytophaga ochracea DSM 7271   \n",
       "\n",
       "                      Organism name Other clinical Other environmental  \\\n",
       "0   Flavobacterium johnsoniae UW101            NaN                 NaN   \n",
       "1  Capnocytophaga ochracea DSM 7271            NaN                 NaN   \n",
       "\n",
       "  Oxygen requirement Paired ASM comp   pH Phenotypes  Plasmids Project name  \\\n",
       "0            Aerobic       identical  NaN        NaN       NaN          NaN   \n",
       "1        Facultative       identical  NaN        NaN       NaN          NaN   \n",
       "\n",
       "  Project type Publication         Publication date  \\\n",
       "0          NaN         NaN  2014-01-28T00:00:00.000   \n",
       "1          NaN    21304645  2009-02-19T00:00:00.000   \n",
       "\n",
       "  Quality assessment method and version Ref biomaterial Refseq accessions  \\\n",
       "0                                   NaN             NaN         NC_009441   \n",
       "1                                   NaN             NaN         NC_013162   \n",
       "\n",
       "         Refseq category  Refseq cds Rel to oxygen  \\\n",
       "0  representative genome      5017.0           NaN   \n",
       "1  representative genome      2171.0           NaN   \n",
       "\n",
       "     Relation to type material Release type        Salinity Samp size  \\\n",
       "0  assembly from type material        Major  Non-halophilic       NaN   \n",
       "1  assembly from type material        Major             NaN       NaN   \n",
       "\n",
       "  Sample accession Sample name Sample type Seq rel date  \\\n",
       "0     SAMN02598357    CP000685         NaN   2007/05/01   \n",
       "1     SAMN00001911        FWCB         NaN   2009/08/26   \n",
       "\n",
       "                                  Sequencing centers Sequencing depth  \\\n",
       "0                         DOE Joint Genome Institute              NaN   \n",
       "1  US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...              NaN   \n",
       "\n",
       "  Sequencing method Sequencing platform Sequencing status Serovar  Sop  \\\n",
       "0               NaN                 NaN          complete     NaN  NaN   \n",
       "1               NaN                 NaN          complete     NaN  NaN   \n",
       "\n",
       "  Source mat ID  Species taxid Specific host Sporulation        SRA accession  \\\n",
       "0           NaN          986.0           NaN          No                  NaN   \n",
       "1           NaN         1018.0           NaN          No  SRR013476,SRR013477   \n",
       "\n",
       "       Strain Strain name alias Subject ID Submission model  \\\n",
       "0  ATCC 17061               NaN        NaN          Generic   \n",
       "1    DSM 7271               NaN        NaN          Generic   \n",
       "\n",
       "  Submission package                                Submitter Subsrc note  \\\n",
       "0        Generic.1.0            US DOE Joint Genome Institute         NaN   \n",
       "1        Generic.1.0  US DOE Joint Genome Institute (JGI-PGF)         NaN   \n",
       "\n",
       "  Supplier name     Taxid Temperature Temperature optimum Temperature range  \\\n",
       "0           NaN  376686.0         NaN               20-30        Mesophilic   \n",
       "1           NaN  521097.0         NaN               35-37        Mesophilic   \n",
       "\n",
       "  Title Trophic level Type strain                             Type material  \\\n",
       "0   NaN           NaN         Yes  type strain of Flavobacterium johnsoniae   \n",
       "1   NaN           NaN         Yes    type strain of Capnocytophaga ochracea   \n",
       "\n",
       "  Value Version status WGS master Marine_coordinates MAG_words  MAG  \\\n",
       "0   NaN         latest        NaN                NaN       NaN  NaN   \n",
       "1   NaN         latest        NaN                NaN       NaN  NaN   \n",
       "\n",
       "        Human-associated_words  Human-associated Terrestrial_words  \\\n",
       "0                          NaN               NaN          ['soil']   \n",
       "1  ['\"human', 'homo sapiens\"']  Human-associated               NaN   \n",
       "\n",
       "   Terrestrial Marine_words Marine Freshwater_words Freshwater  \\\n",
       "0  Terrestrial          NaN    NaN              NaN        NaN   \n",
       "1          NaN          NaN    NaN              NaN        NaN   \n",
       "\n",
       "             Origin    Conflict_words Conflict?  \n",
       "0       Terrestrial       Terrestrial       NaN  \n",
       "1  Human-associated  Human-associated       NaN  "
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "metadata_class = pd.read_csv(\"Metadata_classification.csv\")\n",
    "metadata_class.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Yes    7\n",
       "Name: Conflict?, dtype: int64"
      ]
     },
     "execution_count": 87,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "metadata_class[\"Conflict?\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Unclassified    7\n",
       "Name: Conflict?, dtype: int64"
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "metadata_class[\"Conflict?\"] = [\"Unclassified\" if str(x) == \"Yes\" else None for x in metadata_class['Conflict?']]\n",
    "metadata_class[\"Conflict?\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Unclassified    1594\n",
       "Marine           660\n",
       "Non_marine       432\n",
       "Name: Origin, dtype: int64"
      ]
     },
     "execution_count": 89,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "metadata_class[\"Origin\"] = metadata_class[\"Conflict?\"].combine_first(metadata_class[\"Origin\"])\n",
    "metadata_class[\"Origin\"] = [\"Non_marine\" if  str(x) == \"Terrestrial\" or str(x) ==\"Freshwater\" or str(x) ==\"Human-associated\" else \"Marine\" if str(x)==\"Marine\" else \"Unclassified\" for x in metadata_class['Origin']]\n",
    "metadata_class[\"Origin\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata_class.drop(columns=[\"MAG_words\",\"Human-associated_words\",\"Human-associated\",\"Terrestrial_words\",\"Terrestrial\",\"Marine_words\",\"Marine\",\"Freshwater_words\",\"Freshwater\",\"Conflict_words\",\"Conflict?\"], inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Assembly accession</th>\n",
       "      <th>Alias</th>\n",
       "      <th>Altitude</th>\n",
       "      <th>Anonymized name</th>\n",
       "      <th>Arrayexpress species</th>\n",
       "      <th>Arrayexpress strain or line</th>\n",
       "      <th>ASM name</th>\n",
       "      <th>Assembly method</th>\n",
       "      <th>Assembly level</th>\n",
       "      <th>Biomaterial provider</th>\n",
       "      <th>Bioproject accession</th>\n",
       "      <th>Biosample accession</th>\n",
       "      <th>Biosample model</th>\n",
       "      <th>Biosample title</th>\n",
       "      <th>Biotic relationship</th>\n",
       "      <th>Body sample site</th>\n",
       "      <th>Cell shape</th>\n",
       "      <th>Chromosomes</th>\n",
       "      <th>Collected by</th>\n",
       "      <th>Collection date</th>\n",
       "      <th>Comment paragraph</th>\n",
       "      <th>Completion date</th>\n",
       "      <th>Contact email</th>\n",
       "      <th>Contact lab</th>\n",
       "      <th>Contigs</th>\n",
       "      <th>Country</th>\n",
       "      <th>Culture collection</th>\n",
       "      <th>Depth</th>\n",
       "      <th>Derived from</th>\n",
       "      <th>Description</th>\n",
       "      <th>Disease</th>\n",
       "      <th>Elevation</th>\n",
       "      <th>ENA checklist</th>\n",
       "      <th>ENA-first-public</th>\n",
       "      <th>ENA-last-update</th>\n",
       "      <th>Env biome</th>\n",
       "      <th>Env broad scale</th>\n",
       "      <th>Env feature</th>\n",
       "      <th>Env local scale</th>\n",
       "      <th>Env material</th>\n",
       "      <th>Env medium</th>\n",
       "      <th>Environment</th>\n",
       "      <th>Environmental sample</th>\n",
       "      <th>Estimated size</th>\n",
       "      <th>Excluded from refseq</th>\n",
       "      <th>FDA cvm ID</th>\n",
       "      <th>Finishing strategy (depth of coverage)</th>\n",
       "      <th>First name</th>\n",
       "      <th>Ftp path</th>\n",
       "      <th>Gbrs paired ASM</th>\n",
       "      <th>Genbank accessions</th>\n",
       "      <th>Gene calling method</th>\n",
       "      <th>Genome ID</th>\n",
       "      <th>Genome name</th>\n",
       "      <th>Genome quality flags</th>\n",
       "      <th>Genome rep</th>\n",
       "      <th>Genome status</th>\n",
       "      <th>Genotype</th>\n",
       "      <th>Geo loc name</th>\n",
       "      <th>Geographic location</th>\n",
       "      <th>Geographic location (country and/or atlantic ocean)</th>\n",
       "      <th>GOLD stamp ID</th>\n",
       "      <th>Gram stain</th>\n",
       "      <th>Habitat</th>\n",
       "      <th>Host</th>\n",
       "      <th>Host age</th>\n",
       "      <th>Host disease</th>\n",
       "      <th>Host disease outcome</th>\n",
       "      <th>Host gender</th>\n",
       "      <th>Host health</th>\n",
       "      <th>Host name</th>\n",
       "      <th>Host scientific name</th>\n",
       "      <th>Host sex</th>\n",
       "      <th>Host taxid</th>\n",
       "      <th>Host tissue sampled</th>\n",
       "      <th>Identification method</th>\n",
       "      <th>Identified by</th>\n",
       "      <th>Infraspecific name</th>\n",
       "      <th>INSDC center alias</th>\n",
       "      <th>INSDC center name</th>\n",
       "      <th>INSDC first public</th>\n",
       "      <th>INSDC last update</th>\n",
       "      <th>INSDC status</th>\n",
       "      <th>Investigation type</th>\n",
       "      <th>Isol growth condt</th>\n",
       "      <th>Isolate</th>\n",
       "      <th>Isolation comments</th>\n",
       "      <th>Isolation country</th>\n",
       "      <th>Isolation site</th>\n",
       "      <th>Isolation source</th>\n",
       "      <th>Last name</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Locus tag prefix</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Mapping method and version</th>\n",
       "      <th>Material</th>\n",
       "      <th>Metagenome source</th>\n",
       "      <th>Metagenomic</th>\n",
       "      <th>Misc param</th>\n",
       "      <th>Misc param: hmp supersite</th>\n",
       "      <th>Mlst</th>\n",
       "      <th>Motility</th>\n",
       "      <th>Name</th>\n",
       "      <th>Ncbi taxon ID</th>\n",
       "      <th>Note</th>\n",
       "      <th>Num replicons</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Organism name</th>\n",
       "      <th>Other clinical</th>\n",
       "      <th>Other environmental</th>\n",
       "      <th>Oxygen requirement</th>\n",
       "      <th>Paired ASM comp</th>\n",
       "      <th>pH</th>\n",
       "      <th>Phenotypes</th>\n",
       "      <th>Plasmids</th>\n",
       "      <th>Project name</th>\n",
       "      <th>Project type</th>\n",
       "      <th>Publication</th>\n",
       "      <th>Publication date</th>\n",
       "      <th>Quality assessment method and version</th>\n",
       "      <th>Ref biomaterial</th>\n",
       "      <th>Refseq accessions</th>\n",
       "      <th>Refseq category</th>\n",
       "      <th>Refseq cds</th>\n",
       "      <th>Rel to oxygen</th>\n",
       "      <th>Relation to type material</th>\n",
       "      <th>Release type</th>\n",
       "      <th>Salinity</th>\n",
       "      <th>Samp size</th>\n",
       "      <th>Sample accession</th>\n",
       "      <th>Sample name</th>\n",
       "      <th>Sample type</th>\n",
       "      <th>Seq rel date</th>\n",
       "      <th>Sequencing centers</th>\n",
       "      <th>Sequencing depth</th>\n",
       "      <th>Sequencing method</th>\n",
       "      <th>Sequencing platform</th>\n",
       "      <th>Sequencing status</th>\n",
       "      <th>Serovar</th>\n",
       "      <th>Sop</th>\n",
       "      <th>Source mat ID</th>\n",
       "      <th>Species taxid</th>\n",
       "      <th>Specific host</th>\n",
       "      <th>Sporulation</th>\n",
       "      <th>SRA accession</th>\n",
       "      <th>Strain</th>\n",
       "      <th>Strain name alias</th>\n",
       "      <th>Subject ID</th>\n",
       "      <th>Submission model</th>\n",
       "      <th>Submission package</th>\n",
       "      <th>Submitter</th>\n",
       "      <th>Subsrc note</th>\n",
       "      <th>Supplier name</th>\n",
       "      <th>Taxid</th>\n",
       "      <th>Temperature</th>\n",
       "      <th>Temperature optimum</th>\n",
       "      <th>Temperature range</th>\n",
       "      <th>Title</th>\n",
       "      <th>Trophic level</th>\n",
       "      <th>Type strain</th>\n",
       "      <th>Type material</th>\n",
       "      <th>Value</th>\n",
       "      <th>Version status</th>\n",
       "      <th>WGS master</th>\n",
       "      <th>Marine_coordinates</th>\n",
       "      <th>MAG</th>\n",
       "      <th>Origin</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM1664v1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA16082</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Sample from Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae ATCC 17061. Flavobac...</td>\n",
       "      <td>2007-05-01T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000016645.1</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>Complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>United Kingdom: England</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Multiple</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=UW101; ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>isolated from soil in England and is the type ...</td>\n",
       "      <td>United Kingdom</td>\n",
       "      <td>NaN</td>\n",
       "      <td>soil</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>Flavobacterium johnsoniae UW101</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2014-01-28T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_009441</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>5017.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Major</td>\n",
       "      <td>Non-halophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN02598357</td>\n",
       "      <td>CP000685</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2007/05/01</td>\n",
       "      <td>DOE Joint Genome Institute</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>986.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ATCC 17061</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>US DOE Joint Genome Institute</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>376686.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>20-30</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>type strain of Flavobacterium johnsoniae</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Non_marine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM2328v1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA29403</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic sample from Capnocytophaga ochracea DS...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>2009-04-30T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000023285.1</td>\n",
       "      <td>CP001632</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>Complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Host-associated</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Human, Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>isolated from the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>the human oral cavity</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>Joint Genome Institute, U.S. Department of Energy</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>Capnocytophaga ochracea DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Facultative</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>21304645</td>\n",
       "      <td>2009-02-19T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_013162</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>2171.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Major</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN00001911</td>\n",
       "      <td>FWCB</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2009/08/26</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1018.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>SRR013476,SRR013477</td>\n",
       "      <td>DSM 7271</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>US DOE Joint Genome Institute (JGI-PGF)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>521097.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>35-37</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>type strain of Capnocytophaga ochracea</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Non_marine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GCA_000023465.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM2346v1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA38641</td>\n",
       "      <td>SAMN02603855</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Sample from Zunongwangia profunda SM-A87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Zunongwangia profunda SM-A87.This strain will ...</td>\n",
       "      <td>2010-04-21T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>DSM:18752</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000023465.1</td>\n",
       "      <td>CP001650</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000023465.1</td>\n",
       "      <td>Zunongwangia profunda SM-A87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>Complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aquatic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=SMA-87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>655815.0</td>\n",
       "      <td>type strain of Zunongwangia profunda</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Zunongwangia profunda SM-A87</td>\n",
       "      <td>Zunongwangia profunda SM-A87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>20398413</td>\n",
       "      <td>2014-01-30T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_014041</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>4653.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Major</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN02603855</td>\n",
       "      <td>CP001650</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2010/04/21</td>\n",
       "      <td>ShanDong University|Beijing Institute of Genom...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>398743.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>No</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SM-A87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>ShanDong University</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>655815.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>type strain of Zunongwangia profunda</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Unclassified</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GCA_000023725.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM2372v1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA38559</td>\n",
       "      <td>SAMN02604231</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Sample from Flavobacteriaceae bacterium 3519-10</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacteriaceae bacterium 3519-10.This strai...</td>\n",
       "      <td>2009-08-05T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>Antarctica: Vostok Station</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000023725.1</td>\n",
       "      <td>CP001673</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000023725.1</td>\n",
       "      <td>Flavobacteriaceae bacterium 3519-10</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>Complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Antarctica: Vostok Station</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=3519-10</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>glacial ice, 3519 m depth</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>531844.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Flavobacteriaceae bacterium 3519-10</td>\n",
       "      <td>Flavobacteriaceae bacterium 3519-10</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>18622572</td>\n",
       "      <td>2014-01-30T15:13:19.717</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_013062</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2534.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Major</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN02604231</td>\n",
       "      <td>CP001673</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2009/08/05</td>\n",
       "      <td>University of Nevada Las Vegas</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>531844.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3519</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>University of Nevada Las Vegas</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>531844.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Non_marine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GCA_000024125.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASM2412v1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Complete Genome</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PRJNA13461</td>\n",
       "      <td>SAMN02603916</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Sample from Robiginitalea biformata HTCC2501</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Rod</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Robiginitalea biformata strain HTCC2501. Strai...</td>\n",
       "      <td>2006-02-24T00:00:00Z</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...</td>\n",
       "      <td>GCF_000024125.1</td>\n",
       "      <td>CP001712</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GCA_000024125.1</td>\n",
       "      <td>Robiginitalea biformata HTCC2501</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Full</td>\n",
       "      <td>Complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aquatic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>strain=HTCC2501</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NCBI</td>\n",
       "      <td>313596.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Robiginitalea biformata HTCC2501</td>\n",
       "      <td>Robiginitalea biformata HTCC2501</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Aerobic</td>\n",
       "      <td>identical</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>19767438</td>\n",
       "      <td>2014-01-30T00:00:00.000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NC_013222</td>\n",
       "      <td>representative genome</td>\n",
       "      <td>3211.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>assembly from type material</td>\n",
       "      <td>Major</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SAMN02603916</td>\n",
       "      <td>CP001712</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2009/09/11</td>\n",
       "      <td>J. Craig Venter Institute</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>complete</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>252307.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HTCC2501</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Generic</td>\n",
       "      <td>Generic.1.0</td>\n",
       "      <td>The Gordon and Betty Moore Foundation Marine M...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>313596.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>30</td>\n",
       "      <td>Mesophilic</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Yes</td>\n",
       "      <td>type strain of Robiginitalea biformata</td>\n",
       "      <td>NaN</td>\n",
       "      <td>latest</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Unclassified</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Assembly accession Alias Altitude Anonymized name Arrayexpress species  \\\n",
       "0    GCA_000016645.1   NaN      NaN             NaN                  NaN   \n",
       "1    GCA_000023285.1   NaN      NaN             NaN                  NaN   \n",
       "2    GCA_000023465.1   NaN      NaN             NaN                  NaN   \n",
       "3    GCA_000023725.1   NaN      NaN             NaN                  NaN   \n",
       "4    GCA_000024125.1   NaN      NaN             NaN                  NaN   \n",
       "\n",
       "  Arrayexpress strain or line   ASM name Assembly method   Assembly level  \\\n",
       "0                         NaN  ASM1664v1             NaN  Complete Genome   \n",
       "1                         NaN  ASM2328v1             NaN  Complete Genome   \n",
       "2                         NaN  ASM2346v1             NaN  Complete Genome   \n",
       "3                         NaN  ASM2372v1             NaN  Complete Genome   \n",
       "4                         NaN  ASM2412v1             NaN  Complete Genome   \n",
       "\n",
       "  Biomaterial provider Bioproject accession Biosample accession  \\\n",
       "0                  NaN           PRJNA16082        SAMN02598357   \n",
       "1                  NaN           PRJNA29403        SAMN00001911   \n",
       "2                  NaN           PRJNA38641        SAMN02603855   \n",
       "3                  NaN           PRJNA38559        SAMN02604231   \n",
       "4                  NaN           PRJNA13461        SAMN02603916   \n",
       "\n",
       "  Biosample model                                    Biosample title  \\\n",
       "0             NaN        Sample from Flavobacterium johnsoniae UW101   \n",
       "1             NaN  Generic sample from Capnocytophaga ochracea DS...   \n",
       "2             NaN           Sample from Zunongwangia profunda SM-A87   \n",
       "3             NaN    Sample from Flavobacteriaceae bacterium 3519-10   \n",
       "4             NaN       Sample from Robiginitalea biformata HTCC2501   \n",
       "\n",
       "  Biotic relationship Body sample site Cell shape  Chromosomes Collected by  \\\n",
       "0                 NaN              NaN        Rod          1.0          NaN   \n",
       "1                 NaN              NaN        Rod          1.0          NaN   \n",
       "2                 NaN              NaN        Rod          1.0          NaN   \n",
       "3                 NaN              NaN        NaN          1.0          NaN   \n",
       "4                 NaN              NaN        Rod          1.0          NaN   \n",
       "\n",
       "  Collection date                                  Comment paragraph  \\\n",
       "0             NaN  Flavobacterium johnsoniae ATCC 17061. Flavobac...   \n",
       "1             NaN                   Capnocytophaga ochracea DSM 7271   \n",
       "2             NaN  Zunongwangia profunda SM-A87.This strain will ...   \n",
       "3             NaN  Flavobacteriaceae bacterium 3519-10.This strai...   \n",
       "4             NaN  Robiginitalea biformata strain HTCC2501. Strai...   \n",
       "\n",
       "        Completion date Contact email Contact lab  Contigs  \\\n",
       "0  2007-05-01T00:00:00Z           NaN         NaN      1.0   \n",
       "1  2009-04-30T00:00:00Z           NaN         NaN      1.0   \n",
       "2  2010-04-21T00:00:00Z           NaN         NaN      1.0   \n",
       "3  2009-08-05T00:00:00Z           NaN         NaN      1.0   \n",
       "4  2006-02-24T00:00:00Z           NaN         NaN      1.0   \n",
       "\n",
       "                      Country Culture collection Depth Derived from  \\\n",
       "0                         NaN         ATCC 17061   NaN          NaN   \n",
       "1                         NaN           DSM 7271   NaN          NaN   \n",
       "2                         NaN          DSM:18752   NaN          NaN   \n",
       "3  Antarctica: Vostok Station                NaN   NaN          NaN   \n",
       "4                         NaN                NaN   NaN          NaN   \n",
       "\n",
       "  Description Disease Elevation ENA checklist ENA-first-public  \\\n",
       "0         NaN     NaN       NaN           NaN              NaN   \n",
       "1         NaN     NaN       NaN           NaN              NaN   \n",
       "2         NaN     NaN       NaN           NaN              NaN   \n",
       "3         NaN     NaN       NaN           NaN              NaN   \n",
       "4         NaN     NaN       NaN           NaN              NaN   \n",
       "\n",
       "  ENA-last-update Env biome Env broad scale Env feature Env local scale  \\\n",
       "0             NaN       NaN             NaN         NaN             NaN   \n",
       "1             NaN       NaN             NaN         NaN             NaN   \n",
       "2             NaN       NaN             NaN         NaN             NaN   \n",
       "3             NaN       NaN             NaN         NaN             NaN   \n",
       "4             NaN       NaN             NaN         NaN             NaN   \n",
       "\n",
       "  Env material Env medium Environment Environmental sample Estimated size  \\\n",
       "0          NaN        NaN         NaN                  NaN            NaN   \n",
       "1          NaN        NaN         NaN                  NaN            NaN   \n",
       "2          NaN        NaN         NaN                  NaN            NaN   \n",
       "3          NaN        NaN         NaN                  NaN            NaN   \n",
       "4          NaN        NaN         NaN                  NaN            NaN   \n",
       "\n",
       "  Excluded from refseq FDA cvm ID Finishing strategy (depth of coverage)  \\\n",
       "0                  NaN        NaN                                    NaN   \n",
       "1                  NaN        NaN                                    NaN   \n",
       "2                  NaN        NaN                                    NaN   \n",
       "3                  NaN        NaN                                    NaN   \n",
       "4                  NaN        NaN                                    NaN   \n",
       "\n",
       "  First name                                           Ftp path  \\\n",
       "0        NaN  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "1        NaN  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "2        NaN  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "3        NaN  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "4        NaN  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000...   \n",
       "\n",
       "   Gbrs paired ASM Genbank accessions Gene calling method        Genome ID  \\\n",
       "0  GCF_000016645.1           CP000685                 NaN  GCA_000016645.1   \n",
       "1  GCF_000023285.1           CP001632                 NaN  GCA_000023285.1   \n",
       "2  GCF_000023465.1           CP001650                 NaN  GCA_000023465.1   \n",
       "3  GCF_000023725.1           CP001673                 NaN  GCA_000023725.1   \n",
       "4  GCF_000024125.1           CP001712                 NaN  GCA_000024125.1   \n",
       "\n",
       "                           Genome name Genome quality flags Genome rep  \\\n",
       "0      Flavobacterium johnsoniae UW101                  NaN       Full   \n",
       "1     Capnocytophaga ochracea DSM 7271                  NaN       Full   \n",
       "2         Zunongwangia profunda SM-A87                  NaN       Full   \n",
       "3  Flavobacteriaceae bacterium 3519-10                  NaN       Full   \n",
       "4     Robiginitalea biformata HTCC2501                  NaN       Full   \n",
       "\n",
       "  Genome status Genotype Geo loc name         Geographic location  \\\n",
       "0      Complete      NaN          NaN     United Kingdom: England   \n",
       "1      Complete      NaN          NaN                         NaN   \n",
       "2      Complete      NaN          NaN                         NaN   \n",
       "3      Complete      NaN          NaN  Antarctica: Vostok Station   \n",
       "4      Complete      NaN          NaN                         NaN   \n",
       "\n",
       "  Geographic location (country and/or atlantic ocean) GOLD stamp ID  \\\n",
       "0                                                NaN            NaN   \n",
       "1                                                NaN            NaN   \n",
       "2                                                NaN            NaN   \n",
       "3                                                NaN            NaN   \n",
       "4                                                NaN            NaN   \n",
       "\n",
       "  Gram stain          Habitat Host Host age Host disease Host disease outcome  \\\n",
       "0        NaN         Multiple  NaN      NaN          NaN                  NaN   \n",
       "1        NaN  Host-associated  NaN      NaN          NaN                  NaN   \n",
       "2        NaN          Aquatic  NaN      NaN          NaN                  NaN   \n",
       "3        NaN              NaN  NaN      NaN          NaN                  NaN   \n",
       "4        NaN          Aquatic  NaN      NaN          NaN                  NaN   \n",
       "\n",
       "  Host gender Host health            Host name Host scientific name Host sex  \\\n",
       "0         NaN         NaN                  NaN                  NaN      NaN   \n",
       "1         NaN         NaN  Human, Homo sapiens                  NaN      NaN   \n",
       "2         NaN         NaN                  NaN                  NaN      NaN   \n",
       "3         NaN         NaN                  NaN                  NaN      NaN   \n",
       "4         NaN         NaN                  NaN                  NaN      NaN   \n",
       "\n",
       "   Host taxid Host tissue sampled Identification method Identified by  \\\n",
       "0         NaN                 NaN                   NaN           NaN   \n",
       "1         NaN                 NaN                   NaN           NaN   \n",
       "2         NaN                 NaN                   NaN           NaN   \n",
       "3         NaN                 NaN                   NaN           NaN   \n",
       "4         NaN                 NaN                   NaN           NaN   \n",
       "\n",
       "         Infraspecific name INSDC center alias INSDC center name  \\\n",
       "0  strain=UW101; ATCC 17061                NaN               NaN   \n",
       "1           strain=DSM 7271                NaN               NaN   \n",
       "2             strain=SMA-87                NaN               NaN   \n",
       "3            strain=3519-10                NaN               NaN   \n",
       "4           strain=HTCC2501                NaN               NaN   \n",
       "\n",
       "  INSDC first public INSDC last update INSDC status Investigation type  \\\n",
       "0                NaN               NaN          NaN                NaN   \n",
       "1                NaN               NaN          NaN                NaN   \n",
       "2                NaN               NaN          NaN                NaN   \n",
       "3                NaN               NaN          NaN                NaN   \n",
       "4                NaN               NaN          NaN                NaN   \n",
       "\n",
       "  Isol growth condt Isolate  \\\n",
       "0               NaN     NaN   \n",
       "1               NaN     NaN   \n",
       "2               NaN     NaN   \n",
       "3               NaN     NaN   \n",
       "4               NaN     NaN   \n",
       "\n",
       "                                  Isolation comments Isolation country  \\\n",
       "0  isolated from soil in England and is the type ...    United Kingdom   \n",
       "1                isolated from the human oral cavity               NaN   \n",
       "2                                                NaN               NaN   \n",
       "3                                                NaN               NaN   \n",
       "4                                                NaN               NaN   \n",
       "\n",
       "  Isolation site           Isolation source Last name Latitude  \\\n",
       "0            NaN                       soil       NaN      NaN   \n",
       "1            NaN      the human oral cavity       NaN      NaN   \n",
       "2            NaN                        NaN       NaN      NaN   \n",
       "3            NaN  glacial ice, 3519 m depth       NaN      NaN   \n",
       "4            NaN                        NaN       NaN      NaN   \n",
       "\n",
       "  Locus tag prefix Longitude Mapping method and version Material  \\\n",
       "0              NaN       NaN                        NaN      NaN   \n",
       "1              NaN       NaN                        NaN      NaN   \n",
       "2              NaN       NaN                        NaN      NaN   \n",
       "3              NaN       NaN                        NaN      NaN   \n",
       "4              NaN       NaN                        NaN      NaN   \n",
       "\n",
       "  Metagenome source Metagenomic Misc param Misc param: hmp supersite Mlst  \\\n",
       "0               NaN         NaN        NaN                       NaN  NaN   \n",
       "1               NaN         NaN        NaN                       NaN  NaN   \n",
       "2               NaN         NaN        NaN                       NaN  NaN   \n",
       "3               NaN         NaN        NaN                       NaN  NaN   \n",
       "4               NaN         NaN        NaN                       NaN  NaN   \n",
       "\n",
       "  Motility                                               Name  Ncbi taxon ID  \\\n",
       "0      Yes                                               NCBI       376686.0   \n",
       "1      Yes  Joint Genome Institute, U.S. Department of Energy       521097.0   \n",
       "2       No                                               NCBI       655815.0   \n",
       "3      NaN                                               NCBI       531844.0   \n",
       "4      NaN                                               NCBI       313596.0   \n",
       "\n",
       "                                   Note Num replicons  \\\n",
       "0                                   NaN           NaN   \n",
       "1                                   NaN           NaN   \n",
       "2  type strain of Zunongwangia profunda           NaN   \n",
       "3                                   NaN           NaN   \n",
       "4                                   NaN           NaN   \n",
       "\n",
       "                              Organism                        Organism name  \\\n",
       "0      Flavobacterium johnsoniae UW101      Flavobacterium johnsoniae UW101   \n",
       "1     Capnocytophaga ochracea DSM 7271     Capnocytophaga ochracea DSM 7271   \n",
       "2         Zunongwangia profunda SM-A87         Zunongwangia profunda SM-A87   \n",
       "3  Flavobacteriaceae bacterium 3519-10  Flavobacteriaceae bacterium 3519-10   \n",
       "4     Robiginitalea biformata HTCC2501     Robiginitalea biformata HTCC2501   \n",
       "\n",
       "  Other clinical Other environmental Oxygen requirement Paired ASM comp   pH  \\\n",
       "0            NaN                 NaN            Aerobic       identical  NaN   \n",
       "1            NaN                 NaN        Facultative       identical  NaN   \n",
       "2            NaN                 NaN            Aerobic       identical  NaN   \n",
       "3            NaN                 NaN                NaN       identical  NaN   \n",
       "4            NaN                 NaN            Aerobic       identical  NaN   \n",
       "\n",
       "  Phenotypes  Plasmids Project name Project type Publication  \\\n",
       "0        NaN       NaN          NaN          NaN         NaN   \n",
       "1        NaN       NaN          NaN          NaN    21304645   \n",
       "2        NaN       NaN          NaN          NaN    20398413   \n",
       "3        NaN       NaN          NaN          NaN    18622572   \n",
       "4        NaN       NaN          NaN          NaN    19767438   \n",
       "\n",
       "          Publication date Quality assessment method and version  \\\n",
       "0  2014-01-28T00:00:00.000                                   NaN   \n",
       "1  2009-02-19T00:00:00.000                                   NaN   \n",
       "2  2014-01-30T00:00:00.000                                   NaN   \n",
       "3  2014-01-30T15:13:19.717                                   NaN   \n",
       "4  2014-01-30T00:00:00.000                                   NaN   \n",
       "\n",
       "  Ref biomaterial Refseq accessions        Refseq category  Refseq cds  \\\n",
       "0             NaN         NC_009441  representative genome      5017.0   \n",
       "1             NaN         NC_013162  representative genome      2171.0   \n",
       "2             NaN         NC_014041  representative genome      4653.0   \n",
       "3             NaN         NC_013062                    NaN      2534.0   \n",
       "4             NaN         NC_013222  representative genome      3211.0   \n",
       "\n",
       "  Rel to oxygen    Relation to type material Release type        Salinity  \\\n",
       "0           NaN  assembly from type material        Major  Non-halophilic   \n",
       "1           NaN  assembly from type material        Major             NaN   \n",
       "2           NaN  assembly from type material        Major             NaN   \n",
       "3           NaN                          NaN        Major             NaN   \n",
       "4           NaN  assembly from type material        Major      Mesophilic   \n",
       "\n",
       "  Samp size Sample accession Sample name Sample type Seq rel date  \\\n",
       "0       NaN     SAMN02598357    CP000685         NaN   2007/05/01   \n",
       "1       NaN     SAMN00001911        FWCB         NaN   2009/08/26   \n",
       "2       NaN     SAMN02603855    CP001650         NaN   2010/04/21   \n",
       "3       NaN     SAMN02604231    CP001673         NaN   2009/08/05   \n",
       "4       NaN     SAMN02603916    CP001712         NaN   2009/09/11   \n",
       "\n",
       "                                  Sequencing centers Sequencing depth  \\\n",
       "0                         DOE Joint Genome Institute              NaN   \n",
       "1  US DOE Joint Genome Institute (JGI-PGF)|DOE Jo...              NaN   \n",
       "2  ShanDong University|Beijing Institute of Genom...              NaN   \n",
       "3                     University of Nevada Las Vegas              NaN   \n",
       "4                          J. Craig Venter Institute              NaN   \n",
       "\n",
       "  Sequencing method Sequencing platform Sequencing status Serovar  Sop  \\\n",
       "0               NaN                 NaN          complete     NaN  NaN   \n",
       "1               NaN                 NaN          complete     NaN  NaN   \n",
       "2               NaN                 NaN          complete     NaN  NaN   \n",
       "3               NaN                 NaN          complete     NaN  NaN   \n",
       "4               NaN                 NaN          complete     NaN  NaN   \n",
       "\n",
       "  Source mat ID  Species taxid Specific host Sporulation        SRA accession  \\\n",
       "0           NaN          986.0           NaN          No                  NaN   \n",
       "1           NaN         1018.0           NaN          No  SRR013476,SRR013477   \n",
       "2           NaN       398743.0           NaN          No                  NaN   \n",
       "3           NaN       531844.0           NaN         NaN                  NaN   \n",
       "4           NaN       252307.0           NaN         NaN                  NaN   \n",
       "\n",
       "       Strain Strain name alias Subject ID Submission model  \\\n",
       "0  ATCC 17061               NaN        NaN          Generic   \n",
       "1    DSM 7271               NaN        NaN          Generic   \n",
       "2      SM-A87               NaN        NaN          Generic   \n",
       "3        3519               NaN        NaN          Generic   \n",
       "4    HTCC2501               NaN        NaN          Generic   \n",
       "\n",
       "  Submission package                                          Submitter  \\\n",
       "0        Generic.1.0                      US DOE Joint Genome Institute   \n",
       "1        Generic.1.0            US DOE Joint Genome Institute (JGI-PGF)   \n",
       "2        Generic.1.0                                ShanDong University   \n",
       "3        Generic.1.0                     University of Nevada Las Vegas   \n",
       "4        Generic.1.0  The Gordon and Betty Moore Foundation Marine M...   \n",
       "\n",
       "  Subsrc note Supplier name     Taxid Temperature Temperature optimum  \\\n",
       "0         NaN           NaN  376686.0         NaN               20-30   \n",
       "1         NaN           NaN  521097.0         NaN               35-37   \n",
       "2         NaN           NaN  655815.0         NaN                 NaN   \n",
       "3         NaN           NaN  531844.0         NaN                 NaN   \n",
       "4         NaN           NaN  313596.0         NaN                  30   \n",
       "\n",
       "  Temperature range Title Trophic level Type strain  \\\n",
       "0        Mesophilic   NaN           NaN         Yes   \n",
       "1        Mesophilic   NaN           NaN         Yes   \n",
       "2        Mesophilic   NaN           NaN         NaN   \n",
       "3               NaN   NaN           NaN         NaN   \n",
       "4        Mesophilic   NaN           NaN         Yes   \n",
       "\n",
       "                              Type material Value Version status WGS master  \\\n",
       "0  type strain of Flavobacterium johnsoniae   NaN         latest        NaN   \n",
       "1    type strain of Capnocytophaga ochracea   NaN         latest        NaN   \n",
       "2      type strain of Zunongwangia profunda   NaN         latest        NaN   \n",
       "3                                       NaN   NaN         latest        NaN   \n",
       "4    type strain of Robiginitalea biformata   NaN         latest        NaN   \n",
       "\n",
       "  Marine_coordinates  MAG        Origin  \n",
       "0                NaN  NaN    Non_marine  \n",
       "1                NaN  NaN    Non_marine  \n",
       "2                NaN  NaN  Unclassified  \n",
       "3                NaN  NaN    Non_marine  \n",
       "4                NaN  NaN  Unclassified  "
      ]
     },
     "execution_count": 91,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "metadata_class.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata_class.to_csv(\"Metadata_final.csv\", index=False)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
