{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 1 Dataset Creation"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Author: Sandra Godinho Silva \\\n",
    "Most updated version: 0.2 from 07/09/2020"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#import libraries\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import seaborn as sns"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1 - Create input dataset\n",
    "### 1.1 Download Genomes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### **Some notes:**\n",
    "**Download done on 7 September 2020:** \\\n",
    "Taxon **Flavobacteriia** (txid 117743) \\\n",
    "Latest GenBank: 3,493 genomes\\\n",
    "Latest RefSeq: 2,093 genomes\n",
    "\n",
    "**Download GenBank in fasta format files** \\\n",
    "RefSeq genomes are a subset of the available GenBank genomes, so the number of RefSeq genomes is lower. \\\n",
    "Our goal is to get the maximum number of genomes. The filtering step for quality and taxonomy will be done afterwards. \\\n",
    "As we are going to use the genomes in an annotated format (fasta) and the main difference between GenBank and RefSeq is the annotation pipeline, downloading GenBank assemblies fits our needs. \\\n",
    "https://www.ncbi.nlm.nih.gov/genome/doc/ftpfaq/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1.2 Create dataset table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.set_option('display.max_rows', 100)\n",
    "pd.set_option('display.max_columns', 1000)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0    GCA_000016645.1\n",
       "1    GCA_000023285.1\n",
       "2    GCA_000023465.1\n",
       "3    GCA_000023725.1\n",
       "4    GCA_000024125.1\n",
       "Name: gbk_files.txt, dtype: object"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gbk = pd.read_csv(\"gbk_files.txt\")\n",
    "gbk = gbk[\"gbk_files.txt\"].str.replace(\".gbk\", \"\")\n",
    "gbk.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>I</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GCA_000023465.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GCA_000023725.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GCA_000024125.1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                 I\n",
       "0  GCA_000016645.1\n",
       "1  GCA_000023285.1\n",
       "2  GCA_000023465.1\n",
       "3  GCA_000023725.1\n",
       "4  GCA_000024125.1"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "assemblies = pd.read_csv(\"assembly_ids.csv\", header=None)\n",
    "assemblies = assemblies.rename(columns={0:\"I\"})\n",
    "assemblies.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 2680 entries, 0 to 2679\n",
      "Data columns (total 1 columns):\n",
      " #   Column  Non-Null Count  Dtype \n",
      "---  ------  --------------  ----- \n",
      " 0   I       2680 non-null   object\n",
      "dtypes: object(1)\n",
      "memory usage: 21.1+ KB\n"
     ]
    }
   ],
   "source": [
    "assemblies.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "merge = pd.merge(assemblies, gbk, how=\"outer\", left_on=\"I\", right_on=\"gbk_files.txt\", indicator=\"merge\")\n",
    "merge[merge[\"merge\"] == \"left_only\"][\"I\"].to_csv(\"Missing_to_prokka.txt\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>I</th>\n",
       "      <th>faa.txt</th>\n",
       "      <th>merge</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [I, faa.txt, merge]\n",
       "Index: []"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "faa = pd.read_csv(\"faa.txt\")\n",
    "faa = faa[\"faa.txt\"].str.replace(\".faa\", \"\")\n",
    "faa.head()\n",
    "merge = pd.merge(assemblies, faa, how=\"outer\", left_on=\"I\", right_on=\"faa.txt\", indicator=\"merge\")\n",
    "merge[merge[\"merge\"] == \"left_only\"]#[\"I\"]#.to_csv(\"Missing_to_prokka.txt\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>I</th>\n",
       "      <th>gbk.txt</th>\n",
       "      <th>merge</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [I, gbk.txt, merge]\n",
       "Index: []"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gbk = pd.read_csv(\"gbk.txt\")\n",
    "gbk = gbk[\"gbk.txt\"].str.replace(\".gbk\", \"\")\n",
    "gbk.head()\n",
    "merge = pd.merge(assemblies, gbk, how=\"outer\", left_on=\"I\", right_on=\"gbk.txt\", indicator=\"merge\")\n",
    "merge[merge[\"merge\"] == \"left_only\"]#[\"I\"]#.to_csv(\"Missing_to_prokka.txt\", index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2 - Run CheckM - Get genome quality\n",
    "https://github.com/Ecogenomics/CheckM/wiki"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2.1 Run CheckM"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2.2. Upload CheckM results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "checkm = pd.read_csv(\"output_checkm.tsv\", sep='\\t')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2.3 Calculation of the Quality score"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "checkm[\"Quality_score\"] = checkm[\"Completeness\"] - 5*checkm[\"Contamination\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "checkm = checkm.rename(columns = {'Strain heterogeneity': 'Strain_heterogeneity', 'Bin Id': 'Bin_Id'})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "checkm = checkm[[\"Bin_Id\", \"Marker lineage\", \"Completeness\", \"Contamination\", \"Strain_heterogeneity\", \"Quality_score\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Bin_Id</th>\n",
       "      <th>Marker lineage</th>\n",
       "      <th>Completeness</th>\n",
       "      <th>Contamination</th>\n",
       "      <th>Strain_heterogeneity</th>\n",
       "      <th>Quality_score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1_ASM1664v1_genomic</td>\n",
       "      <td>f__Flavobacteriaceae (UID2817)</td>\n",
       "      <td>99.65</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.0</td>\n",
       "      <td>98.95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000017525.1_ASM1752v1_genomic</td>\n",
       "      <td>k__Bacteria (UID2565)</td>\n",
       "      <td>44.01</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>44.01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GCA_000022605.2_ASM2260v1_genomic</td>\n",
       "      <td>k__Bacteria (UID2565)</td>\n",
       "      <td>98.82</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>98.82</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GCA_000022945.1_ASM2294v1_genomic</td>\n",
       "      <td>k__Bacteria (UID2565)</td>\n",
       "      <td>49.30</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>49.30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GCA_000023285.1_ASM2328v1_genomic</td>\n",
       "      <td>p__Bacteroidetes (UID2605)</td>\n",
       "      <td>100.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>100.00</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                              Bin_Id                  Marker lineage  \\\n",
       "0  GCA_000016645.1_ASM1664v1_genomic  f__Flavobacteriaceae (UID2817)   \n",
       "1  GCA_000017525.1_ASM1752v1_genomic           k__Bacteria (UID2565)   \n",
       "2  GCA_000022605.2_ASM2260v1_genomic           k__Bacteria (UID2565)   \n",
       "3  GCA_000022945.1_ASM2294v1_genomic           k__Bacteria (UID2565)   \n",
       "4  GCA_000023285.1_ASM2328v1_genomic      p__Bacteroidetes (UID2605)   \n",
       "\n",
       "   Completeness  Contamination  Strain_heterogeneity  Quality_score  \n",
       "0         99.65           0.14                   0.0          98.95  \n",
       "1         44.01           0.00                   0.0          44.01  \n",
       "2         98.82           0.00                   0.0          98.82  \n",
       "3         49.30           0.00                   0.0          49.30  \n",
       "4        100.00           0.00                   0.0         100.00  "
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "checkm.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3 - Run GTDBtk - Taxonomy"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1 Run GTDB-Tk v.1.3.0\n",
    "GTDB release 95.0 (GTDB R05-RS95) https://gtdb.ecogenomic.org/ \\\n",
    "https://ecogenomics.github.io/GTDBTk/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.2 Upload GTDB-Tk results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "#GenBank: 3,493 genomes\n",
    "gtdb = pd.read_csv(\"gtdbtk.bac120.summary.tsv\", sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "gtdb[[\"1\", \"Domain\", \"Phyla\", \"Class\", \"Order\", \"Family\", \"Genus\", \"Species\"]] = gtdb.classification.str.split(\".__\", expand=True)\n",
    "gtdb[[\"Domain\", \"Phyla\", \"Class\", \"Order\", \"Family\", \"Genus\"]] = gtdb[[\"Domain\", \"Phyla\", \"Class\", \"Order\", \"Family\", \"Genus\"]].replace(\";\", \"\", regex=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "gtdb = gtdb[[\"user_genome\",\"classification\", \"Domain\", \"Phyla\", \"Class\", \"Order\", \"Family\", \"Genus\", \"Species\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>user_genome</th>\n",
       "      <th>classification</th>\n",
       "      <th>Domain</th>\n",
       "      <th>Phyla</th>\n",
       "      <th>Class</th>\n",
       "      <th>Order</th>\n",
       "      <th>Family</th>\n",
       "      <th>Genus</th>\n",
       "      <th>Species</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_008011715.1_ASM801171v1_genomic</td>\n",
       "      <td>d__Bacteria;p__Proteobacteria;c__Gammaproteoba...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Proteobacteria</td>\n",
       "      <td>Gammaproteobacteria</td>\n",
       "      <td>Burkholderiales</td>\n",
       "      <td>Burkholderiaceae</td>\n",
       "      <td>Pseudoduganella</td>\n",
       "      <td>Pseudoduganella sp003293715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_004151235.1_ASM415123v1_genomic</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Flavobacteriaceae</td>\n",
       "      <td>Flavobacterium</td>\n",
       "      <td>Flavobacterium anhuiense_A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GCA_900101855.1_IMG-taxon_2596583647_annotated...</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Flavobacteriaceae</td>\n",
       "      <td>Flavobacterium</td>\n",
       "      <td>Flavobacterium anhuiense</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GCA_001705175.1_ASM170517v1_genomic</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Flavobacteriaceae</td>\n",
       "      <td>Flavobacterium</td>\n",
       "      <td>Flavobacterium anhuiense</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GCA_000278705.1_ASM27870v1_genomic</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Flavobacteriaceae</td>\n",
       "      <td>Flavobacterium</td>\n",
       "      <td>Flavobacterium anhuiense</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                         user_genome  \\\n",
       "0                GCA_008011715.1_ASM801171v1_genomic   \n",
       "1                GCA_004151235.1_ASM415123v1_genomic   \n",
       "2  GCA_900101855.1_IMG-taxon_2596583647_annotated...   \n",
       "3                GCA_001705175.1_ASM170517v1_genomic   \n",
       "4                 GCA_000278705.1_ASM27870v1_genomic   \n",
       "\n",
       "                                      classification    Domain  \\\n",
       "0  d__Bacteria;p__Proteobacteria;c__Gammaproteoba...  Bacteria   \n",
       "1  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria   \n",
       "2  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria   \n",
       "3  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria   \n",
       "4  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria   \n",
       "\n",
       "            Phyla                Class             Order             Family  \\\n",
       "0  Proteobacteria  Gammaproteobacteria   Burkholderiales   Burkholderiaceae   \n",
       "1    Bacteroidota          Bacteroidia  Flavobacteriales  Flavobacteriaceae   \n",
       "2    Bacteroidota          Bacteroidia  Flavobacteriales  Flavobacteriaceae   \n",
       "3    Bacteroidota          Bacteroidia  Flavobacteriales  Flavobacteriaceae   \n",
       "4    Bacteroidota          Bacteroidia  Flavobacteriales  Flavobacteriaceae   \n",
       "\n",
       "             Genus                      Species  \n",
       "0  Pseudoduganella  Pseudoduganella sp003293715  \n",
       "1   Flavobacterium   Flavobacterium anhuiense_A  \n",
       "2   Flavobacterium     Flavobacterium anhuiense  \n",
       "3   Flavobacterium     Flavobacterium anhuiense  \n",
       "4   Flavobacterium     Flavobacterium anhuiense  "
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gtdb.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "3482"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(gtdb)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2826"
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(gtdb[gtdb[\"Family\"] == \"Flavobacteriaceae\"]) + len(gtdb[gtdb[\"Family\"] == \"Weeksellaceae\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.3 Solve nomenclature conflicts"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Family**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "gtdb = gtdb.astype(str)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "filt = (gtdb[\"Family\"] == \"Flavobacteriaceae\") & (gtdb.Genus ==\"\")\n",
    "gtdb.loc[filt,\"Genus\"] = \"Uncl. Flavobacteriaceae\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "filt = (gtdb[\"Family\"] == \"Weeksellaceae\") & (gtdb.Genus ==\"\")\n",
    "gtdb.loc[filt,\"Genus\"] = \"Uncl. Weeksellaceae\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Genus**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "l = gtdb.Genus.unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "for x in l:\n",
    "    filt = (gtdb[\"Genus\"] == str(x)) & (gtdb.Species == \"\")\n",
    "    s = ' '.join([str(x), \"sp.\"])\n",
    "    gtdb.loc[filt,\"Species\"] = s"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4 - Get more information of the dataset"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.1 - Run BBtools\n",
    "BBtools: https://jgi.doe.gov/data-and-tools/bbtools/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.2 Upload BBtools results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "bbtools = pd.read_csv(\"bbtools.txt\", sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "bbtools[\"file\"] = bbtools[\"filename\"].str.split(\"/\", expand=True)[8].str.replace(\".fna\",\"\")\n",
    "bbtools = bbtools.drop(columns=[\"filename\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>n_scaffolds</th>\n",
       "      <th>n_contigs</th>\n",
       "      <th>scaf_bp</th>\n",
       "      <th>contig_bp</th>\n",
       "      <th>gap_pct</th>\n",
       "      <th>scaf_N50</th>\n",
       "      <th>scaf_L50</th>\n",
       "      <th>ctg_N50</th>\n",
       "      <th>ctg_L50</th>\n",
       "      <th>scaf_N90</th>\n",
       "      <th>scaf_L90</th>\n",
       "      <th>ctg_N90</th>\n",
       "      <th>ctg_L90</th>\n",
       "      <th>scaf_max</th>\n",
       "      <th>ctg_max</th>\n",
       "      <th>scaf_n_gt50K</th>\n",
       "      <th>scaf_pct_gt50K</th>\n",
       "      <th>gc_avg</th>\n",
       "      <th>gc_std</th>\n",
       "      <th>file</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.34113</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>GCA_000016645.1_ASM1664v1_genomic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>245530</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>245530</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.22441</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>GCA_000017525.1_ASM1752v1_genomic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>640935</td>\n",
       "      <td>640935</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>636850</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>99.363</td>\n",
       "      <td>0.27159</td>\n",
       "      <td>0.0135</td>\n",
       "      <td>GCA_000022605.2_ASM2260v1_genomic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>276984</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>276984</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.22595</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>GCA_000022945.1_ASM2294v1_genomic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.39588</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>GCA_000023285.1_ASM2328v1_genomic</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   n_scaffolds  n_contigs  scaf_bp  contig_bp  gap_pct  scaf_N50  scaf_L50  \\\n",
       "0            1          1  6096872    6096872      0.0         1   6096872   \n",
       "1            1          1   245530     245530      0.0         1    245530   \n",
       "2            2          2   640935     640935      0.0         1    636850   \n",
       "3            1          1   276984     276984      0.0         1    276984   \n",
       "4            1          1  2612925    2612925      0.0         1   2612925   \n",
       "\n",
       "   ctg_N50  ctg_L50  scaf_N90  scaf_L90  ctg_N90  ctg_L90  scaf_max  ctg_max  \\\n",
       "0        1  6096872         1   6096872        1  6096872   6096872  6096872   \n",
       "1        1   245530         1    245530        1   245530    245530   245530   \n",
       "2        1   636850         1    636850        1   636850    636850   636850   \n",
       "3        1   276984         1    276984        1   276984    276984   276984   \n",
       "4        1  2612925         1   2612925        1  2612925   2612925  2612925   \n",
       "\n",
       "   scaf_n_gt50K  scaf_pct_gt50K   gc_avg  gc_std  \\\n",
       "0             1         100.000  0.34113  0.0000   \n",
       "1             1         100.000  0.22441  0.0000   \n",
       "2             1          99.363  0.27159  0.0135   \n",
       "3             1         100.000  0.22595  0.0000   \n",
       "4             1         100.000  0.39588  0.0000   \n",
       "\n",
       "                                file  \n",
       "0  GCA_000016645.1_ASM1664v1_genomic  \n",
       "1  GCA_000017525.1_ASM1752v1_genomic  \n",
       "2  GCA_000022605.2_ASM2260v1_genomic  \n",
       "3  GCA_000022945.1_ASM2294v1_genomic  \n",
       "4  GCA_000023285.1_ASM2328v1_genomic  "
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bbtools.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 5 - Create final table"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.1 Merge"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset = pd.merge(checkm, gtdb, how = \"left\", left_on=\"Bin_Id\", right_on=\"user_genome\", indicator=\"merge\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset = pd.merge(dataset, bbtools, how = \"left\", left_on=\"Bin_Id\", right_on=\"file\", indicator=\"merge2\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "both          3482\n",
       "left_only       10\n",
       "right_only       0\n",
       "Name: merge, dtype: int64"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset[\"merge\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "both          3492\n",
       "right_only       0\n",
       "left_only        0\n",
       "Name: merge2, dtype: int64"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset[\"merge2\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset = dataset.drop(columns=[\"merge\", \"merge2\", \"user_genome\", \"file\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Bin_Id</th>\n",
       "      <th>Marker lineage</th>\n",
       "      <th>Completeness</th>\n",
       "      <th>Contamination</th>\n",
       "      <th>Strain_heterogeneity</th>\n",
       "      <th>Quality_score</th>\n",
       "      <th>classification</th>\n",
       "      <th>Domain</th>\n",
       "      <th>Phyla</th>\n",
       "      <th>Class</th>\n",
       "      <th>Order</th>\n",
       "      <th>Family</th>\n",
       "      <th>Genus</th>\n",
       "      <th>Species</th>\n",
       "      <th>n_scaffolds</th>\n",
       "      <th>n_contigs</th>\n",
       "      <th>scaf_bp</th>\n",
       "      <th>contig_bp</th>\n",
       "      <th>gap_pct</th>\n",
       "      <th>scaf_N50</th>\n",
       "      <th>scaf_L50</th>\n",
       "      <th>ctg_N50</th>\n",
       "      <th>ctg_L50</th>\n",
       "      <th>scaf_N90</th>\n",
       "      <th>scaf_L90</th>\n",
       "      <th>ctg_N90</th>\n",
       "      <th>ctg_L90</th>\n",
       "      <th>scaf_max</th>\n",
       "      <th>ctg_max</th>\n",
       "      <th>scaf_n_gt50K</th>\n",
       "      <th>scaf_pct_gt50K</th>\n",
       "      <th>gc_avg</th>\n",
       "      <th>gc_std</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1_ASM1664v1_genomic</td>\n",
       "      <td>f__Flavobacteriaceae (UID2817)</td>\n",
       "      <td>99.65</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.0</td>\n",
       "      <td>98.95</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Flavobacteriaceae</td>\n",
       "      <td>Flavobacterium</td>\n",
       "      <td>Flavobacterium johnsoniae</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.34113</td>\n",
       "      <td>0.0000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000017525.1_ASM1752v1_genomic</td>\n",
       "      <td>k__Bacteria (UID2565)</td>\n",
       "      <td>44.01</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>44.01</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Blattabacteriaceae</td>\n",
       "      <td>Sulcia</td>\n",
       "      <td>Sulcia sp.</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>245530</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>245530</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.22441</td>\n",
       "      <td>0.0000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GCA_000022605.2_ASM2260v1_genomic</td>\n",
       "      <td>k__Bacteria (UID2565)</td>\n",
       "      <td>98.82</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>98.82</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Blattabacteriaceae</td>\n",
       "      <td>Blattabacterium</td>\n",
       "      <td>Blattabacterium sp000022605</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>640935</td>\n",
       "      <td>640935</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>636850</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>99.363</td>\n",
       "      <td>0.27159</td>\n",
       "      <td>0.0135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GCA_000022945.1_ASM2294v1_genomic</td>\n",
       "      <td>k__Bacteria (UID2565)</td>\n",
       "      <td>49.30</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>49.30</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Blattabacteriaceae</td>\n",
       "      <td>Sulcia</td>\n",
       "      <td>Sulcia muelleri</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>276984</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>276984</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.22595</td>\n",
       "      <td>0.0000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GCA_000023285.1_ASM2328v1_genomic</td>\n",
       "      <td>p__Bacteroidetes (UID2605)</td>\n",
       "      <td>100.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>100.00</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Flavobacteriaceae</td>\n",
       "      <td>Capnocytophaga</td>\n",
       "      <td>Capnocytophaga ochracea</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.39588</td>\n",
       "      <td>0.0000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                              Bin_Id                  Marker lineage  \\\n",
       "0  GCA_000016645.1_ASM1664v1_genomic  f__Flavobacteriaceae (UID2817)   \n",
       "1  GCA_000017525.1_ASM1752v1_genomic           k__Bacteria (UID2565)   \n",
       "2  GCA_000022605.2_ASM2260v1_genomic           k__Bacteria (UID2565)   \n",
       "3  GCA_000022945.1_ASM2294v1_genomic           k__Bacteria (UID2565)   \n",
       "4  GCA_000023285.1_ASM2328v1_genomic      p__Bacteroidetes (UID2605)   \n",
       "\n",
       "   Completeness  Contamination  Strain_heterogeneity  Quality_score  \\\n",
       "0         99.65           0.14                   0.0          98.95   \n",
       "1         44.01           0.00                   0.0          44.01   \n",
       "2         98.82           0.00                   0.0          98.82   \n",
       "3         49.30           0.00                   0.0          49.30   \n",
       "4        100.00           0.00                   0.0         100.00   \n",
       "\n",
       "                                      classification    Domain         Phyla  \\\n",
       "0  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "1  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "2  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "3  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "4  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "\n",
       "         Class             Order              Family            Genus  \\\n",
       "0  Bacteroidia  Flavobacteriales   Flavobacteriaceae   Flavobacterium   \n",
       "1  Bacteroidia  Flavobacteriales  Blattabacteriaceae           Sulcia   \n",
       "2  Bacteroidia  Flavobacteriales  Blattabacteriaceae  Blattabacterium   \n",
       "3  Bacteroidia  Flavobacteriales  Blattabacteriaceae           Sulcia   \n",
       "4  Bacteroidia  Flavobacteriales   Flavobacteriaceae   Capnocytophaga   \n",
       "\n",
       "                       Species  n_scaffolds  n_contigs  scaf_bp  contig_bp  \\\n",
       "0    Flavobacterium johnsoniae            1          1  6096872    6096872   \n",
       "1                   Sulcia sp.            1          1   245530     245530   \n",
       "2  Blattabacterium sp000022605            2          2   640935     640935   \n",
       "3              Sulcia muelleri            1          1   276984     276984   \n",
       "4      Capnocytophaga ochracea            1          1  2612925    2612925   \n",
       "\n",
       "   gap_pct  scaf_N50  scaf_L50  ctg_N50  ctg_L50  scaf_N90  scaf_L90  ctg_N90  \\\n",
       "0      0.0         1   6096872        1  6096872         1   6096872        1   \n",
       "1      0.0         1    245530        1   245530         1    245530        1   \n",
       "2      0.0         1    636850        1   636850         1    636850        1   \n",
       "3      0.0         1    276984        1   276984         1    276984        1   \n",
       "4      0.0         1   2612925        1  2612925         1   2612925        1   \n",
       "\n",
       "   ctg_L90  scaf_max  ctg_max  scaf_n_gt50K  scaf_pct_gt50K   gc_avg  gc_std  \n",
       "0  6096872   6096872  6096872             1         100.000  0.34113  0.0000  \n",
       "1   245530    245530   245530             1         100.000  0.22441  0.0000  \n",
       "2   636850    636850   636850             1          99.363  0.27159  0.0135  \n",
       "3   276984    276984   276984             1         100.000  0.22595  0.0000  \n",
       "4  2612925   2612925  2612925             1         100.000  0.39588  0.0000  "
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genome_ID</th>\n",
       "      <th>Bin_Id</th>\n",
       "      <th>Marker lineage</th>\n",
       "      <th>Completeness</th>\n",
       "      <th>Contamination</th>\n",
       "      <th>Strain_heterogeneity</th>\n",
       "      <th>Quality_score</th>\n",
       "      <th>classification</th>\n",
       "      <th>Domain</th>\n",
       "      <th>Phyla</th>\n",
       "      <th>Class</th>\n",
       "      <th>Order</th>\n",
       "      <th>Family</th>\n",
       "      <th>Genus</th>\n",
       "      <th>Species</th>\n",
       "      <th>n_scaffolds</th>\n",
       "      <th>n_contigs</th>\n",
       "      <th>scaf_bp</th>\n",
       "      <th>contig_bp</th>\n",
       "      <th>gap_pct</th>\n",
       "      <th>scaf_N50</th>\n",
       "      <th>scaf_L50</th>\n",
       "      <th>ctg_N50</th>\n",
       "      <th>ctg_L50</th>\n",
       "      <th>scaf_N90</th>\n",
       "      <th>scaf_L90</th>\n",
       "      <th>ctg_N90</th>\n",
       "      <th>ctg_L90</th>\n",
       "      <th>scaf_max</th>\n",
       "      <th>ctg_max</th>\n",
       "      <th>scaf_n_gt50K</th>\n",
       "      <th>scaf_pct_gt50K</th>\n",
       "      <th>gc_avg</th>\n",
       "      <th>gc_std</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GCA_000016645.1</td>\n",
       "      <td>GCA_000016645.1_ASM1664v1_genomic</td>\n",
       "      <td>f__Flavobacteriaceae (UID2817)</td>\n",
       "      <td>99.65</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.0</td>\n",
       "      <td>98.95</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Flavobacteriaceae</td>\n",
       "      <td>Flavobacterium</td>\n",
       "      <td>Flavobacterium johnsoniae</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>6096872</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.34113</td>\n",
       "      <td>0.0000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GCA_000017525.1</td>\n",
       "      <td>GCA_000017525.1_ASM1752v1_genomic</td>\n",
       "      <td>k__Bacteria (UID2565)</td>\n",
       "      <td>44.01</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>44.01</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Blattabacteriaceae</td>\n",
       "      <td>Sulcia</td>\n",
       "      <td>Sulcia sp.</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>245530</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>245530</td>\n",
       "      <td>245530</td>\n",
       "      <td>245530</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.22441</td>\n",
       "      <td>0.0000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GCA_000022605.2</td>\n",
       "      <td>GCA_000022605.2_ASM2260v1_genomic</td>\n",
       "      <td>k__Bacteria (UID2565)</td>\n",
       "      <td>98.82</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>98.82</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Blattabacteriaceae</td>\n",
       "      <td>Blattabacterium</td>\n",
       "      <td>Blattabacterium sp000022605</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>640935</td>\n",
       "      <td>640935</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>636850</td>\n",
       "      <td>636850</td>\n",
       "      <td>636850</td>\n",
       "      <td>1</td>\n",
       "      <td>99.363</td>\n",
       "      <td>0.27159</td>\n",
       "      <td>0.0135</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GCA_000022945.1</td>\n",
       "      <td>GCA_000022945.1_ASM2294v1_genomic</td>\n",
       "      <td>k__Bacteria (UID2565)</td>\n",
       "      <td>49.30</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>49.30</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Blattabacteriaceae</td>\n",
       "      <td>Sulcia</td>\n",
       "      <td>Sulcia muelleri</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>276984</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>276984</td>\n",
       "      <td>276984</td>\n",
       "      <td>276984</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.22595</td>\n",
       "      <td>0.0000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GCA_000023285.1</td>\n",
       "      <td>GCA_000023285.1_ASM2328v1_genomic</td>\n",
       "      <td>p__Bacteroidetes (UID2605)</td>\n",
       "      <td>100.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>100.00</td>\n",
       "      <td>d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...</td>\n",
       "      <td>Bacteria</td>\n",
       "      <td>Bacteroidota</td>\n",
       "      <td>Bacteroidia</td>\n",
       "      <td>Flavobacteriales</td>\n",
       "      <td>Flavobacteriaceae</td>\n",
       "      <td>Capnocytophaga</td>\n",
       "      <td>Capnocytophaga ochracea</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>2612925</td>\n",
       "      <td>1</td>\n",
       "      <td>100.000</td>\n",
       "      <td>0.39588</td>\n",
       "      <td>0.0000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         Genome_ID                             Bin_Id  \\\n",
       "0  GCA_000016645.1  GCA_000016645.1_ASM1664v1_genomic   \n",
       "1  GCA_000017525.1  GCA_000017525.1_ASM1752v1_genomic   \n",
       "2  GCA_000022605.2  GCA_000022605.2_ASM2260v1_genomic   \n",
       "3  GCA_000022945.1  GCA_000022945.1_ASM2294v1_genomic   \n",
       "4  GCA_000023285.1  GCA_000023285.1_ASM2328v1_genomic   \n",
       "\n",
       "                   Marker lineage  Completeness  Contamination  \\\n",
       "0  f__Flavobacteriaceae (UID2817)         99.65           0.14   \n",
       "1           k__Bacteria (UID2565)         44.01           0.00   \n",
       "2           k__Bacteria (UID2565)         98.82           0.00   \n",
       "3           k__Bacteria (UID2565)         49.30           0.00   \n",
       "4      p__Bacteroidetes (UID2605)        100.00           0.00   \n",
       "\n",
       "   Strain_heterogeneity  Quality_score  \\\n",
       "0                   0.0          98.95   \n",
       "1                   0.0          44.01   \n",
       "2                   0.0          98.82   \n",
       "3                   0.0          49.30   \n",
       "4                   0.0         100.00   \n",
       "\n",
       "                                      classification    Domain         Phyla  \\\n",
       "0  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "1  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "2  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "3  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "4  d__Bacteria;p__Bacteroidota;c__Bacteroidia;o__...  Bacteria  Bacteroidota   \n",
       "\n",
       "         Class             Order              Family            Genus  \\\n",
       "0  Bacteroidia  Flavobacteriales   Flavobacteriaceae   Flavobacterium   \n",
       "1  Bacteroidia  Flavobacteriales  Blattabacteriaceae           Sulcia   \n",
       "2  Bacteroidia  Flavobacteriales  Blattabacteriaceae  Blattabacterium   \n",
       "3  Bacteroidia  Flavobacteriales  Blattabacteriaceae           Sulcia   \n",
       "4  Bacteroidia  Flavobacteriales   Flavobacteriaceae   Capnocytophaga   \n",
       "\n",
       "                       Species  n_scaffolds  n_contigs  scaf_bp  contig_bp  \\\n",
       "0    Flavobacterium johnsoniae            1          1  6096872    6096872   \n",
       "1                   Sulcia sp.            1          1   245530     245530   \n",
       "2  Blattabacterium sp000022605            2          2   640935     640935   \n",
       "3              Sulcia muelleri            1          1   276984     276984   \n",
       "4      Capnocytophaga ochracea            1          1  2612925    2612925   \n",
       "\n",
       "   gap_pct  scaf_N50  scaf_L50  ctg_N50  ctg_L50  scaf_N90  scaf_L90  ctg_N90  \\\n",
       "0      0.0         1   6096872        1  6096872         1   6096872        1   \n",
       "1      0.0         1    245530        1   245530         1    245530        1   \n",
       "2      0.0         1    636850        1   636850         1    636850        1   \n",
       "3      0.0         1    276984        1   276984         1    276984        1   \n",
       "4      0.0         1   2612925        1  2612925         1   2612925        1   \n",
       "\n",
       "   ctg_L90  scaf_max  ctg_max  scaf_n_gt50K  scaf_pct_gt50K   gc_avg  gc_std  \n",
       "0  6096872   6096872  6096872             1         100.000  0.34113  0.0000  \n",
       "1   245530    245530   245530             1         100.000  0.22441  0.0000  \n",
       "2   636850    636850   636850             1          99.363  0.27159  0.0135  \n",
       "3   276984    276984   276984             1         100.000  0.22595  0.0000  \n",
       "4  2612925   2612925  2612925             1         100.000  0.39588  0.0000  "
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "l1 = dataset[\"Bin_Id\"].str.split(\"_\", expand=True).iloc[:,0:2]\n",
    "dataset[\"Genome_ID\"]=l1.apply(lambda two: '_'.join(two.dropna().astype(str).values), axis=1)\n",
    "l2 = dataset.columns.tolist() # list the columns in the df\n",
    "l2.insert(0, l2.pop(l2.index('Genome_ID'))) # Assign new position for column Bin\n",
    "dataset = dataset.reindex(columns= l2)\n",
    "dataset.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.2 Classification of genomes\n",
    "High - Medium - Low"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "def classifier(row):\n",
    "    if (row[\"Quality_score\"] > 50) and (row[\"Completeness\"] >90) and (row[\"Contamination\"] <5)  and (row[\"Strain_heterogeneity\"] <10) and (row[\"n_contigs\"]<500): #and (row[\"scaf_N50\"] > 10)\n",
    "        return \"High\"\n",
    "    elif (row[\"Quality_score\"] > 50) and (row[\"Completeness\"] >80) and (row[\"Contamination\"] <10)  and (row[\"Strain_heterogeneity\"] <50):\n",
    "        return \"Medium\"\n",
    "    elif  (row[\"Completeness\"] >50) and (row[\"Contamination\"] <10) and  (row[\"Quality_score\"] > 50):\n",
    "        return \"Low\"\n",
    "    else:\n",
    "        return \"Error\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset[\"Classification_quality\"] = dataset.apply(classifier, axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "High      2150\n",
       "Low        651\n",
       "Medium     436\n",
       "Error      255\n",
       "Name: Classification_quality, dtype: int64"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset[\"Classification_quality\"].value_counts()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.3 Remove genomes below minimum quality"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset = dataset[~(dataset[\"Classification_quality\"] ==\"Error\")]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5.4 Only include Flavobacteriaceae and Weeksleaceae families"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset = dataset[dataset.Family.str.contains(\"Flavobacteriaceae\", na=False) | dataset.Family.str.contains(\"Weeksellaceae\", na=False)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Flavobacteriaceae    1924\n",
       "Weeksellaceae         757\n",
       "Name: Family, dtype: int64"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset[\"Family\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset.to_csv(\"Dataset.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2681"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(dataset)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 6 - Table with Genus that have at least x genomes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "175\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "2681"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset.groupby('Genus')[\"Genus\"].nunique().count()\n",
    "print(dataset.groupby('Genus')[\"Genus\"].nunique().count())\n",
    "len(dataset)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "150"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset[dataset[\"Family\"]==\"Flavobacteriaceae\"].groupby('Genus')[\"Genus\"].nunique().count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Genus              Genus            \n",
       "AU392              AU392                 2\n",
       "Aequorivita        Aequorivita          42\n",
       "Algibacter         Algibacter            9\n",
       "Algibacter_A       Algibacter_A          1\n",
       "Algibacter_B       Algibacter_B          5\n",
       "                                        ..\n",
       "YIM-102668         YIM-102668            2\n",
       "Zeaxanthinibacter  Zeaxanthinibacter     3\n",
       "Zhouia             Zhouia                2\n",
       "Zobellia           Zobellia              6\n",
       "Zunongwangia       Zunongwangia         13\n",
       "Name: Genus, Length: 175, dtype: int64"
      ]
     },
     "execution_count": 44,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset.groupby('Genus')[\"Genus\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Family             Family           \n",
      "Flavobacteriaceae  Flavobacteriaceae    1650\n",
      "Weeksellaceae      Weeksellaceae         710\n",
      "Name: Family, dtype: int64\n",
      "60\n",
      "2\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "2360"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset_reduced_8 = dataset.groupby(['Genus']).filter(lambda x:x['Genus'].count()>7)\n",
    "print(dataset_reduced_8.groupby('Family')[\"Family\"].value_counts())\n",
    "print(dataset_reduced_8.groupby('Genus')[\"Genus\"].nunique().count())\n",
    "print(dataset_reduced_8.groupby('Family')[\"Genus\"].nunique().count())\n",
    "len(dataset_reduced_8)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Genus                    Genus                  \n",
       "Aequorivita              Aequorivita                 42\n",
       "Algibacter               Algibacter                   9\n",
       "Apibacter                Apibacter                   21\n",
       "Aquimarina               Aquimarina                  39\n",
       "Arenibacter              Arenibacter                 22\n",
       "BACL21                   BACL21                      15\n",
       "CG1-02-35-72             CG1-02-35-72                 8\n",
       "CG2-30-34-30             CG2-30-34-30                 8\n",
       "Capnocytophaga           Capnocytophaga              75\n",
       "Cellulophaga             Cellulophaga                21\n",
       "Chryseobacterium         Chryseobacterium           294\n",
       "Croceivirga              Croceivirga                  8\n",
       "Dokdonia                 Dokdonia                    11\n",
       "Elizabethkingia          Elizabethkingia            193\n",
       "Empedobacter             Empedobacter                44\n",
       "Epilithonimonas          Epilithonimonas             31\n",
       "Euzebyella               Euzebyella                   9\n",
       "Flavobacterium           Flavobacterium             589\n",
       "Gelidibacter             Gelidibacter                12\n",
       "Gillisia                 Gillisia                    11\n",
       "Gramella                 Gramella                    16\n",
       "Jejuia                   Jejuia                       9\n",
       "Kaistella                Kaistella                   48\n",
       "Kordia                   Kordia                       8\n",
       "Lacinutrix               Lacinutrix                   9\n",
       "Leeuwenhoekiella         Leeuwenhoekiella            38\n",
       "Lutibacter               Lutibacter                  21\n",
       "MAG-120531               MAG-120531                  29\n",
       "MAG-121220-bin8          MAG-121220-bin8             11\n",
       "MED-G11                  MED-G11                      8\n",
       "MED-G14                  MED-G14                     12\n",
       "MS024-2A                 MS024-2A                    20\n",
       "Maribacter               Maribacter                  45\n",
       "Marinirhabdus            Marinirhabdus               14\n",
       "Marixanthomonas          Marixanthomonas              9\n",
       "Mesonia                  Mesonia                     19\n",
       "Muricauda                Muricauda                   53\n",
       "Muriicola                Muriicola                    8\n",
       "Myroides                 Myroides                    50\n",
       "Nonlabens                Nonlabens                   28\n",
       "Olleya                   Olleya                      13\n",
       "Ornithobacterium         Ornithobacterium             9\n",
       "Planobacterium           Planobacterium              11\n",
       "Polaribacter             Polaribacter                57\n",
       "Psychroflexus            Psychroflexus               17\n",
       "Psychroserpens           Psychroserpens              12\n",
       "Riemerella               Riemerella                  59\n",
       "SHLJ01                   SHLJ01                      12\n",
       "Salegentibacter          Salegentibacter             19\n",
       "Tenacibaculum            Tenacibaculum               68\n",
       "UBA11891                 UBA11891                     8\n",
       "UBA3537                  UBA3537                     12\n",
       "UBA6231                  UBA6231                     13\n",
       "UBA724                   UBA724                       9\n",
       "UBA7446                  UBA7446                     12\n",
       "UBA8316                  UBA8316                     12\n",
       "Uncl. Flavobacteriaceae  Uncl. Flavobacteriaceae     21\n",
       "Winogradskyella          Winogradskyella             55\n",
       "Xanthomarina             Xanthomarina                11\n",
       "Zunongwangia             Zunongwangia                13\n",
       "Name: Genus, dtype: int64"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset_reduced_8.groupby('Genus')[\"Genus\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset_reduced_8.to_csv(\"Dataset_reduced.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [],
   "source": [
    "dataset_reduced_8[\"Genome_ID\"].to_csv(\"Dataset_reduced_8.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "Int64Index: 2360 entries, 0 to 3491\n",
      "Data columns (total 35 columns):\n",
      " #   Column                  Non-Null Count  Dtype  \n",
      "---  ------                  --------------  -----  \n",
      " 0   Genome_ID               2360 non-null   object \n",
      " 1   Bin_Id                  2360 non-null   object \n",
      " 2   Marker lineage          2360 non-null   object \n",
      " 3   Completeness            2360 non-null   float64\n",
      " 4   Contamination           2360 non-null   float64\n",
      " 5   Strain_heterogeneity    2360 non-null   float64\n",
      " 6   Quality_score           2360 non-null   float64\n",
      " 7   classification          2360 non-null   object \n",
      " 8   Domain                  2360 non-null   object \n",
      " 9   Phyla                   2360 non-null   object \n",
      " 10  Class                   2360 non-null   object \n",
      " 11  Order                   2360 non-null   object \n",
      " 12  Family                  2360 non-null   object \n",
      " 13  Genus                   2360 non-null   object \n",
      " 14  Species                 2360 non-null   object \n",
      " 15  n_scaffolds             2360 non-null   int64  \n",
      " 16  n_contigs               2360 non-null   int64  \n",
      " 17  scaf_bp                 2360 non-null   int64  \n",
      " 18  contig_bp               2360 non-null   int64  \n",
      " 19  gap_pct                 2360 non-null   float64\n",
      " 20  scaf_N50                2360 non-null   int64  \n",
      " 21  scaf_L50                2360 non-null   int64  \n",
      " 22  ctg_N50                 2360 non-null   int64  \n",
      " 23  ctg_L50                 2360 non-null   int64  \n",
      " 24  scaf_N90                2360 non-null   int64  \n",
      " 25  scaf_L90                2360 non-null   int64  \n",
      " 26  ctg_N90                 2360 non-null   int64  \n",
      " 27  ctg_L90                 2360 non-null   int64  \n",
      " 28  scaf_max                2360 non-null   int64  \n",
      " 29  ctg_max                 2360 non-null   int64  \n",
      " 30  scaf_n_gt50K            2360 non-null   int64  \n",
      " 31  scaf_pct_gt50K          2360 non-null   float64\n",
      " 32  gc_avg                  2360 non-null   float64\n",
      " 33  gc_std                  2360 non-null   float64\n",
      " 34  Classification_quality  2360 non-null   object \n",
      "dtypes: float64(8), int64(15), object(12)\n",
      "memory usage: 663.8+ KB\n"
     ]
    }
   ],
   "source": [
    "dataset_reduced_8.info()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 7 - Statistics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "with pd.ExcelWriter('Counts.xlsx') as writer:  \n",
    "    dataset[\"Species\"].value_counts().to_excel(writer, sheet_name=\"Species_counts\")\n",
    "    dataset[\"Genus\"].value_counts().to_excel(writer, sheet_name=\"Genus_counts\")\n",
    "    dataset[\"Family\"].value_counts().to_excel(writer, sheet_name=\"Family_counts\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Completeness</th>\n",
       "      <th>Contamination</th>\n",
       "      <th>Strain_heterogeneity</th>\n",
       "      <th>Quality_score</th>\n",
       "      <th>n_scaffolds</th>\n",
       "      <th>n_contigs</th>\n",
       "      <th>scaf_bp</th>\n",
       "      <th>contig_bp</th>\n",
       "      <th>gap_pct</th>\n",
       "      <th>scaf_N50</th>\n",
       "      <th>scaf_L50</th>\n",
       "      <th>ctg_N50</th>\n",
       "      <th>ctg_L50</th>\n",
       "      <th>scaf_N90</th>\n",
       "      <th>scaf_L90</th>\n",
       "      <th>ctg_N90</th>\n",
       "      <th>ctg_L90</th>\n",
       "      <th>scaf_max</th>\n",
       "      <th>ctg_max</th>\n",
       "      <th>scaf_n_gt50K</th>\n",
       "      <th>scaf_pct_gt50K</th>\n",
       "      <th>gc_avg</th>\n",
       "      <th>gc_std</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>mean</th>\n",
       "      <td>96.07</td>\n",
       "      <td>0.84</td>\n",
       "      <td>9.98</td>\n",
       "      <td>91.85</td>\n",
       "      <td>81.36</td>\n",
       "      <td>105.55</td>\n",
       "      <td>3578576.36</td>\n",
       "      <td>3574200.99</td>\n",
       "      <td>0.16</td>\n",
       "      <td>14.38</td>\n",
       "      <td>953412.35</td>\n",
       "      <td>17.61</td>\n",
       "      <td>877064.74</td>\n",
       "      <td>44.14</td>\n",
       "      <td>732178.26</td>\n",
       "      <td>55.14</td>\n",
       "      <td>676630.70</td>\n",
       "      <td>1176042.33</td>\n",
       "      <td>1094902.15</td>\n",
       "      <td>11.48</td>\n",
       "      <td>78.22</td>\n",
       "      <td>0.36</td>\n",
       "      <td>0.03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>std</th>\n",
       "      <td>8.50</td>\n",
       "      <td>1.18</td>\n",
       "      <td>23.27</td>\n",
       "      <td>11.39</td>\n",
       "      <td>132.49</td>\n",
       "      <td>216.76</td>\n",
       "      <td>1102195.71</td>\n",
       "      <td>1103229.29</td>\n",
       "      <td>0.66</td>\n",
       "      <td>30.13</td>\n",
       "      <td>1430573.91</td>\n",
       "      <td>42.49</td>\n",
       "      <td>1394955.02</td>\n",
       "      <td>91.05</td>\n",
       "      <td>1445028.39</td>\n",
       "      <td>134.23</td>\n",
       "      <td>1408055.72</td>\n",
       "      <td>1364424.27</td>\n",
       "      <td>1334361.10</td>\n",
       "      <td>9.13</td>\n",
       "      <td>31.98</td>\n",
       "      <td>0.04</td>\n",
       "      <td>0.02</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>min</th>\n",
       "      <td>50.31</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>50.01</td>\n",
       "      <td>1.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>569462.00</td>\n",
       "      <td>569462.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>2659.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>1884.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>545.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>518.00</td>\n",
       "      <td>11420.00</td>\n",
       "      <td>9123.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.28</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25%</th>\n",
       "      <td>98.35</td>\n",
       "      <td>0.09</td>\n",
       "      <td>0.00</td>\n",
       "      <td>91.32</td>\n",
       "      <td>14.00</td>\n",
       "      <td>17.00</td>\n",
       "      <td>2805039.00</td>\n",
       "      <td>2800951.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>2.00</td>\n",
       "      <td>83912.00</td>\n",
       "      <td>2.00</td>\n",
       "      <td>76644.00</td>\n",
       "      <td>6.00</td>\n",
       "      <td>21204.00</td>\n",
       "      <td>7.00</td>\n",
       "      <td>19917.00</td>\n",
       "      <td>233227.00</td>\n",
       "      <td>218957.00</td>\n",
       "      <td>2.00</td>\n",
       "      <td>71.32</td>\n",
       "      <td>0.33</td>\n",
       "      <td>0.01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50%</th>\n",
       "      <td>99.52</td>\n",
       "      <td>0.49</td>\n",
       "      <td>0.00</td>\n",
       "      <td>96.37</td>\n",
       "      <td>39.00</td>\n",
       "      <td>44.00</td>\n",
       "      <td>3545181.00</td>\n",
       "      <td>3542912.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>5.00</td>\n",
       "      <td>294681.00</td>\n",
       "      <td>5.00</td>\n",
       "      <td>259832.00</td>\n",
       "      <td>14.00</td>\n",
       "      <td>85343.00</td>\n",
       "      <td>15.00</td>\n",
       "      <td>75351.00</td>\n",
       "      <td>634411.00</td>\n",
       "      <td>578043.00</td>\n",
       "      <td>11.00</td>\n",
       "      <td>94.89</td>\n",
       "      <td>0.35</td>\n",
       "      <td>0.03</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75%</th>\n",
       "      <td>100.00</td>\n",
       "      <td>1.08</td>\n",
       "      <td>0.00</td>\n",
       "      <td>98.75</td>\n",
       "      <td>90.00</td>\n",
       "      <td>101.00</td>\n",
       "      <td>4276845.00</td>\n",
       "      <td>4275762.00</td>\n",
       "      <td>0.02</td>\n",
       "      <td>12.00</td>\n",
       "      <td>821621.00</td>\n",
       "      <td>13.00</td>\n",
       "      <td>679776.00</td>\n",
       "      <td>39.00</td>\n",
       "      <td>235479.00</td>\n",
       "      <td>42.00</td>\n",
       "      <td>188725.00</td>\n",
       "      <td>1481894.00</td>\n",
       "      <td>1222563.00</td>\n",
       "      <td>18.00</td>\n",
       "      <td>99.15</td>\n",
       "      <td>0.37</td>\n",
       "      <td>0.04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>max</th>\n",
       "      <td>100.00</td>\n",
       "      <td>9.68</td>\n",
       "      <td>100.00</td>\n",
       "      <td>100.00</td>\n",
       "      <td>1512.00</td>\n",
       "      <td>5612.00</td>\n",
       "      <td>7041311.00</td>\n",
       "      <td>7041311.00</td>\n",
       "      <td>11.48</td>\n",
       "      <td>418.00</td>\n",
       "      <td>6653812.00</td>\n",
       "      <td>1103.00</td>\n",
       "      <td>6653812.00</td>\n",
       "      <td>1182.00</td>\n",
       "      <td>6653812.00</td>\n",
       "      <td>3546.00</td>\n",
       "      <td>6653812.00</td>\n",
       "      <td>6653812.00</td>\n",
       "      <td>6653812.00</td>\n",
       "      <td>54.00</td>\n",
       "      <td>100.00</td>\n",
       "      <td>0.55</td>\n",
       "      <td>0.17</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      Completeness  Contamination  Strain_heterogeneity  Quality_score  \\\n",
       "mean         96.07           0.84                  9.98          91.85   \n",
       "std           8.50           1.18                 23.27          11.39   \n",
       "min          50.31           0.00                  0.00          50.01   \n",
       "25%          98.35           0.09                  0.00          91.32   \n",
       "50%          99.52           0.49                  0.00          96.37   \n",
       "75%         100.00           1.08                  0.00          98.75   \n",
       "max         100.00           9.68                100.00         100.00   \n",
       "\n",
       "      n_scaffolds  n_contigs     scaf_bp   contig_bp  gap_pct  scaf_N50  \\\n",
       "mean        81.36     105.55  3578576.36  3574200.99     0.16     14.38   \n",
       "std        132.49     216.76  1102195.71  1103229.29     0.66     30.13   \n",
       "min          1.00       1.00   569462.00   569462.00     0.00      1.00   \n",
       "25%         14.00      17.00  2805039.00  2800951.00     0.00      2.00   \n",
       "50%         39.00      44.00  3545181.00  3542912.00     0.00      5.00   \n",
       "75%         90.00     101.00  4276845.00  4275762.00     0.02     12.00   \n",
       "max       1512.00    5612.00  7041311.00  7041311.00    11.48    418.00   \n",
       "\n",
       "        scaf_L50  ctg_N50     ctg_L50  scaf_N90    scaf_L90  ctg_N90  \\\n",
       "mean   953412.35    17.61   877064.74     44.14   732178.26    55.14   \n",
       "std   1430573.91    42.49  1394955.02     91.05  1445028.39   134.23   \n",
       "min      2659.00     1.00     1884.00      1.00      545.00     1.00   \n",
       "25%     83912.00     2.00    76644.00      6.00    21204.00     7.00   \n",
       "50%    294681.00     5.00   259832.00     14.00    85343.00    15.00   \n",
       "75%    821621.00    13.00   679776.00     39.00   235479.00    42.00   \n",
       "max   6653812.00  1103.00  6653812.00   1182.00  6653812.00  3546.00   \n",
       "\n",
       "         ctg_L90    scaf_max     ctg_max  scaf_n_gt50K  scaf_pct_gt50K  \\\n",
       "mean   676630.70  1176042.33  1094902.15         11.48           78.22   \n",
       "std   1408055.72  1364424.27  1334361.10          9.13           31.98   \n",
       "min       518.00    11420.00     9123.00          0.00            0.00   \n",
       "25%     19917.00   233227.00   218957.00          2.00           71.32   \n",
       "50%     75351.00   634411.00   578043.00         11.00           94.89   \n",
       "75%    188725.00  1481894.00  1222563.00         18.00           99.15   \n",
       "max   6653812.00  6653812.00  6653812.00         54.00          100.00   \n",
       "\n",
       "      gc_avg  gc_std  \n",
       "mean    0.36    0.03  \n",
       "std     0.04    0.02  \n",
       "min     0.28    0.00  \n",
       "25%     0.33    0.01  \n",
       "50%     0.35    0.03  \n",
       "75%     0.37    0.04  \n",
       "max     0.55    0.17  "
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "statistics = dataset.describe().loc[[\"mean\", \"std\", \"min\", \"25%\", \"50%\", \"75%\", \"max\"]].round(2)\n",
    "statistics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead tr th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe thead tr:last-of-type th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th colspan=\"3\" halign=\"left\">Completeness</th>\n",
       "      <th colspan=\"3\" halign=\"left\">Contamination</th>\n",
       "      <th colspan=\"3\" halign=\"left\">Strain_heterogeneity</th>\n",
       "      <th colspan=\"3\" halign=\"left\">Quality_score</th>\n",
       "      <th colspan=\"3\" halign=\"left\">n_scaffolds</th>\n",
       "      <th colspan=\"3\" halign=\"left\">n_contigs</th>\n",
       "      <th colspan=\"3\" halign=\"left\">scaf_bp</th>\n",
       "      <th colspan=\"3\" halign=\"left\">contig_bp</th>\n",
       "      <th colspan=\"3\" halign=\"left\">gap_pct</th>\n",
       "      <th colspan=\"3\" halign=\"left\">scaf_N50</th>\n",
       "      <th colspan=\"3\" halign=\"left\">scaf_L50</th>\n",
       "      <th colspan=\"3\" halign=\"left\">ctg_N50</th>\n",
       "      <th colspan=\"3\" halign=\"left\">ctg_L50</th>\n",
       "      <th colspan=\"3\" halign=\"left\">scaf_N90</th>\n",
       "      <th colspan=\"3\" halign=\"left\">scaf_L90</th>\n",
       "      <th colspan=\"3\" halign=\"left\">ctg_N90</th>\n",
       "      <th colspan=\"3\" halign=\"left\">ctg_L90</th>\n",
       "      <th colspan=\"3\" halign=\"left\">scaf_max</th>\n",
       "      <th colspan=\"3\" halign=\"left\">ctg_max</th>\n",
       "      <th colspan=\"3\" halign=\"left\">scaf_n_gt50K</th>\n",
       "      <th colspan=\"3\" halign=\"left\">scaf_pct_gt50K</th>\n",
       "      <th colspan=\"3\" halign=\"left\">gc_avg</th>\n",
       "      <th colspan=\"3\" halign=\"left\">gc_std</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th></th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "      <th>mean</th>\n",
       "      <th>min</th>\n",
       "      <th>max</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Classification_quality</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>High</th>\n",
       "      <td>99.27</td>\n",
       "      <td>90.14</td>\n",
       "      <td>100.0</td>\n",
       "      <td>0.50</td>\n",
       "      <td>0.0</td>\n",
       "      <td>4.16</td>\n",
       "      <td>0.07</td>\n",
       "      <td>0.0</td>\n",
       "      <td>9.09</td>\n",
       "      <td>96.79</td>\n",
       "      <td>71.57</td>\n",
       "      <td>100.0</td>\n",
       "      <td>47.00</td>\n",
       "      <td>1.0</td>\n",
       "      <td>434.0</td>\n",
       "      <td>52.56</td>\n",
       "      <td>1.0</td>\n",
       "      <td>465.0</td>\n",
       "      <td>3798407.46</td>\n",
       "      <td>1378189.0</td>\n",
       "      <td>7005797.0</td>\n",
       "      <td>3796823.28</td>\n",
       "      <td>1378189.0</td>\n",
       "      <td>7005740.0</td>\n",
       "      <td>0.04</td>\n",
       "      <td>0.0</td>\n",
       "      <td>8.13</td>\n",
       "      <td>6.04</td>\n",
       "      <td>1.0</td>\n",
       "      <td>76.0</td>\n",
       "      <td>1131606.39</td>\n",
       "      <td>12138.0</td>\n",
       "      <td>6363829.0</td>\n",
       "      <td>6.73</td>\n",
       "      <td>1.0</td>\n",
       "      <td>80.0</td>\n",
       "      <td>1035143.71</td>\n",
       "      <td>11189.0</td>\n",
       "      <td>6209424.0</td>\n",
       "      <td>18.60</td>\n",
       "      <td>1.0</td>\n",
       "      <td>253.0</td>\n",
       "      <td>866615.86</td>\n",
       "      <td>3054.0</td>\n",
       "      <td>6363829.0</td>\n",
       "      <td>20.92</td>\n",
       "      <td>1.0</td>\n",
       "      <td>261.0</td>\n",
       "      <td>795670.62</td>\n",
       "      <td>2624.0</td>\n",
       "      <td>6209424.0</td>\n",
       "      <td>1391776.28</td>\n",
       "      <td>51808.0</td>\n",
       "      <td>6363829.0</td>\n",
       "      <td>1292321.30</td>\n",
       "      <td>51384.0</td>\n",
       "      <td>6209424.0</td>\n",
       "      <td>12.45</td>\n",
       "      <td>1.0</td>\n",
       "      <td>54.0</td>\n",
       "      <td>90.24</td>\n",
       "      <td>1.68</td>\n",
       "      <td>100.0</td>\n",
       "      <td>0.35</td>\n",
       "      <td>0.28</td>\n",
       "      <td>0.55</td>\n",
       "      <td>0.03</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Medium</th>\n",
       "      <td>93.03</td>\n",
       "      <td>80.03</td>\n",
       "      <td>100.0</td>\n",
       "      <td>1.93</td>\n",
       "      <td>0.0</td>\n",
       "      <td>9.56</td>\n",
       "      <td>18.88</td>\n",
       "      <td>0.0</td>\n",
       "      <td>47.22</td>\n",
       "      <td>83.39</td>\n",
       "      <td>50.14</td>\n",
       "      <td>100.0</td>\n",
       "      <td>171.02</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1512.0</td>\n",
       "      <td>234.81</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1782.0</td>\n",
       "      <td>3494803.61</td>\n",
       "      <td>1043041.0</td>\n",
       "      <td>7041311.0</td>\n",
       "      <td>3485573.28</td>\n",
       "      <td>1043041.0</td>\n",
       "      <td>7041311.0</td>\n",
       "      <td>0.29</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.20</td>\n",
       "      <td>30.85</td>\n",
       "      <td>1.0</td>\n",
       "      <td>353.0</td>\n",
       "      <td>552570.71</td>\n",
       "      <td>3040.0</td>\n",
       "      <td>6653812.0</td>\n",
       "      <td>39.04</td>\n",
       "      <td>1.0</td>\n",
       "      <td>370.0</td>\n",
       "      <td>522564.81</td>\n",
       "      <td>3040.0</td>\n",
       "      <td>6653812.0</td>\n",
       "      <td>99.93</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1121.0</td>\n",
       "      <td>437711.30</td>\n",
       "      <td>545.0</td>\n",
       "      <td>6653812.0</td>\n",
       "      <td>128.07</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1177.0</td>\n",
       "      <td>418300.08</td>\n",
       "      <td>545.0</td>\n",
       "      <td>6653812.0</td>\n",
       "      <td>704654.60</td>\n",
       "      <td>13713.0</td>\n",
       "      <td>6653812.0</td>\n",
       "      <td>658304.79</td>\n",
       "      <td>13713.0</td>\n",
       "      <td>6653812.0</td>\n",
       "      <td>12.06</td>\n",
       "      <td>0.0</td>\n",
       "      <td>50.0</td>\n",
       "      <td>54.48</td>\n",
       "      <td>0.00</td>\n",
       "      <td>100.0</td>\n",
       "      <td>0.36</td>\n",
       "      <td>0.28</td>\n",
       "      <td>0.47</td>\n",
       "      <td>0.02</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Low</th>\n",
       "      <td>83.07</td>\n",
       "      <td>50.31</td>\n",
       "      <td>100.0</td>\n",
       "      <td>1.68</td>\n",
       "      <td>0.0</td>\n",
       "      <td>9.68</td>\n",
       "      <td>50.60</td>\n",
       "      <td>0.0</td>\n",
       "      <td>100.00</td>\n",
       "      <td>74.69</td>\n",
       "      <td>50.01</td>\n",
       "      <td>99.2</td>\n",
       "      <td>176.99</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1506.0</td>\n",
       "      <td>260.01</td>\n",
       "      <td>1.0</td>\n",
       "      <td>5612.0</td>\n",
       "      <td>2590496.53</td>\n",
       "      <td>569462.0</td>\n",
       "      <td>7026888.0</td>\n",
       "      <td>2576486.22</td>\n",
       "      <td>569462.0</td>\n",
       "      <td>6570299.0</td>\n",
       "      <td>0.60</td>\n",
       "      <td>0.0</td>\n",
       "      <td>11.48</td>\n",
       "      <td>41.63</td>\n",
       "      <td>1.0</td>\n",
       "      <td>418.0</td>\n",
       "      <td>408083.12</td>\n",
       "      <td>2659.0</td>\n",
       "      <td>5602501.0</td>\n",
       "      <td>53.25</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1103.0</td>\n",
       "      <td>392454.35</td>\n",
       "      <td>1884.0</td>\n",
       "      <td>5602501.0</td>\n",
       "      <td>123.54</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1182.0</td>\n",
       "      <td>314657.00</td>\n",
       "      <td>1241.0</td>\n",
       "      <td>5602501.0</td>\n",
       "      <td>162.97</td>\n",
       "      <td>1.0</td>\n",
       "      <td>3546.0</td>\n",
       "      <td>305079.77</td>\n",
       "      <td>518.0</td>\n",
       "      <td>5602501.0</td>\n",
       "      <td>505158.89</td>\n",
       "      <td>11420.0</td>\n",
       "      <td>5602501.0</td>\n",
       "      <td>484987.41</td>\n",
       "      <td>9123.0</td>\n",
       "      <td>5602501.0</td>\n",
       "      <td>6.38</td>\n",
       "      <td>0.0</td>\n",
       "      <td>42.0</td>\n",
       "      <td>38.91</td>\n",
       "      <td>0.00</td>\n",
       "      <td>100.0</td>\n",
       "      <td>0.36</td>\n",
       "      <td>0.28</td>\n",
       "      <td>0.53</td>\n",
       "      <td>0.02</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.12</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                       Completeness               Contamination             \\\n",
       "                               mean    min    max          mean  min   max   \n",
       "Classification_quality                                                       \n",
       "High                          99.27  90.14  100.0          0.50  0.0  4.16   \n",
       "Medium                        93.03  80.03  100.0          1.93  0.0  9.56   \n",
       "Low                           83.07  50.31  100.0          1.68  0.0  9.68   \n",
       "\n",
       "                       Strain_heterogeneity              Quality_score         \\\n",
       "                                       mean  min     max          mean    min   \n",
       "Classification_quality                                                          \n",
       "High                                   0.07  0.0    9.09         96.79  71.57   \n",
       "Medium                                18.88  0.0   47.22         83.39  50.14   \n",
       "Low                                   50.60  0.0  100.00         74.69  50.01   \n",
       "\n",
       "                              n_scaffolds              n_contigs               \\\n",
       "                          max        mean  min     max      mean  min     max   \n",
       "Classification_quality                                                          \n",
       "High                    100.0       47.00  1.0   434.0     52.56  1.0   465.0   \n",
       "Medium                  100.0      171.02  1.0  1512.0    234.81  1.0  1782.0   \n",
       "Low                      99.2      176.99  1.0  1506.0    260.01  1.0  5612.0   \n",
       "\n",
       "                           scaf_bp                         contig_bp  \\\n",
       "                              mean        min        max        mean   \n",
       "Classification_quality                                                 \n",
       "High                    3798407.46  1378189.0  7005797.0  3796823.28   \n",
       "Medium                  3494803.61  1043041.0  7041311.0  3485573.28   \n",
       "Low                     2590496.53   569462.0  7026888.0  2576486.22   \n",
       "\n",
       "                                             gap_pct             scaf_N50  \\\n",
       "                              min        max    mean  min    max     mean   \n",
       "Classification_quality                                                      \n",
       "High                    1378189.0  7005740.0    0.04  0.0   8.13     6.04   \n",
       "Medium                  1043041.0  7041311.0    0.29  0.0   3.20    30.85   \n",
       "Low                      569462.0  6570299.0    0.60  0.0  11.48    41.63   \n",
       "\n",
       "                                      scaf_L50                     ctg_N50  \\\n",
       "                        min    max        mean      min        max    mean   \n",
       "Classification_quality                                                       \n",
       "High                    1.0   76.0  1131606.39  12138.0  6363829.0    6.73   \n",
       "Medium                  1.0  353.0   552570.71   3040.0  6653812.0   39.04   \n",
       "Low                     1.0  418.0   408083.12   2659.0  5602501.0   53.25   \n",
       "\n",
       "                                        ctg_L50                     scaf_N90  \\\n",
       "                        min     max        mean      min        max     mean   \n",
       "Classification_quality                                                         \n",
       "High                    1.0    80.0  1035143.71  11189.0  6209424.0    18.60   \n",
       "Medium                  1.0   370.0   522564.81   3040.0  6653812.0    99.93   \n",
       "Low                     1.0  1103.0   392454.35   1884.0  5602501.0   123.54   \n",
       "\n",
       "                                      scaf_L90                    ctg_N90  \\\n",
       "                        min     max       mean     min        max    mean   \n",
       "Classification_quality                                                      \n",
       "High                    1.0   253.0  866615.86  3054.0  6363829.0   20.92   \n",
       "Medium                  1.0  1121.0  437711.30   545.0  6653812.0  128.07   \n",
       "Low                     1.0  1182.0  314657.00  1241.0  5602501.0  162.97   \n",
       "\n",
       "                                       ctg_L90                       scaf_max  \\\n",
       "                        min     max       mean     min        max        mean   \n",
       "Classification_quality                                                          \n",
       "High                    1.0   261.0  795670.62  2624.0  6209424.0  1391776.28   \n",
       "Medium                  1.0  1177.0  418300.08   545.0  6653812.0   704654.60   \n",
       "Low                     1.0  3546.0  305079.77   518.0  5602501.0   505158.89   \n",
       "\n",
       "                                               ctg_max                      \\\n",
       "                            min        max        mean      min        max   \n",
       "Classification_quality                                                       \n",
       "High                    51808.0  6363829.0  1292321.30  51384.0  6209424.0   \n",
       "Medium                  13713.0  6653812.0   658304.79  13713.0  6653812.0   \n",
       "Low                     11420.0  5602501.0   484987.41   9123.0  5602501.0   \n",
       "\n",
       "                       scaf_n_gt50K            scaf_pct_gt50K               \\\n",
       "                               mean  min   max           mean   min    max   \n",
       "Classification_quality                                                       \n",
       "High                          12.45  1.0  54.0          90.24  1.68  100.0   \n",
       "Medium                        12.06  0.0  50.0          54.48  0.00  100.0   \n",
       "Low                            6.38  0.0  42.0          38.91  0.00  100.0   \n",
       "\n",
       "                       gc_avg             gc_std             \n",
       "                         mean   min   max   mean  min   max  \n",
       "Classification_quality                                       \n",
       "High                     0.35  0.28  0.55   0.03  0.0  0.17  \n",
       "Medium                   0.36  0.28  0.47   0.02  0.0  0.16  \n",
       "Low                      0.36  0.28  0.53   0.02  0.0  0.12  "
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "statistics_classification = dataset.groupby(\"Classification_quality\").describe()\n",
    "statistics_classification2 = statistics_classification.T.round(2)\n",
    "col = [\"High\", \"Medium\", \"Low\"]\n",
    "statistics_classification2 = statistics_classification2[col]\n",
    "statistics_classification = statistics_classification2.loc(axis=0)[:, [\"mean\", \"min\", \"max\"]]\n",
    "statistics_classification.T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "with pd.ExcelWriter('Statistics.xlsx') as writer:  \n",
    "    statistics.to_excel(writer, sheet_name=\"Statistics\")\n",
    "    statistics_classification.to_excel(writer, sheet_name=\"Statistics_per_quality\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "-----------------------"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## References"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**CheckM** \\\n",
    "Parks DH, Imelfort M, Skennerton CT, Hugenholtz P, Tyson GW. 2015. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Genome Research, 25: 10431055. \\\n",
    "3rd party dependencies: \\\n",
    "pplacer: Matsen FA, Kodner RB, Armbrust EV. 2010. pplacer: linear time maximum-likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree. BMC Bioinformatics 11: doi:10.1186/1471-2105-11-538. \\\n",
    "prodigal: Hyatt D, Locascio PF, Hauser LJ, Uberbacher EC. 2012. Gene and translation initiation site prediction in metagenomic sequences. Bioinformatics 28: 22232230. \\\n",
    "HMMER: http://hmmer.org/ \\\n",
    "\n",
    "**GTDB-Tk** \\\n",
    "Chaumeil PA, et al. 2019. GTDB-Tk: A toolkit to classify genomes with the Genome Taxonomy Database. Bioinformatics, btz848.\\\n",
    "3rd party dependencies: \\\n",
    "Matsen FA, et al. 2010. pplacer: linear time maximum-likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree. BMC Bioinformatics, 11:538. \\\n",
    "Jain C, et al. 2019. High-throughput ANI Analysis of 90K Prokaryotic Genomes Reveals Clear Species Boundaries. Nat. Communications, doi: 10.1038/s41467-018-07641-9. \\\n",
    "Hyatt D, et al. 2010. Prodigal: prokaryotic gene recognition and translation initiation site identification. BMC Bioinformatics, 11:119. doi: 10.1186/1471-2105-11-119. \\\n",
    "Price MN, et al. 2010. FastTree 2 - Approximately Maximum-Likelihood Trees for Large Alignments. PLoS One, 5, e9490. \\\n",
    "Eddy SR. 2011. Accelerated profile HMM searches. PLOS Comp. Biol., 7:e1002195.\n",
    "Ondov BD, et al. 2016. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biol 17, 132. doi: doi: 10.1186/s13059-016-0997-x. \n",
    "\n",
    "**BBtools**\n",
    "https://jgi.doe.gov/data-and-tools/bbtools/\n",
    "Bushnell, B. BBMap short read aligner, and other bioinformatic tools. Available online: http://sourceforge.net/projects/bbmap "
   ]
  }
 ],
 "metadata": {
  "interpreter": {
   "hash": "44aa9334d14e77acd0ab58a32227ca6642aa33f62a1f1c24b03364be7e1b2260"
  },
  "kernelspec": {
   "display_name": "Python 3.8.5 64-bit ('base': conda)",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
